{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import pandas as pd\n",
    "def write_fasta(input_dict):\n",
    "    with open(input_dict['save_path'], 'w') as f:\n",
    "        f.write('>' + input_dict['title'] + '\\n')\n",
    "        f.write(input_dict['sequence'] + '\\n')\n",
    "\n",
    "def mutate_sequence(original_sequence, mutation_label, correction):\n",
    "    standard_amino_acids = 'ACDEFGHIKLMNPQRSTVWY'\n",
    "    running_sequence = original_sequence\n",
    "    for label in mutation_label:\n",
    "        label = label.upper().strip()\n",
    "        res1, pos, res2 = label[0], int(label[1:-1]), label[-1]\n",
    "        pos = pos + correction\n",
    "        # Check if res1 and res2 are standard amino acids\n",
    "        if res1 not in standard_amino_acids or res2 not in standard_amino_acids:\n",
    "            raise ValueError('res1 and res2 should be standard amino acids.')\n",
    "        if original_sequence[pos-1] != res1:\n",
    "            raise ValueError('The residue at position does not match res1.')\n",
    "        running_sequence = running_sequence[:pos-1] + res2 + running_sequence[pos:]\n",
    "    return running_sequence\n",
    "\n",
    "def truncate_sequence(original_sequence, label): # label = 'start-end'\n",
    "    if label is None:\n",
    "        return original_sequence\n",
    "    start, end = label.split('-')\n",
    "    if end == 'end':\n",
    "        end = len(original_sequence)\n",
    "    elif start == 'start':\n",
    "        start = 1\n",
    "    start, end = int(start), int(end)\n",
    "    truncated_sequence = original_sequence[start-1:end]\n",
    "    return truncated_sequence\n",
    "\n",
    "def generate_config(config_dict):\n",
    "    \"\"\"\n",
    "    Generates a configuration string from a given dictionary input.\n",
    "\n",
    "    Args:\n",
    "    config_dict (dict): A dictionary containing the configuration data.\n",
    "\n",
    "    Returns:\n",
    "    str: A string representing the configuration in the specified format.\n",
    "    \"\"\"\n",
    "    config_str = \"defaults:\\n  - base\\n\\n\"\n",
    "    config_str += f\"job_name: \\\"{config_dict['job_name']}\\\"\\n\"\n",
    "    \n",
    "    for input_type in ['protein_inputs', 'na_inputs', 'sm_inputs']:\n",
    "        if input_type in config_dict:\n",
    "            config_str += f\"{input_type}:\\n\"\n",
    "            for key, value in config_dict[input_type].items():\n",
    "                if input_type == 'sm_inputs' or input_type == 'na_inputs':\n",
    "                    config_str += f\"  {key}:\\n    input: {value['input']}\\n    input_type: \\\"{value['input_type']}\\\"\\n\"\n",
    "                else:  # protein_inputs or na_inputs\n",
    "                    config_str += f\"  {key}:\\n    fasta_file: {value['fasta_file']}\\n\"\n",
    "    \n",
    "    return config_str"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index(['database_id', 'database', 'literature', 'protein', 'state',\n",
      "       'match_uni', 'complex_uni', 'uni_sequence', 'modification', 'mutation',\n",
      "       'truncation', 'complex_state', 'D_Conc', 'pH', 'temp', 'data_type',\n",
      "       'check', 'structure_file', 'note', 'chain_identifier',\n",
      "       'correction_value', 'protein_chain', 'NA_chain', 'SM_chain'],\n",
      "      dtype='object')\n",
      "(154, 24)\n",
      "(213, 24)\n"
     ]
    }
   ],
   "source": [
    "# load the data\n",
    "import pandas as pd\n",
    "\n",
    "work_fpath = '/home/lwang/models/HDX_LSTM/data/Latest_set/merged_data-NAcomplex.xlsx'\n",
    "work_df = pd.read_excel(work_fpath, sheet_name='Sheet1')\n",
    "print(work_df.columns)\n",
    "df1 = work_df.dropna(subset=['structure_file']).drop_duplicates(subset=['structure_file'])\n",
    "df2 = work_df.dropna(subset=['structure_file']).drop_duplicates(subset=['protein', 'state'])\n",
    "print(df1.shape)\n",
    "print(df2.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/lwang/models/HDX_LSTM/data/Latest_set/HDX_files/PXD041774_revised.xlsx\n",
      "21799\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "import pandas as pd\n",
    "\n",
    "HDX_dir = f'/home/lwang/models/HDX_LSTM/data/Latest_set/HDX_files'\n",
    "groups = work_df.dropna(subset=['structure_file']).groupby(['database_id'])\n",
    "\n",
    "count = 0\n",
    "for id, group in groups:\n",
    "    group['label'] = group['protein']+group['state']\n",
    "    labels = group['label'].unique()\n",
    "    fname = id[0]\n",
    "    fpath = f'{HDX_dir}/{fname}_revised.xlsx'\n",
    "    if not os.path.exists(fpath):\n",
    "        print(fpath)\n",
    "        continue\n",
    "    hdx_df = pd.read_excel(fpath, sheet_name='Sheet1')\n",
    "    hdx_df['label'] = hdx_df['protein']+hdx_df['state']\n",
    "\n",
    "    count += hdx_df[hdx_df['label'].isin(labels)].groupby(['label', 'sequence']).ngroups\n",
    "print(count)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "'''\n",
    "FOR SINGLE CHAIN PROTEIN\n",
    "'''\n",
    "# output fasta files\n",
    "fail_dict = {}\n",
    "for index, row in work_df.iterrows():\n",
    "    # create a dictionary to store the information\n",
    "    database_id = row['database_id'].replace(' ', '')\n",
    "    protein_id = row['protein'].replace(' ', '')\n",
    "    state = row['state'].replace(' ', '')\n",
    "    chain = 'A'\n",
    "    correction = int(row['correction_value'])\n",
    "    original_sequence = row['uni_sequence'].strip()\n",
    "    current_label = database_id + '_' + protein_id + '_' + state + '_' + chain\n",
    "    if os.path.isfile(os.path.join('/home/lwang/models/HDX_LSTM/data/Fullset/fasta_files/protein', f'{current_label}.fasta')):\n",
    "        continue\n",
    "\n",
    "    mutation_label = str(row['mutation']).strip().split('/')\n",
    "    #print(f'mutation_label: {mutation_label}')\n",
    "    truncation_label = str(row['truncation']).strip()\n",
    "    print(f'current_label: {current_label}')\n",
    "    modified_sequence = original_sequence\n",
    "\n",
    "    if 'nan' not in mutation_label:\n",
    "        modified_sequence = mutate_sequence(original_sequence, mutation_label,0)        \n",
    "    if truncation_label != 'nan':\n",
    "        modified_sequence = truncate_sequence(modified_sequence, truncation_label)\n",
    "    fasta_dict = {\n",
    "        'title': current_label,\n",
    "        'sequence': modified_sequence,\n",
    "        'save_path': os.path.join('/home/lwang/models/HDX_LSTM/data/Fullset/fasta_files/protein', f'{current_label}.fasta')\n",
    "    }\n",
    "    write_fasta(fasta_dict)\n",
    "print(fail_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# generate config files\n",
    "for fasta_file in os.listdir('/home/lwang/models/HDX_LSTM/data/Fullset/fasta_files/protein'):\n",
    "    if fasta_file.endswith('.fasta'):\n",
    "        parts = fasta_file.split('_')\n",
    "        pdb_fname = '_'.join(parts[:-1])\n",
    "        print(f'pdb_fname: {pdb_fname}')\n",
    "        config_dict = {\n",
    "            'job_name': pdb_fname,\n",
    "            'protein_inputs': {\n",
    "                'A': {\n",
    "                    'fasta_file': f'/home/lwang/models/HDX_LSTM/data/Fullset/fasta_files/protein/{fasta_file}'\n",
    "                }\n",
    "            }\n",
    "        }\n",
    "        config_str = generate_config(config_dict)\n",
    "        with open(f'/home/lwang/models/HDX_LSTM/data/Fullset/fasta_files/config/{pdb_fname}.yaml', 'w') as f:\n",
    "            f.write(config_str)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(78, 23)\n",
      "current_label: PXD019367_GoA_alpha_Alone_A\n"
     ]
    }
   ],
   "source": [
    "'''\n",
    "FOR COMPLEX\n",
    "'''\n",
    "# load the data\n",
    "work_fpath = '/home/lwang/models/HDX_LSTM/data/Fullset/merged_data.xlsx'\n",
    "work_df = pd.read_excel(work_fpath, sheet_name='Sheet1')\n",
    "\n",
    "work_df = work_df[work_df['check'] != 'yes']\n",
    "work_df = work_df[work_df['complex_state'] == 'protein complex']\n",
    "\n",
    "print(work_df.shape)\n",
    "\n",
    "for index, row in work_df.iterrows():\n",
    "    # create a dictionary to store the information\n",
    "    database_id = row['database_id'].replace(' ', '')\n",
    "    protein_id = row['protein'].replace(' ', '')\n",
    "    state = row['state'].replace(' ', '')\n",
    "    chain = 'A'\n",
    "    original_sequence = row['uni_sequence'].strip()\n",
    "    current_label = database_id + '_' + protein_id + '_' + state + '_' + chain\n",
    "    if os.path.isfile(os.path.join('/home/lwang/models/HDX_LSTM/data/Fullset/fasta_files/protein', f'{current_label}.fasta')):\n",
    "        continue\n",
    "\n",
    "    mutation_label = str(row['mutation']).strip().split('/')\n",
    "    #print(f'mutation_label: {mutation_label}')\n",
    "    truncation_label = str(row['truncation']).strip()\n",
    "    print(f'current_label: {current_label}')\n",
    "    modified_sequence = original_sequence\n",
    "\n",
    "    if 'nan' not in mutation_label:\n",
    "        modified_sequence = mutate_sequence(original_sequence, mutation_label,0)        \n",
    "    if truncation_label != 'nan':\n",
    "        modified_sequence = truncate_sequence(modified_sequence, truncation_label)\n",
    "    fasta_dict = {\n",
    "        'title': current_label,\n",
    "        'sequence': modified_sequence,\n",
    "        'save_path': os.path.join('/home/lwang/models/HDX_LSTM/data/Fullset/fasta_files/protein', f'{current_label}.fasta')\n",
    "    }\n",
    "    write_fasta(fasta_dict)\n",
    "    break\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "current_label: PXD019367_GoA-beta_Alone_C\n"
     ]
    }
   ],
   "source": [
    "# write other chains\n",
    "chain = 'C'\n",
    "current_label = database_id + '_' + protein_id + '_' + state + '_' + chain\n",
    "seq = 'MGCTLSAEERAALERSKAIEKNLKEDGISAAKDVKLLLLGAGESGKSTIVKQMKIIHEDGFSGEDVKQYKPVVYSNTIQSLAAIVRAMDTLGIEYGDKERKADAKMVCDVVSRMEDTEPFSAELLSAMMRLWGDSGIQECFNRSREYQLNDSAKYYLDSLDRIGAADYQPTEQDILRTRVKTTGIVETHFTFKNLHFRLFDVGGQRSERKKWIHCFEDVTAIIFCVALSGYDQVLHEDETTNRMHESLMLFDSICNNKFFIDTSIILFLNKKDLFGEKIKKSPLTICFPEYTGPNTYEDAAAYIQAQFESKNRSPNKEIYCHMTCATDTNNIQVVFDAVTDIIIANNLRGCGLY'\n",
    "#seq = truncate_sequence(seq, '135-467')\n",
    "print(f'current_label: {current_label}')\n",
    "fasta_dict = {\n",
    "    'title': current_label,\n",
    "    'sequence': seq,\n",
    "    'save_path': os.path.join('/home/lwang/models/HDX_LSTM/data/Fullset/fasta_files/protein', f'{current_label}.fasta')\n",
    "}\n",
    "write_fasta(fasta_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [],
   "source": [
    "# save NA\n",
    "save_folder = '/home/lwang/models/HDX_LSTM/data/extra_NA+SM/fasta_files/nucleic_acid'\n",
    "\n",
    "chain = 'C'\n",
    "NA_sequence = 'GGGACTTTCC'\n",
    "\n",
    "input_dict = {\n",
    "    'title': f'{database_id}_{protein}_{state}_{chain}',\n",
    "    'sequence': NA_sequence,\n",
    "    'save_path': os.path.join(save_folder, f'{database_id}_{protein}_{state}_{chain}.fasta')\n",
    "}\n",
    "write_fasta(input_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import os\n",
    "\n",
    "def generate_config_from_excel(excel_file_path):\n",
    "    \"\"\"\n",
    "    Reads an Excel file to generate configuration strings based on the contents.\n",
    "\n",
    "    Args:\n",
    "    excel_file_path (str): The file path to the Excel document containing the configuration data.\n",
    "\n",
    "    Returns:\n",
    "    list of str: A list of configuration strings for each row in the Excel document.\n",
    "    \"\"\"\n",
    "    # Read the Excel file\n",
    "    df = pd.read_excel(excel_file_path)\n",
    "    df = df.dropna(subset=['chain_identifier'], how='all')\n",
    "    df = df.drop_duplicates(subset=['apo_identifier'], keep='first')\n",
    "    protein_fasta_folder = '/home/lwang/models/HDX_LSTM/data/extra_NA+SM/fasta_files/protein'\n",
    "    na_fasta_folder = '/home/lwang/models/HDX_LSTM/data/extra_NA+SM/fasta_files/nucleic_acid'\n",
    "    sm_sdf_folder = '/home/lwang/models/HDX_LSTM/data/extra_NA+SM/fasta_files/ligand'\n",
    "\n",
    "    for index, row in df.iterrows():\n",
    "        try:        \n",
    "            pdb_name = row['apo_identifier'].replace(' ','').split('.')[0]\n",
    "            print(pdb_name)\n",
    "            config_dict = {\"job_name\": f\"{pdb_name}\"}\n",
    "            database_id = row['database_id'].replace(' ','')\n",
    "            protein = row['protein'].replace(' ','')\n",
    "            state = row['state'].replace(' ','')    \n",
    "\n",
    "            # Process 'protein_inputs'\n",
    "            if pd.notna(row['protein_chain']):\n",
    "                protein_chains = row['protein_chain'].split(',')\n",
    "                config_dict['protein_inputs'] = {}\n",
    "                for chain in protein_chains:\n",
    "                    chain = chain.strip()\n",
    "                    fname = f\"{database_id}-{protein}-{state}-{chain}\"\n",
    "                    if not os.path.isfile(f\"{protein_fasta_folder}/{fname}.fasta\"):\n",
    "                        raise FileNotFoundError(f\"File {protein_fasta_folder}/{fname}.fasta not found.\")\n",
    "                    config_dict['protein_inputs'][chain] = {\n",
    "                        \"fasta_file\": f\"{protein_fasta_folder}/{fname}.fasta\"\n",
    "                    }\n",
    "            \n",
    "            # Process 'NA_inputs'\n",
    "            if pd.notna(row['NA_chain']):\n",
    "                na_chains = row['NA_chain'].split(',')\n",
    "                config_dict['na_inputs'] = {}\n",
    "                for chain in na_chains:\n",
    "                    chain = chain.strip()\n",
    "                    fname = f\"{database_id}-{protein}-{state}-{chain}\"\n",
    "                    config_dict['na_inputs'][chain] = {\n",
    "                        \"input\": f\"{na_fasta_folder}/{fname}.fasta\",\n",
    "                        'input_type': 'dna'\n",
    "                    }\n",
    "\n",
    "            # Process 'SM_inputs'\n",
    "            if pd.notna(row['SM_chain']):\n",
    "                sm_chains = row['SM_chain'].split(',')\n",
    "                config_dict['sm_inputs'] = {}\n",
    "                for chain in sm_chains:\n",
    "                    chain = chain.strip()\n",
    "                    fname = f\"{database_id}-{protein}-{state}-{chain}\"\n",
    "                    config_dict['sm_inputs'][chain] = {\n",
    "                        \"input\": f\"{sm_sdf_folder}/{fname}.sdf\",\n",
    "                        \"input_type\": \"sdf\"\n",
    "                    }\n",
    "        except FileNotFoundError as e:\n",
    "            print(e)\n",
    "            continue\n",
    "\n",
    "            # Generate the configuration for the current row\n",
    "        config_str = generate_config(config_dict)\n",
    "        with open(f\"/home/lwang/models/HDX_LSTM/data/extra_NA+SM/config/{pdb_name}.yaml\", 'w') as f:\n",
    "            f.write(config_str)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd \n",
    "import os\n",
    "\n",
    "fpath = '/home/lwang/models/HDX_LSTM/data/Fullset/merged_data.xlsx'\n",
    "df = pd.read_excel(fpath, sheet_name='Sheet1')\n",
    "df = df.dropna(subset=['apo_identifier'], how='all')\n",
    "df = df.drop_duplicates(subset=['apo_identifier'], keep='first')\n",
    "print(df.shape)\n",
    "\n",
    "#check pdb files\n",
    "for pdb_fname in df['apo_identifier']:\n",
    "    pdb_fname = pdb_fname.replace(' ','')\n",
    "    pdb_fpath = os.path.join('/home/lwang/models/HDX_LSTM/data/Fullset/structrues', f'{pdb_fname}')\n",
    "    if not os.path.isfile(pdb_fpath):\n",
    "        print(f'{pdb_fname} not found.')\n",
    "        continue\n",
    "\n",
    "# check HDX file\n",
    "for index, row in df.iterrows():\n",
    "    # check if HDX data is available\n",
    "    database_id = row['database_id']\n",
    "    protein = row['protein']\n",
    "    state = row['state']\n",
    "    HDX_fpath = os.path.join('/home/lwang/models/HDX_LSTM/data/Fullset/HDX_files', f'{database_id}.xlsx')\n",
    "    if not os.path.isfile(HDX_fpath):\n",
    "        print(f'{database_id} not found.')\n",
    "    else:\n",
    "        print(f'{database_id}:')\n",
    "        temp_df = pd.read_excel(HDX_fpath, sheet_name='Sheet1')\n",
    "        if protein not in temp_df['protein'].values:\n",
    "            print(f'{protein} not found in {database_id}.xlsx')\n",
    "        if state not in temp_df['state'].values:\n",
    "            print(f'{state} not found in {database_id}.xlsx')\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\" embedding.pt dictionary\\ndata_to_save = {\\n    'res_idx': res_idx,  [#res]\\n    'res_name': res_name,  [#res]\\n    'chain_label': chain_label, [#res]\\n    'embedding': protein_embedding\\n}\\n\""
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# t-SNE plot of pep embeddings\n",
    "\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "import os\n",
    "import torch\n",
    "from torch.nn.functional import pad\n",
    "from sklearn.manifold import TSNE\n",
    "from bioinfokit.visuz import cluster\n",
    "from Bio.PDB.Polypeptide import three_to_one\n",
    "import warnings\n",
    "\n",
    "warnings.filterwarnings(\"ignore\")\n",
    "def read_HDX_table(HDX_df, embedding_mtx, res_dict, proteins, states, correction): # read the HDX table file and return the residue information as node_list\n",
    "    feat_mtx = []\n",
    "    y_label = []\n",
    "    for index, (protein, state) in enumerate(zip(proteins, states)):\n",
    "        temp_HDX_df = HDX_df[(HDX_df['state']==state) & (HDX_df['protein']==protein)]\n",
    "        temp_HDX_df = temp_HDX_df.sort_values(by=['start', 'end'], ascending=[True, True])\n",
    "        #print(protein, state, chain, temp_HDX_df.shape)\n",
    "        temp_feat_mtx = []\n",
    "        for i, row in temp_HDX_df.iterrows():\n",
    "            sequence = row['sequence'].strip()\n",
    "            start_pos = int(row['start'])+correction[index]\n",
    "            end_pos = int(row['end'])+correction[index]\n",
    "            if (end_pos-start_pos+1) >30 or len(sequence) != (end_pos-start_pos+1):\n",
    "                continue\n",
    "            elif start_pos<= 0 or end_pos > embedding_mtx.shape[0]:\n",
    "                continue\n",
    "\n",
    "            pdb_seq = [three_to_one(res_dict[i]) for i in range(start_pos, end_pos+1) if i in res_dict]\n",
    "            pdb_seq = ''.join(pdb_seq)\n",
    "            if pdb_seq != sequence:\n",
    "                #print(f'HDX sequence:{sequence}, pdb sequence:{pdb_seq}')\n",
    "                continue\n",
    "\n",
    "            hdx_value = float(row['%d'])/100\n",
    "            bottom_pad_len = 30 - (end_pos-start_pos+1)\n",
    "            mtx = embedding_mtx[start_pos-1:end_pos, :]\n",
    "            mtx = pad(mtx, (0,0,0,bottom_pad_len), 'constant', 0)\n",
    "            \n",
    "            temp_feat_mtx.append(mtx.flatten())\n",
    "            y_label.append(hdx_value)\n",
    "        feat_mtx.extend(temp_feat_mtx)\n",
    "    return feat_mtx, y_label\n",
    "\n",
    "''' embedding.pt dictionary\n",
    "data_to_save = {\n",
    "    'res_idx': res_idx,  [#res]\n",
    "    'res_name': res_name,  [#res]\n",
    "    'chain_label': chain_label, [#res]\n",
    "    'embedding': protein_embedding\n",
    "}\n",
    "'''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SASDHC5_fit1_model1_revised not found.\n",
      "4x06_clean_Hplus not found.\n",
      "6h9v_clean_Hplus not found.\n",
      "DAT+DA_complex not found.\n",
      "AcrB_homotrimer not found.\n",
      "Hdock_TRAPP_3+2+5 not found.\n",
      "5IV9 not found.\n",
      "1t3d_hexamer not found.\n",
      "CD47_BRIL not found.\n",
      "(3791, 2610)\n",
      "3791\n",
      "(478, 2610)\n",
      "478\n"
     ]
    }
   ],
   "source": [
    "def load_data(work_fpath, work_dir):\n",
    "    df = pd.read_excel(work_fpath, sheet_name='Sheet1')\n",
    "    df = df.dropna(subset=['apo_identifier'], how='all')\n",
    "    df = df.drop_duplicates(subset=['apo_identifier'], keep='first')\n",
    "    X_data = []\n",
    "    Y_data = []\n",
    "    for index, row in df.iterrows():\n",
    "        pdb_fname = row['apo_identifier'][:-4]\n",
    "        database_id = row['database_id']\n",
    "        protein = row['protein']\n",
    "        state = row['state']\n",
    "        chain = row['chain_identifier']\n",
    "        correction = int(row['correction_value'])\n",
    "        embed_fpath = os.path.join(work_dir, 'embedding_files', f'{pdb_fname}.pt')\n",
    "        if not os.path.isfile(embed_fpath):\n",
    "            print(f'{pdb_fname} not found.')\n",
    "            continue\n",
    "\n",
    "        # load the embedding file\n",
    "        #print(f'loading {pdb_fname}.pt')\n",
    "        embedding_dict = torch.load(embed_fpath)\n",
    "        chain_label = np.array(embedding_dict['chain_label'])\n",
    "        chain_mask = (chain_label == chain)\n",
    "        embedding_mtx = embedding_dict['embedding'][chain_mask]\n",
    "        res_list = np.array(embedding_dict['res_name'])[chain_mask]\n",
    "        res_idx = np.array(embedding_dict['res_idx'])[chain_mask]\n",
    "        res_dict = {idx: res for idx, res in zip(res_idx, res_list)}\n",
    "\n",
    "        # load the HDX file\n",
    "        HDX_fpath = os.path.join(work_dir, 'HDX_files', f'{database_id}.xlsx')\n",
    "        HDX_df = pd.read_excel(HDX_fpath, sheet_name='Sheet1')\n",
    "        HDX_df = HDX_df.drop_duplicates(subset=['protein', 'state', 'start', 'end'], keep='last')\n",
    "\n",
    "        data, y_label = read_HDX_table(HDX_df, embedding_mtx, res_dict, [protein], [state], [correction]) # return list of np array vector\n",
    "        X_data.extend(data)\n",
    "        Y_data.extend(y_label)\n",
    "        #print(len(data), len(y_label))\n",
    "    return X_data, Y_data\n",
    "\n",
    "# load the data\n",
    "work_fpath = '/home/lwang/AI-HDX-main/HDX_MS_dataset/full_dataset/merged_data_oldVer.xlsx'\n",
    "X_data, Y_data = load_data(work_fpath, '/home/lwang/models/HDX_LSTM/data/Fullset')\n",
    "X_data = np.row_stack(X_data)\n",
    "print(X_data.shape)\n",
    "print(len(Y_data))\n",
    "\n",
    "COVID_fpath = '/home/lwang/models/HDX_LSTM/data/COVID_SPIKE/COVID_record.xlsx'\n",
    "COVID_X_data, COVID_Y_data = load_data(COVID_fpath, '/home/lwang/models/HDX_LSTM/data/COVID_SPIKE')\n",
    "COVID_X_data = np.row_stack(COVID_X_data)\n",
    "print(COVID_X_data.shape)\n",
    "print(len(COVID_Y_data))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(4269, 2610)\n",
      "(4269,)\n",
      "[t-SNE] Computing 91 nearest neighbors...\n",
      "[t-SNE] Indexed 4269 samples in 0.007s...\n",
      "[t-SNE] Computed neighbors for 4269 samples in 0.893s...\n",
      "[t-SNE] Computed conditional probabilities for sample 1000 / 4269\n",
      "[t-SNE] Computed conditional probabilities for sample 2000 / 4269\n",
      "[t-SNE] Computed conditional probabilities for sample 3000 / 4269\n",
      "[t-SNE] Computed conditional probabilities for sample 4000 / 4269\n",
      "[t-SNE] Computed conditional probabilities for sample 4269 / 4269\n",
      "[t-SNE] Mean sigma: 54.207937\n",
      "[t-SNE] KL divergence after 250 iterations with early exaggeration: 81.955505\n",
      "[t-SNE] KL divergence after 2250 iterations: 1.575898\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'\\nfrom sklearn.decomposition import PCA\\npca_scores = PCA(n_components=50).fit_transform(X_data)\\ndf_pc = pd.DataFrame(pca_scores)\\ntsne_em = TSNE(n_components=3, perplexity=30.0, n_iter=5000, early_exaggeration=12, learning_rate=368, verbose=1).fit_transform(df_pc.iloc[:, :50])\\n'"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "Y_data = np.array(Y_data)\n",
    "# classify Y labels into 5 classes\n",
    "Y_bin = np.digitize(Y_data, [0.2, 0.4, 0.6, 0.8])\n",
    "\n",
    "# process COVID data\n",
    "COVID_Y_bin = np.full(len(COVID_Y_data), 5)\n",
    "Y_bin = np.append(Y_bin, COVID_Y_bin)\n",
    "X_data = np.row_stack([X_data, COVID_X_data])\n",
    "\n",
    "print(X_data.shape)\n",
    "print(Y_bin.shape)\n",
    "tsne_em = TSNE(n_components=3, perplexity=30.0, n_iter=8000, early_exaggeration=12, learning_rate=368, verbose=1).fit_transform(X_data)\n",
    "\n",
    "'''\n",
    "from sklearn.decomposition import PCA\n",
    "pca_scores = PCA(n_components=50).fit_transform(X_data)\n",
    "df_pc = pd.DataFrame(pca_scores)\n",
    "tsne_em = TSNE(n_components=3, perplexity=30.0, n_iter=5000, early_exaggeration=12, learning_rate=368, verbose=1).fit_transform(df_pc.iloc[:, :50])\n",
    "'''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(4269, 3)\n",
      "(478, 3)\n",
      "(478,)\n"
     ]
    }
   ],
   "source": [
    "\n",
    "print(tsne_em.shape)\n",
    "tsne_training = tsne_em[:len(Y_data)]\n",
    "tsne_covid = tsne_em[len(Y_data):]\n",
    "\n",
    "training_bin = Y_bin[:len(Y_data)]\n",
    "covid_bin = np.digitize(COVID_Y_data, [0.2, 0.4, 0.6, 0.8])\n",
    "\n",
    "print(tsne_covid.shape)\n",
    "print(covid_bin.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n",
      "findfont: Font family 'Arial' not found.\n"
     ]
    }
   ],
   "source": [
    "from bioinfokit.visuz import cluster\n",
    "\n",
    "cluster.tsneplot(score=tsne_em, colorlist=Y_bin, legendpos='upper right', \n",
    "colordot=('#407DD7', '#40B6BD', '#A982C0', '#EEB2E2', '#D46853', '#1A2B47'), valphadot=0.8, show=False,\n",
    "legendanchor=(1.35, 1), figname = '/home/lwang/models/HDX_LSTM/data/Fullset/tsne_2d_plexi30')\n",
    "\n",
    "cluster.tsneplot(score=tsne_covid, colorlist=covid_bin, legendpos='upper right', \n",
    "colordot=('#407DD7', '#40B6BD', '#A982C0', '#EEB2E2'), valphadot=1, show=False,\n",
    "legendanchor=(1.35, 1), figname = '/home/lwang/models/HDX_LSTM/data/Fullset/tsne_2d_plexi30_covid')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAjsAAAGwCAYAAABPSaTdAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/bCgiHAAAACXBIWXMAAA9hAAAPYQGoP6dpAAA4VUlEQVR4nO3deXyNd/7//+eRTXYSJFKpUEktQZsog6ml1mqjmBmU2jtlikpR6judsdRY26gy1ZVoR6sbRj9DiVpqmRbR2NLBEFslUhWJJZJIrt8ffs70iC0nJznJlcf9dju321zv632u8zrvycx5el/v67oshmEYAgAAMKlKzi4AAACgJBF2AACAqRF2AACAqRF2AACAqRF2AACAqRF2AACAqRF2AACAqbk6u4CyoKCgQGfOnJGvr68sFouzywEAAPfAMAxdvHhRISEhqlTp9vM3hB1JZ86cUWhoqLPLAAAAdjh16pRq1ap12/2EHUm+vr6Srg+Wn5+fk6sBAAD3IisrS6Ghodbf8dsh7EjWU1d+fn6EHQAAypm7LUFhgTIAADA1wg4AADA1wg4AADA11uwAAAopKChQbm6us8tABefm5iYXF5diH4ewAwCwkZubq5SUFBUUFDi7FEBVqlRRcHBwse6DR9gBAFgZhqHU1FS5uLgoNDT0jjdqA0qSYRi6cuWK0tPTJUk1a9a0+1iEHQCA1bVr13TlyhWFhITIy8vL2eWggvP09JQkpaenq0aNGnaf0iKyAwCs8vPzJUnu7u5OrgS47kbozsvLs/sYhB0AQCE8JxBlhSP+Fgk7AADA1Ag7AADA1FigDAC4q5iY0v28r74q3c+7WVhYmGJjYxUbG3tP/Tdv3qz27dsrIyNDVapUKdHaUHTM7AAAyr127drdczC5F7t27dJzzz13z/1btWql1NRU+fv7O6yGkuDocSovmNkBAFQIhmEoPz9frq53/+mrXr16kY7t7u6u4OBge0tDCWNmBwBQrg0ePFhbtmzR/PnzZbFYZLFYdPz4cW3evFkWi0Xr1q1Ts2bN5OHhoa1bt+ro0aN66qmnFBQUJB8fHz3yyCPasGGDzTHDwsL0xhtvWLctFovef/999ezZU15eXgoPD9fq1aut+2981oULFyRJ8fHxqlKlitatW6cGDRrIx8dHXbt2VWpqqvU9165d0wsvvKAqVaooMDBQEydO1KBBg9SjR4/bftcTJ04oJiZGVatWlbe3txo1aqQ1a9ZY9ycnJ6tbt27y8fFRUFCQBgwYoHPnzt1xnCoCZnaAuyjJtQrOXpcAmMH8+fN1+PBhRUZGatq0aZKuz8zc+CGfMGGCXnvtNdWtW1dVqlTR6dOn1a1bN02fPl2VK1fW0qVLFRMTo0OHDun++++/7edMnTpVc+bM0dy5c7VgwQL1799fJ06cUEBAwC37X7lyRa+99po++ugjVapUSc8884zGjx+vZcuWSZJmz56tZcuWacmSJWrQoIHmz5+vVatWqX379retYeTIkcrNzdW3334rb29vJScny8fHR5KUmpqqtm3b6o9//KPi4uKUnZ2tiRMnqnfv3tq4ceNtx6kiIOwAAMo1f39/ubu7y8vL65ankqZNm6ZOnTpZtwMDA9W0aVPr9vTp07Vy5UqtXr1ao0aNuu3nDB48WE8//bQkacaMGVqwYIF27typrl273rJ/Xl6e3n77bT3wwAOSpFGjRllDhiQtWLBAkyZNUs+ePSVJCxcutJmluZWTJ0/qd7/7nRo3bixJqlu3rnXfokWLFBUVpRkzZljbFi9erNDQUB0+fFgRERF3HCcz4zQWAMDUmjVrZrN9+fJlTZgwQQ0bNlSVKlXk4+Oj//znPzp58uQdj9OkSRPrf/b29pavr6/1uU234uXlZQ060vVnO93on5mZqbNnz6p58+bW/S4uLoqOjr5jDS+88IKmT5+u1q1ba/Lkydq3b591X2JiojZt2iQfHx/rq379+pKko0eP3vG4ZkfYAQCYmre3t832Sy+9pC+//FJ/+9vftHXrViUlJalx48bKzc2943Hc3Nxsti0Wyx2fDH+r/oZhFGr7tZv33+zZZ5/VsWPHNGDAAO3fv1/NmjXTggULJEkFBQWKiYlRUlKSzevIkSNq06bNHY9rdoQdAEC55+7ubn2u191s3bpVgwcPVs+ePdW4cWMFBweX+kJdf39/BQUFaefOnda2/Px8/fDDD3d9b2hoqEaMGKEVK1Zo3Lhxeu+99yRJUVFROnjwoMLCwlSvXj2b143AV5RxMhPCDgCg3AsLC9P333+v48eP69y5c3eccalXr55WrFihpKQk7d27V/369btj/5IyevRozZw5U//85z916NAhjRkzRhkZGXd8FlRsbKzWrVunlJQU7dmzRxs3blSDBg0kXV+8fP78eT399NPauXOnjh07pvXr12vo0KHWgFOUcTITFigDAO6qrF85OH78eA0aNEgNGzZUdna2UlJSbtt33rx5Gjp0qFq1aqVq1app4sSJysrKKsVqr5s4caLS0tI0cOBAubi46LnnnlOXLl3k4uJy2/fk5+dr5MiROn36tPz8/NS1a1fNmzdPkhQSEqLt27dr4sSJ6tKli3JyclS7dm117dpVlSpdn9u41TiFhYWVxtd1KotxtxOEFUBWVpb8/f2VmZkpPz8/Z5eDMoZLz1GRXL16VSkpKapTp44qV67s7HIqlIKCAjVo0EC9e/fWq6++6uxyyow7/U3e6+83MzsAADjBiRMntH79erVt21Y5OTlauHChUlJS1K9fP2eXZjqs2QEAwAkqVaqk+Ph4PfLII2rdurX279+vDRs2WNfgwHGY2QEAwAlCQ0O1fft2Z5dRITCzAwAATI2wAwAATI2wAwAATI2wAwAATI2wAwAATI2wAwCo8Nq1a6fY2FjrdlhYmN544w2n1VMapkyZooceesjZZZQKLj0HANxdSd5K/FZK+fbiK1asKPSU8vJiypQpWrVqlZKSkor0vvHjx2v06NElU9Q9CgsLU2xsrE3QLAmEHQBAhRcQEODsEgrJy8sr0QDm4+MjHx+fEjt+WcJpLABAuffFF1+ocePG8vT0VGBgoDp27KjLly9LkgYPHqwePXpo6tSpqlGjhvz8/DR8+HDl5uZa33/zaaybLVmyRP7+/kpISJAkJScnq1u3bvLx8VFQUJAGDBigc+fO3fb98fHxqlKlilatWqWIiAhVrlxZnTp10qlTp6x9bpxWWrx4serWrSsPDw8ZhqGTJ0/qqaeeko+Pj/z8/NS7d2+dPXvWetypU6dq7969slgsslgsio+PlyRlZmbqueees37nxx57THv37i30eTfcGKfXXntNNWvWVGBgoEaOHKm8vLzbfq+9e/eqffv28vX1lZ+fn6Kjo7V7927r/h07dqhNmzby9PRUaGioXnjhBet/L+3atdOJEyf04osvWmsvKYQdAEC5lpqaqqefflpDhw7Vjz/+qM2bN6tXr1769XOuv/nmG/3444/atGmTPvnkE61cuVJTp069p+O/9tprGj9+vNatW6dOnTopNTVVbdu21UMPPaTdu3fr66+/1tmzZ9W7d+87HufKlSv629/+pqVLl2r79u3KyspS3759bfr897//1WeffaYvv/zSelqqR48eOn/+vLZs2aKEhAQdPXpUffr0kST16dNH48aNU6NGjZSamqrU1FT16dNHhmHoiSeeUFpamtasWaPExERFRUWpQ4cOOn/+/G1r3LRpk44ePapNmzZp6dKlio+Pt4anW+nfv79q1aqlXbt2KTExUS+//LJ1Nmr//v3q0qWLevXqpX379unTTz/Vtm3bNGrUKEnXTx3WqlVL06ZNs9ZeUjiNBQAo11JTU3Xt2jX16tVLtWvXliQ1btzYpo+7u7sWL14sLy8vNWrUSNOmTdNLL72kV199VZUq3f7f/ZMmTdLSpUu1efNm6zEXLVqkqKgozZgxw9pv8eLFCg0N1eHDhxUREXHLY+Xl5WnhwoVq0aKFJGnp0qVq0KCBdu7cqebNm0uScnNz9dFHH6l69eqSpISEBO3bt08pKSkKDQ2VJH300Udq1KiRdu3apUceeUQ+Pj5ydXVVcHCw9bM2btyo/fv3Kz09XR4eHpKuh7ZVq1bpiy++0HPPPXfLGqtWraqFCxfKxcVF9evX1xNPPKFvvvlGf/zjH2/Z/+TJk3rppZdUv359SVJ4eLh139y5c9WvXz/rjFl4eLjefPNNtW3bVosWLVJAQIBcXFzk6+trU3tJYGYHAFCuNW3aVB06dFDjxo31hz/8Qe+9954yMjIK9fHy8rJut2zZUpcuXbI5jXSz119/Xe+88462bdtmE54SExO1adMm65oXHx8f64/90aNHb3s8V1dXNWvWzLpdv359ValSRT/++KO1rXbt2tagI0k//vijQkNDrUFHkho2bFjofTdLTEzUpUuXFBgYaFNnSkrKHWts1KiRXFxcrNs1a9ZUenr6bfuPHTtWzz77rDp27KhZs2bZHDsxMVHx8fE2n9+lSxcVFBQoJSXltscsCYQdAEC55uLiooSEBK1du1YNGzbUggUL9OCDD97TD+qd1ok8+uijys/P12effWbTXlBQoJiYGCUlJdm8jhw5ojZt2hT5837d5u3tbbPPMIxbvud27b+usWbNmoVqPHTokF566aXbvu/mBdEWi0UFBQW37T9lyhQdPHhQTzzxhDZu3KiGDRtq5cqV1hqGDx9u8/l79+7VkSNH9MADD9z2mCWB01gAgHLPYrGodevWat26tf7617+qdu3aWrlypcaOHSvp+kLa7OxseXp6SpK+++47+fj4qFatWrc9ZvPmzTV69Gh16dJFLi4u1pAQFRWlL7/8UmFhYXJ1vfef0WvXrmn37t3WU1aHDh3ShQsXrLNCt9KwYUOdPHlSp06dss7uJCcnKzMzUw0aNJB0/RRdfn6+zfuioqKUlpYmV1dXhYWF3XON9oiIiFBERIRefPFFPf3001qyZIl69uypqKgoHTx4UPXq1bvte29Ve0lgZgcAUK59//33mjFjhnbv3q2TJ09qxYoV+vnnn61hQLq+FmbYsGFKTk7W2rVrNXnyZI0aNeqO63Wk66e71q5dq2nTpmnevHmSpJEjR+r8+fN6+umntXPnTh07dkzr16/X0KFD7/jD7ebmptGjR+v777/Xnj17NGTIEP3mN7+xhp9b6dixo5o0aaL+/ftrz5492rlzpwYOHKi2bdtaT4mFhYUpJSVFSUlJOnfunHJyctSxY0e1bNlSPXr00Lp163T8+HHt2LFDr7zyis3VUsWRnZ2tUaNGafPmzTpx4oS2b9+uXbt2Wcd94sSJ+ve//62RI0daZ75Wr15tc2+fsLAwffvtt/rpp5/ueDVbcRF2AADlmp+fn7799lt169ZNEREReuWVV/T666/r8ccft/bp0KGDwsPD1aZNG/Xu3VsxMTGaMmXKPR2/devW+te//qW//OUvevPNNxUSEqLt27crPz9fXbp0UWRkpMaMGSN/f/87hicvLy9NnDhR/fr1U8uWLeXp6anly5ff8bMtFotWrVqlqlWrqk2bNurYsaPq1q2rTz/91Nrnd7/7nbp27ar27durevXq+uSTT2SxWLRmzRq1adNGQ4cOVUREhPr27avjx48rKCjonr733bi4uOiXX37RwIEDFRERod69e+vxxx+3XuXWpEkTbdmyRUeOHNGjjz6qhx9+WH/5y19Us2ZN6zGmTZum48eP64EHHrBZq+RoFuPX1+ZVUFlZWfL391dmZqb8/PycXQ7KmJK8cWwp3yQWuKurV68qJSVFderUUeXKlZ1djkMMHjxYFy5c0KpVq5xWQ3x8vGJjY3XhwgWn1VBe3elv8l5/v5nZAQAApkbYAQAApkbYAQCYWnx8vFNPYUn/O5UG5yDsAAAAUyPsAAAK4doVlBWO+Fsk7AAArG48KuDXTwQHnOnKlSuSCt/duSi4gzIAwMrV1VVeXl76+eef5ebmdteb7gElxTAMXblyRenp6apSpYrNM7uKirADALCyWCyqWbOmUlJSdOLECWeXA6hKlSrFfio6YQcAYMPd3V3h4eGcyoLTubm5FWtG5wbCDgCgkEqVKpnmDsoAJ2MBAICpMbMDOFFJPXeLZ24BwP8wswMAAEyNsAMAAEyNsAMAAEytzISdmTNnymKxKDY21tpmGIamTJmikJAQeXp6ql27djp48KDN+3JycjR69GhVq1ZN3t7e6t69u06fPl3K1QMAgLKqTISdXbt26d1331WTJk1s2ufMmaO4uDgtXLhQu3btUnBwsDp16qSLFy9a+8TGxmrlypVavny5tm3bpkuXLunJJ59Ufn5+aX8NAABQBjk97Fy6dEn9+/fXe++9p6pVq1rbDcPQG2+8oT//+c/q1auXIiMjtXTpUl25ckUff/yxJCkzM1MffPCBXn/9dXXs2FEPP/yw/vGPf2j//v3asGHDbT8zJydHWVlZNi8AAGBOTg87I0eO1BNPPKGOHTvatKekpCgtLU2dO3e2tnl4eKht27basWOHJCkxMVF5eXk2fUJCQhQZGWntcyszZ86Uv7+/9RUaGurgbwUAAMoKp4ad5cuXa8+ePZo5c2ahfWlpaZKkoKAgm/agoCDrvrS0NLm7u9vMCN3c51YmTZqkzMxM6+vUqVPF/SoAAKCMctpNBU+dOqUxY8Zo/fr1d7wlucVisdk2DKNQ283u1sfDw0MeHh5FKxhlXkndoA8AUL45bWYnMTFR6enpio6Olqurq1xdXbVlyxa9+eabcnV1tc7o3DxDk56ebt0XHBys3NxcZWRk3LYPAACo2JwWdjp06KD9+/crKSnJ+mrWrJn69++vpKQk1a1bV8HBwUpISLC+Jzc3V1u2bFGrVq0kSdHR0XJzc7Ppk5qaqgMHDlj7AACAis1pp7F8fX0VGRlp0+bt7a3AwEBre2xsrGbMmKHw8HCFh4drxowZ8vLyUr9+/SRJ/v7+GjZsmMaNG6fAwEAFBARo/Pjxaty4caEFzwAAoGIq0w8CnTBhgrKzs/X8888rIyNDLVq00Pr16+Xr62vtM2/ePLm6uqp3797Kzs5Whw4dFB8fLxcXFydWDgAAygqLYRiGs4twtqysLPn7+yszM1N+fn7OLgd2YoHy//DUcwAVwb3+fjv9PjsAAAAlibADAABMjbADAABMjbADAABMjbADAABMjbADAABMjbADAABMjbADAABMrUzfQRmAfUryBovcsBBAecPMDgAAMDXCDgAAMDXCDgAAMDXCDgAAMDXCDgAAMDXCDgAAMDXCDgAAMDXCDgAAMDXCDgAAMDXCDgAAMDXCDgAAMDWejQWgSErquVs8cwtASWFmBwAAmBphBwAAmBphBwAAmBphBwAAmBphBwAAmBpXYwEoE0rqKi+JK72Aio6ZHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGqEHQAAYGquzi4AFUtMjLMrAABUNMzsAAAAUyPsAAAAUyPsAAAAUyPsAAAAUyPsAAAAU+NqLACmV1JXAX71VckcF4BjMbMDAABMjbADAABMjbADAABMjbADAABMjbADAABMjbADAABMjbADAABMjbADAABMzalhZ9GiRWrSpIn8/Pzk5+enli1bau3atdb9hmFoypQpCgkJkaenp9q1a6eDBw/aHCMnJ0ejR49WtWrV5O3tre7du+v06dOl/VUAAEAZ5dSwU6tWLc2aNUu7d+/W7t279dhjj+mpp56yBpo5c+YoLi5OCxcu1K5duxQcHKxOnTrp4sWL1mPExsZq5cqVWr58ubZt26ZLly7pySefVH5+vrO+FgAAKEMshmEYzi7i1wICAjR37lwNHTpUISEhio2N1cSJEyVdn8UJCgrS7NmzNXz4cGVmZqp69er66KOP1KdPH0nSmTNnFBoaqjVr1qhLly739JlZWVny9/dXZmam/Pz8Suy7oeRu2w84A4+LAJzrXn+/y8yanfz8fC1fvlyXL19Wy5YtlZKSorS0NHXu3Nnax8PDQ23bttWOHTskSYmJicrLy7PpExISosjISGufW8nJyVFWVpbNCwAAmJPTw87+/fvl4+MjDw8PjRgxQitXrlTDhg2VlpYmSQoKCrLpHxQUZN2XlpYmd3d3Va1a9bZ9bmXmzJny9/e3vkJDQx38rQAAQFnh9LDz4IMPKikpSd99953+9Kc/adCgQUpOTrbut1gsNv0NwyjUdrO79Zk0aZIyMzOtr1OnThXvSwAAgDLL6WHH3d1d9erVU7NmzTRz5kw1bdpU8+fPV3BwsCQVmqFJT0+3zvYEBwcrNzdXGRkZt+1zKx4eHtYrwG68AACAOTk97NzMMAzl5OSoTp06Cg4OVkJCgnVfbm6utmzZolatWkmSoqOj5ebmZtMnNTVVBw4csPYBAAAVm6szP/z//b//p8cff1yhoaG6ePGili9frs2bN+vrr7+WxWJRbGysZsyYofDwcIWHh2vGjBny8vJSv379JEn+/v4aNmyYxo0bp8DAQAUEBGj8+PFq3LixOnbs6MyvBgAAyginhp2zZ89qwIABSk1Nlb+/v5o0aaKvv/5anTp1kiRNmDBB2dnZev7555WRkaEWLVpo/fr18vX1tR5j3rx5cnV1Ve/evZWdna0OHTooPj5eLi4uzvpaAACgDClz99lxBu6zU3q4zw7MhPvsAM5V7u6zAwAAUBIIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNTsCjspKSmOrgMAAKBE2BV26tWrp/bt2+sf//iHrl696uiaAAAAHMausLN37149/PDDGjdunIKDgzV8+HDt3LnT0bUBAAAUm11hJzIyUnFxcfrpp5+0ZMkSpaWl6be//a0aNWqkuLg4/fzzz46uEwAAwC7FWqDs6uqqnj176rPPPtPs2bN19OhRjR8/XrVq1dLAgQOVmprqqDoBAADsUqyws3v3bj3//POqWbOm4uLiNH78eB09elQbN27UTz/9pKeeespRdQIAANjFrgeBxsXFacmSJTp06JC6deumDz/8UN26dVOlStezU506dfTOO++ofv36Di0WAACgqOwKO4sWLdLQoUM1ZMgQBQcH37LP/fffrw8++KBYxQEAABSXXWHnyJEjd+3j7u6uQYMG2XN4AAAAh7Frzc6SJUv0+eefF2r//PPPtXTp0mIXBQAA4Ch2hZ1Zs2apWrVqhdpr1KihGTNmFLsoAAAAR7Er7Jw4cUJ16tQp1F67dm2dPHmy2EUBAAA4il1hp0aNGtq3b1+h9r179yowMLDYRQEAADiKXWGnb9++euGFF7Rp0ybl5+crPz9fGzdu1JgxY9S3b19H1wgAAGA3u67Gmj59uk6cOKEOHTrI1fX6IQoKCjRw4EDW7AAAgDLFrrDj7u6uTz/9VK+++qr27t0rT09PNW7cWLVr13Z0fQAAAMViV9i5ISIiQhEREY6qBQAAwOHsCjv5+fmKj4/XN998o/T0dBUUFNjs37hxo0OKAwAAKC67ws6YMWMUHx+vJ554QpGRkbJYLI6uCwAAwCHsCjvLly/XZ599pm7dujm6HgAAAIey69Jzd3d31atXz9G1AAAAOJxdYWfcuHGaP3++DMNwdD0AAAAOZddprG3btmnTpk1au3atGjVqJDc3N5v9K1ascEhxAAAAxWVX2KlSpYp69uzp6FoAAAAczq6ws2TJEkfXAQAAUCLsWrMjSdeuXdOGDRv0zjvv6OLFi5KkM2fO6NKlSw4rDgAAoLjsmtk5ceKEunbtqpMnTyonJ0edOnWSr6+v5syZo6tXr+rtt992dJ0AAAB2sWtmZ8yYMWrWrJkyMjLk6elpbe/Zs6e++eYbhxUHAABQXHZfjbV9+3a5u7vbtNeuXVs//fSTQwqDc8XEOLsCAAAcw66ZnYKCAuXn5xdqP336tHx9fYtdFAAAgKPYFXY6deqkN954w7ptsVh06dIlTZ48mUdIAACAMsWu01jz5s1T+/bt1bBhQ129elX9+vXTkSNHVK1aNX3yySeOrhG3wakmAADuzq6wExISoqSkJH3yySfas2ePCgoKNGzYMPXv399mwTIAAICz2RV2JMnT01NDhw7V0KFDHVkPAACAQ9kVdj788MM77h84cKBdxQAAADiaXWFnzJgxNtt5eXm6cuWK3N3d5eXlRdgBAABlhl1XY2VkZNi8Ll26pEOHDum3v/0tC5QBAECZYvezsW4WHh6uWbNmFZr1AQAAcCaHhR1JcnFx0ZkzZxx5SAAAgGKxa83O6tWrbbYNw1BqaqoWLlyo1q1bO6QwACjrSvJeV199VXLHBioau8JOjx49bLYtFouqV6+uxx57TK+//roj6gIAAHAIu8JOQUGBo+sAAAAoEQ5dswMAAFDW2DWzM3bs2HvuGxcXZ89HAAAAOIRdYeeHH37Qnj17dO3aNT344IOSpMOHD8vFxUVRUVHWfhaLxTFVAgAA2MmusBMTEyNfX18tXbpUVatWlXT9RoNDhgzRo48+qnHjxjm0SAAAAHtZDMMwivqm++67T+vXr1ejRo1s2g8cOKDOnTuXu3vtZGVlyd/fX5mZmfLz83N2OfesJC97BeBcXHoO3N29/n7btUA5KytLZ8+eLdSenp6uixcv2nNIAACAEmFX2OnZs6eGDBmiL774QqdPn9bp06f1xRdfaNiwYerVq5ejawQAALCbXWt23n77bY0fP17PPPOM8vLyrh/I1VXDhg3T3LlzHVogAFRE3J0ZcBy71uzccPnyZR09elSGYahevXry9vZ2ZG2lhjU7ACoSwg7MokTX7NyQmpqq1NRURUREyNvbW8XITQAAACXCrrDzyy+/qEOHDoqIiFC3bt2UmpoqSXr22We57BwAAJQpdoWdF198UW5ubjp58qS8vLys7X369NHXX3/tsOIAAACKy64FyuvXr9e6detUq1Ytm/bw8HCdOHHCIYUBAAA4gl0zO5cvX7aZ0bnh3Llz8vDwKHZRAAAAjmJX2GnTpo0+/PBD67bFYlFBQYHmzp2r9u3bO6w4AACA4rLrNNbcuXPVrl077d69W7m5uZowYYIOHjyo8+fPa/v27Y6uEQAAwG52zew0bNhQ+/btU/PmzdWpUyddvnxZvXr10g8//KAHHnjA0TUCAADYrcgzO3l5eercubPeeecdTZ06tSRqAgAAcJgiz+y4ubnpwIEDslgsxf7wmTNn6pFHHpGvr69q1KihHj166NChQzZ9DMPQlClTFBISIk9PT7Vr104HDx606ZOTk6PRo0erWrVq8vb2Vvfu3XX69Oli1wcAAMo/u05jDRw4UB988EGxP3zLli0aOXKkvvvuOyUkJOjatWvq3LmzLl++bO0zZ84cxcXFaeHChdq1a5eCg4PVqVMnm6erx8bGauXKlVq+fLm2bdumS5cu6cknn1R+fn6xawQAAOWbXc/GGj16tD788EPVq1dPzZo1K/RMrLi4OLuK+fnnn1WjRg1t2bJFbdq0kWEYCgkJUWxsrCZOnCjp+ixOUFCQZs+ereHDhyszM1PVq1fXRx99pD59+kiSzpw5o9DQUK1Zs0ZdunS56+fybCwAFQnPxoJZ3Ovvd5HW7Bw7dkxhYWE6cOCAoqKiJEmHDx+26VOc01uZmZmSpICAAElSSkqK0tLS1LlzZ2sfDw8PtW3bVjt27NDw4cOVmJhoXUd0Q0hIiCIjI7Vjx45bhp2cnBzl5ORYt7OysuyuGQAAlG1FCjvh4eFKTU3Vpk2bJF1/PMSbb76poKCgYhdiGIbGjh2r3/72t4qMjJQkpaWlSVKh4wcFBVnv1JyWliZ3d3dVrVq1UJ8b77/ZzJkzWVwNAEAFUaQ1Ozef8Vq7dq3N+priGDVqlPbt26dPPvmk0L6bZ4sMw7jrDNKd+kyaNEmZmZnW16lTp+wvHAAAlGl2LVC+wY7lPrc0evRorV69Wps2bbJ53lZwcLAkFZqhSU9Pt872BAcHKzc3VxkZGbftczMPDw/5+fnZvAAAgDkVKexYLJZCsyXFWaNjGIZGjRqlFStWaOPGjapTp47N/jp16ig4OFgJCQnWttzcXG3ZskWtWrWSJEVHR8vNzc2mT2pqqg4cOGDtAwAAKq4irdkxDEODBw+2Puzz6tWrGjFiRKGrsVasWHFPxxs5cqQ+/vhj/fOf/5Svr691Bsff31+enp6yWCyKjY3VjBkzFB4ervDwcM2YMUNeXl7q16+fte+wYcM0btw4BQYGKiAgQOPHj1fjxo3VsWPHonw9AABgQkUKO4MGDbLZfuaZZ4r14YsWLZIktWvXzqZ9yZIlGjx4sCRpwoQJys7O1vPPP6+MjAy1aNFC69evl6+vr7X/vHnz5Orqqt69eys7O1sdOnRQfHy8XFxcilUfAAAo/+y6z47ZcJ8dABUJ99mBWdzr73exFigDAACUdYQdAABgaoQdAABgaoQdAABgaoQdAABgaoQdAABgaoQdAABgaoQdAABgaoQdAABgaoQdAABgaoQdAABgaoQdAABgaoQdAABgaoQdAABgaoQdAABgaoQdAABgaoQdAABgaoQdAABgaoQdAABgaq7OLgAAULpiYkrmuF99VTLHBYqLmR0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqhB0AAGBqrs4uwOxiYpxdAQAAFRszOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNScGna+/fZbxcTEKCQkRBaLRatWrbLZbxiGpkyZopCQEHl6eqpdu3Y6ePCgTZ+cnByNHj1a1apVk7e3t7p3767Tp0+X4rcAAABlmVPDzuXLl9W0aVMtXLjwlvvnzJmjuLg4LVy4ULt27VJwcLA6deqkixcvWvvExsZq5cqVWr58ubZt26ZLly7pySefVH5+fml9DQAAUIZZDMMwnF2EJFksFq1cuVI9evSQdH1WJyQkRLGxsZo4caKk67M4QUFBmj17toYPH67MzExVr15dH330kfr06SNJOnPmjEJDQ7VmzRp16dLllp+Vk5OjnJwc63ZWVpZCQ0OVmZkpPz8/h36vmBiHHg4AyqyvvnJ2BahosrKy5O/vf9ff7zK7ZiclJUVpaWnq3Lmztc3Dw0Nt27bVjh07JEmJiYnKy8uz6RMSEqLIyEhrn1uZOXOm/P39ra/Q0NCS+yIAAMCpymzYSUtLkyQFBQXZtAcFBVn3paWlyd3dXVWrVr1tn1uZNGmSMjMzra9Tp045uHoAAFBWuDq7gLuxWCw224ZhFGq72d36eHh4yMPDwyH1AQCAsq3MzuwEBwdLUqEZmvT0dOtsT3BwsHJzc5WRkXHbPgAAoGIrs2GnTp06Cg4OVkJCgrUtNzdXW7ZsUatWrSRJ0dHRcnNzs+mTmpqqAwcOWPsAAICKzamnsS5duqT//ve/1u2UlBQlJSUpICBA999/v2JjYzVjxgyFh4crPDxcM2bMkJeXl/r16ydJ8vf317BhwzRu3DgFBgYqICBA48ePV+PGjdWxY0dnfS0AAFCGODXs7N69W+3bt7dujx07VpI0aNAgxcfHa8KECcrOztbzzz+vjIwMtWjRQuvXr5evr6/1PfPmzZOrq6t69+6t7OxsdejQQfHx8XJxcSn17wMAAMqeMnOfHWe61+v07cF9dgBUFNxnB6Wt3N9nBwAAwBEIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNQIOwAAwNRcnV0AAMAcYmJK7thffVVyx4b5MbMDAABMjZkdlGt/2emYf0q+2px/NgKAWTGzAwAATI2wAwAATI2wAwAATI2wAwAATI2wAwAATI2wAwAATI2wAwAATI377KBIuK8NzIa/acD8CDtwCkf9wACoGErqURQ8hqJi4DQWAAAwNcIOAAAwNU5jASbGehQAYGYHAACYHDM7AEoVs00AShthB1DZ+wHmajUAcBzCDoByiUAI4F6xZgcAAJgaMzuAAzHbAABlD2GnguBHGABQURF2ANwVYRlAecaaHQAAYGrM7JRx/IsaAIDiYWYHAACYGmEHAACYGmEHAACYGmEHAACYGmEHAACYGldjAYADlLWHyQL4H2Z2AACAqRF2AACAqXEaCwDKEE6HwZliSug+tl85+c+RsAMAQAkwa3AojziNBQAATI2ZHQBAhVVSsy8oW5jZAQAApmaasPPWW2+pTp06qly5sqKjo7V161ZnlwQAAMoAU5zG+vTTTxUbG6u33npLrVu31jvvvKPHH39cycnJuv/++51dHgCUOq7qMi9OvRWdKWZ24uLiNGzYMD377LNq0KCB3njjDYWGhmrRokXOLg0AADhZuZ/Zyc3NVWJiol5++WWb9s6dO2vHjh23fE9OTo5ycnKs25mZmZKkrKwsh9eXl1e8918qKOYBAKAYXvyuq7NLsDEn+jOHHGdCYm+HHMdR9ZhdCfy8/v/HvX5gwzDu2K/ch51z584pPz9fQUFBNu1BQUFKS0u75XtmzpypqVOnFmoPDQ0tkRqLY52zCwCAsmSdv2MO45CjyGH1mJ1/CQ/TxYsX5X+HDyn3YecGi8Vis20YRqG2GyZNmqSxY8datwsKCnT+/HkFBgbe9j2lJSsrS6GhoTp16pT8/PycWktFwHiXPsa89DHmpY8xLx2GYejixYsKCQm5Y79yH3aqVasmFxeXQrM46enphWZ7bvDw8JCHh4dNW5UqVUqqRLv4+fnxP5BSxHiXPsa89DHmpY8xL3l3mtG5odwvUHZ3d1d0dLQSEhJs2hMSEtSqVSsnVQUAAMqKcj+zI0ljx47VgAED1KxZM7Vs2VLvvvuuTp48qREjRji7NAAA4GSmCDt9+vTRL7/8omnTpik1NVWRkZFas2aNateu7ezSiszDw0OTJ08udJoNJYPxLn2MeeljzEsfY162WIy7Xa8FAABQjpX7NTsAAAB3QtgBAACmRtgBAACmRtgBAACmRtgpZW+99Zbq1KmjypUrKzo6Wlu3br1j/y1btig6OlqVK1dW3bp19fbbb5dSpeZRlDFfsWKFOnXqpOrVq8vPz08tW7bUunU8tKOoivp3fsP27dvl6uqqhx56qGQLNKGijnlOTo7+/Oc/q3bt2vLw8NADDzygxYsXl1K15lDUMV+2bJmaNm0qLy8v1axZU0OGDNEvv/xSStVWcAZKzfLlyw03NzfjvffeM5KTk40xY8YY3t7exokTJ27Z/9ixY4aXl5cxZswYIzk52XjvvfcMNzc344svvijlysuvoo75mDFjjNmzZxs7d+40Dh8+bEyaNMlwc3Mz9uzZU8qVl19FHfMbLly4YNStW9fo3Lmz0bRp09Ip1iTsGfPu3bsbLVq0MBISEoyUlBTj+++/N7Zv316KVZdvRR3zrVu3GpUqVTLmz59vHDt2zNi6davRqFEjo0ePHqVcecVE2ClFzZs3N0aMGGHTVr9+fePll1++Zf8JEyYY9evXt2kbPny48Zvf/KbEajSboo75rTRs2NCYOnWqo0szLXvHvE+fPsYrr7xiTJ48mbBTREUd87Vr1xr+/v7GL7/8UhrlmVJRx3zu3LlG3bp1bdrefPNNo1atWiVWI/6H01ilJDc3V4mJiercubNNe+fOnbVjx45bvuff//53of5dunTR7t27lZeXV2K1moU9Y36zgoICXbx4UQEBASVRounYO+ZLlizR0aNHNXny5JIu0XTsGfPVq1erWbNmmjNnju677z5FRERo/Pjxys7OLo2Syz17xrxVq1Y6ffq01qxZI8MwdPbsWX3xxRd64oknSqPkCs8Ud1AuD86dO6f8/PxCDycNCgoq9BDTG9LS0m7Z/9q1azp37pxq1qxZYvWagT1jfrPXX39dly9fVu/evUuiRNOxZ8yPHDmil19+WVu3bpWrK/+XVFT2jPmxY8e0bds2Va5cWStXrtS5c+f0/PPP6/z586zbuQf2jHmrVq20bNky9enTR1evXtW1a9fUvXt3LViwoDRKrvCY2SllFovFZtswjEJtd+t/q3bcXlHH/IZPPvlEU6ZM0aeffqoaNWqUVHmmdK9jnp+fr379+mnq1KmKiIgorfJMqSh/5wUFBbJYLFq2bJmaN2+ubt26KS4uTvHx8czuFEFRxjw5OVkvvPCC/vrXvyoxMVFff/21UlJSeIZjKeGfUaWkWrVqcnFxKZT609PTC/3r4Ibg4OBb9nd1dVVgYGCJ1WoW9oz5DZ9++qmGDRumzz//XB07dizJMk2lqGN+8eJF7d69Wz/88INGjRol6foPsWEYcnV11fr16/XYY4+VSu3llT1/5zVr1tR9990nf39/a1uDBg1kGIZOnz6t8PDwEq25vLNnzGfOnKnWrVvrpZdekiQ1adJE3t7eevTRRzV9+nRm6ksYMzulxN3dXdHR0UpISLBpT0hIUKtWrW75npYtWxbqv379ejVr1kxubm4lVqtZ2DPm0vUZncGDB+vjjz/mfHoRFXXM/fz8tH//fiUlJVlfI0aM0IMPPqikpCS1aNGitEovt+z5O2/durXOnDmjS5cuWdsOHz6sSpUqqVatWiVarxnYM+ZXrlxRpUq2P7kuLi6S/jdjjxLkrJXRFdGNSxU/+OADIzk52YiNjTW8vb2N48ePG4ZhGC+//LIxYMAAa/8bl56/+OKLRnJysvHBBx9w6XkRFXXMP/74Y8PV1dX4+9//bqSmplpfFy5ccNZXKHeKOuY342qsoivqmF+8eNGoVauW8fvf/944ePCgsWXLFiM8PNx49tlnnfUVyp2ijvmSJUsMV1dX46233jKOHj1qbNu2zWjWrJnRvHlzZ32FCoWwU8r+/ve/G7Vr1zbc3d2NqKgoY8uWLdZ9gwYNMtq2bWvTf/PmzcbDDz9suLu7G2FhYcaiRYtKueLyryhj3rZtW0NSodegQYNKv/ByrKh/579G2LFPUcf8xx9/NDp27Gh4enoatWrVMsaOHWtcuXKllKsu34o65m+++abRsGFDw9PT06hZs6bRv39/4/Tp06VcdcVkMQzmzwAAgHmxZgcAAJgaYQcAAJgaYQcAAJgaYQcAAJgaYQcAAJgaYQcAAJgaYQcAAJgaYQcAAJgaYQcA7iIsLExvvPGGs8sAYCfCDgCHGzx4sHr06FGoffPmzbJYLLpw4YLNtsViUaVKleTv76+HH35YEyZMUGpqqs17+/TpoxYtWig/P9/alpeXp6ioKD3zzDMl+XUAlHOEHQBOd+jQIZ05c0a7du3SxIkTtWHDBkVGRmr//v3WPm+99ZZOnDihWbNmWdteffVVpaWlacGCBc4oG0A5QdgB4HQ1atRQcHCwIiIi1LdvX23fvl3Vq1fXn/70J2ufwMBAvfvuu5o2bZr27dunxMREzZw5U++//76qVq16y+O+8847uu+++1RQUGDT3r17dw0aNEiSdPToUT311FMKCgqSj4+PHnnkEW3YsOG2tR4/flwWi0VJSUnWtgsXLshisWjz5s3WtuTkZHXr1k0+Pj4KCgrSgAEDdO7cOTtGB0BxEXYAlDmenp4aMWKEtm/frvT0dGt79+7d1bdvXw0cOFADBw7UoEGD1K1bt9se5w9/+IPOnTunTZs2WdsyMjK0bt069e/fX5J06dIldevWTRs2bNAPP/ygLl26KCYmRidPnrS7/tTUVLVt21YPPfSQdu/era+//lpnz55V79697T4mAPu5OrsAAOb0f//3f/Lx8bFp+/V6m7upX7++pOszKTVq1LC2z58/XyEhIfLz81NcXNwdjxEQEKCuXbvq448/VocOHSRJn3/+uQICAqzbTZs2VdOmTa3vmT59ulauXKnVq1dr1KhR91zvry1atEhRUVGaMWOGtW3x4sUKDQ3V4cOHFRERYddxAdiHmR0AJaJ9+/ZKSkqyeb3//vv3/H7DMCRJFovFpv3jjz+WxWLRuXPn9J///Oeux+nfv7++/PJL5eTkSJKWLVumvn37ysXFRZJ0+fJlTZgwQQ0bNlSVKlXk4+Oj//znP8Wa2UlMTNSmTZvk4+Njfd0Ib0ePHrX7uADsw8wOgBLh7e2tevXq2bSdPn36nt//448/Srp+2fcNx44d04QJE7Rw4UJt375dgwcP1g8//CAPD4/bHicmJkYFBQX617/+pUceeURbt261mRF66aWXtG7dOr322muqV6+ePD099fvf/165ubm3PF6lStf/jXgjjEnXrwr7tYKCAsXExGj27NmF3l+zZs27f3kADkXYAVDmZGdn691331WbNm1UvXp1SdcDxJAhQ9SuXTsNGTJEv/vd7xQZGanJkyfbXKF1M09PT/Xq1UvLli3Tf//7X0VERCg6Otq6f+vWrRo8eLB69uwp6foanuPHj9/2eDfqSU1N1cMPPyxJNouVJSkqKkpffvmlwsLC5OrK/80CzsZpLABOl56errS0NB05ckTLly9X69atde7cOS1atMjaZ/78+dq/f7/ee+89SZKfn5/ef/99vf7669q5c+cdj9+/f3/961//0uLFiwvdk6devXpasWKFkpKStHfvXvXr16/Q1Vu/5unpqd/85jeaNWuWkpOT9e233+qVV16x6TNy5EidP39eTz/9tHbu3Kljx45p/fr1Gjp0aJHWLQFwDMIOAKd78MEHFRISoujoaM2aNUsdO3bUgQMH1LBhQ0nS4cOH9ec//1kLFy60OQ3UuXNnDRkyRIMHD7auybmVxx57TAEBATp06JD69etns2/evHmqWrWqWrVqpZiYGHXp0kVRUVF3rHfx4sXKy8tTs2bNNGbMGE2fPt1mf0hIiLZv3678/Hx16dJFkZGRGjNmjPz9/a2nwQCUHovx6xPPAAAAJsM/MQAAgKkRdgAAgKkRdgAAgKkRdgAAgKkRdgAAgKkRdgAAgKkRdgAAgKkRdgAAgKkRdgAAgKkRdgAAgKkRdgAAgKn9fwU86WjSp5AOAAAAAElFTkSuQmCC",
      "text/plain": [
       "<Figure size 640x480 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import matplotlib.pyplot as plt\n",
    "\n",
    "plt.figure()\n",
    "# plot the bar chart for Y_data and COVID_Y_data\n",
    "Y_data = np.array(Y_data)\n",
    "COVID_Y_data = np.array(COVID_Y_data)\n",
    "#Y_bin = np.digitize(Y_data, [0.2, 0.4, 0.6, 0.8])\n",
    "#Y_bin = np.append(Y_bin, covid_bin)\n",
    "#Y_data = np.append(Y_data, COVID_Y_data)\n",
    "plt.hist(Y_data, bins=20, color='blue', alpha=0.7, label='training set')\n",
    "plt.hist(COVID_Y_data, bins=20, color='red', alpha=0.7, label='spike protein set')\n",
    "plt.xlabel('HDX value')\n",
    "plt.ylabel('Frequency')\n",
    "plt.legend()\n",
    "#plt.show()\n",
    "plt.savefig('/home/lwang/models/HDX_LSTM/data/Fullset/HDX_hist.png')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "py3.9",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.19"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
