{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "36\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "import shutil\n",
    "\n",
    "pdb_path = '/home/lwang/AI-HDX-main/HDX_MS_dataset/database_collection'\n",
    "# Get a list of all pdb files in the subfolder\n",
    "save_path = '/home/lwang/AI-HDX-main/HDX_MS_dataset/database_collection/feature/HDX_files'\n",
    "subfolder = os.listdir(pdb_path)\n",
    "count = 0\n",
    "for folder in subfolder:\n",
    "    if folder.split('.')[0] == '':\n",
    "        continue\n",
    "    elif folder == 'feature':\n",
    "        continue\n",
    "    \n",
    "    if os.path.isfile(f'{pdb_path}/{folder}/apo_revised_modified.xlsx'):\n",
    "        shutil.copy(os.path.join(pdb_path, folder, f'apo_revised_modified.xlsx'), os.path.join(save_path, f'{folder}.xlsx'))\n",
    "        count += 1\n",
    "print(count)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "feature output: dssp, rigidity"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "start processing model_3_revised\n",
      "start processing model_4_revised\n",
      "start processing model_5_revised\n",
      "start processing model_6_revised\n",
      "start processing model_7_revised\n",
      "start processing model_8_revised\n",
      "start processing model_9_revised\n",
      "start processing model_10_revised\n",
      "start processing model_11_revised\n",
      "start processing model_12_revised\n",
      "start processing model_13_revised\n",
      "start processing model_14_revised\n",
      "start processing model_15_revised\n",
      "start processing model_16_revised\n",
      "start processing model_17_revised\n",
      "start processing model_18_revised\n",
      "start processing model_19_revised\n",
      "start processing model_20_revised\n",
      "start processing model_21_revised\n",
      "start processing model_22_revised\n",
      "start processing model_23_revised\n",
      "start processing model_24_revised\n",
      "start processing model_25_revised\n",
      "start processing model_26_revised\n",
      "start processing model_27_revised\n",
      "start processing model_28_revised\n",
      "start processing model_29_revised\n",
      "start processing model_30_revised\n",
      "start processing model_31_revised\n",
      "start processing model_32_revised\n",
      "start processing model_33_revised\n",
      "start processing model_34_revised\n",
      "start processing model_35_revised\n",
      "start processing model_36_revised\n",
      "start processing model_37_revised\n",
      "start processing model_38_revised\n",
      "start processing model_39_revised\n",
      "start processing model_40_revised\n",
      "start processing model_41_revised\n",
      "start processing model_42_revised\n",
      "start processing model_43_revised\n",
      "start processing model_44_revised\n",
      "start processing model_45_revised\n",
      "start processing model_46_revised\n",
      "start processing model_47_revised\n",
      "start processing model_48_revised\n",
      "start processing model_49_revised\n",
      "start processing model_50_revised\n",
      "start processing model_51_revised\n",
      "start processing model_52_revised\n",
      "start processing model_53_revised\n",
      "start processing model_54_revised\n",
      "start processing model_55_revised\n",
      "start processing model_57_revised\n",
      "start processing model_58_revised\n",
      "start processing model_60_revised\n",
      "start processing model_62_revised\n",
      "start processing model_63_revised\n",
      "start processing model_65_revised\n",
      "start processing model_66_revised\n",
      "start processing model_68_revised\n",
      "start processing model_70_revised\n",
      "start processing model_71_revised\n",
      "start processing model_73_revised\n",
      "start processing model_74_revised\n",
      "start processing model_76_revised\n",
      "start processing model_78_revised\n",
      "start processing model_79_revised\n",
      "start processing model_81_revised\n",
      "start processing model_82_revised\n",
      "start processing model_84_revised\n",
      "start processing model_86_revised\n",
      "start processing model_87_revised\n",
      "start processing model_89_revised\n",
      "start processing model_90_revised\n",
      "start processing rtt_bcd_6thl\n",
      "start processing rtt_bcd_AFpred_rank1\n",
      "start processing rtt_bcd_AFpred_rank2\n",
      "start processing rtt_bcd_AFpred_rank3\n",
      "start processing rtt_bcd_AFpred_rank4\n",
      "start processing rtt_bcd_AFpred_rank5\n",
      "start processing model_1_revised\n",
      "start processing model_2_revised\n",
      "start processing model_56_revised\n",
      "start processing model_59_revised\n",
      "start processing model_61_revised\n",
      "start processing model_64_revised\n",
      "start processing model_67_revised\n",
      "start processing model_69_revised\n",
      "start processing model_72_revised\n",
      "start processing model_75_revised\n",
      "start processing model_77_revised\n",
      "start processing model_80_revised\n",
      "start processing model_83_revised\n",
      "start processing model_85_revised\n",
      "start processing model_88_revised\n",
      "start processing model_91_revised\n",
      "start processing model_92_revised\n",
      "start processing model_93_revised\n",
      "start processing model_94_revised\n",
      "start processing model_95_revised\n",
      "start processing model_96_revised\n",
      "start processing model_97_revised\n",
      "start processing model_98_revised\n",
      "start processing model_99_revised\n",
      "start processing model_100_revised\n",
      "start processing rtt_3FSS\n",
      "start processing bcd_6NZ2\n",
      "start processing rtt+bcd_complex_revised\n",
      "[]\n"
     ]
    }
   ],
   "source": [
    "## output dssp (RSA) for each amino acid  ##\n",
    "from RUN_FIRST import pdb_clean\n",
    "from output_dssp import output_dssp\n",
    "import pandas as pd\n",
    "import os\n",
    "import warnings\n",
    "from Bio import PDB\n",
    "\n",
    "warnings.filterwarnings(\"ignore\")\n",
    "\n",
    "root_dir = '/home/lwang/models/HDX_LSTM/data/RTT_BCD'\n",
    "pdb_dir = f'{root_dir}/structure'\n",
    "dssp_dir = f'{root_dir}/dssp_files'\n",
    "if os.path.isdir(dssp_dir) == False:\n",
    "    os.mkdir(dssp_dir)\n",
    "\n",
    "#fname = 'COVID_record.xlsx'\n",
    "#apo_df = pd.read_excel(f'{root_dir}/{fname}', sheet_name='Sheet1')\n",
    "#apo_df = apo_df.dropna(subset=['apo_identifier'])\n",
    "fail_list = []\n",
    "\n",
    "#for index, row in apo_df.iterrows():\n",
    "#    pdb_fname = row['apo_identifier'].strip().split('.')[0]\n",
    "    #chain_id = row['chain_identifier'].strip()\n",
    "    #uni_id = row['match_uni'].strip()\n",
    "for pdb in os.listdir(pdb_dir):\n",
    "    pdb_fname = pdb.split('.')[0]        \n",
    "    if os.path.isfile(f'{root_dir}/structure/{pdb_fname}.pdb'):\n",
    "        try:\n",
    "            parser = PDB.PDBParser()\n",
    "            filepath = f'{root_dir}/structure/{pdb_fname}.pdb'\n",
    "            structure = parser.get_structure('AF_structure', filepath)\n",
    "            chain_ids = list(structure.get_chains())\n",
    "\n",
    "            if not os.path.isfile(f'{root_dir}/structure/{pdb_fname}_clean.pdb'):\n",
    "                pdb_clean(f'{root_dir}/structure/{pdb_fname}.pdb', f'{root_dir}/structure/{pdb_fname}_clean.pdb')\n",
    "            print('start processing', f'{pdb_fname}')    \n",
    "\n",
    "            \n",
    "            for chain_id in chain_ids:\n",
    "                chain_id = chain_id.get_id()\n",
    "                if os.path.isfile(f'{root_dir}/dssp_files/{pdb_fname}_{chain_id}.dssp.txt'):\n",
    "                    continue\n",
    "            \n",
    "            output_dssp(f'{pdb_fname}', f'{root_dir}/structure/{pdb_fname}_clean.pdb', f'{root_dir}/dssp_files', \n",
    "                        dssp_path='/home/lwang/models/mambaforge/envs/liyao_env/bin/mkdssp')\n",
    "\n",
    "        except Exception as e:\n",
    "            fail_list.append(pdb_fname)\n",
    "            print(e)\n",
    "            continue\n",
    "    else:\n",
    "        print(f'{root_dir}/structure/{pdb_fname}.pdb does not exist')\n",
    "print(fail_list) #FIXME: add b-factor col to the pdb file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio.Data.PDBData import protein_letters_1to3\n",
    "from Bio.PDB.Polypeptide import is_aa, protein_letters_3to1_extended\n",
    "non_StdAA = ['TRO', 'PYR', 'SNN', 'PTR', 'PCA']\n",
    "\n",
    "def get_pdb_seq(structure, chain_id = 'A', fill_gap = True):\n",
    "    if len(structure) == 0: #if succussfully read the pdb file\n",
    "        return None, None\n",
    "    \n",
    "    chains = list(structure.get_chains())\n",
    "\n",
    "    for chain in chains:\n",
    "        if not chain_id == chain.id:\n",
    "            continue\n",
    "\n",
    "        sequence = \"\"\n",
    "\n",
    "        previous_residue_id = None\n",
    "        residue_indices = {}\n",
    "\n",
    "        for residue in chain:\n",
    "            residue_name = residue.get_resname()\n",
    "            #if residue_name == \"MSE\":\n",
    "            #    residue_name = \"MET\"\n",
    "\n",
    "            if not is_aa(residue_name, standard=False) and residue_name != \"MET\":\n",
    "                continue  # Skip the non-amino acid residue\n",
    "            else:\n",
    "                if residue_name in non_StdAA:\n",
    "                    aa = '-'\n",
    "                else:\n",
    "                    aa = protein_letters_3to1_extended.get(residue_name)\n",
    "                idx = int(residue.id[1])\n",
    "                residue_indices[idx] = protein_letters_3to1_extended.get(residue_name) # this makes sequence have '-' while residue_list transform to normal aa\n",
    "\n",
    "            # Check for discontinuity\n",
    "            current_residue_id = residue.get_id()[1]\n",
    "            if fill_gap:\n",
    "                if previous_residue_id is not None and current_residue_id != previous_residue_id + 1:\n",
    "                    gap_size = current_residue_id - previous_residue_id - 1\n",
    "                    sequence += \"-\" * gap_size\n",
    "            try:\n",
    "                sequence += aa\n",
    "            except Exception as e:\n",
    "                print(residue_name, aa, '//')\n",
    "                return False, False\n",
    "            previous_residue_id = current_residue_id\n",
    "\n",
    "    return sequence, residue_indices"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "start processing 1EUG\n",
      "start processing 1UGH\n",
      "start processing 1UGI\n",
      "start processing 2ERK\n",
      "start processing 3DVL\n",
      "start processing 3KVD\n",
      "start processing 4KSO\n",
      "start processing 4QTE\n",
      "start processing 5K4I\n",
      "start processing 5UMO\n",
      "start processing 6GDQ\n",
      "start processing 6OPH\n",
      "start processing 6OPK\n",
      "start processing 8U8J\n",
      "start processing 8U8K\n",
      "[]\n"
     ]
    }
   ],
   "source": [
    "### output fasta file for hhblits ###\n",
    "#from predataprocess import get_pdb_seq\n",
    "from Bio import PDB\n",
    "import pandas as pd\n",
    "import os\n",
    "import warnings\n",
    "warnings.filterwarnings(\"ignore\")\n",
    "\n",
    "root_dir = '/home/lwang/models/HDX_LSTM/data/Latest_test'\n",
    "fail_list = []\n",
    "pdb_dir = f'{root_dir}/revised_structure'\n",
    "pdb_list = os.listdir(pdb_dir)\n",
    "for pdb_name in pdb_list:\n",
    "    pdb_fname = pdb_name.split('.')[0]\n",
    "    #if pdb_fname.startswith('model'):\n",
    "    #    continue\n",
    "    print('start processing', f'{pdb_fname}')        \n",
    "    if os.path.isfile(f'{pdb_dir}/{pdb_fname}.pdb'):\n",
    "        #try:\n",
    "        parser = PDB.PDBParser()\n",
    "        filepath = f'{pdb_dir}/{pdb_fname}.pdb'\n",
    "        structure = parser.get_structure('AF_structure', filepath)\n",
    "        chain_ids = list(structure.get_chains())\n",
    "        for chain_id in chain_ids:\n",
    "            chain_id = chain_id.get_id()\n",
    "            if chain_id == 'Z': #skip the heteroatoms chain\n",
    "                continue\n",
    "            sequence, residue_indices = get_pdb_seq(structure, chain_id)\n",
    "            if os.path.isdir(f'{root_dir}/fasta_files') == False:\n",
    "                os.mkdir(f'{root_dir}/fasta_files')\n",
    "            with open(f'{root_dir}/fasta_files/{pdb_fname}_{chain_id}.fasta', 'w') as f:\n",
    "                f.write(f'>{pdb_fname}_{chain_id}\\n')\n",
    "                f.write(sequence)\n",
    "        #except Exception as e:\n",
    "        #    print(e)\n",
    "        #    fail_list.append(pdb_fname+'_'+chain_id)\n",
    "        #    continue\n",
    "    else:\n",
    "        print(f'{pdb_dir}/{pdb_fname}.pdb does not exist')\n",
    "print(fail_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "start processing bcd_6NZ2\n",
      "start processing rtt_3FSS\n",
      "start processing rtt_bcd_6thl\n",
      "start processing rtt+bcd_complex_revised\n",
      "start processing model_1_revised\n",
      "start processing model_2_revised\n",
      "start processing model_3_revised\n",
      "start processing model_4_revised\n",
      "start processing model_5_revised\n",
      "start processing model_6_revised\n",
      "start processing model_7_revised\n",
      "start processing model_8_revised\n",
      "start processing model_9_revised\n",
      "start processing model_10_revised\n",
      "start processing model_11_revised\n",
      "start processing model_12_revised\n",
      "start processing model_13_revised\n",
      "start processing model_14_revised\n",
      "start processing model_15_revised\n",
      "start processing model_16_revised\n",
      "start processing model_17_revised\n",
      "start processing model_18_revised\n",
      "start processing model_19_revised\n",
      "start processing model_20_revised\n",
      "start processing model_21_revised\n",
      "start processing model_22_revised\n",
      "start processing model_23_revised\n",
      "start processing model_24_revised\n",
      "start processing model_25_revised\n",
      "start processing model_26_revised\n",
      "start processing model_27_revised\n",
      "start processing model_28_revised\n",
      "start processing model_29_revised\n",
      "start processing model_30_revised\n",
      "start processing model_31_revised\n",
      "start processing model_32_revised\n",
      "start processing model_33_revised\n",
      "start processing model_34_revised\n",
      "start processing model_35_revised\n",
      "start processing model_36_revised\n",
      "start processing model_37_revised\n",
      "start processing model_38_revised\n",
      "start processing model_39_revised\n",
      "start processing model_40_revised\n",
      "start processing model_41_revised\n",
      "start processing model_42_revised\n",
      "start processing model_43_revised\n",
      "start processing model_44_revised\n",
      "start processing model_45_revised\n",
      "start processing model_46_revised\n",
      "start processing model_47_revised\n",
      "start processing model_48_revised\n",
      "start processing model_49_revised\n",
      "start processing model_50_revised\n",
      "start processing model_51_revised\n",
      "start processing model_52_revised\n",
      "start processing model_53_revised\n",
      "start processing model_54_revised\n",
      "start processing model_55_revised\n",
      "start processing model_56_revised\n",
      "start processing model_57_revised\n",
      "start processing model_58_revised\n",
      "start processing model_59_revised\n",
      "start processing model_60_revised\n",
      "start processing model_61_revised\n",
      "start processing model_62_revised\n",
      "start processing model_63_revised\n",
      "start processing model_64_revised\n",
      "start processing model_65_revised\n",
      "start processing model_66_revised\n",
      "start processing model_67_revised\n",
      "start processing model_68_revised\n",
      "start processing model_69_revised\n",
      "start processing model_70_revised\n",
      "start processing model_71_revised\n",
      "start processing model_72_revised\n",
      "start processing model_73_revised\n",
      "start processing model_74_revised\n",
      "start processing model_75_revised\n",
      "start processing model_76_revised\n",
      "start processing model_77_revised\n",
      "start processing model_78_revised\n",
      "start processing model_79_revised\n",
      "start processing model_80_revised\n",
      "start processing model_81_revised\n",
      "start processing model_82_revised\n",
      "start processing model_83_revised\n",
      "start processing model_84_revised\n",
      "start processing model_85_revised\n",
      "start processing model_86_revised\n",
      "start processing model_87_revised\n",
      "start processing model_88_revised\n",
      "start processing model_89_revised\n",
      "start processing model_90_revised\n",
      "start processing model_91_revised\n",
      "start processing model_92_revised\n",
      "start processing model_93_revised\n",
      "start processing model_94_revised\n",
      "start processing model_95_revised\n",
      "start processing model_96_revised\n",
      "start processing model_97_revised\n",
      "start processing model_98_revised\n",
      "start processing model_99_revised\n",
      "start processing model_100_revised\n"
     ]
    }
   ],
   "source": [
    "### output fasta file for hhblits ###\n",
    "from Bio import PDB\n",
    "from Bio.PDB.Polypeptide import CaPPBuilder\n",
    "import warnings\n",
    "import os\n",
    "import pandas as pd\n",
    "\n",
    "warnings.filterwarnings(\"ignore\")\n",
    "root_dir = '/home/lwang/models/HDX_LSTM/data/Latest_set'\n",
    "fail_list = []\n",
    "#pdb_list = os.listdir(f'{root_dir}/structure')\n",
    "df = pd.read_excel(f'{root_dir}/merged_data.xlsx', sheet_name='Sheet1')\n",
    "df = df.dropna(subset=['structure_file'])\n",
    "df = df.drop_duplicates(subset=['structure_file'])\n",
    "pdb_list = df['apo_identifier'].tolist()\n",
    "for pdb_name in pdb_list:\n",
    "    pdb_fname = pdb_name.split('.')\n",
    "    if len(pdb_fname) > 1:\n",
    "        pdb_fname = ''.join(pdb_fname[:-1])\n",
    "    else:\n",
    "        pdb_fname = pdb_fname[0]\n",
    "    print('start processing', f'{pdb_fname}')        \n",
    "    if os.path.isfile(f'{root_dir}/structure/{pdb_fname}.pdb'):\n",
    "        try:\n",
    "            parser = PDB.PDBParser()\n",
    "            filepath = f'{root_dir}/structure/{pdb_fname}.pdb'\n",
    "            structure = parser.get_structure('AF_structure', filepath)\n",
    "            chains = structure.get_chains()\n",
    "            for chain in chains:\n",
    "                chain_id = chain.get_id()\n",
    "                hhm_path = f'{root_dir}/hhm_files/{pdb_fname}_{chain_id}.hhm'\n",
    "                if os.path.isfile(hhm_path):\n",
    "                    continue\n",
    "\n",
    "                ppb = CaPPBuilder()\n",
    "                seq = ''\n",
    "                for pp in ppb.build_peptides(chain, aa_only=False):\n",
    "                    seq+=str(pp.get_sequence())\n",
    "                if os.path.isdir(f'{root_dir}/fasta_files') == False:\n",
    "                    os.mkdir(f'{root_dir}/fasta_files')\n",
    "                with open(f'{root_dir}/fasta_files/{pdb_fname}_{chain_id}.fasta', 'w') as f:\n",
    "                    f.write(f'>{pdb_fname}_{chain_id}\\n')\n",
    "                    f.write(seq)\n",
    "        except Exception as e:\n",
    "            print(e)\n",
    "            fail_list.append(pdb_fname)\n",
    "            continue"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "### secondary structure one-hot encoding ###\n",
    "root_dir = '/home/lwang/models/HDX_LSTM/data/test_set'\n",
    "dssp_dir = f'{root_dir}/dssp_files'\n",
    "\n",
    "def one_of_k_encoding(x, allowable_set):\n",
    "    if x not in allowable_set:\n",
    "        raise Exception(\"input {0} not in allowable set{1}:\".format(x, allowable_set))\n",
    "    return [int(x == s) for s in allowable_set]\n",
    "\n",
    "folder_list = os.listdir(dssp_dir)\n",
    "for folder in folder_list:\n",
    "    if folder == '.DS_Store':\n",
    "        continue\n",
    "\n",
    "    dssp_file = f'{dssp_dir}/{folder}'\n",
    "    with open(dssp_file, 'r') as f:\n",
    "        for line in f:\n",
    "            content = line.split()\n",
    "            dssp_value = round(float(content[3]), 4)\n",
    "            ss_type = content[2]\n",
    "            ss_list = ['H', 'B', 'E', 'G', 'I', 'T', 'S', 'P', '-']\n",
    "            ss_encoding = one_of_k_encoding(ss_type, ss_list)\n",
    "            print(ss_type)\n",
    "            print(ss_encoding)\n",
    "/home/lwang/AI-HDX-main/HDX_MS_dataset/database_collection/feature/fasta_files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "### add H to pdb file, then run FIRST ###\n",
    "### usually need to run separately in terminal ###\n",
    "import os\n",
    "import shutil\n",
    "from RUN_FIRST import RUN_FIRST, copyfiles, pdb_clean, hbplus\n",
    "\n",
    "root_dir = '/home/lwang/models/HDX_LSTM/data/COVID_SPIKE'\n",
    "proflex_dir = f'/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/ProFlex-master/proflex'\n",
    "hbplus_dir = '/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus'\n",
    "\n",
    "pdb_dir = f'{root_dir}/structure'\n",
    "save_dir = f'{root_dir}/proflex_files'\n",
    "\n",
    "if os.path.isdir(save_dir) == False:\n",
    "    os.mkdir(save_dir)\n",
    "\n",
    "fname = 'COVID_record.xlsx'\n",
    "apo_df = pd.read_excel(f'{root_dir}/{fname}', sheet_name='Sheet1')\n",
    "apo_df = apo_df.dropna(subset=['chain_identifier'])\n",
    "\n",
    "for index, row in apo_df.iterrows():\n",
    "    pdb_fname = row['apo_identifier'].strip().split('.')[0]\n",
    "    #chain_id = row['chain_identifier'].strip()\n",
    "    #uni_id = row['match_uni'].strip()\n",
    "\n",
    "    pdb_fpath = f'{root_dir}/structure/{pdb_fname}.pdb'\n",
    "\n",
    "    proflex_name = f'{pdb_fname}_clean_Hplus_proflexdataset'\n",
    "    if os.path.isfile(f'{save_dir}/{proflex_name}'):\n",
    "        continue           \n",
    "    \n",
    "    if os.path.isfile(pdb_fpath):\n",
    "        try:\n",
    "            print('start processing', f'{pdb_fname}') \n",
    "            pdb_folder = f'{root_dir}/structure/'\n",
    "            if pdb_clean(pdb_fpath, f'{hbplus_dir}/{pdb_fname}_clean.pdb') == False:\n",
    "                raise Exception('pdb_clean failed')\n",
    "            hbplus(hbplus_dir, f'{pdb_fname}_clean', pdb_folder)\n",
    "\n",
    "            pdb_fname = pdb_fname+'_clean_Hplus'       \n",
    "            pdb_clean(f'{pdb_folder}/{pdb_fname}.pdb', f'{proflex_dir}/{pdb_fname}.pdb')\n",
    "            RUN_FIRST(proflex_dir, f'{pdb_fname}.pdb', '-h')\n",
    "            copyfiles(pdb_fname, proflex_dir, save_dir)\n",
    "\n",
    "        except Exception as e:\n",
    "            fail_list.append(pdb_fname)\n",
    "            print(e)\n",
    "            continue\n",
    "print(fail_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "### add H to single pdb file ###\n",
    "from RUN_FIRST import hbplus, pdb_clean\n",
    "\n",
    "hbplus_dir = '/Users/liyao/Desktop/Tsuda_Lab/Source_code/hbplus/'\n",
    "pdb_dir = f'/Users/liyao/Desktop/Tsuda_Lab/Source_code/AI-HDX-main/HDX_MS_dataset/database_collection/feature/structure/B5BTR7'\n",
    "file = 'AF-B5BTR7-F1-model_v4.pdb'\n",
    "\n",
    "pdb_clean(f'{pdb_dir}/{file}', f'{hbplus_dir}/{file}')\n",
    "hbplus(hbplus_dir, file, pdb_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "start processing Wuhan_spike\n",
      "extracted ss list shape: (3198,)\n",
      "processing Wuhan_spike\n",
      "start processing Omicron_spike\n",
      "extracted ss list shape: (3307,)\n",
      "processing Omicron_spike\n",
      "start processing Delta_spike\n",
      "extracted ss list shape: (3228,)\n",
      "processing Delta_spike\n"
     ]
    }
   ],
   "source": [
    "### convert proflex output to ASSURR flexibility index ###\n",
    "import os\n",
    "import pandas as pd\n",
    "from Bio import PDB\n",
    "from prot_rigidity import ASSURR_flexibility\n",
    "\n",
    "root_dir = '/home/lwang/models/HDX_LSTM/data/COVID_SPIKE'\n",
    "proflex_dir = f'{root_dir}/proflex_files'\n",
    "dssp_dir = f'{root_dir}/dssp_files'\n",
    "save_dir = f'{root_dir}/rigidity_files'\n",
    "if os.path.isdir(save_dir) == False:\n",
    "    os.mkdir(save_dir)\n",
    "\n",
    "fname = 'COVID_record.xlsx'\n",
    "apo_df = pd.read_excel(f'{root_dir}/{fname}', sheet_name='Sheet1')\n",
    "apo_df = apo_df.dropna(subset=['chain_identifier'])\n",
    "fail_list = []\n",
    "\n",
    "for index, row in apo_df.iterrows():\n",
    "    pdb_fname = row['apo_identifier'].strip().split('.')[0]\n",
    "    print('start processing', f'{pdb_fname}')        \n",
    "    if os.path.isfile(f'{root_dir}/structure/{pdb_fname}.pdb'):\n",
    "        parser = PDB.PDBParser()\n",
    "        filepath = f'{root_dir}/structure/{pdb_fname}.pdb'\n",
    "        structure = parser.get_structure('AF_structure', filepath)\n",
    "        #chain_ids = list([chain.id for chain in structure.get_chains()])\n",
    "        chain_ids = ['A', 'B', 'C']\n",
    "        dssp_file = [f'{dssp_dir}/{pdb_fname}_{chain_id}.dssp.txt' for chain_id in chain_ids]\n",
    "        ASSURR_flexibility(pdb_fname, dssp_file, proflex_dir, save_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# calculate Relative fractional uptake (RFU) by PyHDX and correct for temperature and pH\n",
    "from pyhdx import read_dynamx, HDXMeasurement\n",
    "from pyhdx.process import filter_peptides, apply_control, correct_d_uptake\n",
    "from pyhdx.plot import peptide_coverage\n",
    "import proplot as pplt\n",
    "from pathlib import Path"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAB9AAAAPoCAYAAACGXmWqAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjQuMywgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/MnkTPAAAACXBIWXMAAB7CAAAewgFu0HU+AACrxElEQVR4nOzde3ycdZk3/s+khzRpCj2EIi0EbKDQgqhFXEE8gKsoUhd/Kj7VqrCIsIviPiq6VFddF1bAw7qgLOoKWosPyuIJgQIqJxUEEcopbUGElmMNpS3pIW0y8/sjNktJ06btTCZp3u/Xa15N78P3viaTmfueuea6voW2trZSAAAAAAAAAGCIq6l2AAAAAAAAAAAwEEigAwAAAAAAAEAk0AEAAAAAAAAgiQQ6AAAAAAAAACSRQAcAAAAAAACAJBLoAAAAAAAAAJBEAh0AAAAAAAAAkkigAwAAAAAAAEASCXQAAAAAAAAASCKBDgAAAAAAAABJJNABAAAAAAAAIIkEOgAAAAAAAAAkkUAHAACAAW/9+vV57LHHsnLlym3ed/ny5Xn66adTLBa3ed/Vq1fn8ccfz/r167d5361ZuXJlnnjiiXR0dJR9bAAAANheEugAAADb4Oyzz05DQ8MWby960Yvyile8Ih/72MeyePHiaoe8TV772tdmypQpmTJlynbtv/F3cPPNN5c5sqHpiiuuyFFHHZUJEybkgAMOyOTJk/M3f/M3+eEPf7jF/dasWZPPfvazmTJlSpqamtLc3Jzdd989J554Yh599NGtHvfiiy/OK17xiuy+++7Zf//9M2HChBxzzDG56aabduj+dHR05Ctf+UoOPPDATJ48OVOnTk1jY2OOP/743HPPPTs0NgAAAJSDBDoAAECZtbW1ZeHChfnWt76Vww47LJdeemm1Q9rEm9/85jQ0NOTss8/usa61tTXLli3LsmXLqhBZ/3n00Ue7k/19SShXw0c/+tF84AMfyO23355SqdS9/P77789JJ52UT33qU5vdb+XKlXn961+fr371q5s8jmvXrs3ll1+eww8/PHfddVevx/3Qhz6U008/PQsXLuxeViqVcvPNN2fmzJm5+OKLt+v+bNiwIX/3d3+Xz33uc5v8zjs6OnL11VfnyCOPzPz587drbAAAACgXCXQAAIDttGDBgh63O+64I1dccUVOPPHEDBs2LO3t7TnttNNy9913VztcBpGvf/3r+c53vpNCoZAPf/jDuffee7Ns2bLccMMNOeyww5Ik3/jGNzabcP7IRz6SBx54IIVCIWeeeWb+9Kc/5cknn8zFF1+csWPHZuXKlXnPe96TtWvX9tj329/+dn7wgx8kSf7u7/4ud955Z/7yl7/k2muvzYEHHphisZiPf/zj21Ut/oUvfKG7gv2DH/xgHnjggTz99NP58Y9/nD333DPt7e35+7//+zz11FPbPDYAAACUiwQ6AADAdmpubu5xmzZtWo4++uhccMEFueyyy5J0Vdj+53/+Z5Wj7ZsHHnggbW1taWtrq3YoQ9YzzzzT3R1gzpw5Oeecc/LiF7849fX1OfTQQ/PjH/84++yzT5Lk/PPP32Tfe++9Nz/5yU+SJJ/4xCfy6U9/OrvvvnvGjBmT448/Pt///veTJEuXLs13v/vdTfZtb2/PueeemyQ5/PDDM2/evOy///6pq6vLq1/96lx55ZUZP358NmzYkHPOOWeb7tPTTz+diy66KEny7ne/O1/72tfS1NSU0aNH501velN++tOfZuTIkVm1atWgea4AAACwc5JABwAAqJC3vOUtecUrXpEk+e1vf1vlaBgs5s6dm+eeey577LFHPv7xj/dYP2bMmHz0ox/NS17ykqxevTqdnZ3d6y677LKUSqXU19fnYx/7WI99jzzyyBx++OFJkh/96EebrLvhhhu6q7/PPPPMFAqFTdZPnDgxJ598cpLkmmuuyXPPPdfn+/TTn/60u+J9zpw5PdYfcMABecc73tEd1/Nb1gMAAEB/kkAHAACooClTpiTpmlt8c1asWJGzzz47hx9+eCZNmpRJkyblsMMOy+c///leW1lPnz49DQ0Nueeee7J27dqce+65eeUrX5mJEydmr732yhvf+MZccsklKRaLm+y3ce7z3/zmN0mSL37xi2loaMj06dO7tznllFPS0NCQU045ZbPHXrBgQU466aRMnTo148ePT3Nzc9773vfm9ttv79Pv49e//nVmz56d/fbbL+PHj8+0adPyjne8I9dcc80Wk6Z33nlnTj755Bx44IEZP358dtttt8yYMSOf/OQnt6nl98a5zw888MDuZQceeGCvc8JXw1VXXZUkefvb356RI0dudpuTTz45t956a2666aYMGzase/mvfvWrJMnrX//6jBkzZrP7vu1tb0vS9Tt99tlne+y766675nWve90W992wYUN3O/a++PWvf52k63fd3Ny82W1mzpyZpKta/b777uvz2AAAAFBOw6sdAAAAwM7siSeeSJI0NDT0WPfHP/4x73jHO/KXv/xlk+X33ntv7r333lx00UX53ve+l6OPPnqzYz/zzDM5+eSTc//993cvW7NmTW699dbceuutufzyy3P55Zdn9OjRZbkv3/zmN3PGGWdskph/+umn87Of/Sy/+MUv8vWvf73XfTs7O/PRj360R9vwpUuXZunSpbn22mtzzDHH5JJLLukR79e//vXMmTOnxxcCFi9enMWLF+eyyy7Ltddem2nTpu34nayyDRs25A9/+EOS5JWvfOU27dvZ2ZnFixcnSV7+8pf3ut3GdcViMYsWLcqrXvWqJF3t+5PkpS99aWpqNv99+4MOOigjR47M+vXr09LSkmOPPbZPsW0ce8aMGb1u8/x1LS0teclLXtKnsQEAAKCcVKADAABUyF133ZVbb701SXLooYdusu6JJ57Icccdl7/85S+ZNGlSvvSlL+V3v/tdbr311nz1q1/NpEmT0tbWluOPP747ofpCH/nIR3L//ffnLW95S372s5/l9ttvz2WXXZYjjjgiSXLzzTfnIx/5SPf2F198cRYsWNDdVv7UU0/NggULMn/+/K3el+uvvz4f//jHUywWM3HixHz5y1/O7373u9x8882ZM2dORo4cmdNPP73X/T//+c93J8/f+c535mc/+1n+8Ic/5Morr8zxxx+fJLn66qtz4oknbrLfggULupPnBx10UL7zne/k1ltvzY033pizzjor9fX1Wb58eT7xiU9s9T4kyeTJk7NgwYJcc8013cuuueaaLFiwIP/wD//QpzEq6aGHHkpHR0eSru4FDz30UP7xH/8xBxxwQMaPH5999tkn7373u3PDDTf02Hfp0qVZv359kmTvvffu9RhNTU3dP//5z3/u/vnhhx/e6r7Dhg3LpEmTeuy7JR0dHVmyZEmPY7/QpEmTMnz48E1iAQAAgP6mAh0AAKCMNmzYkEcffTS/+MUv8uUvf7l7furTTjttk+3+5V/+JcuXL8/uu++eW265Jbvvvnv3upe85CV5+9vfnje+8Y156KGHcsYZZ2w2YfrII4/k/e9/fy688MLuZdOnT88xxxyT97///fnpT3+aH/3oR/nwhz+cGTNmdCc+R40alSQZN25cr+20n69UKuWTn/xkkmT33XfPDTfcsEkidMaMGTn88MO723u/0OLFi/Of//mfSbrm1v70pz/dve6AAw7IkUcemUMOOSSf+tSncvXVV+f666/PG9/4xiTJT37ykxSLxeyyyy6ZP39+xo4d273vK17xinR0dOTzn/98brnllqxZsyb19fVbvC/Dhw9Pc3Nzd6I26Urqbilp3J8ef/zx7p9vu+22fO5zn+ueOzzpmgrgqquuylVXXZV/+qd/yllnndW9buXKld0/jx8/vtdjPH/d81u4b9x/S/tuXP/II49ssu+WPPfcc93Pgy2NXVNTk1133TXPPPNMVqxY0aexAQAAoNwk0AEAALbT5tqyb86cOXNy1FFHdf//2WefzY9//OMkXfOQPz95vtFuu+2Wc889N+94xztyxx135E9/+lOPZPfYsWNz7rnn9ti3pqYm559/fq6//vqsXr06l1566RZbZ2/N73//+zz44INJkn/913/dbBXx61//+rz3ve/N97///R7rvve976VYLGbq1Kn553/+580e47TTTsuPfvSj3Hnnnbnsssu6E+irVq1K0lWd/Pzk+UazZs3Ki170ou29a9vlueeey7Jly7Zpn758UWHj2BudeeaZGTlyZM4444y85S1vSUNDQ+6///6ce+65WbhwYb72ta9ln332yQc/+MEkXe37N6qtre31GHV1dd0/P3+fjT9vad/kf7+A8fx9t+T5XwDYuO/WYlu9enWfxgYAAIByk0AHAACogJqamhx66KH5+Mc/nmOOOWaTdb///e+zYcOGFAqFvOENb+h1jNe+9rXdPy9YsKBHEvYtb3lLxowZs9l9x48fnze/+c254oor8vvf/34H7klXJXSSjBgxIu94xzt63e7d7373ZhPot9xyS5LkqKOOyrBhw3rd/7WvfW3uvPPOLFiwoHvZxnnNFy5cmHPOOScf/ehHN0kAT548ObNnz962O7SDfvazn+XUU0/dpn3a2tr6tN26deu6f66pqclPf/rTvPrVr+5eNn369Lz5zW/O61//+ixatChnnXVW3vve96auri6FQqF7u1Kp1OsxNraIf+F2G/d/4VzzL7SxmnxLx3i+58e1tbE3xtbXsQEAAKDcJNABAAC20/MTvc83YsSINDY29tpOfOP8zqVSqc+tw5988skeyw4++OAt7nPwwQfniiuu6J5/ens99thjSbqqqJ+fvH6hl7zkJZtdvnGu7IsuuigXXXTRVo/31FNPdf/83ve+NxdeeGEefPDBnHXWWfnqV7+aI444IocffngOP/zwHHrooRkxYsS23J0BbeTIkd0/z5o1a5Pk+UZjxozJnDlz8oEPfCCtra353e9+lze84Q0ZPXp09zbPT8S/0PPXPf9vdPTo0VmxYkXa29u3GOPG/bfWLn9zx9hSXEm6j/38+wIAAAD9SQIdAABgO/W1LfcLPb9Nd19trl32LrvsssV9dt111yT/2wZ9e21swd3X473Qth7/+e276+vrc/311+eLX/xiLrvssqxcuTLXXXddrrvuuu6Y3vnOd+bMM8/MHnvssU3H2V6zZ8+uWNX78zsKHHnkkb1u95rXvKb754ULF+YNb3jDJvOLb6nF/PPXPX+f8ePHZ8WKFVttT79x/dbmSt9ozJgxGTFiRDZs2LDFsdevX9899/m4ceP6NDYAAACUmwQ6AABAP9tYXTt+/PjccMMNfdpnc8nKrc1BvbFt+I5W826c631rifDe2pSPHj06q1atypw5c/Lud797q8d7fsvvJGlsbMxXvvKV/Pu//3tuu+223HLLLfnd736X22+/PatWrcrFF1+cX/ziF7n11ls3O5/8YPL8+eU3N+f7Rs9PMG/8gsOkSZNSX1+fNWvW5JFHHul13+d3JNh33327f25ubs7DDz+cRx99tNd9161b150Ef/6+W1JTU5MXv/jFWbx48RbHXrJkSXfr9r6ODQAAAOUmgQ4AANDPJk+enCRZuXJl9thjjz63wn6hlpaWLa6/9957k3QlVnfEPvvskyT505/+lDVr1vQa7/3337/Z5ZMnT86qVauyfv367a7aT5La2tq87nWvy+te97okXZXql156aT75yU9m2bJlueCCC3LWWWdt9/h99dxzz221SvuF+nq/m5ubM3LkyKxfvz5PPPFEr9v95S9/6f55woQJSbq+eHDQQQfl9ttvzx/+8Ide9924bvjw4TnggAO6lx988MG5/vrrs2DBgqxfv36TdvIb3XXXXd1zoPfWsn9zDj744CxevHiLcd15553dP2/L2AAAAFBONdUOAAAAYKh51atelUKhkM7Ozu5W5Jvzu9/9Lscee2xmzpyZ5cuX91h/9dVXZ/369Zvdd8WKFbn66quTJH/zN3+zQ/EedthhSZINGzbk8ssv73W73ta96lWvSpJcc801WzzOhz/84Rx77LH5xje+0b3s+OOPzxvf+MZ885vf7LH96NGj86EPfSh/+7d/m2TrXygol5/97Gd56Utfuk23vhoxYkRe+9rXJkl+8Ytf9Lrd/Pnzu38+9NBDu39+4xvfmCT5zW9+k6effrrX+JPk1a9+9SYt4zfuu3bt2l4fq4371tfXb9JGfmve9KY3JUkefvjh3H333Zvd5qc//WmS5MUvfnH233//Po8NAAAA5SSBDgAA0M/22GOPvOENb0iS/Mu//Mtmq5nb2tryqU99KjfeeGNWrVq12RbuTz31VD7/+c/3WF4sFnP66ad3z7Xe23zdG9tlb83LXvayTJ8+PUny+c9/frPtwW+99dZ897vf3ez+73vf+5J0Vaj/53/+52a3ufzyy/Pd7343N9544ybtux9//PHceuut+eY3v5n29vYe+61bty6LFi1K0vc5uV+or7+H/vKe97wnSdcXDq644ooe6x9//PGcffbZSboem4MOOqh73Xvf+96MGDEiHR0d+cIXvtBj38svvzwLFixIkpxwwgmbrDv88MMzderUJMnZZ5+ddevWbbL+4YcfziWXXJIk+T//5/9k1KhRfb5PM2fO7H58Pve5z6VYLG6y/rbbbstVV1212bgAAACgP0mgAwAAVMHZZ5+dUaNG5c9//nOOOOKIXHjhhVmwYEHuueee/OAHP8gb3vCG3HXXXampqem1LXlNTU3OP//8vP/9788NN9yQlpaWXHXVVTnmmGPy4x//OEny//1//193Bfjz90u62nGvWLEizz777FbjPffcc5N0tQ5//etfn2984xu55557cvfdd+dLX/pSjjvuuBQKhUycOLHHvq985Su75z7/9Kc/nRNOOCG//OUv09LSkptvvjlnnHFGTj755CTJkUcemaOPPrp735kzZyZJFi9enLe//e35+c9/nnvvvTcLFizID3/4w7zpTW/Kn//85yRdSd2+2vg7SJLf/va3Wb16da9zuL/Q7Nmz09bWtk23bfHOd74zM2bMSJKcdNJJOfPMM3P77bfn/vvvz8UXX5wjjzwyTz/9dEaMGJEvfelLm+zb1NSUU045JUnyve99L6ecckp+//vfZ8GCBTn33HO71x1yyCF5xzve0eN38m//9m9JkgceeCBvfetbc/311+e+++7LvHnz8uY3vzmrV6/O2LFj88lPfrJH3BdccEGmTp2aqVOndv/9bTRmzJj88z//c5LkV7/6VY4//vjcfPPNuffee/Nf//Vfecc73pFisbhJ/AAAAFANhba2toH1VXsAAIAB7Oyzz84Xv/jFJNnmxOgLXXfddXnf+96X1atXb3b9qFGj8vWvf71HYnj69OlZsmRJPv3pT+eHP/xhHnrooc3uf8QRR+Tyyy/fpE13knzsYx/Lt771re7/NzU15YEHHkiSnHLKKbn00kvz3ve+t0fb9G9/+9v5xCc+0T0H9vMNHz48F154YebOnZvf/OY3ufrqq7tbkSfJmjVr8oEPfGCLbdwPP/zw/PCHP8y4ceO6l61evTozZ87M7bff3ut+NTU1+cxnPrPZpG5vOjs709TUlJUrV3YvO/PMM/PpT3+6z2NU0hNPPJFjjjmm18e2rq4u3/rWt/L2t7+9x7oNGzZk1qxZm7R5f7599903V111VSZPnrzZ9V/60pfyhS98YbOV+WPGjMkPfvCDHHnkkT3WPf+5cdFFF22288FHPvKR7ir2F5o4cWJ+/vOfb1JRDwAAAP1NBToAAECVvOlNb8rdd9+d008/PQcccEBGjx6d2tra7Lvvvjn55JPz+9//fotV1XvttVd++9vf5jOf+UymTZuWUaNGZdddd81hhx2W888/P1dddVWP5HmSzJkzJzNnzsyuu+6akSNH9ppIfaGTTz45v/nNbzJr1qxMnjw5I0eOzG677Za3ve1t+dWvftXdenxz6uvrc/nll2fu3Lk5+uijs9tuu2X48OEZN25cXve61+Wiiy7K/PnzN0meJ13znF977bX5yle+kte85jWZMGFChg8fnrq6uuy333454YQTcvPNN29T8jxJhg0blrlz5+aggw5KbW1tdtlll0yYMGGbxqikSZMm5dZbb82//uu/5mUve1l22WWX1NbWprm5OaecckruvPPOzSbPk6551C+//PJceOGFedWrXpWxY8emvr4+06ZNy5lnnpnf/OY3W3zMzzjjjPziF7/Isccem4kTJ6a2tjZNTU058cQT87vf/W6zyfO+uuCCC3LppZfmyCOPzPjx4zNq1Kg0Nzfn9NNPz2233SZ5DgAAQNWpQAcAABhkNlag91blCwAAAMD2UYEOAAAAAAAAAJFABwAAAAAAAIAkEugAAAAAAAAAkEQCHQAAAAAAAACSSKADAAAAAAAAQJKk0NbWVqp2EAAAAAAAAABUV0dHR8aPH5/dd989Dz74YLXDqQoV6AAAAAAAAADknHPOSbFYrHYYVSWBDgAAAAAAADDE3XDDDfnyl79c7TCqbni1AwAAAAAAAACg//3whz/M1772tTz88MNZvXp1tcMZECTQAQAAAAAAAIagn/zkJ7n33nurHcaAUmhraytVOwgAAAAAAAAA+tdDDz2UBx54oPv///Zv/5aWlpbsscceefDBB6sYWfWoQAcAAAAAAAAYgvbdd9/su+++3f+/8MILqxjNwFDRBPqUKVOyZs2a7LnnnpU8DAAAAAAAAMCAs3DhwiTJyJEjyzbm+vXr+zze8uXLy3bcoaKiCfQ1a9Zkw4YNqampqeRhdkobNmzIQw89lFJJh/3+VCgUsu+++2bEiBHVDmWzOjs7kyTDhg2rciTAC5Xr+en1f+Ab6OcKNs85FAY2z1GoDNeW1eF6kf7kHAoDg3Mug9lQvnbpz/PoyJFJc/OGsoz1pz95ram0iibQ99xzz9TU1OT++++v5GF2Sn/84x9zyCGH5OLPNOSAvXXa7w8LH+3I35/VlssuuywzZsyodjibtWzZsiTJxIkTqxwJ8ELlen5ufP1/+4dfmt0mN5QjNMroL4+35SdfXzCgzxVsnnMoDGyeo1AZG68tpxz5+Ywau892jdHRvipJMrx2lzJGtvNat+KRPHzD510v0m+cQ2Fg2HjO/d63a3PA1P8tKFz+bFeSa/y4QrVCgy1auLiYD5zcPmSvXfrrPFpTU5Pm5uTuu+vKMt7LXrY2f/rTCJXlFSQzO8AdsPfwvHyqhwlgqNltckP2mLJrtcMAAGAnMGrsPhndeMB27bthbdeHciPqxpczJADYKR0wtSYzXva/lazL/lJMkkzcTZdegMHEqzYAAAAAAAAARAIdAAAAAAAAAJJo4Q4AAAAAAABQWcVqB0BfqUAHAAAAAAAAgEigAwAAAAAAAEASLdwBAAAAAAAAKqpQKlQ7BPpIBToAAAAAAAAARAU6AAAAAAAAAEnmz59f7RCqTgU6AAAAAAAAAEQFOgAAAAAAAEBllaodAH2lAh0AAAAAAAAAIoEOAAAAAAAAAEm0cAcAAAAAAAConFKh61ausagoFegAAAAAAAAAEAl0AAAAAAAAAEiihTsAAAAAAABAxRSSFIrlG4vKUoEOAAAAAAAAAJFABwAAAAAAAIAkEugAAAAAAAAAkMQc6AAAAAAAAACVVap2APSVCnQAAAAAAAAAiAQ6AAAAAAAAACTRwh0AAAAAAACgogqlQrVDoI9UoAMAAAAAAABAJNABAAAAAAAAIIkW7gAAAAAAAACVVap2APSVCnQAAAAAAAAAiAQ6AAAAAAAAACSRQAcAAAAAAACAJOZABwAAAAAAAKicUpJiGceiolSgAwAAAAAAAEAk0AEAAAAAAAAgiRbuAAAAAAAAABVVKBWqHQJ9pAIdAAAAAAAAACKBDgAAAAAAAABJtHAHAAAAAAAAqKxStQOgr1SgAwAAAAAAAEAk0AEAAAAAAAAgiQQ6AAAAAAAAACQxBzoAAAAAAABAZZkDfdCQQB/gFj7aUZFxn3qmmJVtxYqMPVg98mRntUMA6PbgXcvS+nhbtcPgBZ5dtiZJ0tLSUuVIujQ2NqapqanaYQAAA9zKpb/L2hWPbNe+neu7rkmHjWwoY0Q7r/XPPZFk4FwvVpJrUYCeFi7e9DP35c92ZcvGj+uZNXvq6WJWrOyXsBhixu6avGj3vjWgfuHfLINHqVTKlVde2WN5c3Nzpk+fXoWIdi4S6ANUY2Nj6utH5e/PqkzypKYmKXpd7KGmJmlvb692GMAQ1t7enpqa5IYfPVjtUNiC2bNnVzuEJEl9/ai0tCzywSUAsFnt7e1JoSaP/+Fb1Q5lyBko14uVNGpUfRYtanEtCpD//Tz/Ayev6/M+PqOnUrb1b6u+flQaGxsrFxAV0dHRkVmzZvVYfsQRR2T+/PlViGjnIoE+QDU1NaWlZVFaW1vLPnZLS0tmz56d/z6zIfs3+RPYaNGSjnzwi22pra2tdijAEFZbW5tiMfnnj09I054jqh0OA9iSxzbknK88k9bWVh9aAgCbVVtbm5SKmTbjjNQ3bN/1wob1zyVJRowcU87QGOTWtC1Jyx+/5FoU4K96+zx/+fLlSZLx48dvsnzjZ/RzL6zLtKl9qxSGvmhZXMz7/3Ft5s2bl2nTpvVpH11l+kkpSbFQtrGGDx+euXPn9ljV3NxcnmMMcbKnA1hTU1NFX7T2bxqel031JwAwEDXtOSL77Tuy2mEAALATqG9oypix+27XvuvbVyRJRtaOLV9AALAT2tzn+cuWLUuSTJw4cbP7TJtakxkHD6t4bAw906ZNy4wZM6odBhVUKBQyc+bMaoex0/LVJgAAAAAAAACIBDoAAAAAAAAAJNHCHQAAAAAAAKCiCqVqR0BfqUAHAAAAAAAAgEigAwAAAAAAAEASLdwBAAAAAAAAKksL90FDBToAAAAAAAAARAIdAAAAAAAAAJJo4Q4AAAAAAABQWVq4Dxoq0AEAAAAAAAAgEugAAAAAAAAAkEQCHQAAAAAAAACSmAMdAAAAAAAAoLKKhWpHQB+pQAcAAAAAAACASKADAAAAAAAAQBIt3AEAAAAAAAAqp/TXW7nGoqJUoAMAAAAAAABAJNABAAAAAAAAIIkW7gAAAAAAAACVpfX6oKECHQAAAAAAAAAigQ4AAAAAAAAASSTQAQAAAAAAACCJOdABAAAAAAAAKssc6IOGCnQAAAAAAAAAiAQ6AAAAAAAAACTRwh0AAAAAAACgogqlQrVDoI9UoAMAAAAAAABAJNABAAAAAAAAIIkEOgAAAAAAAAAkMQc6AAAAAAAAQGUVqx0AfaUCHQAAAAAAAAAigQ4AAAAAAAAASbRwBwAAAAAAAKic0l9v5RqLilKBDgAAAAAAAACRQAcAAAAAAACAJFq4AwAAAAAAAFSW1uuDhgp0AAAAAAAAAIgEOgAAAAAAAAAkkUAHAAAAAAAAgCTmQB/SFi3pqHYIA8q2/j6WLFmS1tbWCkWzecuXL0+SjB8/vl+Py8DW2NiYpqamaodBmd1+59osWbqh2mEwgD21rOu81dLSUrFjeH0BgJ3DM8vuyJrnlm7Xvh0bVidJho8YXc6QGOTWrn0qybZdi7q2BOjpml92ZOGDndUOg53Inx/tmmTbOXogKiSlQtnGKpVKufLKK3usaW5uzvTp08t0nKFLAn0IamxsTH39qHzwi23VDmXAqa8flcbGxq1ut2TJkkybtn/WrFnXD1HBltXXj0pLyyIXOTuJ9vb2FGqS785bWe1QGCRmz55dsbHr6kdlodcXABi02tvbkxTyyMK51Q6FndS2XIuOGlWXRYsWurYESNc5uqYm+ew57dUOhZ3Utpyjfb48OHV0dGTWrFk9lh9xxBGZP39+FSLauUigD0FNTU1paVnU79XTg0Ffv2nV2tqaNWvW5fT/Oyl77jWyHyLr0vZc17cRG8YM67djMrA9tnR9zv+PJ9La2uoCZydRW1ubUjGZffoBmbhnfbXDYQhb9tiazDt/odcXABjEamtrk5Ry4IEfTH39Hts1xoa/VqCPUIHODliz5sncf/9/u7YE+Kva2toUi8ncb47KAfubaZfqWbiomPefss45ehAaPnx45s7t+UXZ5ubmKkSz85FAH6Kampq8GJbBnnuNzJTmun473soVXe16dx3rqQs7u4l71mevKWOqHQYAADuB+vo9sssue2/XvuvXr0qSjBy5SzlDAgCSHLB/TWa8VLEUDBml8g1VKBQyc+bM8g3IJny1CQAAAAAAAAAigQ4AAAAAAAAASbRwBwAAAAAAAKisMrZwp7JUoAMAAAAAAABAJNABAAAAAAAAIIkEOgAAAAAAAAAkMQc6AAAAAAAAQOWUkhTLOBYVpQIdAAAAAAAAACKBDgAAAAAAAABJtHAHAAAAAAAAqKiS1uuDhgp0AAAAAAAAAIgEOgAAAAAAAAAkkUAHAAAAAAAAgCTmQAcAAAAAAACoLHOgDxoq0AEAAAAAAAAgEugAAAAAAAAAkEQLdwAAAAAAAIDKKhWqHQF9pAIdAAAAAAAAACKBDgAAAAAAAABJtHAHAAAAAAAAqKxStQOgr1SgAwAAAAAAAEAk0AEAAAAAAAAgiQQ6AAAAAAAAACQxBzoAAAAAAABA5ZRSvjnQzaVecSrQAQAAAAAAACAS6AAAAAAAAACQRAt3AAAAAAAAgMrSen3QUIEOAAAAAAAAAJFABwAAAAAAAIAkWrgDAAAAAAAAVFRJC/dBQwU6AAAAAAAAAEQCHQAAAAAAAACSSKADAAAAAAAAQBJzoAMAAAAAAABUVrHaAdBXKtABAAAAAAAAIBLoAAAAAAAAAJBEC3cAAAAAAACAyipVOwD6SgU6AAAAAAAAAEQCHQAAAAAAAACSaOEOO+SPf2jLY4+199vx1qzpTJLU1w/rt2MysC17akOSpKWlpc/7NDY2pqmpqVIhUSYtf1yepx9bU+0wGMKWL1uXpO+vL15bAGDgWrPmye3ed8OG1UmSESNG91jX3r4yHR2uWV9o+PD61NbuWu0wBpQd+RsE2JktXFSsdggMcf4GB69SqZQrr7yyx/Lm5uZMnz69ChHtXCTQYTu0t7enpia57Aet1Q4FkiSzZ8/u87b19aPS0rJIomuAam9vT6EmueayR6odCiTp++tLXd2oLFzotQUABpLGxsaMGlWX++//7wodoRATOW6O38vmjBpVl8bGxmqHATAgNDY2pr5+VN5/yrpqhwKprx/lHN0fSinfJWIp6ejoyKxZs3qsOuKIIzJ//vwyHWjokkCH7VBbW5tiMfn0x8dl7z3772m06rmub4PtMsbsC2yfRx/ryNlfeTatra2SXANUbW1tSsXkPadPy8TJPat8YCBa9vjq/OD8Fq8tADDANDU1ZdGihWlt3f4vfy9fvjxJMn78+E2Wt7S0ZPbs2Tlg3yNTXzd2R8LcqaxZuyILH7oh8+bNy7Rp06odzoCiYxHA/2pqakpLy6IdOkdDuThHD07Dhw/P3Llzeyxvbm6uQjQ7Hwl02AF77zk8U/cd2W/He3ZFVwv3cWO1cIed3cTJo7PnlDHVDgMAgEGuqalphz4QXbZsWZJk4sSJm11fXzc2Y0arWHqhadOmZcaMGdUOA4ABbEfP0cDQVigUMnPmzGqHsdOSQAcAAAAAAACooFKpUO0Q6CN9oAEAAAAAAAAgEugAAAAAAAAAkEQLdwAAAAAAAIDKKlU7APpKBToAAAAAAAAARAIdAAAAAAAAAJJIoAMAAAAAAABAEnOgAwAAAAAAAFRUyRzog4YKdAAAAAAAAACIBDoAAAAAAAAAJNHCHQAAAAAAAKCytHAfNFSgAwAAAAAAAEAk0AEAAAAAAAAgiRbuAAAAAAAAABVTSlIqUwt3neArTwU6AAAAAAAAAEQCHQAAAAAAAACSSKADAAAAAAAAQBJzoAMAAAAAAABUTinlm7zcJOgVpwIdAAAAAAAAACKBDgAAAAAAAABJtHAHAAAAAAAAqKiS1uuDhgp0AAAAAAAAAIgEOgAAAAAAAAAkkUAHAAAAAAAAgCTmQAcAAAAAAACoLHOgDxoq0AEAAAAAAAAgEugAAAAAAAAAkEQLdwAAAAAAAICKKmnhPmioQAcAAAAAAACASKADAAAAAAAAQBIt3AEAAAAAAAAqSgv3wUMFOgAAAAAAAMAQVSwWc8IJJ2S33XZLQ0NDxowZk8mTJ+cLX/jCdo03f/78HHzwwdl11127x2tubs5FF11U5sgrQwIdAAAAAAAAYAgqFovZf//98z//8z9Zu3ZtkqRUKmXlypU577zzcuyxx27TeOecc07e+c535uGHH05nZ2f3eE8//XQ+8YlP5N3vfnfZ70O5SaADAAAAAAAADEGzZs3Kk08+mSQ5/vjjs2jRotx222055JBDkiQ33nhjvvGNb/RprIULF+ass85KkowdOzbf+ta38sgjj+SnP/1p9ttvvyTJVVddlYsvvrgC96R8JNABAAAAAAAAhphVq1blmmuuSZIcd9xxufjiizN58uQcdNBBuemmmzJlypQkXVXlffG5z30uSTJ8+PDcc889ec973pPGxsb87d/+be666640NjYmSb7yla9U4N6UjwQ6AAAAAAAAwBDz7W9/O8ViMYVCYbPzk5933nlJkmeffTaPPvroVsd74IEHkiT77bdfxo8f32P9m970piTJU089tSNhV5wEOgAAAAAAAMAQc/XVVydJ9thjjzQ0NPRY/+Y3vznDhg1Lknzve9/b6niFQmGTf19o41gDnQQ6AAAAAAAAQAWVSuW5ldOSJUuSJFOnTu11m41t1++5556tjnfggQcmSRYvXpzly5f3WH/dddcl6UrYD2QS6AAAAAAAAABDzMqVK5MkTU1NvW4zYcKEJMnSpUu3Ot55552XESNGpKOjIy996Utz2WWXpbW1Nb/61a8yY8aMPP3000mSc889twzRV87wagcAg9ltd67Lo0s39NvxVq8pJklG1/vuC9vnyWWdSZKWlpY+79PY2LjFkyeVsezx1X3abtWz7Vm3pqPC0fB8o+qHZ5dxtdUOY0Dp698rALDzWf7s0qxZu6LaYQwY69Y9l2Tb3nMNVt4rAgBUz/r16zc7x/jmbK4SfKOOjq7PlseOHdvrNqNHj06SrF27dqvH2muvvXLdddflmGOOybPPPpsPfvCDm6yvqanJ+eefn7e+9a19iLx6JNBhO7S3t6emJrl43nPVDgW2y+zZs/u8bX39qLS0LPLBSD9pbGxMXd2o/OD8vn3gViiUv20PW+Z3vnl1daO62zkBADu/9vb2JIU88tgfqh3KgLQt77kGq1Gj6rJo0ULvFQEA+qKc7dfL+NlksdhVuDlq1KhetxkxYkSSje8Btu7aa6/tddtisZibbropJ5xwwrYF2s8k0GE71NbWplhMvvDxMXnxXsP67bgrn+t6Idt1jAp0+sefl3bms195Lq2trT4U6SdNTU1ZuHBRWltbt7ptS0tLZs+endec+pqMnbxrP0THisdX5paLbsm8efMybdq0aoczoKhAAoChpba2Nkkp06cel9H1vkQ31Kxe05oHFv/Ue0UAgCoZOXLkFivL+6qmpivftKXq8o3J8I2J9C2ZM2dOzj///CTJy1/+8syZMyeHHnpoFixYkPPPPz+/+tWvcvnll2fVqlW54oordjj+SpFAhx3w4r2G5YB9t/6CUS7LV3Ql0MePlUCHnVlTU9M2fQg1dvKumbDPhApGxAtNmzYtM2bMqHYYAABVN7q+MWMa9qh2GAAAwHYYPnx41q9fnxUrVvS6zerVXdM31tXVbXGsFStW5Otf/3qS5H3ve1/+67/+q3vdG97whrzhDW/I1772tXzmM5/Jtddem/vuuy8HHXTQjt+JCpCFAwAAAAAAABhidt21q7PpkiVLet1mY6X7pEmTtjjWvHnzUiwWUygUcsEFF2x2m3/6p3/qbhd/0UUXbU/I/UICHQAAAAAAAKBCSumaA70stzLGtffeeydJFi9e3Os2zzzzTJLk4IMP3uJYDz/8cJKu6Z6GD++9CfrGpP1jjz22TbH2Jwl0AAAAAAAAgCHmrW99a5LkqaeeSltbW4/11113XTo7O5MkJ5544hbH2muvvZJ0zZne0dHR63YrV65Mkuy+++7bFXN/kEAHAAAAAAAAGGJOPvnk1NTUpFQq5bTTTuux/lOf+lSSZMKECZkyZcoWx3rve9+bJCmVSjn11FM3u81XvvKVrFu3LknXPOkDlQQ6AAAAAAAAQAWVq4V7OTU0NHRXoV9xxRU59dRT8+STT+a+++7LUUcdlQcffDBJ8tnPfnaT/caNG5dx48blqKOO6l42ceLEvPrVr06SXHbZZXnNa16T+fPnZ/ny5fnVr36VY489Np/73OeSJM3NzTniiCPKe2fKqPcG9AAAAAAAAADstC699NIccMABeeKJJzJv3rzMmzdvk/VHH310TjrppE2WbdiwIUnS2tq6yfKrrroqL3/5y/PnP/85d911V975znf2ON6ECRNyww03lPlelJcKdAAAAAAAAIAhqKamJgsXLsy73vWu1NXVJUkKhULGjh2bOXPm5IorrujzWMOHD8+9996bT37yk9ltt91SKBS6xxs3blxOPvnk/PnPf8748eMrcl/KRQU6AAAAAAAAQAWVu/16OdXU1OSSSy7p8/ZtbW1bXP/Zz362R9v3wUQFOgAAAAAAAABEAh0AAAAAAAAAkkigAwAAAAAAAEASc6ADAAAAAAAAVNQAngKdF1CBDgAAAAAAAACRQAcAAAAAAACAJFq4AwAAAAAAAFRUSQ/3QUMFOgAAAAAAAABEAh0AAAAAAAAAkkigAwAAAAAAAEASc6ADAAAAAAAAVJQ50AcPFegAAAAAAAAAEAl0AAAAAAAAAEiihTsAAAAAAABAxZT+eivXWFSWCnQAAAAAAAAAiAQ6AAAAAAAAACTRwh0AAAAAAACgckpJUQ/3QUMFOgAAAAAAAABEAh0AAAAAAAAAkkigAwAAAAAAAEASc6ADAAAAAAAAVFTJ3OWDhgp0AAAAAAAAAIgEOgAAAAAAAAAk0cIdAAAAAAAAoKKK1Q6APlOBDgAAAAAAAACRQAcAAAAAAACAJFq4AwAAAAAAAFRMKUmpVCrbWFSWCnQAAAAAAAAAiAQ6AAAAAAAAACSRQAcAAAAAAACAJOZABwAAAAAAAKioYrUDoM8k0GEH/PYP6/PI0o5+O17bmq6X14Z6zSPoH48/3fU319LSUuVIBr7ly5cnScaPH99jXWNjY5qamip27BWPr6zY2GtWrM36NesrNv5g07asrdohAAAMKKvXtFY7BKqgPx73JUuWpLXV39fWbOm96FBR6ffcADAQlUqlXHnllT2WNzc3Z/r06VWIaOcigQ7bob29PTU1yUXz1lQ7FOgXs2fPrnYIg1p9/ai0tCwq+xv6xsbG1NXV5ZaLbinruJsoFJJSqXLjD0KFQiHt7e3VDgMAoKoaGxszalRdHlj802qHQpWMGlWXxsbGioy9ZMmS7H/AAVm3dm1FxmfnMqquLosWLpREB2BI6ejoyKxZs3osP+KIIzJ//vwqRLRzkUCH7VBbW5tiMfnyJ0Zn372G9dtxVzzXVQ08dowKdBhoent+PrS0M5/48uq0traW/c18U1NTFi5cWLGqjJaWlsyePTuHfuiojNljXEWOMdg89+SzueNbv05tbW21QwEAqKqmpqYsWlS5a1EGvkpW/ba2tmbd2rWZ+PaPZORukytyjJ1F59quLlnD6hqqHEl1rP/L41n2kwsq8p4bAMqtWMY6peHDh2fu3Lk9ljc3N5fvIEOYBDrsgH33GpYD9+2/p9EzK7oSdBPGSqDDQFOt52dTU1PFPyQYs8e4jNtnt4oeAwCAwac/rkUZ2kbuNjm1e0ypdhgDWsfqrim9ho/etcqRAAD9qVAoZObMmdUOY6clCwcAAAAAAAAAkUAHAAAAAAAAgCRauAMAAAAAAABUTClJsYxjUVkq0AEAAAAAAAAgEugAAAAAAAAAkEQLdwAAAAAAAICKKlcLdypPBToAAAAAAAAARAIdAAAAAAAAAJJo4Q4AAAAAAABQUZ0pVTsE+kgFOgAAAAAAAABEAh0AAAAAAAAAkkigAwAAAAAAAEASc6ADAAAAAAAAVEwpSbFMU6CbSb3yVKADAAAAAAAAQCTQAQAAAAAAACCJFu4AAAAAAAAAFdVZ7QDoMxXoAAAAAAAAABAJdAAAAAAAAABIooU7AAAAAAAAQMWUUr4W7qUyjUPvVKADAAAAAAAAQCTQAQAAAAAAACCJBDoAAAAAAAAAJDEHOgAAAAAAAEBFFVOodgj0kQp0AAAAAAAAAIgEOgAAAAAAAAAk0cIdAAAAAAAAoGJKSTrLOBaVpQIdAAAAAAAAACKBDgAAAAAAAABJJNABAAAAAAAAIIk50AEAAAAAAAAqqjOFaodAH6lABwAAAAAAAIBIoAMAAAAAAABAEi3cAQAAAAAAACqmlKRYKk8L91JZRmFLVKADAAAAAAAAQCTQAQAAAAAAACCJFu4AAAAAAAAAFdWprnnQ8EgBAAAAAAAAQCTQAQAAAAAAACCJBDoAAAAAAAAAJDEHOuyQh5Z29uvxVjxXTJKMHbPpd1/+sryYVatL/RoLA8cuowvZbbzvQ1Vbb8/P/n6dqITnnny22iEMGH4XO6clS5aktbW12mEMao2NjWlqaqp2GADATmb1g3dlfevj1Q5jQCuuW5MkqRlVX+VIqmPDs8uSJC0tLVWOZPBzTQ9QWaUU0plC2cYqlUq58sore6xrbm7O9OnTy3KcoUwCHbZDY2Nj6utH5RNfXl3tUJIkNYWkKH8+ZHn8B776+lFpbGysdhjbrLGxMXV1dbnjW7+udigDSl1d3aB8PNm8JUuWZP8DDsi6tWurHcqgNqquLosWLvSBGwBQFu3t7SnUFPLsDT+sdigMErNnz652CINeXV1dFrqmBxg0Ojo6MmvWrB7LjzjiiMyfP78KEe1cJNBhOzQ1NaWlZVG/V6stX748STJ+/PjuZS0tLZk9e3a+esboNO81rF/jofr+tLQzH/vS6sybNy/Tpk2rdjhD2uaenxsN1m9xNzU1ZeHChSpzX2CwPp5sXmtra9atXZt9TzwudS/yxYjtsfap1jx0yU/T2trquQEAlEVtbW1KxVJefeprs+ukXasdzoDW3taeJKltqK1yJAxmK59Ymd9edLNreoBBZPjw4Zk7d26P5c3NzVWIZucjgQ7bqampqd8vKJct62pLNXHixB7rmvcaloP29ZQeqqZNm5YZM2ZUO4whbUvPz8GsGq91UA11L2pMQ9Me1Q4DAIDn2XXSrpmwjy85bsnaVV2dlOp2qatyJADA1hTL1MI9SQqFQmbOnFm28diUSXMBAAAAAAAAIBLoAAAAAAAAAJBEC3cAAAAAAACAyiklnaUy1TWXyjMMvVOBDgAAAAAAAACRQAcAAAAAAACAJBLoAAAAAAAAAJDEHOgAAAAAAAAAFVNKUixTXbMp0CtPBToAAAAAAAAARAIdAAAAAAAAAJJo4Q4AAAAAAABQQYV0plC2sagsFegAAAAAAAAAEAl0AAAAAAAAAEgigQ4AAAAAAAAAScyBDgAAAAAAAFAxpSTFMtU1l8oyCluiAh0AAAAAAAAAIoEOAAAAAAAAAEm0cAcAAAAAAACoqM6SuubBwiMFAAAAAAAAAJFABwAAAAAAAIAkWrgDAAAAAAAAVFAhxbLVNRfKNA69UYEOAAAAAAAAAJFABwAAAAAAAIAkEugAAAAAAAAAkMQc6AAAAAAAAAAVU0rSWaa5y0tlGYUtUYEOAAAAAAAAAJFABwAAAAAAAIAkWrgDAAAAAAAAVFSxpK55sPBIAQAAAAAAAEAk0AEAAAAAAAAgiRbuAAAAAAAAABVTStJZprrmUllGYUtUoAMAAAAAAABAJNABAAAAAAAAIIkEOgAAAAAAAAAkMQc6AAAAAAAAQAUVUiyVq665EDOhV5YKdAAAAAAAAACIBDoAAAAAAAAAJNHCHQAAAAAAAKBiSkk6y1TXrHl75alABwAAAAAAAIBIoAMAAAAAAABAEgl0AAAAAAAAAEhiDnTYadz0h/X509LOaodBP3vs6a7HvKWlpc/7NDY2pqmpqVIhAQxKz973UNY+1VrtMAalda0rkmzbuaiSnOcAYOex8omVFRl37Yo1Wb9mfUXG7m/r13bdj5F1I6scSfWMrB+ZurH11Q5jUKvUcw2ATRVTKNtYpVIpV155ZY/lzc3NmT59etmOM1RJoMMg197enpqa5Ktz11U7FKpo9uzZfd62vn5UWloWSS4ApOs8WqhJHrvyxmqHMuhty7mokurqRmXhQuc5ABjMGhsbU1dXl99edHNlDlAoJKVSZcam/3k8y6Kuri6NjY3VDgOAPuro6MisWbN6LD/iiCMyf/78KkS0c5FAh0GutrY2xWLy9TMasm/TsGqHwwD30JLOfPhLbWltbZVYAEjXebRUTP7P6Qdm4mRVK4PdssfX5LLz73eeA4BBrqmpKQsXLkxra/k7BLW0tGT27Nl52clvSsMe48s+fn9bv7qroGLk6FFVjqQ62p5cnru/fV3mzZuXadOmVTucQU0nJ4DBZfjw4Zk7d26P5c3NzVWIZucjgQ47iX2bhuXgfT2lAWB7TJxcn8lTdql2GAAA/FVTU1NFk3kNe4zPrntPrNj4/aV91ZokSe0uQ/vLoNOmTcuMGTOqHQYAbEEhnaWaso1VKBQyc+bMMo3HC5XrkQIAAAAAAACAQU0CHQAAAAAAAACihTsAAAAAAABAxZSSFMtU11wqyyhsiQp0AAAAAAAAAIgEOgAAAAAAAAAkkUAHAAAAAAAAgCTmQAcAAAAAAACoqM6SuubBwiMFAAAAAAAAAJFABwAAAAAAAIAkWrgDAAAAAAAAVEwphRTLVNdcSqEs49A7FegAAAAAAAAAEAl0AAAAAAAAAEiihTsAAAAAAABARXWqax40PFIAAAAAAAAAEAl0AAAAAAAAAEgigQ4AAAAAAAAAScyBDgAAAAAAAFA5paRYKlNdc6k8w9A7FegAAAAAAAAAEAl0AAAAAAAAAEiihTsAAAAAAABAxZSSFFMo21hUlgp0AAAAAAAAAIgEOgAAAAAAAAAkkUAHAAAAAAAAgCTmQAcAAAAAAACooEI6S+Wqay7PXOr0TgU6AAAAAAAAAEQCHQAAAAAAAACSaOEOAAAAAAAAUDGlJMUy1TWXyjIKW6ICHQAAAAAAAAAigQ4AAAAAAAAASbRwBwAAAAAAAKiozpK65sHCIwUAAAAAAAAwRBWLxZxwwgnZbbfd0tDQkDFjxmTy5Mn5whe+sN1j/vu//3v23nvvjBkzJg0NDRk/fnze/va3Z82aNWWMvDJUoAMAAAAAAAAMQcViMfvvv3+efPLJ7mWlUikrV67Meeedl9tvvz2/+MUvtmnMI488Mnfccccmy9avX5/rr78+U6dOzQMPPJBddtmlLPFXggp0AAAAAAAAgCFo1qxZ3cnz448/PosWLcptt92WQw45JEly44035hvf+Eafxzv99NO7k+dvf/vbc/fdd+fuu+/O8ccfnyRZsWJF988DlQQ6AAAAAAAAQIWUUkhnaspyK6VQtrhWrVqVa665Jkly3HHH5eKLL87kyZNz0EEH5aabbsqUKVOSJOecc06fx/vud7+bJPmnf/qnfP/738++++6bfffdNxdffHHe9a53JUl++9vfZt26dWW7H+UmgQ4AAAAAAAAwxHz7299OsVhMoVDIRRdd1GP9eeedlyR59tln8+ijj251vC9+8YspFosZPXp0zjrrrB7rv/KVr6S+vj61tbW59dZbd/wOVIgEOgAAAAAAAMAQc/XVVydJ9thjjzQ0NPRY/+Y3vznDhg1Lknzve9/b6nhXXXVVkuR1r3vdZtePHz8+y5YtS2tra4488sjtDbvihlc7AAAAAAAAAICdWbE08OqalyxZkiSZOnVqr9s0Njbm6aefzj333LPV8R5//PEkyVFHHVWeAKtk4D1SAAAAAAAAAFTUypUrkyRNTU29bjNhwoQkydKlS7c4VrFYTHt7e5LkkEMOybnnnpu99torY8aMSUNDQyZMmJC3vOUtefLJJ8sUfeWoQAcAAAAAAAAYYjo6OpIkY8eO7XWb0aNHJ0nWrl27xbGenxj/5Cc/mTvuuGOT9e3t7bnlllty4IEH5sYbb8zBBx+8nVFXngQ67CRuuGN9HlzaWe0wGOAee6rrb6SlpaXP+zQ2Nm7x22cAO4OFdz2TZY+vrnYY7KDly9Yl2bbzXCU5hwLAwNX25PI+bbdu5ep0rGmvcDTbb8ParthG1NVWOZLqWNO6qtohAECfdZaxMfj69eszfvz4Pm27fHnv1z3FYjFJMmrUqF63GTFiRJJ0V5f35plnnun++Y477kh9fX0+/elP533ve19Wr16d//iP/8i3v/3trF+/Psccc0wee+yxPsVfDRLoMMi1t7enpiY5d+6Wv/kDzzd79uw+b1tfPyotLYskAICdUnt7ewo1hVx32cPVDoUy2pbzXCXV1Y3KwoXOoQAwkDQ2NmZUXV3u/vZ1fdq+UEhKpQoHxQ4p1BS2+oE+ALB5NTVdSf0tVZdvPM9uTKT3ZsOGDd0/jxw5Mg888EAaGxuTJOPHj89Xv/rV7LPPPpkzZ05WrFiR73znOznppJN29C5UhAQ6DHK1tbUpFpP/+mRD9tvLU5ryenBpR/7hvLa0trb68B/YKdXW1qZULOXvPvyyNE4eU+1w2Im0Pv5cfvb1u51DAWCAaWpqyqKFC9Pa2rrVbVtaWjJ79uwc/Y9/k/GTd+mH6Lbdurb1SZJRDSOrHEl1LH98Va698PeprR2aFfgADF0jR47cYmV5Xw0fPjzr16/PihUret1m9equro11dXVbHOv5FfHvfve7u5Pnz3f66afn7LPPzurVq/PjH/9YAh2orP32Gp6X7ucpDQDbo3HymOzx4l2rHQYAAP2gqalpm77gNn7yLpn44nEVjGj7rVnZNYVN/a69t10FAOjNrrvumjVr1mTJkiW9brMxUT9p0qQtjjV58uTun1/3utf1ut3uu++ehx9+OE899dQ2Rtt/ytdsHwAAAAAAAIBNlFJIsVSeWymFssW19957J0kWL17c6zYb5zY/+OCDtzjWyJEjM3JkV1ecNWvW9Lrdxs4xW2sJX00S6AAAAAAAAABDzFvf+tYkyVNPPZW2trYe66+77rp0dnYmSU488cStjrcxIX/NNdf0us2jjz6aJNl33323Od7+IoEOAAAAAAAAMMScfPLJqampSalUymmnndZj/ac+9akkyYQJEzJlypStjvee97wnSXLttddutqr9rLPO6q5O/8d//McdCb2iJNABAAAAAAAAKqgzNWW5lVNDQ0N3FfoVV1yRU089NU8++WTuu+++HHXUUXnwwQeTJJ/97Gc32W/cuHEZN25cjjrqqE2Wn3HGGWloaEixWMyrXvWqfOlLX8ry5cvz0EMP5QMf+EDOOeecJMn06dNz+OGHl/W+lJMEOgAAAAAAAMAQdOmll2bSpElJknnz5mW//fbLq171qtx+++1JkqOPPjonnXTSJvts2LAhGzZsSGtra4/xfvazn2XYsGFZv359/vVf/zVNTU152cteliuuuCJJsttuu+WXv/xlhe/VjpFABwAAAAAAABiCampqsnDhwrzrXe9KXV1dkqRQKGTs2LGZM2dOd+K7r/7mb/4m99xzTw499NCMGDGie/kuu+ySE044IQ8++GB22WWXst6Hchte7QAAAAAAAAAAdlalJJ2l8tQ1l8oyyqZqampyySWX9Hn7tra2La7fe++9c8MNN+xoWFWjAh0AAAAAAAAAIoEOAAAAAAAAAEkk0AEAAAAAAAAgiTnQAQAAAAAAACqokGLZ6poLZRqH3qhABwAAAAAAAIBIoAMAAAAAAABAEi3cAQAAAAAAACqmVEo6S+Wpay6VyjIMW6ACHQAAAAAAAAAigQ4AAAAAAAAASSTQAQAAAAAAACCJOdABAAAAAAAAKqpTXfOg4ZECAAAAAAAAgEigAwAAAAAAAEASLdwBAAAAAAAAKqaUQjpL5alrLqVQlnHonQp0AAAAAAAAAIgEOgAAAAAAAAAk0cIdAAAAAAAAoKKK6poHDY8UAAAAAAAAAEQCHQAAAAAAAACSSKADAAAAAAAAQBJzoAMAAAAAAABUVGepUO0Q6CMV6AAAAAAAAAAQCXQAAAAAAAAASKKFOwAAAAAAAEDFlJJ0lqmuuVSWUdgSFegAAAAAAAAAEAl0AAAAAAAAAEiihTsAAAAAAABABRXSWSpXXXOhTOPQGxXoAAAAAAAAABAJdAAAAAAAAABIIoEOAAAAAAAAAEnMgQ4AAAAAAABQMaWkbHOgl8oyCluiAh0AAAAAAAAAogIddhq/uqM9Dy7tqHYY7GSWPNWZJGlpaalyJAPf8uXLkyTjx4/vsa6xsTFNTU39HRKwDR666+m0Pt5W7TDYiaxYtjrJtp1DnS8AYGBa/viqPm+7esXatK/eUMFoNtW+putYtfUj+u2Y/aF29IiMHlu31e225bEBgJ1JqVTKlVde2WN5c3Nzpk+fXoWIdi4S6DDItbe3p6Ym+eLctdUOhZ3Y7Nmzqx3CoFZfPyotLYskRWAAam9vT6GmkJt+tLjaobCT2pZzaF1dXRYuXOh8AQADRGNjY+rq6nLthb/v8z6FQlLSV3WHbcvvsa6uLo2NjZUNCADKoLOMjcE7Ojoya9asHsuPOOKIzJ8/v2zHGaok0GGQq62tTbGY/NenGjJ1L09pqJZnnysmScaN2fQiaPHSjvzDuW1pbW2VEIEBqLa2NqViKa865cjsMmlstcNhCFv1xIrc9s0bnC8AYABpamrKwoUL09ra2qftW1paMnv27Hzgo/vlRXvWVzi6LqvbuirQRzfsPBXoTz22Jt/7zwczb968TJs2bavb6+IDwFA0fPjwzJ07t8fy5ubmKkSz85Ftg53E1L2G56X7eUpDtfxlRVcCfbex5fsWIdB/dpk0NuP3UbUCAMCmmpqatjk5+6I967PXlIYKRbSp51auT5KM2XVkvxyvP02bNi0zZsyodhgAMCAVCoXMnDmz2mHstHzKDwAAAAAAAABRgQ4AAAAAAABQMaUU0lkqT11zKYWyjEPvVKADAAAAAAAAQCTQAQAAAAAAACCJFu4AAAAAAAAAFVVU1zxoeKQAAAAAAAAAIBLoAAAAAAAAAJBEC3cAAAAAAACAiimVks5SeeqaS6WyDMMWqEAHAAAAAAAAgEigAwAAAAAAAEASCXQAAAAAAAAASGIOdAAAAAAAAIAKKpRtDvSkUKZx6I0KdAAAAAAAAACIBDoAAAAAAAAAJNHCHQAAAAAAAKBiSkmKpfK0Xi+VZRS2RAU6AAAAAAAAAEQCHQAAAAAAAACSaOEOAAAAAAAAUFGd6poHDY8UAAAAAAAAAEQCHQAAAAAAAACSSKADAAAAAAAAQBJzoAMAAAAAAABUUCHFUrnqmgtlGofeqEAHAAAAAAAAgEigAwAAAAAAAEASLdwBAAAAAAAAKqaUpLNMLdxLZRmFLVGBDgAAAAAAAACRQAcAAAAAAACAJBLoAAAAAAAAAJDEHOgAAAAAAAAAFVVU1zxoeKQAAAAAAAAAICrQAQAAAAAAABhAPvKRj2zXfhdccMEOH1sCHQAAAAAAAKBCSimks1SexuClFMoyzkB3ySWXbNd+EugAAAAAAAAA7FR22WWXFAqb/7JAsVjMmjVr0tnZmSQZMWJEXvKSl5Tt2BLoAAAAAAAAAAwYTzzxxFa3ueOOO/KhD30oDz74YJLk5ptvLsuxy9MrAAAAAAAAAICeSkmxVFOWW0rVvjMDx6GHHpq77rorL3vZy/LHP/4xH/vYx8oyrgQ6AAAAAAAAAIPSf/zHfyRJLr300rKMJ4EOAAAAAAAAwKDU3NycJFm9enVZxpNABwAAAAAAAGBQmjt3blnHG17W0QAAAAAAAADYRLGkrrnc2tra8p3vfCef//znkyRjxowpy7gS6LCT+OUd7Vm8tKPaYcCQtWp1MUmyy+hNL4KWPNWZJGlpaenzWI2NjWlqaipfcMBWrXpiRZ+2W7tiTTasWV/ZYAahEfUjUze2vtphDGp9/RsEAAa++/+4PE89tqZfjrV2TddnQXX1O8/HvM8sW5ek7++jvYcGYCgqlUq58soreyxvbm7O9OnTqxBR+TU0NGzzPqeeempZjr3zXFnBENXe3p6amuSL31tb7VCALZg9e3aft62vH5WWlkU+AIB+0NjYmLq6utz2zRv6tkOhkJRKlQ1qMPJ7KYu6uro0NjZWOwwAYDtt/IzmF/9vabVD2Sn09X10Xf2oLPQeGoAhpqOjI7Nmzeqx/Igjjsj8+fOrEFF1FQqFvOc978nnPve5sowngQ6DXG1tbYrF5JLPNOSAvT2loVqWr+qqQB+/y4614Vn4aEdOPKstra2t3vxDP2hqasrChQvT2tq61W1bWloye/bsHPKhv82YPcb1Q3SDw3NPPps7v/XLzJs3L9OmTat2OIOa6ikAGNw2fkbzj/93r0zas7Zfjtn2XFcFesOYofmZ0BOPtefC/1jqPTQAA14phXSWqYV7KYUMHz58s/N+Nzc3l+UYA8H111/fp+1Gjx6dgw46KDU15WuRPzSvrGAndMDew/Py/T2loVqWPduVQJ84zjw2MNg0NTVt04dtY/YYl7H77FbBiAanadOmZcaMGdUOAwCg6ibtWZsXN/fP9DYrV3Ql0Hcd6zMhABhKCoVCZs6cWe0wKuqwww6r2rF9yg8AAAAAAABAVUydOjV77rnnJssmT57cY1l/8dVEAAAAAAAAgAoqlgrVDmHAeuKJJ5IkDzzwQKZPn54kWblyZdXikUAHAAAAAAAAoCpqa2vT3t6e17zmNT2qzl/60pf2eZxCoZC77757h+ORQAcAAAAAAACgKj70oQ/lggsuSHt7e/70pz9tsu6F/+8PEugAAAAAAAAAVMUXv/jFHHbYYfl//+//ZcWKFUmSW265JUnymte8pt/jkUAHAAAAAAAAqJBSkmJqyjbWzuhtb3tb3va2t3X/v6GhIUlyzTXX9Hss5XmkAAAAAAAAAKCMVq1a1e/HlEAHAAAAAAAAYMAYNmxYkuSmm27q92NLoAMAAAAAAABUULFUU5bbULGxnfvZZ5/d78ceOr9lAAAAAAAAAAa873//+3nNa16T++67L4cddlh++ctf9tuxh/fbkQAAAAAAAABgK6ZMmZKkq5X7vffem+OOO677/4VCodf9VqxYscPHlkAHAAAAAAAAYMBYtmzZZpd3dnZW/NgS6AAAAAAAAAAVUyjj/OW9V1/vTM4999yqHVsCHQAAAAAAAIAB47TTTqvascv1VQcAAAAAAAAAGNRUoAMAAAAAAABUSKmUsrVwL5XKMgxboAIdAAAAAAAAACKBDgAAAAAAAABJtHAHAAAAAAAAqKhytXCn8jxSAAAAAAAAABAJdAAAAAAAAABIIoEOAAAAAAAAAEnMgQ4AAAAAAABQQYUyzoFeKNM49EYFOgAAAAAAAABEAh0AAAAAAAAAkmjhDgAAAAAAAFAxpaRsLdxLZRmFLVGBDgAAAAAAAACRQAcAAAAAAACAJFq4AwAAAAAAAFRUuVq4U3keKQAAAAAAAACIBDoAAAAAAAAAJJFABwAAAAAAAIAk5kAHAAAAAAAAqKBCSmWbA71QpnHojQp0AAAAAAAAAIgEOgAAAAAAAAAk0cIdAAAAAAAAoHJKSbFcLdxL5RmG3qlABwAAAAAAAIBIoAMAAAAAAABAEgl0AAAAAAAAAEhiDnTYacy/rT2LHu2odhgwZK1aXUyS7DJ6x76b9ucnO5MkLS0tfdq+sbExTU1NO3RMYNs89+SzFRt73co12bCmvWLjV8Ka1lVJ+v66Re+8pgPAzmHBnc/licfW9cux1qzpeg9ZXz+sX4430Pzl6Q1JKnct6voMgHIpJSmVaQ70rrFKufLKK3usa25uzvTp08tynKFMAh0Gufb29tTUJP/6nbXVDgUoo9mzZ/dpu/r6UWlpWeQNPfSDxsbG1NXV5c5v/bJixygUklKpYsNXVF9ft+hdXf2oLPSaDgCD1sbPaC7/wdPVDmXIqdS1qPfcAAxUHR0dmTVrVo/lRxxxRObPn1+FiHYuEugwyNXW1qZYTM4/oyH77TU0v20MA8GK57oq0MeO6b/ZUR5c2pnTv9SW1tZWb+ahHzQ1NWXhwoVpbW2tyPgtLS2ZPXt2Tv2npkzac1RFjsHA9cRj63LR15Z4TQeAQWzjZzRf/sTo7NtPn9FU473oUPHQ0s584surXZ8BMCANHz48c+fO7bG8ubm5CtHsfCTQYSex317D8pJ9PaWhWlpXdH1o0TjWhxawM2tqaqr4h2eT9hyVfZrrK3oMAAAqZ9+9huXAfvqM5pm/vhed4L0oAAxwhbK1cE8KKRQKmTlzZpnG44VcWQEAAAAAAABAJNABAAAAAAAAIIkW7gAAAAAAAAAVVSxbC3cqzSMFAAAAAAAAAJFABwAAAAAAAIAkEugAAAAAAAAAkMQc6AAAAAAAAAAVVTIH+qDhkQIAAAAAAACASKADAAAAAAAAQBIt3AEAAAAAAAAqqFDGFu6FMo1Db1SgAwAAAAAAAEAk0AEAAAAAAAAgiRbuAAAAAAAAAJVTSlKuFu6l8gxD71SgAwAAAAAAAEAk0AEAAAAAAAAgiQQ6AAAAAAAAACQxBzoAAAAAAABAxZSSlMo0B3opSaEsI9EbFegAAAAAAAAAEAl0AAAAAAAAAEiihTsAAAAAAABABRVSKparrlkD90pTgQ4AAAAAAAAAkUAHAAAAAAAAgCQS6AAAAAAAAACQxBzoAAAAAAAAAJVVUtc8WHikAAAAAAAAACAS6AAAAAAAAACQRAt3AAAAAAAAgMrSwn3Q8EgBAAAAAAAAQCTQAQAAAAAAACCJFu4AAAAAAAAAlVMqJKVh5RuLilKBDgAAAAAAAACRQAcAAAAAAAAYsorFYk444YTstttuaWhoyJgxYzJ58uR84QtfqHZoVSGBDgAAAAAAADAEFYvF7L///vmf//mfrF27NklSKpWycuXKnHfeeTn22GN3+Bjr16/Pi170ojQ0NOTnP//5Do9XaRLoAAAAAAAAAJVUqinPrcxmzZqVJ598Mkly/PHHZ9GiRbnttttyyCGHJEluvPHGfOMb39ihY7zzne9MW1vbDsfaXyTQAQAAAAAAAIaYVatW5ZprrkmSHHfccbn44oszefLkHHTQQbnpppsyZcqUJMk555yz3cf4n//5n/z6178uS7z9RQIdAAAAAAAAYIj59re/nWKxmEKhkIsuuqjH+vPOOy9J8uyzz+bRRx/d5vFXrFiRU045ZYfj7G8S6AAAAAAAAAAVU0iKNeW5pVC2qK6++uokyR577JGGhoYe69/85jdn2LBhSZLvfe972zz+W9/61rS3t+eAAw7YsUD7mQQ6AAAAAAAAwBCzZMmSJMnUqVN73aaxsTFJcs8992zT2Oeff34WLFiQESNGdCfqBwsJdAAAAAAAAIAhZuXKlUmSpqamXreZMGFCkmTp0qV9Hnfp0qX5l3/5lyTJl7/85UycOHEHoux/w6sdAFAeDy7trHYISZJly4tZtbpU7TCGlF1GFzJxvO9DVduK54pJkrFj+u+xGCjPe6C8nnhsXUXGXfHshqxZ7XWjP9WPHpax40b0adtKPe4AQP+78Q/r86d+er/23F8/gxkzunytXOmy9Omux7ClpaXKkfSPxsbGLSZPANhxhVL5Pjtev359xo8f36dtly9f3uu6jo6OJMnYsWN73Wb06NFJkrVr1/Y5vre85S3p7OzMK1/5ypx00kl93m+gkECHQa6xsTH19aNy+pfaqh1KkqSmJikWqx3F0OJ3PrTV14/qbqEDDG6NjY2pqx+Vi762pCLj1xSSou+49att/Z3XeU0HgEGtvb09NTXJ177vi3E7k9mzZ1c7hH5RXz8qLS2LJNEBhpjiX5MLo0aN6nWbESO6igPa29v7NOacOXPyyCOPZNSoUfn5z3++40FWgQQ6DHJNTU1paVmU1tbWaoeSlpaWzJ49O3P/qy7TpqqI7g8ti4t5/z+szbx58zJt2rRqhzOkbfwWX1+/9VcuviEOO4+mpqYsrNA5feM5+hufbMjUvbwF6A+Ll3bktPPatukc7TUdAAa32traFIvJNz/VkP2b+uea69m/dkMb14/d0Nj5LFrSkVPObUtra6vrUYBBYuTIkVusLO+rmpqua4gtVZdvTJxvTKRvyX333ZcLLrggSfLf//3faWho2OEYq8GnZ7ATaGpqGlAXt9Om1mTGS4dVO4whZdq0aZkxY0a1wxjSli1bliSDbi4XYGCp9Dl96l7Dc/B+3gL0J+doABh69m8anpf20zXXX57tSqDvNk4CHQDYdsOHD8/69euzYsWKXrdZvXp1kqSurm6r4x177LEplUp54xvfmOOOO65MUfY/n54BAAAAAAAAVEgh5ZsDvVCWUbrsuuuuWbNmTZYs6X1KwY2V7pMmTdriWP/3//7ftLa2ZsyYMbn88svLGGX/k0AHAAAAAAAAGGL23nvvPPnkk1m8eHGv2zzzzDNJkoMPPniLY7W0tCRJnnvuuYwdO7bX7d7znvck6Zp3fSBMT7w5evsAAAAAAAAADDFvfetbkyRPPfVU2traeqy/7rrr0tnZmSQ58cQT+zW2apJABwAAAAAAAKiUUiEp1pTnVipfE/eTTz45NTU1KZVKOe2003qs/9SnPpUkmTBhQqZMmbLFsebPn5+2trZebxv94Ac/SFtb24CtPk8k0AEAAAAAAACGnIaGhu4q9CuuuCKnnnpqnnzyydx333056qij8uCDDyZJPvvZz26y37hx4zJu3LgcddRR/R5zfzAHOgAAAAAAAMAQdOmll+aAAw7IE088kXnz5mXevHmbrD/66KNz0kknbbJsw4YNSTKgq8h3hAp0AAAAAAAAgAoqlGrKciu3mpqaLFy4MO9617tSV1fXFWuhkLFjx2bOnDm54ooryn7MgU4FOgAAAAAAAMAQVVNTk0suuaTP2z9/TvNK7lMtKtABAAAAAAAAIBLoAAAAAAAAAJBEC3cAAAAAAACAiiqUhlU7BPpIBToAAAAAAAAARAIdAAAAAAAAAJJo4Q4AAAAAAABQQYUUiuWqay6UaRx6owIdAAAAAAAAACKBDgAAAAAAAABJJNABAAAAAAAAIIk50AEAAAAAAAAqp5TUlMpU11wqzzD0TgU6AAAAAAAAAEQCHQAAAAAAAACSaOEOAAAAAAAAUDGFJIVieeqaC9HFvdJUoAMAAAAAAABAJNABAAAAAAAAIIkW7gAAAAAAAAAVVEihVK665kKZxqE3KtABAAAAAAAAIBLoAAAAAAAAAJBEAh0AAAAAAAAAkpgDHQAAAAAAAKCiaorDqh0CfaQCnf+/vTuPs7qu9wf+mmEZGAbQYcB9MFFZXEosbxrZ1XLJPVMTRaPMJDN/XVtcU7OyW3n12s0lNVPErmWuGGrlmrmWpZkDYiqoIIuIiOzM+f0xdyZpBhzwnDnAPJ+PxzzE7/L5vr9z+HA+5/s+788HAAAAAAAAgEigAwAAAAAAAEASU7gDAAAAAAAAlE6hIhWNRaprLlQUpx1WSgU6AAAAAAAAAEQCHQAAAAAAAACSmMIdAAAAAAAAoKQqTL2+zlCBDgAAAAAAAACRQAcAAAAAAACAJBLoAAAAAAAAAJDEGugAAAAAAAAAJVORpKJQnLrmiiSForTEyqhABwAAAAAAAIBIoAMAAAAAAABAElO4AyVw5++XZeJzy8sdRqfw4tSmiVoaGhpKdo26urrU19eXrH0AOs7vn1icyS8vK3cYncKU15rGQqvzHu09FwDWD5Omdtx46423GpMkG85WJ9VeM+Y05s35jeUOY63SPHYFoLQqGiuK1lahUMj48eNbbR80aFCGDRtWtOt0VhLoQNEsXrw4lZXJ2d9fXO5QOp1Ro0aVrO3q6h5paJjkgT7AOqz5PfoHYxeWO5ROZ3Xeo73nAsC6ra6uLtXVPXLCD+aXOxRWobIyaZQ/b6WysulzAwDrhmXLlmXkyJGtto8YMSJ33XVXGSJav0igA0VTVVWVxsZk7BU9MmRb33xeH0x8rjHHfnFRZs+e7WE+wDqs+T36jK/Vpn7zbuUOhzZMfWVpzv+vOd5zAWAdVl9fn4aGSZk9e3aHXXPOnDlJktra2g675rqsoaEho0aNyjXXdM+QIZ5dNZs4sTGjRy9JVVVVuUMBoJ26du2asWPHtto+aNCgMkSz/pFAB4puyLaVGf6BLuUOAwD4F/Wbd8u2W3cvdxgAAOut+vr6Dv0y3MyZM5MkAwYM6LBrrg+GDKnMTjtJoAOw7qqoqMiBBx5Y7jDWWxLoAAAAAAAAAKVSSCoKRVoDvVCcZlg5X7MDAAAAAAAAgEigAwAAAAAAAEASU7gDAAAAAAAAlFBFKhqLNIV7itUOK6MCHQAAAAAAAAAigQ4AAAAAAAAASUzhDgAAAAAAAFBSFQVTr68rVKADAAAAAAAAQCTQAQAAAAAAACCJBDoAAAAAAAAAJLEGOgAAAAAAAEBpWQN9naECHQAAAAAAAAAigQ4AAAAAAAAASUzhDgAAAAAAAFA6haSisXhtUVoq0AEAAAAAAAAgEugAAAAAAAAAkMQU7gAAAAAAAAAlU5GkokhTr1cUpxlWQQU6AAAAAAAAAEQCHQAAAAAAAACSSKADAAAAAAAAQBJroAMAAAAAAACUVsHq5esKFegAAAAAAAAAEAl0AAAAAAAAAEhiCncAAAAAAACAkqoolDsC2ksFOgAAAAAAAABEAh0AAAAAAAAAkkigAwAAAAAAAEASa6ADAAAAAAAAlJY10NcZKtABAAAAAAAAIBLoAAAAAAAAAJDEFO4AAAAAAAAAJVVhCvd1hgp0AAAAAAAAAIgEOgAAAAAAAAAkMYU7AAAAAAAAQOkUkorGIs3hbir4klOBDgAAAAAAAACRQAcAAAAAAACAJBLoAAAAAAAAAJDEGugAAAAAAAAApWXt8nWGCnQAAAAAAAAAiAp0oATu+t2yTHyusdxhUAQvTWl6HRsaGsocydpvzpw5SZLa2tpW++rq6lJfX9/RIQG0MvWVpeUOgZVofm0663vuqt5H1xbezwFg/TFxoudW77Qu/z6mTp2a2bNnlzuMlVoXxrkUh88LdLRCoZDx48e32j5o0KAMGzasDBGtXyTQgaJZvHhxulQmZ39vSblDochGjRpV7hDWadXVPdLQMMkgGiiburq6VFf3yPn/NafcobAKFanwnrsW69GjZyZNmuj9HADWYc3j4tGjF5U7lLVOdXWP1NXVlTuM1TJ16tQMHTo4CxZ4PSk/z/9oj4oiTuG+bNmyjBw5stX2ESNG5K677irehTopCXSgaKqqqrK8MRn3g14ZOqhLucOBDjXnzaZva9f2XXF1lIZ/LM+oU9/O7NmzDaCBsqmvr09Dw6S1ujKjs2toaMioUaMybNtD0qt63XpwWQxLly5MknTr1rPMkbTt7QWz8+xzt3o/B4B1nHHxyq2L1bOzZ8/OggWLcs3PqjJ0cEW5w2nTnDeasmW1G66d8VEcDZMKGX3cIp8X6FBdu3bN2LFjW20fNGhQGaJZ/0igA0U3dFCXDB/mnxc6l5mvNyXQB/SrfJcjAcqjvr7eB/l1QK/quvSu2aTcYXS4JUveTpJ0796rzJEAAOs74+L1z9DBFdlpp7WzmGfmzKYE+oABEujrt+XlDoBOqKKiIgceeGC5w1hvyXABAAAAAAAAlFKhiHO4U1LK5AAAAAAAAAAgEugAAAAAAAAAkEQCHQAAAAAAAACSWAMdAAAAAAAAoLSsgb7OUIEOAAAAAAAAAJFABwAAAAAAAIAkpnAHAAAAAAAAKJ1CUlGsGdzNBF9yKtABAAAAAAAAIBLoAAAAAAAAAJBEAh0AAAAAAAAAklgDHQAAAAAAAKCECknBIujrChXoAAAAAAAAABAJdAAAAAAAAABIYgp3AAAAAAAAgJKpSFJRpCncK4rSCquiAh0AAAAAAAAAIoEOAAAAAAAAAElM4Q4AAAAAAABQWkWawp3SU4EOAAAAAAAAAJFABwAAAAAAAIAkEugAAAAAAAAAkMQa6AAAAAAAAAAlVWEN9HWGCnQAAAAAAAAAiAQ6AAAAAAAAACQxhTsAAAAAAABAaZnCfZ2hAh0AAAAAAAAAIoEOAAAAAAAAAElM4Q4AAAAAAABQOoUUbwp3M8GXnAp0AAAAAAAAAIgEOgAAAAAAAAAkkUAHAAAAAAAAgCTWQAcAAAAAAAAooUIqirZ4uUXQS00FOgAAAAAAAABEAh0AAAAAAAAAkpjCHQAAAAAAAKC0CqZeX1eoQAcAAAAAAACASKADAAAAAAAAQBIJdAAAAAAAAABIYg10AAAAAAAAgJKqKDSWOwTaSQIdKLoJDy5JwwvLyx0GdKg35xeSJH1rKlbY/uIrTX2hoaGh3W3V1dWlvr6+eMEBsM54e8Hsdh23eMn8LFu2qMTRdJxlyxYnSbp2rSpzJG1buGhuuUMAAGAl7vzt8kyctHYmpebNa3pe1KdPxbscybrsxSlNr/PqPP8rJc8WO4dCoZDx48e32j5o0KAMGzasDBGtXyTQgaJZvHhxulQm3/rx+vMwF4pl1KhR7T62urpHGhomGegCdCJ1dXXp0aNnnn3u1naeUZGkUMKIaK0iixcvLncQAAD8n8WLF6eyMjn3vKXlDgWSrN7zv1LybLFzWLZsWUaOHNlq+4gRI3LXXXeVIaL1iwQ6UDRVVVVZ3phcd3F1hm5dWe5woEPNmdv0TefaDd7b3/2G5xtzzP9bkNmzZxvkAnQi9fX1mTRpYmbPfvcK9IaGhowaNSrDBn86var7d0B0pbd06cIkSbduPcscSdveXjArz066KVVVa2eFPABAZ1RVVZXGxuSaa7pnyJC181nknDlNX3qtrVWBTseYOLExo0cv8mxxbVUo3hfhu3btmrFjx7baPmjQoKJdozOTQAeKbujWlRm+g39e6Fxmzm5KoA+oe68f2Ja992AAWCfV19ev1gOOXtX907tm0xJG1HGWLJmfJOnevabMkQAAsK4ZMqQyO+20dibQZ85sSpYNGCCBDhRXRUVFDjzwwHKHsd5aO99VAAAAAAAAAKCDKREFAAAAAAAAKKUiTuFOaalABwAAAAAAAIBIoAMAAAAAAABAEgl0AAAAAAAAAEhiDXQAAAAAAACAEiqkotBYtLYoLRXoAAAAAAAAABAJdAAAAAAAAABIYgp3AAAAAAAAgBIz9fq6QgU6AAAAAAAAAEQCHQAAAAAAAACSmMIdAAAAAAAAoHQKSQqNxWuLklKBDgAAAAAAAACRQAcAAAAAAACAJBLoAAAAAAAAAJDEGugAAAAAAAAAJVVRsHj5ukIFOgAAAAAAAABEAh0AAAAAAAAAkpjCHQAAAAAAAKCECkmhsXhtUVIq0AEAAAAAAAAgEugAAAAAAAAAkMQU7gAAAAAAAAAlVqwp3Ck1FegAAAAAAAAAEAl0AAAAAAAAAEgigQ4AAAAAAAAASayBDgAAAAAAAFBahUK5I6CdVKADAAAAAAAAQCTQAQAAAAAAACCJKdwBAAAAAAAASqYihVQUGovWFqWlAh0AAAAAAAAAIoEOAAAAAAAAAEkk0AEAAAAAAAA6rcbGxowePTr9+/dPTU1Nevfunc022yznnXdeuUMrCwl0AAAAAAAAgJJqLNJPkaNqbMzgwYPz61//OgsXLkySFAqFvPnmm/nhD3+YAw44YLXbvOKKK7LtttumT58+LQn5TTfdNCeeeGKWLVtW7FsoOgl0AAAAAAAAgE5o5MiRmT59epLkiCOOyKRJk/Loo49m5513TpLcf//9ueSSS1arvVNOOSXTpk1LY2NTwr9QKGTevHkZO3ZsBg0alLlz5xb9PopJAh0AAAAAAACgk5k3b17uvPPOJMkhhxySq6++Optttlm23377PPDAA9lqq62SJP/5n//ZrvZuvvnmjB8/Pkmy0UYbZdy4cZk2bVrGjx+fj3zkI0mS119/PXvuuWcJ7qZ4JNABAAAAAAAASqnQWJyfIrryyivT2NiYioqKXH755a32//CHP0ySvPHGG5kyZcq7tvfd7343SdKrV69MmjQphxxySPr06ZM99tgjd999d8aMGZMkee655/LXv/61eDdSZBLoAAAAAAAAAJ3MhAkTkiSbbLJJampqWu3fd99906VLlyTJtdde+67tvfTSS0mSgw8+OF27dm21/4ILLmhp7xe/+MWahl1yEugAAAAAAAAAnczUqVOTJNtuu+1Kj6mrq0uSPP300+/a3pIlS5Iku+2220qP6dGjR5Km6ePXVq1T/wAAAAAAAAAURyFJoVC8torkzTffTJLU19ev9Jh+/fplxowZefnll9+1vS996UspFAr5+Mc/3ub+2bNn5+23306SDBs2bA0i7hgS6EDRNTzfmGRZu459bWZj5s4r4r/2UCZvvtX097hv74pW+zboU5GNB7Rv0pem/gMA7+7tBbNK1vbiJW9l2bJFJWv/XzVfq2vXHh12zdWxcNEb5Q4BAICVuOuuZZk4ce2cbHfe/z337NOn9fMiKIWXXmp6ttjQ0NDuc+rq6laZPGXttGTJktTW1rbr2Dlz5qx037JlTbmcDTbYYKXH9OrVK0mycOHCd73Wj370o1XuP+yww5IklZWVOfbYY9+1vXKRQAeKpq6uLtXVPXLM/1vQ7nMqK5NG+ULWc6v797y6ukfLtDgA8K/q6urSo0fPPDvpphJepSJF/Ur7+qCiMosXLy53FAAA/J/FixensjI599z2FfJAZzJq1Kh2H1td3SMNDZMk0Tupxv97cN08rXpbunXrliTv6TPx3Llzs99++7VMA3/44YevMmlfbhLoQNHU19enoWFSZs+e3a7jGxoaMmrUqFz3/ZoMfV+XEkcHpTXnzaaBRm3fFb/x3PDi8hxz+vyMGzcuQ4cObVdbvvUJwKrU19dn0qSJ7R5zra7mMdqwHU9Or5rNSnKNf7V0yfwkSbfuNR1yvdX19vxX8+zTP05VVVW5QwEA4P9UVVWlsTG55uruGTJk7axAnzOn6UuptbUq0Fk7TZzYmNGfX5TZs2d7HrmO6d69+yory9ursrLp389VVZc3J86bE+mr6+tf/3quvPLKLF++PEnyb//2b/nZz362Rm11FAl0oKjq6+tX+4126Pu6ZPgw/xyxbpv5elMCfUC/tj+wDR06NMOHD+/IkABYj63JmGt19arZLL37blXSazRbsrhpzbXuVX075HoAAKw/hgypzE47rZ0J9JkzmxLoAwZIoAOFJMWajrd4M8Z17do1S5Ysydy5c1d6TPOa5T179lyttm+99daMGTMm8+c3fWm+srIyX/va13LOOeescbwdRcYKAAAAAAAAoJPp27dvFixYkKlTp670mOZK90033bTd7R5xxBGZMGFCy//vuuuuGTt2bDbZZJM1D7YDSaADAAAAAAAAdDIDBw7M9OnT89xzz630mNdffz1JsuOOO7arzd12261lrfO6urrcfPPN69zsrGvnvCYAAAAAAAAA64tCY3F+imj//fdPkrz22mstU62/029/+9uWtcs/97nPvWt7/+///b+W5Pkee+yRl156aZ1LnicS6AAAAAAAAACdzvHHH5/KysoUCoV8+ctfbrX/1FNPTZL069cvW2211bu2N3bs2CTJ4MGDM378+OIG24Ek0AEAAAAAAAA6mZqampYq9JtuuiljxozJ9OnT88wzz2TPPffM5MmTkyRnn332CudtuOGG2XDDDbPnnnu2bHv44YezdOnSJMkPf/jDDrqD0rAGOgAAAAAAAEAJVRR5+vViuf766zNkyJBMmzYt48aNy7hx41bYv88+++S4445bYVtzonz27Nkt2x588MGWPx988MHvet1jjz02l1566XsJvWRUoAMAAAAAAAB0QpWVlZk4cWIOP/zw9OzZM0lSUVGRDTbYIGeccUZuuummdrUzc+bMUobZoVSgAwAAAAAAAHRSlZWV+fnPf97u4+fPn99q24UXXpgLL7ywmGGVjQp0AAAAAAAAAIgKdAAAAAAAAIASKiQp1hrohSK1w8qoQAcAAAAAAACASKADAAAAAAAAQBJTuAMAAAAAAACUVqFYU7hTairQAQAAAAAAACAS6AAAAAAAAACQRAIdAAAAAAAAAJJYAx0AAAAAAACgtKyBvs5QgQ4AAAAAAAAAkUAHAAAAAAAAgCSmcAcAAAAAAAAooUKSYk3hXihSO6yMCnQAAAAAAAAAiAQ6AAAAAAAAACQxhTsAAAAAAABAaRWKNYU7paYCHQAAAAAAAAAigQ4AAAAAAAAASSTQAQAAAAAAACCJNdABAAAAAAAASswa6OsKFegAAAAAAAAAEAl0AAAAAAAAAEhiCncAAAAAAACA0ikUksLy4rVFSalABwAAAAAAAIBIoAMAAAAAAABAElO4AwAAAAAAAJRYY7kDoJ1UoAMAAAAAAABAJNABAAAAAAAAIIkEOgAAAAAAAAAksQY6AAAAAAAAQAkVkkKx1kAvFKkdVkYFOgAAAAAAAABEAh0AAAAAAAAAkpjCHVgLNLy4vNwhdCqvzW7M3LdM8VJsb85vmn6nb82K30178dWmv98NDQ0luW5dXV3q6+tL0jYAndvb81/tsGstXTI/SdKte02HXXN1rMnvYurUqZk9e3YJomFljIsAoHOaOLH9UyK/9lohc9/suOdi8+Y1XatPn4oOuyblsUHfimy88br3Oq9O/6EYivf7LhQKGT9+fKvtgwYNyrBhw4p2nc5KAh0om7q6ulRX98gxp88vdyidSmVl0mhc1OFGjRpVknarq3ukoWGSh8UAFE1dXV169OiZZ5/+cblDWav06NEzdXV17Tp26tSpGTp0cBYsWFTiqHgn4yIA6Fyany2O/nz7x1yei1Eq6/LfrerqHu3+rMPaY9myZRk5cmSr7SNGjMhdd91VhojWLxLoQNnU19enoWGSypwO1NDQkFGjRuXqs2oyZKC3gGKaM69phFzbp+NWR5k4ZVk+/935mT17tgfFABRNfX19Jk2a2KFjtDlz5iRJamtrO+yaq2t1qptnz56dBQsWZezlPTJkWyundYSJzzXm2DGLjIsAoBNZ3WeLzc/Frrm6KkMHd0yl8Jw3mirQazdc9yqTab+GSYWM/vzijBs3LkOHDi13OKvNTE7rpq5du2bs2LGttg8aNKgM0ax/ZE+Asqqvr/fmXAZDBnbNTtt6CyimmW80JdAHbOghOQDrvo4eo82cOTNJMmDAgA67ZkcYsm1lhr+/S7nDAABYb63JuHXo4IrstFPHjNFmzmxKoA8YIIG+fmtawnHo0KEZPnx4mWOhs6ioqMiBBx5Y7jDWW7InAAAAAAAAAKVUWEfn+e+ElMkBAAAAAAAAQCTQAQAAAAAAACCJKdwBAAAAAAAASqiQpFhTuBeK1A4rowIdAAAAAAAAACKBDgAAAAAAAABJTOEOAAAAAAAAUFqF5eWOgHZSgQ4AAAAAAAAAkUAHAAAAAAAAgCQS6AAAAAAAAACQxBroAAAAAAAAACXWWO4AaCcV6AAAAAAAAAAQCXQAAAAAAAAASGIKdwAAAAAAAIASKqRQKNYU7oUitcPKqEAHAAAAAAAAgEigAwAAAAAAAEASU7gDAAAAAAAAlNjycgdAO6lABwAAAAAAAIBIoAMAAAAAAABAEgl0AAAAAAAAAEhiDXQAAAAAAACAEiokhcbitUVJqUAHAAAAAAAAgEigAwAAAAAAAEASU7gDAAAAAAAAlFixpnCn1FSgAwAAAAAAAEAk0AEAAAAAAAAgiQQ6AAAAAAAAACSxBjoAAAAAAABA6RSSFJYXry1KSgU6AAAAAAAAAEQCHQAAAAAAAACSmMIdAAAAAAAAoGQKKaSQxqK1RWmpQAcAAAAAAACASKADAAAAAAAAQBJTuAMAAAAAAACUVqE4U7hTeirQAQAAAAAAACAS6AAAAAAAAACQRAIdAAAAAAAAAJJYAx0AAAAAAACghApJlhexLUpJBToAAAAAAAAARAU6QKc0ccqykrT72uuNeXN+Y0naXtvNe7vpvvv0av3dtL41ldm4X/G/s1aq1xEAKI67fr8sE5/rnGOjjvbS1Kbfc0NDQ7vPqaurS319falCAgDWUnf+dnkmTuqYMdq8eU1Von36VHTI9SiPF6c0vc6rMxZdmxgXr5sKhULGjx/favugQYMybNiwMkS0fpFAB+hE6urqUl3dI5//7vyStF9ZmTR6RtxKKX8v1dU9UldXV5rGAYA1snjx4lRWJmefv6TcoXQ6o0aNavex1dU90tAwycNCAOgkmsdo5357ablDYT21OmPRtYlxcccpFIr3kHjZsmUZOXJkq+0jRozIXXfdVbTrdFYS6ACdSH19fRoaJmX27NlFb7uhoSGjRo3K2PNqMmTLzvf2Mmde0+Cnts+KleYTX1qWY8+en3HjxmXo0KFFv65viALA2qeqqiqNjcnYK3pkyLZWTlsbTXyuMcd+cVFmz55tLAUAnUTzGO2an1Vl6OCOGaPNeaOpMrl2QxXorJ0aJjVm9HHGxeuirl27ZuzYsa22Dxo0qAzRrH86X4YDoJOrr68v6WBoyJZdM3xI53t7mTmnKYE+oLbtD2BDhw7N8OHDOzIkAKDMhmxbmeEf6FLuMAAAeIehgyuzUweN0WbO/L/nRQN8qRIoroqKihx44IHlDmO91fkyHAAAAAAAAAAdanm5A6CdfO0JAAAAAAAAACKBDgAAAAAAAABJJNABAAAAAAAAIIk10AEAAAAAAABKqJCksYhtUUoq0AEAAAAAAAAgEugAAAAAAAAAkMQU7gAAAAAAAAAlVSgUawp3Sk0FOgAAAAAAAABEAh0AAAAAAAAAkkigAwAAAAAAAEASa6ADAAAAAAAAlFAhyfIitkUpqUAHAAAAAAAAgEigAwAAAAAAAEASU7gDAAAAAAAAlFSh0FjuEGgnFegAAAAAAAAAEAl0AAAAAAAAAEhiCncAAAAAAACAElte7gBoJxXoAAAAAAAAABAJdAAAAAAAAABIIoEOAAAAAAAAAEmsgQ4AAAAAAABQQoUU0li0tigtFegAAAAAAAAAEAl0AAAAAAAAAEhiCncAAAAAAACA0iosL3cEtJMKdAAAAAAAAACIBDoAAAAAAAAAJDGFOwAAAAAAAEAJFVJIY9HaorRUoAMAAAAAAABAJNABAAAAAAAAIIkEOgAAAAAAAAAksQY6AAAAAAAAQIktL3cAtJMKdAAAAAAAAACIBDoAAAAAAAAAJDGFOwAAAAAAAEDJFApJodBYtLYoLRXoAAAAAAAAABAJdAAAAAAAAABIIoEOAAAAAAAAAEmsgQ4AAAAAAABQQoUky4vYFqWkAh0AAAAAAAAAogIdgCKb+NKykrX92uzGzJ3fWLL234t5/xdXn5oVv5v20rSmbxU2NDR0eEwdra6uLvX19eUOAwDWGhOfWzvHLXSu12bq1KmZPXv2Gp8/Z86cJEltbW2xQlrnGOcCrF8aJrVvHPDaa42Z++Z7u9a8eU1Von36VLy3hqBEXpzSecbF65tCoZDx48e32j5o0KAMGzasDBGtXyTQASiKurq6VFf3yLFnzy/ZNSork8Z1dEw3atSocodQctXVPdLQMMnDRQA6vZZx0RcXlTsUVqG6ukfq6urKHUZJTZ06NYOHDM2ihQvKHco6rUfP6kya2GCcC7COax6jjT6ufWO0dfk5FKyOyspk8eLF5Q6jUygUivePyrJlyzJy5MhW20eMGJG77rqraNfprCTQASiK+vr6NDRMek/VLavS0NCQUaNGZdyPemXooC4lucZ7MWdu0+CndoPOuTpKwz+WZ9Q33s7s2bM9WASg0yv1uIji6AxVxbNnz86ihQuy9Ue/nZ4bbLlGbSxbPC9J0rWqTxEjW3csnPtSnv/DOca5AOuB1RmjNT+HGntFjwzZds2f9cx5o6kCvXZDFeisnSY+15hjv7goVVVV5Q6F1dS1a9eMHTu21fZBgwaVIZr1jwQ6AEVTX19f8odKQwd1yfDt1r63r5mvNyXQB/TrnAl0AGBFHTEugvbqucGW6dVvyBqdu3Rh0xTu3Xp23incAVh/rO4Ybci2lRn+gTUv5Jg56/+eF/X3vAgoroqKihx44IHlDmO9tfZlIAAAAAAAAADWG4UUsrxobVFavvYEAAAAAAAAAJFABwAAAAAAAIAkEugAAAAAAAAAkMQa6AAAAAAAAAAl1ljuAGgnFegAAAAAAAAAEAl0AAAAAAAAAEhiCncAAAAAAACAEiqkUFhetLYoLRXoAAAAAAAAABAJdAAAAAAAAABIYgp3AAAAAAAAgJIqpLHcIdBOKtABAAAAAAAAIBLoAAAAAAAAAJBEAh0AAAAAAAAAklgDHQAAAAAAAKCkClle7hBoJxXoAAAAAAAAABAJdAAAAAAAAABIYgp3AAAAAAAAgBIqJIXG4rVFSalABwAAAAAAAIBIoAMAAAAAAABAElO4AwAAAAAAAJRMIUkhy4vWFqWlAh0AAAAAAAAAIoEOAAAAAAAAAEkk0AEAAAAAAAAgiTXQAQAAAAAAAEqokEIai9YWpaUCHQAAAAAAAAAigQ4AAAAAAAAASUzhDgAAAAAAAFBShcLycodAO6lABwAAAAAAAIBIoAMAAAAAAABAEgl0AAAAAAAAgE6rsbExo0ePTv/+/VNTU5PevXtns802y3nnnVfu0MrCGugAAAAAAAAAJVNIIcVaA71QpHaaNDY2ZvDgwZk+ffo/r1Ao5M0338wPf/jDPP7447njjjuKes21nQp0AAAAAAAAgE5o5MiRLcnzI444IpMmTcqjjz6anXfeOUly//3355JLLilniB1OAh0AAAAAAACgk5k3b17uvPPOJMkhhxySq6++Optttlm23377PPDAA9lqq62SJP/5n/9ZzjA7nAQ6AAAAAAAAQAkV0liUn2K68sor09jYmIqKilx++eWt9v/whz9MkrzxxhuZMmVKUa+9NpNABwAAAAAAAOhkJkyYkCTZZJNNUlNT02r/vvvumy5duiRJrr322g6NrZwk0AEAAAAAAAA6malTpyZJtt1225UeU1dXlyR5+umnOySmtUHXcgcAAAAAAAAAsD4rFJaXO4RW3nzzzSRJfX39So/p169fZsyYkZdffrmjwio7FegAAAAAAAAAncyyZcuSJBtssMFKj+nVq1eSZOHChR0R0lqhpBXor7zySpYuXZrtttuulJeBTmP58qZvJzWvNwGdyaJFi5IkB33prVR1ryhzNK39X/dMZ+2ei5cUkiSHH354evToUeZooDXvobB200ehNJrH0JPu+XoqKrutURuFQmOSpKKic9ZgFBqXJjHOZe3lPRRKo/k99FMjF6Z71Zq309mfF7H2W7K46b+ddazTUe+jhUIhixcvzsSJE4vS3uLFi1MoFFJbW9uu4+fMmbPSfY2NTeP9Vb3+3bp1a7luZ1HSBHp1dXUWLFjQ8stfWyxfvjxvvPFGNtxww/V6cOk+1z8vvvhikmSrrbYqcySl1VleU/e5erp3754hQ4YUMbLieuGFF5J03v7ZvVcyZMOmP69t7/trSh9dv3gPXb90lvtMOs+96qPrF/e59ijGGLp5nPu+9bx/Jqt6TfslMc5d13SW+/Qeuv7pLPe6tt9nsZ5DdfbnReub9fE+q3okQ/o2/bl5rLM+3ufKdOT7aKFQKNp4slu3blmyZElR2qqsbPqi7Kqqy5sT582J9M6gYv78+YVyB9HRnn322eyyyy55/PHHM2zYsHKHUzLuc/3zwQ9+MEnypz/9qcyRlFZneU3d5/pF/1z/dJZ77Sz3qY+uXzrLfSad51710fWL+1y/dJb+mXSe19R9rl86Sx/tLK9n0nnutbPcpz66fnGf65/O0kdXZcCAAVmwYEGOOeaYXHbZZW0e88EPfjATJ07M0KFD88QTT3RwhOXROeffAgAAAAAAAOjE+vZtmoJg6tSpKz2meQr4TTfdtENiWhtIoAMAAAAAAAB0MgMHDkySPPfccys95vXXX0+S7Ljjjh0S09pAAh0AAAAAAACgk9l///2TJK+99lrmz5/fav9vf/vbLF++PEnyuc99rkNjKycJdAAAAAAAAIBO5vjjj09lZWUKhUK+/OUvt9p/6qmnJkn69euXrbbaqqPDK5tOmUCvq6vL6aefnrq6unKHUlLuk3VVZ3lN3Sfros70enaWe+0s99lZdJbXs7PcZ9K57rUz6Cyvp/tkXdVZXlP3ybqoM72eneVeO8t9dhad5fV0n6yPampqWqrQb7rppowZMybTp0/PM888kz333DOTJ09Okpx99tnlDLPDVcyfP79Q7iCA9vngBz+YJPnTn/5U5kiAf6V/wtpNH4W1mz4Kay/9E9Zu+iis3fRRWLvpo00aGxszZMiQTJs2rc39++yzT2666aYOjqq8OmUFOgAAAAAAAEBnV1lZmYkTJ+bwww9Pz549kyQVFRXZYIMNcsYZZ3S65HmSdC13AAAAAAAAAACUR2VlZX7+85+XO4y1hgp0AAAAAAAAAIg10AEAAAAAAAAgiQp0AAAAAAAAAEgigQ4AAAAAAAAASSTQAQAAAAAAACCJBDoAAAAAAAAAJJFAh3XavHnzcvbZZ+f9739/+vXrl0GDBmX06NF55plnyh0adHqFQiFXXnllPvrRj2bjjTfOFltskf322y8TJkwod2iw3nn22WdzwgknZMiQIamtrc2QIUPyH//xH5kxY0abx+uf0LHeeOONnHvuuRk+fHj69euXTTfdNHvttVfGjRuXQqHQ5jlLlizJBRdckA996EPp379/ttxyyxx++OF5+OGHOzh66JzWtA/6jAprt9deey3/8R//kWHDhqVfv34ZPHhwTjrppEydOrXcoQExBoaO8uijj+boo4/O1ltvndra2uy4444555xz8tZbb630HP2z86mYP39+208sgA710ksv5eKLL859992XV199NZWVlRk0aFAOOuignHjiienTp88Kx8+aNSt77bVXnn/++VZt9ezZM7/85S+z5557dlT40Ol861vfykUXXZQLLrggY8aMWWFfoVDI0Ucfndtvv32l55566qkdESas926//fYcd9xxWbhwYat9m266aX73u99l4MCBLdv0T+hYU6dOzSc/+clMmTKlzf0HH3xwxo4dmy5durRsW7hwYQ444IA89thjrY7v0qVLLr300hx99NElixnWZ42Njdlmm23y4Q9/ONdff32bx6xpH/QZFd679vTRJHnmmWfyP//zP/nDH/6QGTNmpKqqKoMHD85hhx2WL3zhC6mqqmp1zuTJk7P33ntn1qxZrfbV1tZmwoQJ2X777Yt6P7C+aW8fbctnP/vZ3HTTTbnxxhvzyU9+stV+Y2B4b9rbP3/605/mm9/8ZpYvX95q37Bhw3L33Xdnww03XGG7/tk5qUCHtcDDDz+c3XbbLVdeeWWef/75LFy4MG+//XaefvrpfPe7381HPvKRvPDCCyucc9JJJ+X555/P5ptvnvHjx2fWrFn505/+lL322isLFy7M6NGj88Ybb5TpjmD99vDDD+fiiy9e6f5LLrkkt99+e6qqqnLxxRdn2rRp+cc//pGvfvWrSZLvfve7+cMf/tBB0cL6a9KkSfnCF76QhQsXZsyYMXnmmWcyY8aM3Hrrrdlyyy0zbdq0nHDCCSuco39Cx/r85z+fKVOmZMstt8yNN96YV199Nc8991zOP//89OjRI7fddlur99Szzz47jz32WDbYYINcd911mTFjRv7+97/nqKOOyvLly3PyySe3maAD3t3dd9+90hlamq1pH/QZFd679vTRW265JR/96Edz/fXXZ+rUqVm8eHHmzZuXJ554Iqeeemo+/vGPZ/bs2SucUygU8tnPfjazZs3K9ttvn3vvvTezZs3Kgw8+mJ133jlz5szJMccck2XLlpXy9mCd154+2pZf/epXuemmm1Z5jDEwvDft6Z8PPvhgvvGNbyRJzjzzzEyePDnTpk3Ltddem379+uXZZ5/Naaed1uo8/bNzkkCHMlu0aFFGjx6defPmZejQoS0PGp5//vn89Kc/Tf/+/fPiiy/miCOOaPkg8/TTT+c3v/lNunTpkhtvvDF77LFHevbsmSFDhuSGG27INttskzlz5uSqq64q893B+uett97KF7/4xTQ2Nra5f/HixbnwwguTJN/73vdy3HHHpU+fPtloo43y3e9+N6NGjUqhUMiPfvSjjgwb1kvf+c53smDBgowZMyYXXHBBttxyy/Tq1Suf+MQn8qtf/SpdunTJQw89lKeffjqJ/gkd7amnnsqjjz6aLl265Je//GU++clPpm/fvtl0001z8skn56yzzkqSXH755S3nzJgxI1dffXWS5Morr8ynPvWp9OrVKwMHDswVV1yRj370o1m8eHH++7//uxy3BOu0f/zjH+86y8qa9kGfUeG9a08ffe211/KlL30pS5cuza677pp77703r7/+ep599tn84Ac/SO/evfPXv/41n//851c47ze/+U2efvrp9O7dO7fcckt22WWX9OzZM8OHD89NN92U2traTJ48OTfffHMpbxHWae3po2159dVXc8opp6zyGGNgeG/a2z+/9a1vpbGxMd/97ndz+umnZ5NNNkmfPn3y6U9/umWc+stf/nKF2Vr0z85LAh3K7Pbbb8+0adPSu3fvjB8/vuVBw8Ybb5yjjz4699xzT6qrqzNx4sTceuutSdLygWavvfbKDjvssEJ7VVVV+fKXv5wkue222zr0XqAzOPXUU/PSSy+tdP8f//jHzJw5M7W1ta0eWiTJf/zHfyRJHnjggcydO7dEUcL6b+bMmbn99ttTXV2dM888s9X+YcOGZeTIkdlhhx0yadKkJPondLTmL69svfXW2W677VrtP/DAA5Mk06ZNy+uvv54kmTBhQhYvXpyhQ4e2ObVl88PH8ePHr3T9dOCf/vrXv+Yb3/hG9txzz+y0006tZjb7V2vaB31GhTWzun103LhxmT9/frbYYovcdttt2WWXXVJVVZX6+vp8+ctfzi233JKKiorce++9eeKJJ1rOa+6jI0eOzCabbLJCm3V1dRk9enSStDx3Apqsbh/9V4VCIWPGjHnXz5fGwLD6Vrd/Pvnkk/nzn/+czTffPF/60pda7d9rr73y8Y9/PEOHDm15jpTon52ZBDqU2cMPP5wk2W+//bLxxhu32r/VVlvlU5/6VJLkoYceWuG/e+21V5ttNm9/6qmn8uabbxY9ZuisfvOb32Ts2LHZYYcd8m//9m9tHvPggw8mSXbfffd079691f7Bgwenvr4+y5cvb+n/wOr73e9+l8bGxvz7v/97q7Wpml1++eV55JFHcvjhhyfRP6GjVVY2fdysqKhoc/871z1vPqZ5CYW99967zXN23333VFVV5fXXX8/f//73YoYL66U//vGPueyyy/L444+vdAald1rTPugzKqyZ1e2jzWPUI488MtXV1a32f/jDH85HPvKRJP/sl8m79+1PfOITrc4BVr+P/qvLLrss9913X/bee+/U19ev9DhjYFh9q9s/77rrriTJAQcckK5du7Z5zG233ZZHHnkkI0aMaNmmf3ZeEuhQZq+88kqSpkq5lRkwYECS5O23306SPPfcc0nS6pv9zQYOHJi+ffumUCjkH//4RzHDhU5r1qxZOemkk9K9e/dceeWV6datW5vHTZ48OcnK+2eS7LjjjiscC6y+v/zlL0mS4cOHt/sc/RM61jv7U0NDQ6v9d9xxR5Jk8803T21tbcuxycr7aVVVVbbddtsVjgVW7sgjj8xjjz3W8vOFL3xhlcevaR/0GRXWzOr20ZdffjlJ+54hzZ8/v+W/06dPT7LyPtr8nj1nzpxW66dDZ7a6ffSdJk2alHPOOSe1tbW59NJLV3msMTCsvtXtn3/961+TJDvvvPNqXUf/7Lwk0KHMvvKVr+TnP/95Dj744JUe0/yP+8CBA7NgwYLMmTMnSbLZZput9JxNN900STJ16tTiBQud2Fe+8pXMmjUrZ511VrbffvuVHtf8pRj9E0rr+eefT9L03vjcc8/luOOOy6BBg1JbW5shQ4bkpJNOarXcgv4JHWuHHXbIpz71qSxfvjyf+cxncvfdd2fevHmZPn16Lrnkknz7299Okpx99tkt5zQnBpr7Ylua+3DzscDK9evXL9ttt13LT//+/Vd5/Jr0QZ9RYc2tbh8977zz8vOf/3yFyrh3KhQK+dvf/pYk2XLLLZP8s69WVla2OfNhkmywwQbp1avXCscDq99Hmy1btizHH398Fi5cmAsvvHClfa+ZMTCsvtXtn83J7fr6+jzxxBM58sgjM3DgwPTr1y877rhjTjvttMycObPVefpn59X2PAVAh/nYxz62yv1333137rvvviRN04s0f4M4SWpqalZ6XvO+5qp1YM2NHTs2d9xxR3bdddd89atfXeWxzX1U/4TSmjVrVpKmNaxOPvnkLFy4sGXfK6+8kmuuuSY333xzfvWrX7U8YNQ/oeNdeeWV6dmzZ37xi1/k05/+9Ar7unfvnp/85Cc56qijWrY1973evXuvtM3mB/zvHBcDxbEmfdBnVOg4ba29+k5XXXVVJk+enKqqqpalE5r7XHV19QrLp/yrXr165e233/b+CkXw/e9/P08++WQOO+ywHHbYYe96vDEwlF7zDCsTJkzIT37ykyxfvrxl3wsvvJCf/OQnuemmm3L77bdn6NChLfv0z85LBTqsxa699tocc8wxSZLDDz88H/jAB7JkyZKW/W2t39qseXrpBQsWlDZIWM9NmTIlp556anr16pWf/vSnLeu5rkxzH9U/obSa+89ll12W3r1756qrrsrUqVPz+uuv56677soHPvCBzJs3L0cffXRLVZz+CR3vpZdeapkx4l8tXbo0f/vb31b4AszixYuTZKVLpST/7MPvPA8ojjXpgz6jQvktX748F1xwQb7+9a8naZpBrbnitblfr6p/vnO/91d4b5544olccMEF2WSTTXLRRRe16xxjYCi95kT4xRdfnPe973254YYbMn369MycOTO//vWvs9VWW2X69Ok56qijsnTp0pbz9M/OSwId1kJ//vOfs9dee+XLX/5yFixYkI997GO57LLLkqz4gaf5H++2NO+rqqoqbbCwHmtsbMzxxx+ft956K+eff3622mqrdz2nuY/qn1Bazd8U7tGjR+68884ceeSRqa2tTVVVVUaMGJHbb789G264YV5//fX87Gc/S6J/Qkd7/vnns88+++Txxx/P/vvvn/vuuy/Tp0/Pc889l6uvvjpbbLFFfvrTn+bYY49NY2Njkn/2vXcm5P7VokWLkrx7IgBYfWvSB31GhfK65557suuuu+bcc8/N8uXLc+SRR+acc85p2d/c51bVPxPvr1AMCxYsyPHHH5/ly5fn0ksvzYYbbtiu84yBofSanyP1798/d999dw444ID07t071dXV2XfffXPTTTelW7dumTx5cm655ZaW8/TPzksCHdYis2fPzpgxY/Lv//7veeSRR1JVVZVzzjknt99+e3r06JFkxSnxVjUlyFtvvZXkn9OHAKvv4osvzsMPP5y99947xx13XLvOae6j+ieUVnV1dZLk0EMPzeDBg1vtr62tzec+97kkyf33359E/4SOduaZZ+b111/PUUcdlV/+8pf50Ic+lN69e2fTTTfNEUcckfvuuy/9+vXLnXfemTvuuCPJP/tec19sS3uWYwDWzJr0QZ9RoTymTp2aI444IgcffHCeffbZ9OnTJz/5yU9y1VVXpaKiouW45j63cOHCLFu2bKXteX+F9+6MM87I888/n+OOO65lGYX2MAaG0mvuZ8cdd1w22mijVvu32WabHHTQQUn++Rzpnefpn52PBDqsJe6777586EMfyrhx45I0Tdn+5JNP5hvf+MYKa1RVV1entrY2SfLqq6+utL3p06cnSTbffPMSRg3rrxdffDHf+c53Ultbm0svvbTd5zX3Of0TSqtfv35Jku23336lxzQn1qdNm5ZE/4SOtGzZstx9991JkpNPPrnNYzbaaKMceeSRSZrWoUuSLbbYIsk/+21bmvdtttlmRYsXaLImfdBnVOh4v/rVr7LLLrtkwoQJ6dq1a44//vg89dRTGT16dKtjm/tcY2NjXnvttTbbmzNnTkv1nD4Ka+bhhx/OVVddla222irnn3/+ap1rDAyltybPkRL9szOTQIe1wC233JJPfepTmTVrVrbZZpvcc889+fnPf56BAwe2eXzzP+R/+9vf2tw/ZcqUlm9EbbPNNqUJGtZzL7/8cpYsWZI5c+Zk6623Tk1NzQo/Dz30UJLk61//esu2uXPnZtttt02y8v6ZJM8880yStBwLrL7m97dVTaHVPHtL8xRa+id0nNmzZ7dUua3qIULzw4jmB/rv1k8XL16cyZMnJ0mbs08A782a9kGfUaHjXHLJJfn85z+f+fPn50Mf+lAee+yxXHTRRenfv3+bxzfP/pKsvI82j4H79u3bsnY6sHpeeOGFlv9utNFGrZ4jTZ06NUlT0VRNTc0KY2RjYCi99jxH6tmzZ5IVp2LXPzsvCXQos+effz5f/OIXs2zZsuy///55+OGHs8suu6zynBEjRiRJfv/737e5v3n7jjvu2FIJAHSM3XffPUny4IMPtjkge+655zJ16tR06dKlpS8Dq2/XXXdNkjz22GMrPebJJ59Mkmy99dZJ9E/oSBtssEHLLEovv/zySo9rfpBYV1eXJPnoRz+aJPnd737X5vF/+MMfsmTJktTW1maHHXYoZshA1rwP+owKHeOhhx7K6aefnqRpCtrf//737XpY/259+5577lnhOKBjGQND6X34wx9OsurnSH/+85+T/PM5UqJ/dmYS6FBmP/7xj7Nw4cJst912GTduXMu3nFbl0EMPTdL0j3bzt4SbLV26NJdddlmS5NOf/nTxA4ZOYvfdd8/8+fNX+tP8kPCCCy5o2bbBBhvkIx/5SDbaaKO88cYbueaaa1q1++Mf/zhJsscee3h4CO/B3nvvnZ49e+buu+/OI4880mr/K6+8kmuvvTZJcuCBByaJ/gkdqEePHi0PKC6++OI2j5kxY0ZuuOGGJE39Lkn222+/9OjRIw0NDS1TwL9Tc1uHHHJIKit9nIViW9M+6DMqdIwLLrggjY2N+cQnPpGLL754hSX/VqW5j95www2ZMWPGCvveeOONlnGzPgprbtSoUat8jlRfX58kufHGGzN//vwVlj0xBobSO+igg1JRUZHrr78+zz//fKv9Tz31VO64444kyQEHHNCyXf/svLyaUGbN38QfM2ZMunXr1q5zdthhhxxwwAFZvnx5PvOZz+TBBx/MokWLMnHixIwcOTITJ05M//79c/zxx5cydKAN3bt3zymnnJIkOeOMMzJ27Ni89dZbmTFjRs4555xcc801qayszGmnnVbmSGHdtuGGG+bEE09MY2NjDjvssFxzzTV54403snDhwtx5553Zd999M3fu3Oy444457LDDkuif0NFOO+20VFRU5Je//GWOOOKIPPHEEy0PC2+44YZ87GMfy5w5c7LNNtvk8MMPT5IMGDAgxx13XJLk+OOPz/jx47NgwYJMmTIlY8aMyX333ZeePXu29GWguNa0D/qMCqW3ZMmSPPDAA0mSk08+ebXO3W+//fL+978/8+bNy6GHHponn3wyixYtyl/+8pcceuihmT17doYOHdqSaAc6ljEwlN6QIUPy6U9/OvPnz89+++2Xm2++OfPnz89bb72VX/3qVznooIOybNmy7Lvvvtltt91aztM/O6+u5Q4AOrMFCxa0TFv5la98JV/5yldWefzRRx+dn/70p0mSn/zkJ5k0aVImT56c/fbbb4XjevXqlWuvvTZ9+vQpTeDAKp144ol55JFHcuutt+bEE0/MiSeeuML+8847r6UqD1hzZ511Vv7yl7/k3nvvzUknnZSTTjpphf1bb711/vd//zddu/5zyKt/QsfZY489cuGFF+Yb3/hGJkyYkAkTJrQ6Zquttsqvf/3rFdaY+/a3v50nn3wyjzzySEaOHLnC8V27ds0ll1ySLbfcstThQ6e1pn3QZ1QorRdeeCFLly5N0lRF925OP/30nHnmmUmSioqKXHvttdl7773z1FNPtSxt1Kxfv3657rrr2l3RDhSfMTCU3sUXX5xJkyblb3/7W4499thW+z/0oQ/liiuuaLVd/+ycVKBDGb355ptrfG5dXV0efPDBnHLKKRk0aFCqqqoyYMCAHH744XnggQdafRgCOk5FRUWuu+66/Pd//3d22mmn9OrVKxtssEE+9rGP5aabbspXv/rVcocI64Vu3brllltuyX/9139l5513Tu/evdOzZ88MGzYsZ555Zv7whz9k4MCBK5yjf0LHOv744/PQQw/lmGOOSX19fbp3756ampoMHz483/72t/PHP/4xgwYNWuGcHj165De/+U3OPffcDB06ND179ky/fv2y33775e67784RRxxRpruBzmFN+6DPqFBac+fOfU/nb7311nnkkUdy/PHHZ4sttkhVVVU222yzjB49Oo888kiGDBlSnECBNWIMDKXXt2/f3HfffTnnnHOy3Xbbpbq6Or169crw4cPzgx/8IHfddVebS/rpn51Txfz58wvlDgIAAAAAAAAAyk0FOgAAAAAAAABEAh0AAAAAAAAAkkigAwAAAAAAAEASCXQAAAAAAAAASCKBDgAAAAAAAABJJNABAAAAAAAAIIkEOgAAAAAAAAAkkUAHAAAAAAAAgCQS6AAAAAAAAACQRAIdAAAAAAAAAJJIoAMAAAAAAABAEgl0AAAAAAAAAEgigQ4AAAAAAAAASSTQAQAAAAAAACCJBDoAAAAAAAAAJJFABwAAAAAAAIAkEugAAAAAAAAAkEQCHQAAoGxOOOGE1NTUrPJnk002ye67757/+q//yoIFC8oW67Rp07LTTjtlp512yuWXX75a5+67776pqanJuHHjShQdzX9fpkyZUu5QAAAAYJ3WtdwBAAAAsHJvvfVWnnzyyTz55JP55S9/mbvuuiu1tbUdHsfSpUszefLkJMnrr7/e4dcHAAAA6Agq0AEAAMpsxIgRmT9/fqufuXPn5u9//3tOO+20VFZW5tlnn803v/nNcocLAAAAsN6SQAcAAFhLde3aNQMHDsxZZ52VY445Jkly0003lWUq94EDB7Yk9s8888wOvz4AAABAR5BABwAAWAccdthhSVacSh0AAACA4pJABwAAWAdssskmLX9eunRpq/0LFizIBRdckN122y0bbbRR6uvr84lPfCLXX399li1b1mabc+bMydlnn52dd945dXV16devX3baaad861vfyptvvtnq+GHDhqWmpiYPPvhgq3333XdfDj300NTX16euri4f/OAH86Mf/ajNWJPke9/7XmpqanLCCSe0uX/KlCmpqalJTU3NSmM/55xzsvPOO6d///7ZaqutcsABB+SOO+5IoVBo85yVaY7lBz/4QZLkuuuuy4gRI7LRRhtlk002yQEHHNDmPZ9wwgmpqanJ9773vTbbffDBB1NTU5Nhw4a1eb2LL744CxYsyJlnnpmhQ4emrq4uO++8c66++uqWY8ePH59PfOIT2XjjjTNw4MAceOCB+fOf/7zK+3nyySfzmc98JvX19enfv38++MEP5vzzz8/bb7+90nMmTpyYE044IUOGDEm/fv2y3Xbb5bOf/WyeeuqpNo/fd999U1NTkyeffDIvvfRSDj/88AwYMGClvwsAAABYV3QtdwAAAAC8u9dee63lz5tvvvkK+6ZPn54DDjggkyZNatn29ttv59FHH82jjz6aG264ITfccEN69erVsn/KlCnZe++98+qrr67Q1uTJk3PRRRdlwoQJuf/++9O7d+93je2CCy7Iueeeu8K2iRMn5tvf/nbuv//+lSbw19Szzz6bgw46aIXfycKFCzNz5szcf//9GTlyZC6//PJ06dJltds+7bTT8pOf/GSFbffff38efPDB3Hjjjdlnn33ec/zNFi1alP333z9PPPFEy7ZJkybl5JNPzltvvZUkK0yXP3/+/Nx333155JFH8vjjj2errbZq1eadd96Z008/fYUvLkycODHnn39+br311vzmN79J//79Vzjnpptuyhe+8IUVzpkyZUqmTJmSm2++ORdddFG+8IUvtHkP06dPz2c+85lMnz59zX4JAAAAsJZRgQ4AALAOuO2225Iku+yySzbeeOOW7Y2NjTnmmGMyadKkbLbZZrnuuuvy2muv5dVXX81VV12Vfv365b777stXvvKVFdo77bTT8uqrr6a+vj6//vWvM23atLzyyiu5/vrr07t370yaNClXXnnlu8Z17733tiTP99lnnzz88MN5/fXX85e//CVHHHFEHnjggTz66KNF+z28/fbbOfLII/Paa69lyJAhue222zJr1qy89NJLueCCC9KzZ8/87//+b77zne+sdtsTJkzIZZddlrPPPjv/+Mc/Mm3atIwbNy61tbVpbGzMeeedV7T7SJL/+Z//yauvvppbbrkls2fPzh//+MfssMMOSZJzzz03Z511Vo466qg0NDRk2rRpueKKK9K9e/csWrRopa/Naaedli222CK33nprZs2alcmTJ+db3/pWunTpkmeffbbV34Nnnnkmxx9/fJYuXZr9998/jz76aObMmZO//e1v+eIXv5hCoZCvfvWruffee9u83te+9rX06NEj1157bV588cWcfvrpRf0dAQAAQEeTQAcAAFhLLV68OM8991zOPvvs/OxnP0t1dXUuuuiiFY4ZP358Hn300XTv3j233XZbPvWpT6WmpiZ9+/bNkUcemTvuuCNVVVW58cYb87e//a3lvPvvvz9JcuGFF2bfffdNnz59ssEGG+Tggw9uqTZ+7LHH3jXG5im7P/rRj+ZXv/pVdtxxx1RVVWWbbbbJz372s+y5555F+m00ufrqq/PCCy+ktrY2d955Zz7+8Y+nZ8+eqaury5gxY/KLX/wiSXLJJZdkxowZq9X2n//855x22mn55je/mY022ih9+vTJIYcc0pKMf/bZZ4t6L3Pnzs21116bvfbaKz169Mj73//+ltd36dKl+chHPpIrrrgiW2yxRfr06ZOjjjoqRx11VJKmmQLa0qVLl5Zp33v27JlNNtkkp556av7zP/8zSXLHHXfkmWeeaTn+vPPOy5IlSzJixIjccMMN2X777dO9e/e8733vy4UXXphTTjklSXL22We3eb05c+bkN7/5TT796U+nf//+qaz0mAEAAIB1m0+2AAAAZfbQQw+1rPf9zp9+/fpl+PDhufDCCzNo0KDcc889ef/737/CuTfeeGOS5LOf/WyGDBnSqu0ddtghRxxxRAqFQu64446W7d26dUvSNPX5vzr99NPzj3/8I5dffvkq454xY0ZLkv3cc89tNWV6RUVF0SuSm+/3lFNOaTUVeZLstdde2X333bNw4cLcc889q9V23759c9JJJ7Xa/uEPfzhJ22vPvxfDhw/PrrvuusK2D3zgAy1/Pvnkk1ud07x//vz5bbZ51FFHZeDAga22jxkzJu973/uSNFXaJ8m8efNy9913J2n6IkRFRUWr80477bT06tUrf/3rX/Pyyy+32v/JT34y9fX1bcYCAAAA6yIJdAAAgHXA1KlTWxKf7/SXv/wlSfKRj3xkpefuvPPOSZrWwm62++67J0lOOumk/OAHP1hhX3V1dTbaaKNsuOGGq4ypuaK9b9++2WWXXdo8Ztddd03fvn1X2U57LV26tOWaq7rfD37wg0lWvN/2GDZsWJtrvvfo0WO12mmvrbfeepXX2nbbbVc7lpWt0V5RUZG99947SfL3v/89SfLUU09l+fLl6dWr1wqJ+3eqrq7OsGHDkrT9+9xpp51WGQ8AAACsa7qWOwAAAIDObsSIEbnrrrtabS8UCnn55ZczduzY/OAHP8h3vvOd1NfXZ+TIkS3HzJw5M0kyevTojB49epXXmTVrVsuff/SjH+X555/PM888k+985zv5zne+k/79+2fXXXfNvvvum0MPPTQ1NTWrbG/OnDlJks0226zN6uVmW2yxRd58881VttUeb7zxRksV+B577PGux7/zftujoyupm2cBWNP9bVnVPTRXps+dOzfJP//uvP322+36kkNbv893+5IFAAAArGtUoAMAAKylKioqUl9fn7POOisHH3xwkuTmm29e4Zjly5e3u713Tte+ySab5OGHH87//u//5ogjjsiAAQMya9as3H777TnxxBOz/fbb57777ltle83rXa8qeZ6k1dTu7dHWdOmNjY2r1caCBQtW6/iuXYv7HfNly5YVtb32WNUa5M2/v+b7XJ2/O0nbv881eW0BAABgbaYCHQAAYB2w55575tZbb82UKVNW2F5bW5vXXnstDzzwQMtU7e1VWVmZAw88MAceeGCSZNKkSbnnnntyxRVX5Pnnn8/nPve5/P3vf0+vXr3aPL95DfJXX301hUJhpYn0qVOnrlZcSdpcb3uDDTZIRUVFCoVCXnzxxTbXQF+brMl9v1dTpkzJ9ttvv9J9SbLRRhslSfr165ckGTJkSP70pz91TIAAAACwllOBDgAAsA7YeOONk7SuAt5xxx2TJM8888xKz3311Vfz5JNPZvr06UmSF198MRdffHEuv/zyFY4bPHhwTjzxxNx///3p2bNnZs+evcp2d9hhh1RUVGTu3Ll5/PHH2zzmL3/5S954442VtrFo0aI2tz/22GOttvXo0aNlXfBVxfXCCy/kySefzOuvv77SY4pp8eLFbW5f2e+klH7729+2ub2xsbFl3/Dhw5M0vX5J09+HVVXrP/3003nyySezZMmSIkcLAAAAax8JdAAAgHVAc3X3v05tftBBByVJLrnkkhWmaG+2cOHCfPKTn8zuu++eJ554IkkyY8aMnHnmmfn617+eSZMmtTqne/fuKRQKSVY9bXptbW0++tGPJknOPffcNqcE//73v9/mud27d0+S/PnPf251T3Pnzs2ll17a5nnN93vhhRe2GdvMmTOz++67Z/fdd88rr7yy0tiLoXmN8kceeaTVvhdffDE33HBDSa/flnHjxrWapSBJLrvsskyZMiXdunVr+R0OGDAgu+66axYvXpwf//jHbbZ37733ZrfddsshhxyyyunhAQAAYH3h0y8AAMA6oGfPnkmaksvvdOSRR2abbbbJs88+m9133z133nlnZs2alfnz5+fBBx/MAQcckBdeeCGDBw/OfvvtlyT5wAc+kL59+yZJjjvuuPzxj3/MW2+9lbfffjuPPfZYDj/88CxatCh1dXUt1corc9pppyVJ/vCHP+SII47I008/nSVLluSFF17ImDFjMmHChPTu3bvVec3TjL/00ks5/vjj88ILL2Tx4sV59NFHc+CBB6Zbt26prq5udd6XvvSl9OvXL/fdd1/22WefPPDAA3njjTcyd+7c3HXXXdl3330zd+7c7Lnnnnn/+9+/er/k1dR8D4888ki++c1v5tVXX83ChQvz+9//PgcffHA233zzkl7/X3Xt2jXLli3LgQcemN///vdZtGhRXnvttZx//vk544wzkiQnnXRSBgwY0HLOGWeckYqKinzve9/Ll770pTz99NNZsGBBZsyYkZ/97Gc5+uijkyRf+cpXir5GPAAAAKyNfPoFAABYBzQnPd9+++088cQT+dCHPpSkaVrzX/ziFznooIPS0NCQww8/vNW5m266aX7xi1+0JEB79OiR733veznppJPy17/+Nfvss0+rc3r27JmrrroqVVVVq4xr9913z3nnnZezzz47d999d+6+++4V9n/sYx/L8OHDc9FFF62wfZ999sluu+2Whx9+OL/+9a/z61//umVfTU1NbrnllnzmM59pNbX4gAEDcv311+eII47II488kv33379VTMOGDcuVV165yriLYdSoUfnpT3+ayZMn59JLL12han7AgAG5+eabM2LEiJLH0axPnz4544wz8o1vfCOHHHJIq/377LNPzjrrrBW27bHHHvn+97+f008/Pdddd12uu+66Vud9+tOfzimnnFKqsAEAAGCtogIdAABgHTB48OCWqvHPf/7zK+wbOnRoHnvssXz961/P4MGD07Nnz/To0SPbbbddvvnNb+bxxx/P4MGDVzhn9OjRmTBhQg466KBssskm6datW3r06JFtttkmX/ziF/P444/nE5/4RLtiO+WUU3LHHXdkn332SW1tbaqqqrLNNtvk7LPPzq233tpm5XJlZWVuvfXWfO1rX8uWW26Zbt26pX///jn00ENz//33Z9ddd13p9UaMGJEnnngiJ5xwQrbaaqtUVVWlV69eGT58eL73ve/lwQcfzEYbbdSu2N+Lmpqa/P73v88XvvCFbLrppunWrVs22WSTHHvssXnooYey9dZblzyGfzVmzJjcdttt+fd///f07ds31dXV2WmnnXLRRRflxhtvbPMLESeddFLuueeefOpTn8pGG22Url27pra2NnvuuWfGjh2ba6+9Nl26dOnwewEAAIByqJg/f36h3EEAAAAAAAAAQLmpQAcAAAAAAACASKADAAAAAAAAQBIJdAAAAAAAAABIIoEOAAAAAAAAAEkk0AEAAAAAAAAgiQQ6AAAAAAAAACSRQAcAAAAAAACAJBLoAAAAAAAAAJBEAh0AAAAAAAAAkkigAwAAAAAAAEASCXQAAAAAAAAASCKBDgAAAAAAAABJJNABAAAAAAAAIIkEOgAAAAAAAAAkkUAHAAAAAAAAgCTJ/wcx4n10gop7GgAAAABJRU5ErkJggg==",
      "text/plain": [
       "Figure(nrows=1, ncols=1, figwidth=10.0, figheight=5.0)"
      ]
     },
     "metadata": {
      "image/png": {
       "height": 500,
       "width": 1000
      }
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "fpath = '/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/BiLSTM+GAT_v2/pepGraph_generation/tool/test.csv'\n",
    "\n",
    "data = read_dynamx(fpath, time_conversion=(\"min\", \"s\"))\n",
    "\n",
    "peptides = filter_peptides(data, state=\"SecB WT apo\", query=[\"exposure > 0.\"])\n",
    "peptides[\"state\"].unique(), peptides[\"exposure\"].unique()\n",
    "\n",
    "# Filter out peptides for the full deuteration control\n",
    "fd_df = filter_peptides(data, state=\"Full deuteration control\", exposure=60 * 0.167)\n",
    "fd_df[\"state\"].unique(), fd_df[\"exposure\"].unique()\n",
    "\n",
    "peptides_control = apply_control(peptides, fd_df, nd_control=None)\n",
    "\n",
    "peptides_corrected = correct_d_uptake(peptides_control, drop_first=2, d_percentage=90.0)\n",
    "\n",
    "sequence = \"MSEQNNTEMTFQIQRIYTKDISFEAPNAPHVFQKDWQPEVKLDLDTASSQLADDVYEVVLRVTVTASLGEETAFLCEVQQGGIFSIAGIEGTQMAHCLGAYCPNILFPYARECITSMVSRGTFPQLNLAPVNFDALFMNYLQQQAGEGTEEHQDA\"\n",
    "temperature, pH = 273.15 + 30, 8.0\n",
    "\n",
    "hdxm = HDXMeasurement(\n",
    "    peptides_corrected, sequence=sequence, pH=pH, temperature=temperature, name=\"My HDX measurement\"\n",
    ")\n",
    "\n",
    "fig, ax = pplt.subplots(figsize=(10, 5))\n",
    "i = 4\n",
    "peptide_coverage(ax, hdxm[i].data, 20, cbar=True)\n",
    "t = ax.set_title(f\"Peptides t = {hdxm.timepoints[i]}\")\n",
    "l = ax.set_xlabel(\"Residue number\")\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import os\n",
    "from polyR_model import seq_embedding\n",
    "\n",
    "def prepare_data():\n",
    "    apo_df = pd.DataFrame()\n",
    "    mutation_df = pd.DataFrame()\n",
    "    apo_count = 0\n",
    "    mutation_count = 0 \n",
    "    root_dir = \"/Users/liyao/Desktop/Tsuda_Lab/Source_code/AI-HDX-main/HDX_MS_dataset/database_collection\"\n",
    "    apo_path = f\"{root_dir}/merged_apo.xlsx\"\n",
    "    apo_df = pd.read_excel(apo_path, sheet_name='Sheet1')\n",
    "    apo_df = apo_df.dropna(subset=['chain_identifier'])\n",
    "\n",
    "    manager_path = f\"{root_dir}/manager.txt\"\n",
    "    manager = open(manager_path, 'r')\n",
    "    manager.readline()\n",
    "    available_list = []\n",
    "    for line in manager:\n",
    "        content = line.split(':')\n",
    "        available_list.append(content[0])\n",
    "\n",
    "    input_array = []\n",
    "    truth_array = []\n",
    "    error_id = []\n",
    "\n",
    "    for index, row in apo_df.iterrows():\n",
    "        identidier = row['database_id'].strip()\n",
    "        if identidier not in available_list:\n",
    "            continue\n",
    "        uni_id = row['match_uni'].strip()\n",
    "        state = row['state'].strip()\n",
    "        apo_pdb = row['apo_identifier'].strip().split('.')[0]\n",
    "        chain_id = row['chain_identifier'].strip()\n",
    "\n",
    "        #print(identidier, uni_id, state)\n",
    "\n",
    "        example_input = f\"{root_dir}/{identidier}/apo_revised_modified.xlsx\"\n",
    "        mut_intput = f\"{root_dir}/{identidier}/mutation_revised_modified.xlsx\"\n",
    "        example_embedding = f'{root_dir}/feature/embedding_files/{apo_pdb}_{chain_id}.embedding.txt'\n",
    "        if os.path.isfile(example_embedding) == False or os.path.isfile(example_input) == False:\n",
    "            #print(example_embedding, example_input)\n",
    "            continue\n",
    "        \n",
    "        if os.path.isfile(mut_intput):\n",
    "            temp_apo_df = pd.read_excel(example_input, sheet_name='Sheet1')\n",
    "            #temp_mutation_df = pd.read_excel(mut_intput, sheet_name='Sheet1')\n",
    "\n",
    "            apo_df = pd.concat([apo_df, temp_apo_df], ignore_index=True)\n",
    "            #mutation_df = pd.concat([mutation_df, temp_mutation_df], ignore_index=True)\n",
    "\n",
    "        prot1, truth, start_pos, end_pos, log_t = seq_embedding(example_input, example_embedding, state)\n",
    "        input_array.append(prot1)\n",
    "        truth_array.extend(truth)\n",
    "\n",
    "    #apo_df = apo_df.drop_duplicates(subset=['sequence'], keep='first')\n",
    "    #mutation_df = mutation_df.drop_duplicates(subset=['sequence'], keep='first')\n",
    "    #print(apo_df.shape[0], mutation_df.shape[0])\n",
    "    return input_array, truth_array\n",
    "\n",
    "## FIXME: currently can iterate through all the files, but there are many mismatch records ##\n",
    "## FIXME: consider the exposure time, there are 20k pep apo sequences ##"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(2860, 30, 47) (2860,)\n",
      "training x: (2002, 30)\n",
      "training y: (2002,)\n",
      "test x: (858, 30)\n",
      "test y: (858,)\n",
      "R2 score: 0.5971016425258246\n",
      "RMSE: 0.1270722172104855\n",
      "Pearson: PearsonRResult(statistic=0.7854299226519129, pvalue=1.5764380066335197e-180)\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAjcAAAHFCAYAAAAOmtghAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMiwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8pXeV/AAAACXBIWXMAAA9hAAAPYQGoP6dpAACghklEQVR4nOzdd1xT1/sH8E/CnmHJEBVwg6goDpCqFffE2v7ce7S2Wleto9aq1davXY46qlax1llb66hUxb2wDqRqsVURnKACCgqykvv7IyYQsu5NbibP+/Xqq3K59+bkJuQ+Oec5zxEwDMOAEEIIIcRKCE3dAEIIIYQQPlFwQwghhBCrQsENIYQQQqwKBTeEEEIIsSoU3BBCCCHEqlBwQwghhBCrQsENIYQQQqwKBTeEEEIIsSoU3BBCCCHEqlBwQ4geBAIBq/9OnDhh6qYaVHBwsMLzdXV1RevWrbF582ajPP6mTZsgEAiQkZEh3/bmm2/izTff5HyuL7/8Env27OGtbTIZGRkQCATYtGmT2n2mTp0KgUCAf//9V+0+c+bMgUAgQHJyMuvHDg4OxsiRIzm0lhDLRsENIXpISkpS+K9Hjx5wcnJS2t68eXNTN9XgYmJi5M9XFmyMGDECa9asMUl7Vq9ejdWrV3M+zlDBDRtjxowBAGzcuFHl7yUSCTZv3oyIiIgq8Z4iRFe2pm4AIZYsKipK4edq1apBKBQqba+ssLAQzs7Ohmya0Xl4eCg8706dOiEoKAjfffcd3n//fZXHiMVilJWVwcHBgff2hIWF8X5OQwsPD0erVq3w888/48svv4StreJH9OHDh/HgwQPMnDnTRC0kxDJQzw0hBvbmm28iPDwcp06dQps2beDs7IzRo0cDkA5rzZ8/X+kYVcMIWVlZeO+991CjRg3Y29sjJCQECxYsQFlZmcbH79u3L4KCgiCRSJR+17p1a4UegF27dqF169YQiURwdnZG7dq15W3lysPDAw0aNMDdu3cBlA/LfPXVV1i0aBFCQkLg4OCA48ePAwAuXbqEPn36wMvLC46OjmjWrBl++eUXpfOeP38eMTExcHR0RPXq1TF79myUlpYq7adqWKq4uBiff/45QkND4ejoCG9vb3To0AHnzp0DIH09CgoK8NNPP8mH2Cqeg+1r8OjRI/Tv3x9ubm4QiUQYMGAAsrKyWF23MWPGICsrC3/++afS7+Lj4+Hg4IAhQ4agqKgIH330ESIiIiASieDl5YXo6Gjs3btX62OoGsYDgBMnTqgcRj1y5Ag6duwId3d3ODs7IyYmBkePHlXY5+nTp3j33XdRs2ZNODg4oFq1aoiJicGRI0dYPW9C+EQ9N4QYQWZmJoYOHYoZM2bgyy+/hFDI7XtFVlYWWrVqBaFQiM8++wx16tRBUlISFi1ahIyMDMTHx6s9dvTo0YiLi8OxY8fQqVMn+fZ///0XFy5cwIoVKwBIh9gGDBiAAQMGYP78+XB0dMTdu3dx7NgxnZ5zaWkp7t69i2rVqilsX7FiBerXr49vvvkG7u7uqFevHo4fP45u3bqhdevW+OGHHyASibBjxw4MGDAAhYWF8kAvNTUVHTt2RHBwMDZt2gRnZ2esXr0a27Zt09qesrIydO/eHadPn8aUKVMQGxuLsrIynD9/Hvfu3UObNm2QlJSE2NhYdOjQAXPnzgUAuLu7A2D/Grx69QqdOnXCo0ePsHjxYtSvXx8HDhzAgAEDWF23QYMGYerUqdi4cSN69+4t3/7s2TPs3bsXb731Fjw9PZGXl4fc3FxMnz4dgYGBKCkpwZEjR9CvXz/Ex8dj+PDhrB5Pmy1btmD48OGIi4vDTz/9BDs7O6xduxZdu3bFoUOH0LFjRwDAsGHDkJycjC+++AL169fH8+fPkZycjJycHF7aQQgnDCGENyNGjGBcXFwUtrVv354BwBw9elRpfwDMvHnzlLYHBQUxI0aMkP/83nvvMa6urszdu3cV9vvmm28YAMw///yjtk2lpaWMn58fM3jwYIXtM2bMYOzt7Zns7GyFcz1//lzb01TZ3h49ejClpaVMaWkpk56ezowYMYIBwHz88ccMwzBMeno6A4CpU6cOU1JSonB8w4YNmWbNmjGlpaUK23v16sUEBAQwYrGYYRiGGTBgAOPk5MRkZWXJ9ykrK2MaNmzIAGDS09Pl29u3b8+0b99e/vPmzZsZAMz69es1PhcXFxeFay/D9jVYs2YNA4DZu3evwn7jxo1jADDx8fEaH59hpO8jOzs75vHjx/Jt33//PQOASUxMVHlMWVkZU1payowZM4Zp1qyZwu8qv5/i4+OVrhfDMMzx48cZAMzx48cZhmGYgoICxsvLi+ndu7fCfmKxmGnatCnTqlUr+TZXV1dmypQpWp8bIcZAw1KEGIGnpydiY2N1Pv6PP/5Ahw4dUL16dZSVlcn/6969OwDg5MmTao+1tbXF0KFDsXv3buTl5QGQ5rr8/PPPiIuLg7e3NwCgZcuWAID+/fvjl19+wcOHDzm1MSEhAXZ2drCzs0NISAh++eUXfPjhh1i0aJHCfn369IGdnZ3859u3b+Pff//FkCFDAEDh+fXo0QOZmZn477//AADHjx9Hx44d4efnJz/exsaGVa/In3/+CUdHR52H2di+BsePH4ebmxv69OmjcPzgwYNZP9aYMWNQWlqKn3/+Wb4tPj4eQUFB8p4SQDqMGBMTA1dXV9ja2sLOzg4bNmzAjRs3dHqOlZ07dw65ubkYMWKEwnOWSCTo1q0bLl68iIKCAgBAq1atsGnTJixatAjnz59XOVRIiLFQcEOIEQQEBOh1/OPHj7F//3558CD7r1GjRgCA7OxsjcePHj0aRUVF2LFjBwDg0KFDyMzMxKhRo+T7tGvXDnv27EFZWRmGDx+OGjVqIDw8HNu3b2fVxjfeeAMXL17EpUuXkJqaiufPn2PFihWwt7dX2K/ytXj8+DEAYPr06UrP74MPPlB4fjk5OfD391d6bFXbKnv69CmqV6/OeUiwYjvZvAY5OTkKwReXNsq0bdsW9evXlw91Xb16FcnJyRg1ahQEAgEAYPfu3ejfvz8CAwOxZcsWJCUl4eLFi/LXmg+y1+add95Ret5LliwBwzDIzc0FAOzcuRMjRozAjz/+iOjoaHh5eWH48OGsc40I4RPl3BBiBLIbUmUODg4oLi5W2l45T8HHxwdNmjTBF198ofI81atX1/j4YWFhaNWqFeLj4/Hee+8hPj4e1atXR5cuXRT2i4uLQ1xcHIqLi3H+/HksXrwYgwcPRnBwMKKjozU+hkgkQosWLTTuAyhfCx8fHwDA7Nmz0a9fP5XHNGjQAADg7e2t8mbJ5gZarVo1nDlzBhKJRKcAh+1r4O3tjQsXLujUxopGjx6NWbNm4cKFC9i2bRuEQqFCkvmWLVsQEhKCnTt3KlxTVe+nyhwdHVXuWzlIlr0233//vdoZgLJAzsfHB8uWLcOyZctw79497Nu3D7NmzcKTJ09w8OBB7U+YEB5RcEOICQUHB+Pq1asK244dO4aXL18qbOvVqxcSEhJQp04deHp66vRYo0aNwvvvv48zZ85g//79mDZtGmxsbFTu6+DggPbt28PDwwOHDh3ClStXtAY3umrQoAHq1auHv//+G19++aXGfTt06IB9+/bh8ePH8puqWCzGzp07tT5O9+7dsX37dmzatEnj0JSDgwNevXqltJ3ta9ChQwf88ssv2Ldvn8LQFJuk54pGjBiBTz/9FGvXrsW+ffvQsWNHBAUFyX8vEAhgb2+vENhkZWWxmi0VHBwMQNojJAscAWDfvn0K+8XExMDDwwOpqamYOHEi67bXqlULEydOxNGjR3H27FnWxxHCFwpuCDGhYcOGYe7cufjss8/Qvn17pKamYuXKlRCJRAr7ff7550hMTESbNm0wadIkNGjQAEVFRcjIyEBCQgJ++OEH1KhRQ+NjDRo0CNOmTcOgQYNQXFysNNX8s88+w4MHD9CxY0fUqFEDz58/x/Lly2FnZ4f27dvz/dQVrF27Ft27d0fXrl0xcuRIBAYGIjc3Fzdu3EBycjJ27doFAPj000+xb98+xMbG4rPPPoOzszNWrVolz/vQZNCgQYiPj8f48ePx33//oUOHDpBIJPjrr78QGhqKgQMHAgAaN26MEydOYP/+/QgICICbmxsaNGjA+jUYPnw4li5diuHDh+OLL75AvXr1kJCQgEOHDnG6Jv7+/ujRowfi4+PBMIy8wJ9Mr169sHv3bnzwwQd45513cP/+fSxcuBABAQG4deuWxnO3bNkSDRo0wPTp01FWVgZPT0/8/vvvOHPmjMJ+rq6u+P777zFixAjk5ubinXfega+vL54+fYq///4bT58+xZo1a5CXl4cOHTpg8ODBaNiwIdzc3HDx4kUcPHhQbW8cIQZl6oxmQqyJutlSjRo1Url/cXExM2PGDKZmzZqMk5MT0759eyYlJUVpdgvDMMzTp0+ZSZMmMSEhIYydnR3j5eXFREZGMnPmzGFevnzJqn2DBw9mADAxMTFKv/vjjz+Y7t27M4GBgYy9vT3j6+vL9OjRgzl9+rTW8wYFBTE9e/bUuI9sttTXX3+t8vd///03079/f8bX15exs7Nj/P39mdjYWOaHH35Q2O/s2bNMVFQU4+DgwPj7+zMff/wxs27dOq2zpRiGYV69esV89tlnTL169Rh7e3vG29ubiY2NZc6dOyffJyUlhYmJiWGcnZ0ZAArnYPsaPHjwgHn77bcZV1dXxs3NjXn77beZc+fOsZ4tJbN3714GAOPl5cUUFRUp/f5///sfExwczDg4ODChoaHM+vXrmXnz5jGVP9pVvZ9u3rzJdOnShXF3d2eqVavGfPjhh8yBAwcUZkvJnDx5kunZsyfj5eXF2NnZMYGBgUzPnj2ZXbt2MQzDMEVFRcz48eOZJk2aMO7u7oyTkxPToEEDZt68eUxBQQHr50sIXwQMwzCmC60IIYQQQvhFs6UIIYQQYlUouCGEEEKIVaHghhBCCCFWhYIbQgghhFgVCm4IIYQQYlUouCGEEEKIVamSRfwkEgkePXoENzc3tWXxCSGEEGJeGIbBixcvtK4TVyWDm0ePHqFmzZqmbgYhhBBCdHD//n2NVdmrZHDj5uYGQHpx3N3dTdwaQgghhLCRn5+PmjVryu/j6lTJ4EY2FOXu7k7BDSGEEGJhtKWUUEIxIYQQQqwKBTeEEEIIsSoU3BBCCCHEqlBwQwghhBCrQsENIYQQQqwKBTeEEEIIsSoU3BBCCCHEqlBwQwghhBCrQsENIYQQQqwKBTeEEEIIsSoU3BBCCCHEqlBwQwghhBCrQsENIYQQQvhRVAQ8fmzqVlBwQwghhBAe3LoFREcDb70FlJaatCkU3BBCCCFEPyUlQMeOQEoKkJQEzJtn0uZQcEMIIYQQ/djbA0uXSv/dsCEwaJBJm2Nr0kcnhBBCiHV4+23g55+Bvn0BV1eTNoV6bgghhBDCzebNwIQJytuHDjV5YANQzw0hhBBC2CookAY1P/0k/blVK2DECNO2SQXquSGEEEKIdtevAy1blgc2AJCcbLr2aEDBDSGEEELUYxhgwwZpYHPjhnSbqyuwdSuwfLlp26YGDUsRQgghRLUXL4D335cGMjJNmwK//ALUr2+6dmlBPTeEEEIIUfb330CLFoqBzfjxwPnzZh3YANRzQwghhBBV5s8Hbt6U/tvNDfjxR6B/f5M2iS0KbgghhBCibN064MIFwN9fOgxVp46pW8QaBTeEEEIIAYqLAQeH8p+rVQOOHgVCQhS3WwDKuSGEEEKqMoYBvv8eCA0Fnj5V/F3DhhYX2AAU3BBCCCFV17Nn0mUTJk0C0tOB4cMBicTUrdIbDUsRQgghVdFffwEDBwIZGeXbGjUCxGJAaNl9H5bdekIIIYRwwzDAt98Cb7xRHth4eQH79gHffAPY2Zm0eXygnhtCCCGkqsjJAUaOBP74o3xbmzbA9u1ArVomaxbfqOeGEEIIqQrOnQOaNVMMbGbOBE6csKrABqCeG0IIIaRq+O8/4P596b99fIDNm4Hu3U3bJgOh4IYQQgipCkaOBI4fl+bZbN8OBAaaukUGQ8ENIYQQYo3S0hSrCgsEwNq10oRhW+u+/VPODSGEEGJNxGJg0SKgQQPgt98Uf+fkZPWBDUDBDSGEEGI9Hj8GunYF5s6VBjljxpTn2VQhFNwQQggh1uDoUaBpU+n/AWkhvmnTgOrVTdsuE7D+vilCCCHEmonFwOefAwsXSgv0AUBAALBtG/DmmyZtmqlQcEMIIYRYqkePgCFDpLVqZLp0AX7+GfD1NVmzTI2GpQghhBBLdO4cEBFRHtjY2ACLFwN//lmlAxuAem4IIYQQy1SrVvkK3jVqSGvXvPGGadtkJqjnhhBCCLFENWpIqwz36gVcuUKBTQUU3BBCCCGW4PBhIC9PcVuPHtLVvH18TNMmM0XBDSGEEGLOSkuBjz+W1q8ZO7Z8RpSMQGCadpkxCm4IIYQQc3X3LtCuHfDNN9Kff/0VOHDAtG2yABTcEEIIIeZozx7pbKjz56U/29kBy5YBPXuasFGWgWZLEUIIIeakpASYMQNYvrx8W0gIsHMn0LKl6dplQSi4IYQQQszFnTvAgAHApUvl2955B/jxR0AkMl27LAwFN4QQQog5uH0biIwE8vOlP9vbA0uXAu+/T0nDHFHODSGEEGIO6tQBOnaU/rtuXWmuzQcfUGCjA+q5IYQQQsyBQABs2CCtPLxwIeDmZuoWWSzquSGEEEJMYccOaWG+ijw9pTOiKLDRi1kEN6tXr0ZISAgcHR0RGRmJ06dPa9x/69ataNq0KZydnREQEIBRo0YhJyfHSK0lhBBC9PDqFfDuu8CgQcDQodKVvQmvTB7c7Ny5E1OmTMGcOXNw5coVtG3bFt27d8e9e/dU7n/mzBkMHz4cY8aMwT///INdu3bh4sWLGDt2rJFbTgghhHD0779A69bA+vXSn58+BbZuNW2brJDJg5vvvvsOY8aMwdixYxEaGoply5ahZs2aWLNmjcr9z58/j+DgYEyaNAkhISF444038N577+FSxWlzhBBCiLnZvFk6G+raNenPTk5AfLx0aQXCK5MGNyUlJbh8+TK6dOmisL1Lly44d+6cymPatGmDBw8eICEhAQzD4PHjx/j111/RU0PFxuLiYuTn5yv8RwghhBhFQQEwahQwYgRQWCjd1qiRtJbNyJEmbZq1Mmlwk52dDbFYDD8/P4Xtfn5+yMrKUnlMmzZtsHXrVgwYMAD29vbw9/eHh4cHvv/+e7WPs3jxYohEIvl/NWvW5PV5EEIIISr98w/QqhWwaVP5tjFjgAsXgLAwkzXL2pl8WAoABJXm8DMMo7RNJjU1FZMmTcJnn32Gy5cv4+DBg0hPT8f48ePVnn/27NnIy8uT/3f//n1e208IIYQoKSoCOnUCUlOlP7u4AFu2SKsNOzubtm1WzqR1bnx8fGBjY6PUS/PkyROl3hyZxYsXIyYmBh+/HqNs0qQJXFxc0LZtWyxatAgBAQFKxzg4OMDBwYH/J0AIIYSo4+goXR9qwACgSRPgl1+ABg1M3aoqwaQ9N/b29oiMjERiYqLC9sTERLRp00blMYWFhRAKFZttY2MDQNrjQwghhJiN/v2Bbduk1YYpsDEakw9LTZs2DT/++CM2btyIGzduYOrUqbh37558mGn27NkYPny4fP/evXtj9+7dWLNmDe7cuYOzZ89i0qRJaNWqFapXr26qp0EIIaQqYxhg7VpgwgTl3w0aJJ0ZRYzG5MsvDBgwADk5Ofj888+RmZmJ8PBwJCQkICgoCACQmZmpUPNm5MiRePHiBVauXImPPvoIHh4eiI2NxZIlS0z1FAghhFRl+fnSonw7d0p/jo6WFucjJiNgquBYTn5+PkQiEfLy8uDu7m7q5hBCCLFUycnSoae0tPJtM2cC//uf6dpkxdjev00+LEUIIYRYHIYBVq6U9tLIAhuRCPjtNwpszIDJh6UIIYQQi/L8ubRWze7d5dtatpQOS4WEmKxZpBz13BBCCCFsXbgANGumGNhMnQqcOUOBjRmhnhtCCCGErc8/BzIypP/29JRWHu7Tx5QtIipQcEMIIYSwtWEDEBEh7aXZsQOoVcvULSIqUHBDCCGEqFNUJK00LOPnB5w8KQ1u7OxM1y6iEeXcEEIIIZVJJMBXX0kXt8zJUfxd/foU2Jg5Cm4IIYSQip4+BXr1ktarSU8HRoyQBjvEYtCwFCGEECJz+jQwcCDw6JH0Z4FAmmNT9erdWjQKbgghhBCJBFi8GPjss/JeGl9fYMsWoHNn07aNcEbBDSGEkKrt8WNg2DAgMbF8W4cOwNatQECA6dpFdEY5N4QQQqqu48elw06ywEYgAObPl/5MgY3Fop4bQgghVVdaGpCVJf23vz+wbZu014ZYNApuCCGEVF1jxkh7b54+lebX+PqaukWEBxTcEEIIqTpu3pTWqZERCIAffwQcHAAhZWpYC3olCSGEWL+yMuDTT4GGDYE9exR/5+REgY2VoVeTEEKIdXvwAIiNBb74QlqvZtQoIDPT1K0iBkTBDSGEEOuVkCCdDXX6tPRnW1vgk0+ka0QRq0U5N4QQQqxPaSkwZw7w9dfl22rVkq7kHR1tunYRo6DghhBCiHW5d0+6hEJSUvm2Pn2A+HjAy8t07SJGQ8NShBBCrMfJk9JhKFlgY2cHLF0qTSKmwKbKoJ4bQggh1iMkRPHfO3cCLVuarj3EJKjnhhBCiPWoVQv46SfgnXeA5GQKbKooCm4IIYRYrv37gfx8xW29ewO7dgEeHiZpEjE9Cm4IIYRYnqIiYOJEaaLwe+9J69cQ8hoFN4QQQizLrVtAmzbAqlXSn3fsAI4cMW2biFmh4IYQQojl2LEDiIwErlyR/uzoCKxdC3TqZNp2EbNCs6UIIYSYv1evgClTgHXryrc1aAD88gvQpInJmkXMEwU3hBBCzNt//wH9+wNXr5ZvGzYMWL0acHU1XbuI2aLghhBCiPm6cUM6nbugQPqzk5M012bkSEAgMGnTiPminBtCCCFqiSUMktJysDflIZLSciCWGHlWUsOGQMeO0n+HhQEXL0pX9abAhmhAPTeEEEJUOng9Ewv2pyIzr0i+LUDkiHm9w9AtPMA4jRAIpGtCLV4MzJ8PuLgY53GJRaOeG0IIIUoOXs/E+1uSFQIbAMjKK8L7W5Jx8Hom/w/KMNJA5uhRxe1eXtLVvSmwISxRcEMIIUSBWMJgwf5UqBqAkm1bsD+V3yGqly+B4cOB0aOBIUOArCz+zk2qHApuCCGEKLiQnqvUY1MRAyAzrwgX0nP5ecCrV4EWLYAtW6Q/P34M/PorP+cmVRLl3BBCCFHw5IX6wEaX/dRiGGD9emDSJKC4WLrNzU1ay2bgQFanEEsYXEjPxZMXRfB1c0SrEC/YCCnZuKqj4IYQQogCXzdHXvdTKT9fuibUjh3l25o1kxblq1uX1SnMIuGZmCUaliKEEKKgVYgXAkSOUNf/IYA0iGgV4qXbA1y5Il1CoWJgM3EicO4cp8DG6AnPxGJQcEMIIUSBjVCAeb3DAEApwJH9PK93mG7DPwUFQJcuwO3b0p9FIml+zfffS9eJYsEkCc/EolBwQwghREm38ACsGdoc/iLFgMNf5Ig1Q5vrPuzj4gIsXy79d8uW0l6ct9/mdAqjJzwTi0M5N4QQQlTqFh6AzmH+/CfsDh4M2NoCffsC9vacD8/Ke8VqP70TnonFouCGEEKIWjZCAaLreOt2MMNIe2nS0qTDThX176/TKQ9ez8TCAzdY7atXwjOxaBTcEEIIAcDztOrcXOkaUPv2SX+OiWE9vVsdWRKxtkwaAaTDZzonPFsAc50Cby7touCGEEIIv9Oqk5Kkgcy9e+Xb/vtPr/ZpSiJWReeEZwtgrlPgzaldlFBMCCFVHG/TqiUS4KuvgLZtywMbb2/gwAFg3jy92qgtiVjGy8VOv4RnM2euU+DNrV0U3BBCSBXG27Tq7GygVy9g5kxALJZue+MNICUF6NFD73ayTQ6e26sR58BGLGGQlJaDvSkPkZSWY7ZTyM11Crw5touGpQghpArjMq26VYiX6nyK06eBQYOAhw+lBwkEwCefAPPnS2dF8YBtcrC/O7ckYnMaStGGy2ulcxK4lbSLghtCCKnC2PaIHEnNwrRfUpSDgF6h6LZoUXlgU60asHUr0Lkzr+2UVU3OyitS2UOgSxKxugRl2VCKuQ1vGW3NL47MsV00LEUIIVUY2x6RDWczVOdTbL2C47OWAL6+QIcOwN9/8x7YAPxXTTbHoRRtjLLmlw7MsV0U3BBCSBXGZh2pyvGCY6k0yJHd9j85nwPxqdNAYiIQYLieDrZVk9nk0FhilWODr/mlI3NsFw1LEUJIFSbrEXl/SzIEgEJPhuxn5vVGoUSMD8/txNvXj6L3iGXIc3IrDwJsvRFtY2Pw9mqrmsw2h8Ych1K00fZaAaaZAm+O7WIV3EybNo3ziT/99FN4eVlvASVCCLEWsh6RykGBv8gR3cP9sfFsBqq9zMXy/d+gzb2rAIBvEpZhXL9PpcnDMG4QoK5qMpccGnMcSmFD02tlyiRoc2sXq+Bm2bJliI6Ohj3LNUDOnDmDiRMnUnBDCCEWQl2PyIX0XNzc8juW/vEtqhU+BwCIBUKkBNSHAAyY19/NTR0EaMuhEUCaQ9M5zB82QoFBEpSNxWBrfllRu1gPS/3+++/w9fVlta+bm5vODSKEEGIaSj0iZWVovfE7tN61GMLXY1NZrl6Y1GcGLtQMB2A+QQDX6cjmOJTChV5rfhmQubSLVUJxfHw8RCIR65OuXbsWfn5+OjeKEEKIiT14AMTGQvjll/LA5njtSPQY9b1CYAOYRxCgSw4N2wRlYnlY9dyMGDGC00kHDx6sU2MIIYSYgT//BIYNA3JypD/b2ODfybMxx6sdcl+UyHczdZ5HRbrm0JjTUArhD82WIoQQouju3fLApmZNYOdONIyOxmkzWfFZFX1yaMxlKIXwh1Vw4+npCYGA3Rs4N9d8agIQQgjRwXvvAcePA0VFQHw88HpyiDkHAZaeQ0P4xXq2lExOTg4WLVqErl27Ijo6GgCQlJSEQ4cOYe7cuQZpJCGEEAO6cQMIDS3/WSAAfvoJcHCQT/W2BOY2HZmYjoBhGE61pd9++2106NABEydOVNi+cuVKHDlyBHv27OGzfQaRn58PkUiEvLw8uLu7m7o5hBBiGiUlwKxZwLJlwL590lW9rYBYy/CZtt8T88X2/s05uHF1dUVKSgrq1q2rsP3WrVto1qwZXr58qVuLjYiCG0JIlZeeDgwcCFy4IP3Zy0vag8Oy5IelsqRVwIkytvdvzmtLeXt74/fff1favmfPHnh76zYWu3r1aoSEhMDR0RGRkZE4ffq0xv2Li4sxZ84cBAUFwcHBAXXq1MHGjRt1emxCCKlydu8GmjUrD2zs7YEFC6QrelsxWQVjlQuAbknGweuZJmoZ4Rvn2VILFizAmDFjcOLECXnOzfnz53Hw4EH8+OOPnBuwc+dOTJkyBatXr0ZMTAzWrl2L7t27IzU1FbVq1VJ5TP/+/fH48WNs2LABdevWxZMnT1BWVsb5sQkhxBDMdtijuBiYPh1YubJ8W506wM6dQGSk6dplBFwrGBPLxnlYCgD++usvrFixAjdu3ADDMAgLC8OkSZPQunVrzg1o3bo1mjdvjjVr1si3hYaGom/fvli8eLHS/gcPHsTAgQNx584dnZd3oGEpQggfVAUxialZ5jnscfs2MGAAkJxcvm3AAGDdOsAMPwf5DhCT0nIwaP15rfttHxdltjPCCPv7t051blq3bo2tW7fq3DiZkpISXL58GbNmzVLY3qVLF5w7d07lMfv27UOLFi3w1Vdf4eeff4aLiwv69OmDhQsXwsnJSeUxxcXFKC4ulv+cn5+vd9sJIVWbqtwND2c7PC8sVdpX1cKNRnX4MPDOO8CLF9KfHRyA5cuBd99Vmg1lDr1OhsiLscRVwInudApu0tLSEB8fjzt37mDZsmXw9fXFwYMHUbNmTTRq1Ij1ebKzsyEWi5WWavDz80NWVpbKY+7cuYMzZ87A0dERv//+O7Kzs/HBBx8gNzdXbd7N4sWLsWDBAvZPkBBCNFC3+rSqwAYwg2GP+vUBG5vyf//yC9C0qdJu5pBsy2Vlby4sdRVwohvOCcUnT55E48aN8ddff+G3336Tz466evUq5s2bp1MjKhcIZBhGbdFAiUQCgUCArVu3olWrVujRowe+++47bNq0Ca9evVJ5zOzZs5GXlyf/7/79+zq1kxBCNOVuaFJx4UajCw6WFuMbOhS4fFltYGPqZFtteTGANEAUSzhnU8grGKsLKwWQBnKmXgCU8INzcDNr1iwsWrQIiYmJsLe3l2/v0KEDkpKSOJ3Lx8cHNjY2Sr00T548UbvwZkBAAAIDAxUW8gwNDQXDMHjw4IHKYxwcHODu7q7wHyGE6ELb6tPaGGXY47ffyoegZPr2BX7+GXB1VdrdkEEFF1xW9uZKVsEYgFKAQxWMrQ/n4ObatWt46623lLZXq1YNObK1SFiyt7dHZGQkEhMTFbYnJiaiTZs2Ko+JiYnBo0ePFOrp3Lx5E0KhEDVq1OD0+IQQwpW+wYlBhz0KC4ExY6T5Ne+/D7CcL2LIoIILQ+fF0CrgVQfnnBsPDw9kZmYiJCREYfuVK1cQGBjIuQHTpk3DsGHD0KJFC0RHR2PdunW4d+8exo8fD0A6pPTw4UNs3rwZgHTF8YULF2LUqFFYsGABsrOz8fHHH2P06NFqE4oJIYQvugYnmhZu5EVqKtC/P/DPP9Kft24Fxo0D2rfXeqi5JNsaIy+GVgGvGjgHN4MHD8bMmTOxa9cuCAQCSCQSnD17FtOnT8fw4cM5N2DAgAHIycnB559/jszMTISHhyMhIQFBQUEAgMzMTNy7d0++v6urKxITE/Hhhx+iRYsW8Pb2Rv/+/bFo0SLOj00IIVxpW31aFYMPe2zaBHzwASDLO3R2Bn74gVVgA5hPsq0+K3tzYc4LgBJ+cK5zU1paipEjR2LHjh1gGAa2trYQi8UYPHgwNm3aBBtZRr4Zozo3hBB9yJJvAeXVpxkoTwk32Iyjly+BCROA1z3bAIDGjaWzoRo2ZH0asYTBG0uOaQ0qzsyMNXgPh6ZrC4CGj6o4g6wtxTAM7t27h2rVqiErKwvJycmQSCRo1qwZ6tWrx0vDjYGCG0KIvjRNmzbKsMfVq9IifP/+W77t3Xeli2DqMESvbgq2zHvtQjC7R5hOTdWlLVynpJtDfR5ieAYJbiQSCRwdHfHPP/9YVDBTGQU3hBA+mOyGevUq0Lo1UPT65u/qCqxfL10IUw+LE1Kx9lS6yt8JYNxeEy7X1hzq8xDjMMjCmUKhEPXq1eM8K4oQQqyRLHcjLiIQ0XW8jddTEB4OxMZK/x0RIV1SQc/ARixhsO9vzbVsjDEdXIbttTWH+jzE/HCeCv7VV1/h448/xvXr1w3RHkIIIdoIhcBPPwGzZwNJSQAPPenmMh2cC3Opz0PMD+fZUkOHDkVhYSGaNm0Ke3t7penXubnm88YnhBCLxzDAmjVAaCjQoUP5dh8f4MsveXsYc5kOzgWXgIxmR1UtnIObZcuWGaAZhBBClOTlAWPHAr/+Cvj7A3//Dfj6GuShzGU6OBeWGJAR4+Ac3IwYMcIQ7SCEEILyRNqS8+fRevYEON6/K/1FVhawd6+0MJ8BGKvGDJ8sMSAjxqHTquBisRi///47bty4AYFAgNDQUMTFxcHWVqfTEUIIwetZP/v+Qbdjv2D28XjYS8oAAKVuItht3iRdH8pAZGsvvb8lWV6vR8Yc1l5SNXvKEgMyYhyco5Hr168jLi4OWVlZaNCgAQDp2k7VqlXDvn370LhxY94bSQgh5oyPKeEHr2di1vqT+OrP5ehy67x8e3L1BpjUZyY+rdsa3fhueCWytZcqT6v2N/G0ak1TvdUFZHj989yetBhmVcS5QnFUVBR8fX3x008/wdPTEwDw7NkzjBw5Ek+ePOG8MrgpUJ0bQghf+KixIpYweG/CSszfthA18p/Kt69t1Q9ftxsOsY2t0SoEy9pjLgXx1BUXrFixGIDSayBD9W6si0GK+AGAk5MTLl26hEaNGilsv379Olq2bIlXsrVNzBgFN4QQPrC58bK5qV5IyUCDqHCIigsAALlO7vio51Qcr9NSYb/t46Kq1Kwf2bIQ6mZEVVwW4tD1THyw7YrKfQBatsFaGKSIHwA0aNAAjx8/Vtr+5MkT1K1bl+vpCCHEIvFZYyUTdpjf6T0AwIUaYegxcoVSYAOYZtaPWMIgKS0He1MeIiktx6g1Y9hO9T5/JwcLD9xQuw9A9W6qGs45N19++SUmTZqE+fPnIyoqCgBw/vx5fP7551iyZAny8/Pl+1KvCCHEWvFZY8XXzRG/h8filZ0DEutFQSxUvQCxsWf9mHpZA7bB3Lnb2VTvhijgHNz06tULANC/f38IBNIOP9nIVu/eveU/CwQCiMVivtpJCCFmRecaKxIJsGQJkJkJrFgBoHwa9qEGMWYz60fdkJtsWYOKwzwlZRL8nJSBu7mFCPJyxrDoYNjbch4YUMI2mNt0LoPVfpZY78ac8p8sCefg5vjx44ZoByGEWBSdaqw8eQIMGwYcPiz9+Y03gP79zW4atrYhNwGkwzydw/zx1cEbWH86HRVHfL5IuIFxbfVfRVzbVG+ZghJ2X6Qtrd6NqXvOLBnn4KZ9+/as9vvggw/QqFEj+Pj4cG4UIYSYO841Vk6cAAYPlvbYAIBAAKSXr8BtiGnYun7rZzvkNnnHFfxxVXlhSgkDrD2VjofPi7B8YDOdgzJtQR/bDBpLrHfDpeeMKOM8W4otd3d3pKSkoHbt2oY4vV5othQhhA+yGxCgurdlzdDm6BbqC3zxBbBggXRICgD8/ICtW4GOHZXOydcwhD7f+vemPMTkHSmcH1MVPzd7LIgLR+cwf52fl6rn4u1ij5yCElbHC2BZs6W4zBKrakNUbO/fBispbKCYiRBCDI5tgKG1t8VHAHTpAhw7Vn5Qx47Ali3StaJUsBEK9E561fStf/yWZIyOCUbnMH+1z4vP4ZvHL0owfksyPJzt8LywVL7dy8UefSOqa2yHTLfwAKXgKCvvFab+8rfWx/dwssP/3m5sMYENQAuC8oHWSyDEglGyIf/Y9njIrn1xmQTfvNMUEADZL4vLX4djR4GOQ6R5NgAgFEp7b2bPBmxUz4biA5sp6hvPZmDj2Qy1PTlsc124qBjYAEBuQYnWdlRUOehLSsth9birhjRHTF3LSo+gBUH1R8ENIRaKkg35xzbPQdO1j67jDTAMsHixPLAp8fPHhUXfw6ZDB7QSCGG40Eb7t/6K1OVvaMp1MYRMHfJI2OY8RdW2vJ4NWhBUf/rP1SOEGJ3sJlz5Jia7WR28rpzkSTRjW5Qv4SqLay8QAD//jBJPLyTVb4mo/t9i6G0nDFp/Hm8sOWbQ14fLt3lNBe5kQ27+IsUbqJeLnb5NVItLoT1ZAAaU5zjJsJlhZsrihNrIAjd1fbACSINpS0qQNjbquSHEwnCZpktDVOyxzXP4dO91ldfesaQIRfaO8mufmCvA4v9bgnsefmAE5d8jDT3bxcfVgdP+mvI3VOa65Bdh6s4U/hrMoh3q6DrDzNx7Pc2tNIAlMlhwM3ToUJqJRIgBULKhYbDt8citNEPHVlyGaWe2oOe/Z9B7xDJkAjh/J0d68/RUvlEaMgA9eD0T8/f9o9Ox6p6/rrkuuuKaR6IqANOUe2YpU6zNdYV2S8E5uAkODsbo0aMxcuRI1KpVS+1+a9as0athhBDVKNnQMHTJXwjIf4rv932FFg+l6xot+XMF3u87G0lpOUYPQNXdtNli+/wNkWysSzsqYjvDzNJ6PbkGbqQc55ybjz76CHv37kXt2rXRuXNn7NixA8XFxYZoGyFEBUo2NAw2eQ4V80063v4LCfGT5IFNqdAGlwJDX/+W3W2frwBU001bG675G5pyXVRxdbBhta8x8ki49HqaC1ngFhcRiOg63ioDG3POHzIVzsHNhx9+iMuXL+Py5csICwvDpEmTEBAQgIkTJyI5OdkQbSSEVEDJhobBJkF1UVw4arrY4NNjP2LDbwvhWfQCAPDA3Rf/N+QrbGz1FgI8nBBdm93UY74CUC4zpCrSNX9DXbJxgMgR49qGwMvFXr7tZbEYHs52EDmrT0Q2Vh6JNfZ6HryeiTeWHMOg9ecxeUeKUZLWLYHOs6WaNm2K5cuX4+HDh5g3bx5+/PFHtGzZEk2bNsXGjRupiB8hBsLmmzMlG+pG3U3bX+SINUObo4d7Cfbv+gRjL+6R/+5g/Wj0GLUCf1dvAEB67aPqeBs1AM3Ke8VqP5GjYiaC7Hnpkr/RLTwAZ2bGYvu4KCwfGIHt46Iwt2cYfjydrpSX9LywFM8LS/FO8xoY1SZIacaVPu3gwtp6PWnWpHo6JxSXlpbi999/R3x8PBITExEVFYUxY8bg0aNHmDNnDo4cOYJt27bx2VZCLIahi+vJbsKzdl9TKo6m6Rsy0U5tnsO+vcCoUfB4/hwAUGJjhy86jMZPzXsBAoHSbBtjznapHEyoM6FDXdgIBbyt3l0x10W2ZICmr7W/Jj+Av7sDFsU1hqeLvdHzSDivB2bGLC1/yNg4BzfJycmIj4/H9u3bYWNjg2HDhmHp0qVo2LChfJ8uXbqgXbt2vDaUEEthzGmmlQMbAMgrLDWrWR+WSGWC6sOHwOvABrVrw2bHTnTzCkFzNTdoY8528WI5/XvViTTkvSp/z/x4Jl3eFn0DcrZDY1n5xZiwTfr+jIsIZH1+PljTFGuaNakZ5+CmZcuW6Ny5M9asWYO+ffvCzk75W2JYWBgGDhzISwOJ6VGJf/aMNc1U9q1NFfrWZiAffCBd2VsoBNatg41IhGgth2ia7cLn35W/O7thlIqBDVD+vny3XQj2/Z2pV0DONU/FVO9Pa5libY35Q3ziHNzcuXMHQUFBGvdxcXFBfHy8zo0i5sPci12ZE2N2E9O3NiO4dg1o3Lj859dVh+HgADEDXEjLYRWYqOoF4vvvSjbcwjWpWPZeXXsqXel3XANyLnkqpn5/WsMUa2vLH+Ib58FWbYENsR6UrMaNMaeZ0rc2AyoqAt5/H2jSBPjzT8XfOTri4D9Zes1OMcTflWy4RQB207PZ0LQ0gyraZvGpYsr3J5sp1uaMZk1qxiq48fLyQnZ2NuuT1qpVC3fv3tW5UcT02K6zQ/UUyhkz4KBvbQZy8yYQFQX88IP05+HDgZzyirz6BiZ8/l3Japv8nvwAG07fwatSCaZ0qg+/SkNUHnokmHMJyCvO4mOL3p+603dtLWvHaljq+fPn+PPPPyESiVidNCcnB2KxWK+GEdOiYQ/ujBlwWNOsD7OxdSvw3ntAQYH0Zycn4H//A7yk15CPYUe+/q5UDWvJ+Ls7YGqnegj2cYGvmyMkEgZDNvyl9lxssA3IZfks8/elIitf/TH0/uSHteQPGQLrnJsRI0YYsh3EzNCwB3fGDDisadaHyRUWApMmARs2lG8LDQV++QUID5dv4iMw4ePvStsyC1n5xVh25BbWDG2O6DreSLj6CEIBoE8nK5eAXJbPsvLYLSw9ckvp9/T+5Jc15A8ZAqvgRiKRGLodxMxY4rCHqWd1GTvg4ONbm6mvmcnduAH07w9cv16+beRIYOVKwMVFYVc+AhN9/664LLOwYH8qJBIGE7Zd0XkNKF0DchuhAJM71UcDfzfqVTACtmtrVSUGWxWcWDZLG/Ywl1ldxu4m1udbm7lcM5P54w9gwABpzw0AODsDa9ZI82xU4CPg1/fvim0tGVkv0qd7r2sMbIQCYMwbIfjxdLr8uIptAfQLyKlXgZgKBTdEJUsa9jBWbRm21H2gA0ASy+nDXOjyrc2Y18xse4fCwgDb1x+B4eHSYajQULW78xHw6/t3xXUYOLdAuchjRRIGiG3oh8ggT4MF5NSrQEyBghuiliUkq5lrCfLKH+iqekm8XOyxKC4cPZoY9zoa85qZde9Q7drAxo3AwYPA8uXSnhsN9AlMKgd4qwY3x8ID3P+uDDEM/ORFEeIiAqmHhVgVCm6IRuberWwJs7rU9ZLkFpTgg23JeO9BCGb34DaFVh/GumZm1aPGMMCOHUCfPoq5NG+/Lf1PBVU9Tt3CA/BuuxCsU1H07t12ISqfj7oAb27PUHi6OHD6u9K1WJ8msoCJeliINaHghmhlzh965j6ri00C6NpT6WhawwM9mlQ3SpuMcc3MqkftxQvpFO/t24ERI4BNm7Qeoi4g6dM0QGU1XwbS17FZLU+FAEdTgDdh2xXO6yvZCAWY2zMUH2y7wvoYTTyc7XTKmzPboUZCXuNcoTg5ORnXrl2T/7x371707dsXn3zyCUpK2K1MSwhfzH1WF9sE0E/3XjdaQURjXDNjVmvW6MoVoHlzaWADAD/9BJw/r/EQTYX6VAU2Fc3afU3+OhqqEKanC7tFMtnQJRw5eD1TrwrNhBgD5+Dmvffew82bNwFI15kaOHAgnJ2dsWvXLsyYMYP3BhKiibmXIGfb+5FbUGr4G/1rxrhmJu9RYxhg9WogOhq4fVu6zd1dmjQcFaX2MDYBiSbPC0txPk1a0dhQAR6f1+xZIbf3HS3JQiwF5+Dm5s2biIiIAADs2rUL7dq1w7Zt27Bp0yb89ttvfLePEI3MvQQ5l94PYw2dGeOaGbp3SLb0wN6Uh0hKy1Hs/cjLk9aumTABKC6WbouMBJKTgf/7P43nZdvTpknSHelSNYYK8PjuhWT7+LQkC7EknIMbhmHkRf2OHDmCHj16AABq1qzJaf0pQvgim9XlL1L80PcXORp9GnhlrUK84OViz2pfYw6dGfqaGbJ3SOOwyKVL0mGoX38tP2DyZODsWaBOHa3n5ifAlD5rQwV4smvLF7aPz7YnatPZdApwiMlxTihu0aIFFi1ahE6dOuHkyZNYs2YNACA9PR1+fn68N5AQNsx1VpeNUIBFceH4YFuyxv2MOXQmSwYtLpPgm3eaAgIg+2Ux77V3+KiTVDlx9VlBCSZsU52gu/rrneiybQaEZa9ru3h4APHxQN++rM7dKsSLlwBTlnyvrS4OIF0HSpfqv7Jrq08IwbUQJ9vAb+GBG/jxTLp5TPcnVRbn4GbZsmUYMmQI9uzZgzlz5qBu3boAgF9//RVt2rThvYGEsGWus7p6NAnAew9C1CajCmC8oTNNdWf4vnb61klS1VahQHXuCwPgun8dXAhpiqhbl4DWraVTv4ODWZ9bNj1bW0CiiYu9DaJqS6+jpgBPpqhMgsTULM5BgLprW3kNKU9nOzwrLOWlECeXwM9UBTQJkREwDMNL/2FRURFsbGxgZ2fHx+kMKj8/HyKRCHl5eXB3dzd1c0gVkXD1ET7de12haqwxC9qpm5Ysu7UZ6kaky7RhbYtDquNd8Bx7BH+j5jeLAHvVw4HarkPFOjZcH39qp/qY3Kme0uPN2n0NzwuVqwWzufaarl/l30UGeeLy3WcK+yamZqkNaLn0doolDN5Ycox14CfrGTozM9bkPajEerC9f+sc3JSUlODJkydKi2rWqlVLl9MZFQU3xFRMVR9EdmNSlzNhTjcibW0FAAEjwZiLe3A1oD4u1AxX+N3ygRFqa8ewvQ5ze4YpVRDWxsPZDpc/7ax0/cQSBjH/O4asfPXn8naxR9LsjrC3VUyD5Kuytar3naagR12gJQsMAfaB3/ZxUWbZo0osE9v7N+dhqZs3b2LMmDE4d+6cwnaGYSAQCCAWi7m3lpAqwlRDZ5ZQyVlGW1s9XuXj2wNL0THtIrJcvdBj1PfIdRbJf69p+ITtdfB0sceZmbG4kJ6LI6lZ2HA2Q2u7/9evscoelewXxRoDGwDIKShB1OIj+PKtxvLAQltl69jkahjXtg6rAFnVciC6VI9WNxymiaFmAVIhQaIJ5+Bm1KhRsLW1xR9//IGAgAAIBPRmIsTc6Tot2RQ3EE1tjXyQiu/3fYXqL6QzM/1f5uLNO5ewO7wjqwRZttfhz9f1WlqFeCG6jjdahnipHVqqTFVvCxu5BaXywKJzmL/WytbH/n2KY/8+5Ty0qW/16M5h/nBzsMOuy/exJ+WR1sczxCxAXdcso4Co6uAc3KSkpODy5cto2LChIdpDCDEAXaYlm2rRS1VtFTASvH/+V0w7vQW2jHQoPMfJHVN7fYRTtSNZJ8iyvQ6bk+5ic9JdhdyU+ftSAagObmQBgUQClbO5uFiwPxVujnasg6PMvCKM35KMKR3rokwCAAyia/sgqo63ymuhTy8el8CN62wstnTtdTLrRVwJ7zgHN2FhYVTPhhAj4PNbJptpyRWno5ty0cvKbfUueI7vDnyH9unl0+nP1wzHpN4f44mb9ObLdgYWm+tQkez5TulUX+PQkiwg+HTvdb0CG9l5kl5XOeZi2dHb8n+vPJ4GD2c7/K9fY6VromsvHpckb0MV0NS118msFnElRsE5uFmyZAlmzJiBL7/8Eo0bN1aaHUUJuvyg7lPzYYrXgu9vmRWnJavTp2kAbIQCiCUM5u8z3aKXFdsade8alu//Gn4vpUsESCDA920Gov73/8NykQvn14TN9OyKZM83/pzmNaVkcgv4Wl9P/0mszwtLMX5LMn6odOPWpRePzQKwFbENNrnSpdfJrBZxJUbDObjp1KkTAKBjx44K2ymhmD/UfWo+jP1aiCUMVh67haVHbin9Tt9vmd3CA/BuO/X1dta9XtX638x8Vr0Uhkw+7hYegPW9ayO6Y3+4FBcCAJ66eODzAZ+g59Shel17rkmxDMAq14ZP0bV9sO3CfV6Cpfn7/lG4cWvrvVI1nMR2WYqJHeogpm41g3wBEEsYnL39lNW+FXudLCmZnvCHc3Bz/PhxQ7SDvEbdp+bD2K/FweuZmL8vVW1goe+3TLGEwb6/1S9syACYvCMFxWUStftUxGUWjC69X51iQiFZ9T0wdgyetnoDd5evxbJWobzcNCtWtP7zeiY2J93VeoyHkx3yXpWqDQg8XewUahip4+ZogxdFqr8EygKLqDrerCpbs5GVX6xw49alejTb17qen5tBAgSuSdoVe524DsNRr7l14BzctG/f3hDtIKDuU3Ni7NeCbT5DxfV7fNwcOH34svn2zTawAdgPb3Dq/WIYoMIMTOHoUYC3F6r17o1qNjas28ZGxenRbIKbUTHBWHbkltqAYFFcOBYeuKG1R2Ruz1BM2HYFUHMeWWChrbI1F2dvP1W4WXOtHm3ohVA14ZrrU7nXiUvbqdfcenAObgDg+fPn2LBhA27cuAGBQICwsDCMHj0aIpFI+8FELUvvPrWmbzzGfC245jMA0vV7ZNh++PJZb4TtWlise7/KyoAFC4D8fGD58vIdBQK1a0Pxhe0wzcTYemjg76YxIBAKBVp7RLqFB2CNUMAqsJjdIwxNa3goVbbmauXxNPm/K75f2FYo1mUoiw9c/jbU9Tqxbbumdcuo19zycA5uLl26hK5du8LJyQmtWrUCwzD47rvv8MUXX+Dw4cNo3ry5IdpZJeg6i8EcWNs3HmO+FmzzGdRh++HL57dqtotesur9EolhM3QIcOqU9Jft2wP9+vHWVm24DNNoCwjU9Yh4udhjYVy4/PXhElj0aFIdXcMD5AUFf095qFegU/n9wiY452shVK64/G2o63Vi0/a5PUOx8AD1mlsTofZdFE2dOhV9+vRBRkYGdu/ejd9//x3p6eno1asXpkyZYoAmVh2m7PrVh+zbeeUPIdmH6MHr6vM8zJUxXwt9AyTZB/KC/akQS9R/x5V9g9X3o3lqp/qsAlY2vV/1r5yFpGlEeWBjYwM8eKBnC9UTSxgkpeVgb8pDJKXlyK+XLCjxFym+nv4iR6WgUTacFRcRiGgVtWS6hQdgbs9QeLmUzyTNKSjBwgOpCn8L2s5TkWzfub0b4eKcztg+LgrLB0Zgaqf6nK8B2/dLZVyuEV/Y/m1M7FAXZ2bGqm2DtrZ7ujiw7qkllkGnnpv169fD1rb8UFtbW8yYMQMtWrTgtXFVjam6fvVhqNwUvoa4dD2PMV8LPgIkNsNkXKdBq+LpbIuJsXW17ied2aK+HpatuAzTzmzBB+d/Ld9YowawYwfE0W1wIS2H99f+2esAQ13vIpfeFE0OXs/EhG1XDDa8UXkphQb+rqyrJ8voOqzK1zVii+3fRkxdH61t0NT2vSkPWT2OOfaaE9U4Bzfu7u64d++eUoXi+/fvw83NTadGrF69Gl9//TUyMzPRqFEjLFu2DG3bttV63NmzZ9G+fXuEh4cjJSVFp8c2J6bq+tWHIXJT+Bri0uc8bF8LAEjS80bMtbCcJto+fHVZG6iiL/o21vr8tM1sCch/iu/3fYUWD8vzhtCrF7BpEw5mlmBBpYUt+XrtVakccOi79pcpJgXIbtrn7+S8Lv7HQMIAq0+kaT1Wl5u1LtfIXL5kqGu7pfaaE/U4D0sNGDAAY8aMwc6dO3H//n08ePAAO3bswNixYzFo0CDODdi5cyemTJmCOXPm4MqVK2jbti26d++Oe/fuaTwuLy8Pw4cPV6q3Y+lM0fWrD75zU/ga4uLjPNpeCwB4Y8kxDFp/HpN3pGDQ+vN4Y8kxzsNwskAKgNoho3ea12B1LjYfvt3CA3BmZiymdKzHaYjqvXYh6NGkusZ91F13mdjbF5AQP0ke2JQJbSD5+mtg3z4czCzR+JotP3JLaThJlzZUpOsQjTpcgn0+2QgFiKnrg+ldG2B614ZoW68aq+OMcbM+eD1T578TTX8b+n7hqzhEKZEw8HdXP2QrAPskemIeOPfcfPPNNxAIBBg+fDjKysoAAHZ2dnj//ffxv//9j3MDvvvuO4wZMwZjx44FACxbtgyHDh3CmjVrsHjxYrXHvffeexg8eDBsbGywZ88ezo9rzozd9asPPr/x8PGtVyxhcP5ODmb9dk3teQDgk9+vIbahH+xtNcf36l6LxNQsXmvgqOtRqbi20dm0bI29O0IB8Ixl0bdD17Ow7KhyoUBVvF8nw/Zoovn5aJ3ZwjB476/f4Fn0AgDwwN0X91dvQPSQXlpfewBYeuSmfJu63hxdZp5V7l3UtZdB21BcRYYe3ogM8oSXi73aIoDGGuLmo1YU12nrbNtV+XweznbyzxlL6DUnmnEObuzt7bF8+XIsXrwYaWlpYBgGdevWhbOzM+cHLykpweXLlzFr1iyF7V26dMG5c+fUHhcfH4+0tDRs2bIFixYt0vo4xcXFKC4ulv+cn5/Pua3Gpm/3uLHw2W2s7xAXl0JfuQWliFp8FF++Fc5qiKri4xlq6EFbUKtt+QQJI120cY1Q8w0j4eojTNx+RWNbhAJgeHQQujYK4K+OjkCAyb0/RsKmSbgW0hil69ajU0wou2MrUXdz1Gfm2ZMXRToPZXItMufj4qBTG9mQtUVTYAMY/mbN598Jn1/41AVcea9zlkTOdgr5S4ZaSoIYlk51bgDA2dkZjRs31uvBs7OzIRaL4efnp7Ddz88PWVlZKo+5desWZs2ahdOnTyskNWuyePFiLFiwQK+2EtX4zBPSZ4iLS6EvmdyCEp16WQxZA0dTUNstPACrBjfDxO1XoGkEZcH+VMQ29MPlu8+UbgQHr2fig22aAxtAGih1bcRumrCMqtfFtbgQLx3Kv/hkufugz/Dv8Mn73dCjaaDGYzVRd3PUp0ckI7sAy47c0mm1aa7vvXN3siEUCnjvkWXTFmPdrPn+O+HjCx+bgMvRVoitY1sj+2WxWfeaE81YRQf9+vXDpk2b4O7ujn5a6k/s3r2bcyMEAsU3jmydqsrEYjEGDx6MBQsWoH599lMgZ8+ejWnTpsl/zs/PR82aNTm3k6jGV7exrkNcugxFVMS1l8WU9Yg8XRw0BjayG0bU4iMKtVD83R0xoGVN/HQug/VjcW1/xdfFvqwUs09sRGzaRfQauRwvHFzkv3vg4Y+FCf+ia+Pq8muuS+6HqpujrjkkQgGw7a97nHsZdH3vrTqehlXH03itBcWmLV4udjj5cQetw7F8MMe6XWwCrqz8YggFAsRFBKrdj5g/VsGNSCSSBxvu7u4qAw9d+Pj4wMbGRqmX5smTJ0q9OQDw4sULXLp0CVeuXMHEiRMBABKJBAzDwNbWFocPH0ZsbKzScQ4ODnBwMFw3MOGn21jXIS59hiJ06WUx5cwKtjeCykXesvKLsJxljo2MqiBS0+sre/3sMtKxct8SNMm6DQBY/Of3mBg3U2FZhcrXXJ8ZYxWvSasQL/i7OyArv1jDEcokDPD4hfpj1L1PjFWAkQ02bcktKMXlu8+MMuRtjjOQzDHgIobBKriJj4+X/3vTpk28Pbi9vT0iIyORmJiIt956S749MTERcXFxSvu7u7vj2rVrCttWr16NY8eO4ddff0VISAhvbSPc6dttrOsQ15FU1UOYXHD5MDNlPSJj3Qi8XeyRlfcKSWk58gRq5aq7dngrIhCdwvzlgc5q+zTU2TQV7iXSlbyLbexwPqiJyseoeM0rvvZcVbwmialZKOKwPhZXld8nfBRg5Gt6uClu3JoCXnOs22WOARcxDM45N7Gxsdi9ezc8PDwUtufn56Nv3744duwYp/NNmzYNw4YNQ4sWLRAdHY1169bh3r17GD9+PADpkNLDhw+xefNmCIVChIeHKxzv6+sLR0dHpe2EO3NYG4rrEJdYwuB3lgW4NOHyYWbKekTPCrj1SOgqp6AEU3/5G4B0FomqAnG5BaXYcDYDG85mIMhZiJ9Td6LZzp/kv7/jWR0T42Yh1a+2yseomFQrljAQOdljZJsgbDp3l3XvTcXpuQlXM3lZRVuTyu8TYxVgZMPYN25tydfmWLfLHAMuYhicg5sTJ06gpEQ5C7+oqAinT5/m3IABAwYgJycHn3/+OTIzMxEeHo6EhAQEBQUBADIzM7XWvCGKdAlSzGltKC5DXBfSc/VaZ0fTh5mm62iI6analJRJ8One67yfVxttlW9Dch9iVfz/UOtJ+erVe8LaY06XCShw0DCL8vXLyXWmUUU9wqXvk5wXRZi0M4Xz8Vx4OtsiMshTYZuhCzBy+Vs25o2b7RRvU/ydaMKmh5CmfFsHAcMwrP4mr169CgCIiIjAsWPH4OVV/gciFotx8OBBrF27FhkZGQZpKJ/y8/MhEomQl5cHd3d3UzeHV7oEKeo+qGR/3uZYPFBmb8pDTN6Rwmpfdd8eVT0/ttfRWL1dB69n4pPfr6ud3msqvVNPYvGhlXAteQUAKLK1x7xO72Fnky4KOTaqLB8YAQdbIeeZRqbk4WyH//VrrPAe0GW2lCrbx0Up9Nzo8h7MyC7A0iO3OL3XuRJLGLxRqYp0RbIg6szMWPnfgjn0Cle0OCEV60+nKyTnCwXAuLYhmN0jzGTtItqxvX+z7rmJiIiAQCCAQCBQmbTr5OSE77//XrfWEl7oUjDLFOXi+cS2i31qp/rYcfEeq2+PbK+jvh/YbI/n6+apibeLPXJ0CJx8Cp/LA5vbXjUwoe8s/FctmNWxXs72mPHbVaMGNpVv+lyf9/PCUozfkowfKvwtyXonPvn9mk69iKp6VNi+B9UVo5O1VYbPnhJdpnibU92ug9czse5UutK1ZRhg3al0NKvlabZf5gh7rIOb9PR0MAyD2rVr48KFC6hWrby8t729PXx9fWFjY2OQRhLtdA1SDFmzxRjYDAsEiBwxMbYuJsbW1RpMsL2OEgmDhQdu6DyMx+VbuT7T3LXxcLLDqiHN8SS/SJ5jw0V8ZB9E3buGFw4umNv5fbyyd4SLvQ0KSsRaj/0364VeM410wQCY2zMUPm4O8HVzRGSQJ9p/fZzzsNL8ff8o/C11Cw9AbEM/RC0+yql3TVXuCZf3oKoFOmXF6KZ2qo9gH2fee0osecaRpX+ZI+yxLnYQFBSE4OBgSCQStGjRAkFBQfL/AgICKLAxMV3XtLHkDypA+9ozApTfOGxeF03zdXPEkxfSa1F5PSG21/GDbVd0XreKy7pXXKcac/08/t/bjRFT1wf+IidW+zd6XGkxRoEAE+NmYnrPqXhlL+1Fa1ef3bpG958VcmorX3zcHBAXEYjoOt6wtxVqXddLlaz8YqW/JXtbIb58i9vEBlVrxrF9D36697rG5Sp2XLyHXk2qI7qON683akuecWSqtb+I8XGu5LR48WJs3LhRafvGjRuxZMkSXhpFuNM1SLHkDyoZtouNslnAT58gjs0ijNq+OTIA5vx+HSWvpzNzbQ+XtR+ndqovvzayHjB1t0CnkiJ8e+A7HNg0Ge3uXFb4XamNnfzfHs52GBoVxOrxg7y4LdniYs/PF6jK72V17x9tVL023cIDMLVTPVbHz+0ZijMzY5V6+nStZVSRIW/S2t4r5rzIpKV/mSPscQ5u1q5di4YNGyptb9SoEX744QdeGkW40zVIseQPqopkK11vHxeF5QMjsH1clMKNg21vib5BnLabCpuemJyCErT+MhHLj9zErccv9WqPOkIBUM/XVf6zph6wBk8zsP+nKXj7urTMw3cHvoN7kep2PS8sxZHULHi52GvsCfFwtsOw6GD4u2surunlYoelAyIwtVM9VkNdmmh6L8veP30jNK96XpG690r+K3Z5Nz5uDip7VPj8ImGIm7QhV+o2NGv4MkfY4RzcZGVlISBAOaegWrVqyMzUvoQ9MQxdgxRNH1SA9GY9sGUtPptqMLKkRdmQA9scBqC8t0XbdWTr9K2n2JvyEElpOQq9OIksCw4+KyzD0iO3sPL4bT1boppskc2KvVadw/wxpVM9iJxe98QwDAb8fQh7N09D3dwHAICX9k74vOM45Du6qjotACD+3F3kFpRozGF5XliKbw//q7bgnmxI8cu3GqNP0+rYcfE+x2eojIHmm66NUID/i2S3LIuXs53a8gFs6y6pu4Gy+Vv2drHX6zH0xba31NxYy5c5oh3n4KZmzZo4e/as0vazZ8+ienX233oIv/T5NqWtW37pkZtKwzeGJpYwSErLURkgcD2Gyzg7m+vIxuoTaUpDX2IJgz0pjzicxfBkQZ1syG7pkVt4/qoULsWFWJnwLZYc/B6OZdIE2VTfEPQesQz7wt7k5bHXnkpXW0NH5Gwnv0nqu8SBjGwWkSZRdbzhzGL4a1HfxnrVXfJ2sVd7A7URCjC3Z5jaWjUAsDAu3OQ3aW29pebIknudCDeci/iNHTsWU6ZMQWlpqXxK+NGjRzFjxgx89NFHvDeQsKdPwSxZ4byVx25h6RHlNYi0rYHDZx0LXWv1qDummGU5flkXvqbrOKBFTWw6l4HnLIceAGngNH5LMt5pHmhWdWpkQd3KY7ex7MhN+c007PEdfL9vCerklvdAbInojoUdx6HYll2Pgb6c7GwQ29APSWk5+JNlUD2qTRC6NArAkdQsbDibofT7vMJSres4JaZmoVDL8Nd77ULQo4nq49kOA8VFVFf793HweiYWHkhV+buKf8tCIUxe/decpnizZW6FBYlhsC7iJ8MwDGbNmoUVK1bIKxU7Ojpi5syZ+OyzzwzSSL5ZcxE/QPdAQ5fiXIDmwILrYpq6FBTUdsyUTvVUBmyVVS6iVrk42vYL9zgvyGgJPJzs5MFat//OYvn+b+Aglv78wt4Js7pNwoHQtkZvl5eLPadgcPu4KLQK8eL0Hq74Gvu4OOCjXX8jK199gCJytEHyZ13VvoeT0nIwaP15Vm1VFRRoq2m0enAz9GhSXWF/c6ksbmnMrbAgYYft/ZtzcCPz8uVL3LhxA05OTqhXr55Frbpt7cGNrnT5YNYUWDAAPJxs8fxVmXy7v7sD5vdppLb3R5fKp9qO8XN3ACDA43zNZekrB20yxiiiZy6Cnj3CH5smw63kFa7618XEPjNxz9P8b5ICAJtHt4JQIMCQDX9p3X/7uCjkvSrRadmHqZ3qY7KaGVGy96OmujmezrZYMbA5cgtLFG6qun650OUmXVImwc9JGbibW4ggL2cMiw6GvS3nLAVCjI73CsWVubq6omXLlroeTswQ12mSbBJ1KwY2gLQ+SOUKrzK6FBRkc0xWfjGmdqqPZUducurCF0sYnL+Tg1m/XTOrwEZWhE7W06AuaNNGAEBUodcGAO56Vsfsbh8i8uENLH5zNEpsteepmAMGwLCNF1jlywDSleQ3ns3Q6botPXITDfxdVQbomhaLlHlWWIZhGy/If5b1tIic7HUqqMl1aEjV0gNfJNygpQeIVWEV3PTr1w+bNm2Cu7s7+vXrp3Hf3bt389IwYnxcp0nqk+g5a/c1pSqgutSgYHtMsI8zp3F2fRZzNBRZL1TDAHdkvywGBED/FjWw4hj3GVUCAGAYfPUyGZMk9VBkV/7a/xHaDn+EtuOt3cakLV9G5veUh3oFrJqq2MpyOubv+4fVMKYsn21UTDCrx9ZnevfihFSsPZWutF3CQL6dAhxiDVgFNyKRCILXi+CJRCKDNoiYDtdVhfX5kH1eWIrzaTmIqecj36ZLDQoux0TX8WaVA2Suw1AMgBfFZRjyo/ZhlwCRI+b2DIWniwOOpGbh95SHCrN4goTF+OH4KjQ8fwxfRXbD5E4Tze75GoIAgJujrV4ryQOKPSiqhoUA6VpFbMjK/u9lOZNO1+ndJWUSrD+tHNhUtP50Oj7q0pCGqIjFYxXcxMfHq/w3sS6autRVDd/oW0Mj6U62QnDDNbgCgMggTwgFmivzCgXS/QDtXfiGXstJXwXF7Hom5vYMlSeeRtfxxic9w3AhPRdHUrNwa/9RfLnzC9TIfwIA6HP5INaFd8N1/7oGa/eEN+tAIBDg5uMXOJz62GCPow0DIL+oTOt+bDx5UaR24Up1U9w1tSunoAReLnZ4VlDK+v3Pxc9JGVorWEsY6X5j2tbW6TEIMRcUnpsxXWq96PsYncP8WRfn0r/gneKRutSguHz3GasP7Mt3n7FqEV81VUzt8z8Ul4CwEQqQV1gM4dLvsGHDNHlg89zRFWPenmvQwAYANp+/i5XHb5s0sOFbRnaByqrXXAObit6KCFQbWGsrQqjN3Vx2a3mx3Y8Qc8aq56ZZs2byYSltkpOT9WoQkTLGFE9Nj3FmZqzW4Rs2yZOaqOpB4VqDgu+1YqxlTRnZwo6yYZNLl2/BaehwzLlZPqR1KTAUk/p8jEfuvgZvzwueekvMhYeTHbZfuMd7D5+7k+ESuNmu5cV1zS9CzBGr4KZv377yfxcVFWH16tUICwtDdHQ0AOD8+fP4559/8MEHHxikkVWNupwPbYX0+HyMVYObw5NFiXd1wYhAoDnnwNPZDlG1VQ8PyQoKspneyvdaMda0poxs2OT3lb9g3rZFqP4iW/67Na3fwbdth6LMRucJk1Vap1Bf/JrMbpkFNmRDTtsv3NO4j6ZEZm2GRQfji4QbWodwh0UHcz43IeaG1SfbvHnz5P8eO3YsJk2ahIULFyrtc/++/uu/VHXaplfr+wHH5jEAYOL2ZIUPQU29RqqCkWcFxfhg2xW1bVjcT3X5ehm201vZ5ulEBnkiKS1Ha7DUKsRLp5wJc5SRXYiTP+3FL1tnwpaRVmnOcXLHtF4f4WTtSIM/vrYA1xz5uzugqEyi8fX3cLZDTF0f3oIb2btwYMtaWHrkptr91E0FZ8veVohxbUNUzpaSGdc2hJKJiVXg/LVt165duHTpktL2oUOHokWLFti4cSMvDauqdKn1wvdjAMoJutp6jVQFIz8IBZi/L1Wh4qumIEk26yQr7xVyC0rg5eoAf3flQERWfyYpLQcAg8hanvjjmvoS/X2aBqD918dZDfElpmZZRWDj52aP7Rfu4XH1BjgX1BTtMq7gr5rhmNR7Oh67+Wg/gZ76RlQ3u3W0tJnbMxQjY0KQmJqF97dIh9dVxWb/69cYIif+lqHw13GZEF3IpnlXrnMjFIDq3BCrwjm4cXJywpkzZ1CvnmKFzjNnzsDR0Xq69E0lK+8Vq/30+YDT5Vg2vUaVp8R2DvNnPbykqa5MxUDk4PVMzNp9jXUA0inMF+tOpSvdpGTrPU3tVB8TY+vKK8Qu2K96TR9LM7h1kHTJCYEQ03pNQ/+riVjb+m2IheyK3Omrhqdl5W24OtjAy9UBF9Jz5Un1mnLeEq7qHrhN7lgXrUK8kf2yWOFvQhqsa6fv0OnsHmH4qEtDqlBMrBrn4GbKlCl4//33cfnyZURFRQGQ5txs3LjRYtaWMlfSBfNusNr31uOXSErL0Wk9FF0/HDX1GumTAK2trkzm616jd9tp7lJX5eiNJxqTPpceuYntF+5ifp9GWivEGpqHsx3yClVPA2ZDKBFjysXf0GrUW3js4yLfnu3iidXR/flpJAteLnaIruONlce5Fxc0lZfFYkzdmQJAe1K9WMKw+jutPLyp7e9Bl1IIurK3FdJ0b2LVdFpb6pdffsHy5ctx44b0Dzw0NBSTJ09G//7G+wDVhzmuLaVr4ThdZlCJJQwiFyXqPPyyfGAE4iIC5T/rsthlxbZoWk+nIm31bPQ1IjoIPyXd5Xycm4MN2tTxxombT1FcplsDp3aqhwb+bhqHQzSp9vIZtp78HvWvX0Cxf3VsXPkbllxk1xPAt9WDm6FreADr15UvfL0/tL1v2a7BtnVMawiFAk5rPsn+lgDVdab4mEygCa05Rcwd2/u3Tu/a/v374+zZs8jNzUVubi7Onj1rMYGNOdKncJwsF+bgdfU5J3zLyC6vg8EmOXnB/lSVNXrEEgabzqazvgEaMrABoFNgAwAvisU4lPpE58AmQOSIibH15DPPKtcYChA54r12IWqPb5ORgj9/moT616XrFdk+zsI/P+/RqS0yutYueq9dCHo0qS4vE2CsNZYFAFYOaobt46IwmuUyBupoe9+yHdbNLihGdB1vxEUEIrqON6seVnXvAVV1pvi2OCEVDef+iYUHbmBz0l0sPHADDef+icUJ1jFUS6oWneaBPn/+HL/++ivu3LmD6dOnw8vLC8nJyfDz80NgYKD2ExAF+hSO02UG1YX0XL2SZisuHKhrArQ5rt1kbKoKE1aeeebj6gAw0hvl1E71sf3CPXmCtlAixswLOzHu5HYIX9+SH7t6YXLv6Thfq4lebWsU6I5/HuYD4NaL1KyWp/zf6soEGMLYtsEKFZndHe2w7Ogtnc+naQjW0GUF1JVCAMBqxp8uaM0pYm04BzdXr15Fp06dIBKJkJGRgbFjx8LLywu///477t69i82bNxuinVZN38JxXGdQ8VGoThZM6VIcz5hrNwlfT0c2hxnJlQsdqitMKJt5dvB6Jqbv+lshMPB0tsWUjnURyrxE1NxJEF04J//dqeBmmNrrI+S4eOjd1rQnBRjZJgi7Lj/Ey2L2Bfhm7b4GNwc7RL3uqVBXJmDhgRu8Bjy/JT/ErO7lQeKHHeth3ek7rBfSVEfV+1tbbgwg7W3TJzem8uxDQxb1NKc1p1St08VXAEeqFs7BzbRp0zBy5Eh89dVXcHNzk2/v3r07Bg8ezGvjqgq+CsexDTT4eDxZMMX1W6w+Q3C65FTENvTFkRtPdHg0/n37f00R4OGkMNVd5GQPsYRhvXjns8IyJK/fiXcPLYNzvnRJiTKBEN+1HYo1Ue+AEfBz83lVKkb8Oe7DdM8LSzFkw18KN15VZQK6vu71e/KiCNkvilkn0quTW1CqENwnpmbpHdgAqv9W2FTmflUqRmJqFrqFB+h9wzZ0UU9zWXPKGFXZSdXBObi5ePEi1q5dq7Q9MDAQWVlZvDSqqmHzTZANH1cHoz7ekxdF6NWkutZzeTnboUwswd6Uh8h+UazzN3ZtBchUOfaveQQ2APCssATXH+VhT8oj5BaUyLdX/gDXFAB6Febhhz1fwrm0GACQ6eqND+Nm4FKNRsZ4CqypuvGqu8mLJQx+PJPOy/tR9jj6TunXNjNJNuSmrixBXmGpfIbfvr8zdb5hG6OopzmsOWWMquykauH8Nc/R0RH5+flK2//77z9Uq1aNl0ZVNWwWjGSF5Z1B0+Nx4evmyOpcuYWlGLbxAibvSNH5G/rUTvUxu0cYfhjaHB7O7NffMXQSMlsCAbDwwA1sPJuhENgAyknhmvKYcp1FWNDxXQDA0Tot0WPUCrMLbADlpNyD1zPxxpJjGLT+PCbvSMGg9efxxpJjOHg9k9f3I8Df4qfaFqnsHOYPRzXDNMzr/9aeUk6YVzcJQNVCuVxy2nRl6jWn9JmUQIg6nIObuLg4fP755ygtlX5bEQgEuHfvHmbNmoW3336b9wZaKy6rcbOd/ZFdUMz68dXNynB10N6ZJ4BiToHsXCIOQQcX/u4OmBhbV/5Ylz/tjK1jW6NLmJ9BHk8dL2c7eDjb6XQD1lRwofIHuNLwYqWDdzbpgjFvz8XYt+fimbNIh9YYh+zGu/LYbZWrZ1e8yat7P3o628HZTvvHlFAARAZJk5n5yCkb2zZYa0/BhfRcZOWz/5uTUXXDVhf8Jaay6w3X5zkPiw6Gtk4fQ645ZYwAjlQ9nIelvvnmG/To0QO+vr549eoV2rdvj6ysLERHR+OLL74wRButDtfVuC+k52Lj2Qyt5+WaS6NuVkZiapba7nZVM3xk8nhetkCWzzCoVS38cfWRwlBGTF0fCAUCHE59zOtjquNiL8Sw6CC8KCpj9VpwVfEDXPY62olLMePkTxAwDBZ1HFe+s0CAo3Vb894GQ4k/q1whGlAeVlH3fjx/JwdDfvxLxRnKSRjg8t1niK7jzXp4VpPKCcqq6BNQVHy9816VqB2SYfte0yePztRrTukyKYEQbTgHN+7u7jhz5gyOHTuG5ORkSCQSNG/eHJ06dTJE+6yOLmPLulYuZZPIqCrZs3OYP9wc7LD1rwycvJWNguLyxExPFzu8FRGokAirT5KwJrKeoKVHyqf0VsxX4Ct3iI2CEgmWHzV8xd0nL4pgJxSiRt5jfL/3KzTL/A8AcL5WExypp3tAE+LtjIzcQpMsZPn8lfqgt/JMP1Xvx+yX7HpHnrwoQsLVR5iz55o+zQWgnKCsCh+J+Vn5Rfjq4L8agz+BhkR6vqoWG2LNKbaJ1IaeWk+qJk7BTVlZGRwdHZGSkoLY2FjExsYaql1WSdfkQE2zM9T1pOg680DVcV4u9oioKULK/TzkFpRgw9kMbDibIT8fX8sWeLnYYVhUMGpXc0FGdiGWHbmpNQjUNmvF0vi4OmDfgtU4sOsriIoLAAAlQltUK3im13nTc6TJoG6ONois5YmYutWw8tgt5BWxn+atC2d7G1azljR9K2d7U9vx130kpfNXlfnJiyKNN2g+guvcl5oT7BmUj0qy/dvX1eweYZjSqQG+TEhFRk4hgr2d8UmPMDjZc1+PjMvnjzGXnSBVB6d+RltbWwQFBUEs1n+KZVUiy69ZmnhT57FlLpVLZb1DbBMZtR2XW1CCY/8+VZsIe4RlXoA2uQWl+OXSfdgJBdhx8R6rBEN118XSCADUchEiZOEnWLJlnjywuSfywztDv8L2iG68PM7LIjFO3sxGYUmZwQMbABAK2N10NQUwspuftjOxCWwCRI6s8soAaSVudUnQgH6J+bK8NS8XdquLj4kJNnjV4oPXMxH77Qn8fP4eTt/Kxs/n7yH22xNaq59Xzh9MuPqI0+cPmwkVfAVwpOrgvLZUfHw8du3ahS1btsDLyzIjaWOuLaVLJd7KazdVpK2rV9taTbJvQWdmxnI6Th0BpENVuQX85tuwsX1clHzYQHZd/ryeic06LqNgKgIAtZ5lYt+Z7yFKvSrffqBBDGZ1n4QXDi7qD9bx8UROdhqHi/jk4mCjMLRZmaezHS592lnjzUvdmktczenREP4iJ3y4/YrG/dQtYqpqjSd1vRR9mgZg3es8FnXrRImc7FmtU7V9XJQ8/84QBe50XR9O1XPXVI9K3eePunNRnRtSGdv7N+ecmxUrVuD27duoXr06goKC4OKi+MGbnJzMvbVWStdKvJW/xXIpAqbLcghc13iqfL7cglI42QnxqlTC+Xh9HEnNkj+HirkalhbcDL73F+bvXwa7ly8AAMU2tlgYOw5bmvWQJlzwjIHmPJjKpnSsh1elZUr5GKwfT8sxbE7J11IOXyT8iwCRIzqH+SIxVXMNJLbDx+oSoW2EAjSr5anU5oqVqcUSBgEiR43PSTYzUVU+Eh90HS5X9/mm6T2iqZq6putICFecg5u4uDgIDPCBa210TbL1crGTT2kFuH+b4TrzgK81nowd2ADAhrMZaBnipXAdjJlkzAdvJxt8dONPeWBz16s6PugzE//41TFxy6T83OzxYcd6sBEK8FGXhvg5KQMZOYXYdfk+ili+5tpybp4Xak/eBcpvfpvOputV0TgrrwhZeUUY1zYYvyU/VOh1DBA5YmDLmgpJ7JWpukGrCzy03bBthAL0aRqgcaZSn6YBBr3B6/qFSJ9JBOo+pwwVwJGqh3NwM3/+fAM0w/roWkgst6AU7b8+Lh+D5jqzisvMg4Srmfhgm2X3tFX+RsmmNL45yX0lRq/oD/Dnv5NwIiQSn3SdgAIHwxRL08WCuHCFaxtWXQQvVwfsvvJA67ECSGe8sVmklW1QbiMUwMdNv6nest6IP65m4fzsTrh895lC4PHH1UeszsOlzepu2GIJg31/a85p2fd3JmZ0CzVYgKPLVGx9CyXSzCdiaKwTigsLCzFhwgQEBgbC19cXgwcPRnZ2tiHbZtH0qckgC15m7b7GuWqntuRLWSJjzosiTNxu2YENoDoBu1t4AFYNbsapkrExuRe9lP+bAfDI3RfdR63A5N7TzSaw8XC2ww+V8kpkybVTd6ZozKGRYQCMahPC6vG43Oz4uDHKeiNktXHiIgLlU9GNOTWZTZBg6AJ2ujxfXT/fKhcAJcRQWAc38+bNw6ZNm9CzZ08MHDgQiYmJeP/99w3ZNoumzwefrHS7pm+86mZWVZx5oO648EB3TNyRYjZLE+ir8gftweuZ+GTPdTyrdP2c7W3grMO0Vr44lhZh8Z8rsHfzVLgWK67T88jd1yD5NVz1jaiOrWNa4/KnnRUCG1WzX7QZHROMibF1WQXbXG52bGdPsaFp1W8+28zl8fXZTxe6PF9dPt9o5hMxJtbBze7du7FhwwasW7cOK1aswIEDB7Bnzx6aFq4Gnx/Amqj60OsWHoB326n/xqwtkdLSVPygPXg9E+O3JKsMDAtLxCgsEaO+r6vBX5fK6mTfx57NH2HQ1cMIeZaJLw+t1J5pawJeLvaIqecjv/nok1shGy7UNF2aATCwZU1O5+VrLSpA86rfqs7P9w3aHArY6fJ8ZZ9vXPA9dZ0QTVgHN/fv30fbtm3lP7dq1Qq2trZ49Ijd+HRVw9timFqo+tBjM45vLfzdHeTfKMUSBrN2a69Oe/PJS6Pm4rx97Sj2b56ChtnSWVyFdg44GRJpFj01lW08m6FQh0SX3Ap1a4+pq0W09MgtxPzvGJYfuaWwaKSqhSRl1NZ9cnfA1E71sHRABLxc1A9Laut94VJXii1Vz8eYvUSacH2+2nqIK5vYoS7OzIylwIYYDeuEYrFYDHt7xYJTtra2KCszfCEwS6Vu+qo/ixkZbFRcLLAivlZFNjSBQP/Oi0Gtasm/UZ6/k8MqedVYnEqKsDBxDd65flS+7T+fWpgQNwu3fWqZsGXqVZ72m5XPPbABlL/py2YNrTx2G0uP3FQ6Liu/SGG7LF+q4utZeZagtplITnZClbVx2Pa+8Dk1WdOsR67Vxw2F6/PtFh4gDSRZfI7F1PWhoShiVKyDG4ZhMHLkSDg4lM9UKCoqwvjx4xVq3ezevZvfFlo4dR8YALDj4n29pixXXCywIktZYI5hgKmd6mHdqTsoYFGiX5Vgn/L3XlIaf6X39VX/aQZW7V2Cejn35du2N+mCBZ3eRZGd+c4Uqbyg48I//uF0vL+GMgUAsOPiPVbnURWkyhLtVw1uBk8XB603YE1fLtgWhuNjajKb9eT0bSdfuD7fibH1sP3CfbVBMC2dQEyFdXAzYsQIpW1Dhw7ltTHWSt0HhqZvbGwDnoqF7GSMNc3SQ02VWy49Msn3nukc2ADArccvkZSW8/rD0zxyWN65dgQLD6+BU5l0wceX9k74pOsE7At706CP6+Fsh8GtamL1iTt6n+tIahY2ns3QekUFkObpfNozFP4iJ70KTGoja8vE7VcUkuE11X0ydWE4tgXyzsyMtcgCdjZCAeb3CdOrh4wQQ+C8/II1MObyC9qo664e2LKWyu57VX6oNCYuW0rB0IXsto5pDeHroYvcl8W4/6wQm86ZpjpwgMgRA1rUxLKj+g318WHMxT2Ye+xHAECqbwgmxM1Cupfq5TT4oEs5f228OCypMbVTfUyMrav1BrY35SEm70jRu22VVXz+5hYgJKXlsF5ewZKL19HSCcRY2N6/KbgxcXADqF5eAQAiFyWyyiEJULFWC19r8ajj6mCLv+d1UZhVo8vaVHyRPXMnlqtQGxTDYP3uhXjs6o2FsWNRbKdf0TltPJztMKpNCN5/sw4uZuRiwtZkvdaN0rQ2kDqabmSy9/fZ20+x8niazu3SRFYw0NHWRmGIxNQ3WLYBnab15CwFl2ViCNEVBTcamFtwo4psSjNbqr758bW0giou9ja4Or+r/MOL7TdUQxJAeqOvXN/GoBgGTbJu4WpAfYXNtuIylNlwLgDOibOdDQpLywM5XYISvqhbYNGQ70F92mUsVaXnhhBjYXv/Zj0VnBiPbJyeC3X1bs7MjMX2cVEYHh3EV/MAAAUlYixN/E8+pTUr7xWv59cFA+BZYSmmdqoHf3fD9pYAgFtxAVbu+wp7Nn+EmIwUhd8ZOrABoBDYAKYLbADVVbN1Lf5n6HYZk7lM9SakqqHgxgzpknipLolYlszc3QDfWlceT8Og9ecRuSgRc/dym1VjSME+Ljg7qyO2j4vC0v5N4eVir/0gjsKzbuPQz1PQ69/TEILBsj++gXOJ6QM8U5LNtNp0Nh0lZRK9Flbkk7pq3sZgzIKAhJByFNyYoSOpWZz2d3GwgYRhNH4zNWTF5OeFpXhZbD71jnzdHOVBnb/ICbkFJfydnGEw4vJ+/LZlOqrnSIvd5Tm44NMuH6DQ3onTqdStfeViwiUi+LDwwA1ELT7KKkCf2KGu0Xra+CqRoKm4oCpsC+RxPS8hRD3D951XcVyT7A5ez8SGsxmcHqOgWIwhP/6lMXnS0lbL1oWqmhp81vxxL3qJr/5cjm43k+TbUgLqY2LcTDwQ+XE6lwCAk50NVo1pjicvi5H7shgPnhVi79+Z/AZjenJxsGG1SGZlbJ9DPT9XxEUEYmJsPVxIz0VW3issPHDDINeAjxIJus4K0jYlnWYbEcIv6rkxoIorKU/ekYJB68/jjSXHFMrbV6RLrk1FsqJg6s6vrQy+pWMA9GkaoBA8ZmQX8HLuiEf/ISF+kkJgs75lX/zfkCWcAxugfKhEKBTgrWaBCPR0wqZzdw0a2AgAeLvYY2n/ptg6pjX83TXngng62+kU2HAhCzgM1tMG/vJa1OUQafu7k5E9x4orkPNxXkKIMgpuDITtB1bFruhNZ9N5KXKmKXmyYpLxqDb8Jhmbg3Wn0uXX9uD1TL2XuACAuH+OY9fWGaiRL11w9LmjK8a8PRdfxI5FqY369YvYePKiiNPilPoMKzIAcgpK4C9yQkw9H8zvozkXxJC9e+oCDrY9ba4O7Ibu+Mpr0VaMD9AtadlQ5yWkqqNhKQNgW5VUIgEWHuB3mmzF5El1U0tl3yCj63jD3laItafSeXt8c7BgfypiG/rp1QtW0dWA+iixsYOdRIxLgaGY1OdjPHL35eXcvm6OnBLI/UWOCA9012tld1kAoXntM/ZFJLnSFHCwHTp6ybJHia8lDLS9Rmz+7ox5XkKqOgpuDIDtB9YH29jXseEqK78ISWk5SoUBK4/5z+4RhqY1PPHp3usKwwHqllYwd7Jr+3NSBm9BY7pXID7pOgGhTzPwTdthvEzzrpgf9MfVR6yOmdihLiZ1rIf2Xx/X67ErBhDqckHYtomNytWONQUcssR3ddW1BQBELN+brg42OPlxB9jb6t9BzbZHiWuOl6HOS0hVR8GNAZjDB9HCP/5RuKGoWmXZ390Rg1rVQrCPM5YPiMC/WS9w/1khgryckfeqFCuO3TZ6u/lyN7dQp+MEjAQD/z6M3Y06KFQW3tuoA/by1LbKPRdseyti6vrg8t1nOgdt6hYxVLX2GR/Jt7LHO/lxB1y++4xVUr2mxHfZEaNiglkNN74sFqtcWFYXbK8H1+tmqPMSUtVRcGMA5vBBVHldIJWrLOcXqR16sPSqG0FezpyP8SrMw3d/fIc30y+jcdZtfNJtIi9tqTzjqHLPxTMWCbSy/JR9f+vWo8I190RbDwpb83qHwd5WyCnA0Laad+cwf8SfzWDVe8PXFw02PUq6rH5tqPMSUtVRcGMAfN0YTMlQ7Xaxt9FrFXA2vFzs4OFsDy8Xe9Yzb1rdv44V+76C/0tpobeBfx/CpsheuFktWGlfexsBSsTsr9C6YS0gFAhU9lyIJQwWHtCeGzS3ZygSU7Pw6Z5rrB5TW0ClDR+lA95tF6Jzrou2qdMj2wSzWiTVx5Wf+jlsepR0SVo21HkJqeoouDEAbR9Ylhrw6EMAwMvFHjlGqOGSW1CKj3b9zWpfoUSMD87vwtQz22DDSAAAT509MKX3dJWBDQB4uzpgbs9QzP79GvJeqS9eKPvWHVXbW+3NiW0y8a0nLznN/CooFmNqp3oI9nHReRFDdT0obAgA7Ps7EzO6hep8Y1Y1XCbTMphlTwaPf2zaepT0CeQMcV5CqjIKbgxE0wfWnO4N8dn+f5SGjqwZAyAuojo2cixQaEjVCp9h6b5v8Mbd8kDobFATTOk1HU9d1d88M/OK4OnigOS5XfD90VsqexDYfuuuuIK1JlyvmwDAjov3lVaL56piD0piahY2ns1gFaAbepZPdkExr/uxpa1HydzOS0hVRcGNAan6wHpWUIKFB1KrVGADSBOaRU78r/GkqzYZKVj+xzeoVvAcACAWCLE8ZhBWRveHRKi9hsqj569wIT0XIdVc0KtJABKuZSosXCkQAOPaah+WyX3J7uabx3HmGp/BRcXSAa1CvDj15Bgqud6UibiaepTM8byEVEVmUcRv9erVCAkJgaOjIyIjI3H69Gm1++7evRudO3dGtWrV4O7ujujoaBw6dMiIreWmYlXSvFclmLDNtKskm8rzwlIsPXITDjxMy9VX1L2r2LJzrjyweezqhSEDF2FFzCBWgQ0ATP/1b3nl6T+uZiqtyC1hFAsKqmOIRT0r4ju4kBWBnNszlNX+hkqup9W2CSGamPxOs3PnTkyZMgVz5szBlStX0LZtW3Tv3h337t1Tuf+pU6fQuXNnJCQk4PLly+jQoQN69+6NK1euGLnl3HCpQmsphK97JwRgP7uquExiyCaxcqFGI5yv1RgAcCq4GXqMXIHztZpwOgfD8oXUVl3WX8RtsU2uDNVzMTImROtCrB7OdpBINC/oqk8baLVt46KFPYklETAM249pw2jdujWaN2+ONWvWyLeFhoaib9++WLx4MatzNGrUCAMGDMBnn33Gav/8/HyIRCLk5eXB3d1dp3ZzlZSWg0HrzxvlsYxp+7go5L0q0Snp1JSqvcxFXOoJbGjZF4zAsDH+9nFRaocbxBIGbyw5pvHaSVfMFuBxPvvZd7JkZn1zbjSRLTECaM7BMeQCkLTgpHHQdSbmgu3926Q9NyUlJbh8+TK6dOmisL1Lly44d+4cq3NIJBK8ePECXl7qu5+Li4uRn5+v8J+xmUNhP0OIP5sONwc7HPvoTTjambwjUImNRIzppzaj+cMbCtufunrhx1b9VAY2slDAheX6Rdpoeu1lPRCahlfm92mkdh0odRhIp48bsueC7UKshlwAsuJaacsHRmD7uCicmRlrlBtuVenJoIU9iSUyaUJxdnY2xGIx/PwUV1X28/NDVlYWq3N8++23KCgoQP/+/dXus3jxYixYsECvtupKLGFwIT0Xtx6/NMnjG9rh1Mc4nPoYrg62KCo1/ZBTRf752Vix/yu0epCKvv8cR8+RK5Dn5Kb9OJEj5vYMxT+Z+Vh1PE3vdmgbGlI3s67yN2NV+7jY20DMMCqv/cIDNyAUCgx6o5clzZ+/k4MJW5NVFtaruJ5a5zB/3gMuUyTiVpWeDLbr5BnidSVEH2YxW0ogUPyjYBhGaZsq27dvx/z587F37174+qpfyHD27NmYNm2a/Of8/HzUrFlT9wazpOoD0Fq9LFZf78UUOqRdxLcHlsLrlbSXzu9lLlo9+AeJ9aI0Hje3ZygCRE68LGjKpbosm6nAlffJyC7A9gv3kJWvesaV7Jv1mqHN1d5wZcF35cdUt10VG6EAQoFAY8Vga1oAUtaTUfmGz+Z6Wxpa2JNYKpMGNz4+PrCxsVHqpXny5IlSb05lO3fuxJgxY7Br1y506tRJ474ODg5wcOCnUilb6j4AiWHZisvw8anNeO/Cbvm2B+6++LDPDFwJbKj1+EfPX2HRgRt6v266JLWy6YGQ7XPweiaWHbmlsZ3avlmr633o0zQA+/7O5NQrUVUWgKxqPRlV5XUl1sekSRL29vaIjIxEYmKiwvbExES0adNG7XHbt2/HyJEjsW3bNvTs2dPQzeTMGmdGWYLAvCfYtXWmQmBzuF4UeoxawSqwAYDfUx7y8rr5ixwN9g2ey/ur4jfritTlUWTmFWHtqXTO+RVVZQFILj0Z1qCqvK7E+ph8WGratGkYNmwYWrRogejoaKxbtw737t3D+PHjAUiHlB4+fIjNmzcDkAY2w4cPx/LlyxEVFSXv9XFycoJIJDLZ86iIbUl9wp8uN5PwdcIyiIoLAAAlQlss7jAK8ZF9pBX1tOBreQgPZzusGtQcUXXUL7mgL13eXxW/WesSfGvrlWCznlrFqeGW2qthrJ4MLsOChkQLexJLZfLgZsCAAcjJycHnn3+OzMxMhIeHIyEhAUFBQQCAzMxMhZo3a9euRVlZGSZMmIAJEybIt48YMQKbNm0ydvNVqupdtMZeP6vay1ys2P81HMukgck9kR8mxs3E1YD6nM6jz/IQstvO//o1Rkw9H53OwfaGpsv7q+I3a12Db035FWwW2nxeWIohG/6y6MRbY/RkmFOyMi3sSSyVyevcmIKh69xYa00bNjyd7fCskN+lJVxfr3Ct6Y065EoCvji8GgcaxGBW90l44eDC+vzeLvb44q1wiJzsWb9ulVcc1/fmw+WGxuX9parezd6Uh5i8I0WndgLA8oERiIsIVPm7g9czMX9fqsY1s2S3QUtMvJXVJdLWk6FrfSF1uXqmvmbmFHCRqo3t/dvkPTfWiE0XvbWKqu2NP6+zm8bPloQpHxaRX0+GURhu2hrRHQ9EfjgZ0pzVMJSMl4sdkmZ3hL2tEGIJAw8nO42zfmTm9gyFv8hJr2EDWU/NkdQsbFDRY6Ru9g3b95e6b9b65kdoP17zu96SE28N2ZNhzsnKtLAnsTTmV3XNCmgqDW/t6lRj32PCVmGJGAAgcraDQ1kJPj+8Bp8c36i4k0CAk7UjWQc2siUjvnyrMexfr3dlIxRgVEwIq+P9RU7yNcOidcivOXg9E28sOYZB68+rDGyA8htn5SUc2L6/1CU1a1uXSR1t6zXJeh3UTU2vyJITb9UVL9Q3idzck5UrrpOny3ueEGOinhsDkX0AVu6irzycYW0YRtobom3Vc6EASotNalPv+SPEH/wWrjeuAwD+a9gcv1VvplM7/dV0qU+MrYv4c+l4rmZojY8ESi5lAtTluagr/OflYoe3IgLRKcxf7TdrNvkxlWnrldB1hqCl5qcZoieDpl0Twh8KbgxO8ePeTiiAh7OdxpunyNkOeYWlBhnScrYXorDEcJWEV51gV9FXFtjYCgE2a2n2ST2JLw+thGvJK+kGR0d8HVsT77SNwpMXRch+UYyFB25oPgmAiR3qIKZuNY03/v/1a4zxr9dMqoiPBEo+gwB9brCaqiKrqnOjLhiU0TVJ2ZKnEPNdGZmmXRPCHwpuDETdt/MnL4rl29SN2f+vn3TFakNUN3a0s0Vhifn0HGkLbBxKizHv6DoM/vuQfNvL4Dpw3bsbwiZNEP16m1jC4Mcz6VoTPad2bqD15t8tPAA/qLjxa7vBs8F3EKDPDVZTcDSjWyinoIlrbwJNIVZmimnX5jLlnBC+UXBjAGwSAz2c7eBgK1TIT/Bzd8CgVrVQXCaBr5sjTn7cARczcjFu8yV53om+cgtKWA0bmYM6Ofexcu8ShD7NkG/7rVEHrHpnGhLDG6PispZ8J3oaKoHS3IIAdcER16CJS28CTSFWzdjTrmkGFLFmFNwYAJvEwGeFpdg6tjXAAEl3spH2tADn7+Rg6ZFb8v0CRI4Y2LIWb4GNzFsRgfJ6LpU/QM1ldle/60ex6PBqOJdKg79COwd81vl9/Nq4E1AElbVW1A216NrjYogFGa01COAyQ5CPHjBrxfd7WJ2qtD4WqZoouDEAtt/ON5y5gxuZL9QGQll5RVh65CafTQMAdArzR8sQL6UPUD93BxSVSdTmAxmLjUSMYckJ8sDmP59amBA3C7d9asn3UXeNzX3KqrUGAdp6HRgAo2OC0VlDojORMvR72JynnBPCFwpuDIDtt/Nj/z7V+HtD9KK4OdrIPygrf4BKGAZDfvzLAI/KjVhogw/jZuBA/CT82SAG8zu9iyI7xWuq7hqbew6BLkGApudkTs/XWL0OVYEheg1laKVvUhVQcGMArUK8NM6IMqXFfRvLb36VP0AX7v/HNI1iGIiKXiLPyU2+6YHID13GrMJjN8WlDCrmoFS+sT8rkM6YMvccAm1BgCzo/OPqI2RkF2L7hXsK5QRkzwlQTjo39fM1954zQlPOSdVAwU0VEhnkATGk5fsr33DEEgY7Lt03epucS17hi0Or0CTrFvoMX4oCB2c42ApQXMYoBTaA9Fvl3J5hSEzNYjWbzFxzCNQFAYmpWXhjyTGNzysrr0jlVHXZ70z9fA3Z62AOzKm3TBc05ZxUBRTcGMCF9Fyz67VxtBXi8t3nuHw3BYDiN3yxhMHGM3dQUMxv4rIq7etXw8mbTyEA0PDJHazcuwR1ch8CABYdXo2pvaejpEw6WKOu9+uTPddYX1+uOQTGvHFVDgLYFvfT9HtT5UxY+g2fLWuYYUQrfZOqgIIbA8h8/srUTVBSVKmgjOwb/rvtQpQKthlSu3o+qOPjDJsf12N6who4iKVBygt7Jxyt2wpA+Q0aatZ05Ro4ss0hMOWNS9fifqoYO2fCGm74bFjLDCNa6ZtUBbS2lAFcuf/M1E3Qinn939pT6UYLbIQCYOnuy2g+8wPM2b9CHthc86uDXiOX44/Qdgrte/6qjNfH15RDILtxVb4WsiGg5UduYm/KQySl5Sis88QXXYv7aWKMnAlN1+39Lck4eD3T4G0wBm0zjADlNcDMmaHWxyLEXFDPjQFYxseb8YVm3saqvUsQ/Lz8hhcf2RuL3xyNEls7gz++phlW2m5clesP8d0rYYhAxNA5E1VpSrE1zjCi5G9izSi4MYAQb/5XxrYkIidb5FXqdRl6JQFzj66Dg1i6Pd/BBR93n4xDDdoYpU1CARAZ5Knyd1x7TQwxDMF3IKJp9W6+WOMNXx1rnWFk7cnfpOqiYSkDGBYdjKr85WdEdLDSNueSV/LAJiWgHnqMXG60wAaQLtR5+a7q4UKuNyRDDEPIkjz5etv0aRpg8G/g1nrDV4VmGBFiWSi4MQB7WyHGvBFs6maYzLrTd5S2rW/1Fo7WaYn1Lfvi/4Z8hQce/kZvl7qbrC43pIq9EnyQJXkC4CXA2fd3psHzP6rSDV9b8CmAcXrLCCHsUHBjILENjX/zNhdFJWI0e/ivwjZGIMS7/T7FF7FjUWqjX36N7Abj4cztPOpusvr0mvDZK6EuyVMXfAZe6rC5bpZ0wxdLGCSl5ahMHNcUfNIMI0LMDwU3BmINXfG6EL16gfW7F+K3LR8j+u5Vhd+JhTZqjuLGX+SIH4Y2x+VPO2P7uCgsHRABLxf1gY4AgLeLPbLyXqmc7aRPrwnfvRLdwgNwZmYsto+LwsQOdfU6l6HfgxWvmzrGGB7jw8HrmXhjyTEMWn8ek3ekYND683hjyTGF2V40w4gQyyFgGDXFRKxYfn4+RCIR8vLy4O7ubpDHSErLwaD15w1ybl29FVEdv6c8Mtj5mz+4ge/3fYXAF9I1s7JcvfDmu+uU1oXS1fDoIHQPD1A5o0M2JRnQPltN3WwnVfVa1JEVOjszM9ZgN++9KQ8xeUeKzsdvHxdllGTRxQmpWHsqXeXvBIDZ3/jV1a+RvaqV219VChYSYo7Y3r+p58ZAnhWUmLoJSt5pUZPXpFUZASPBe3/9il+2zZQHNrlO7pjV7UPeAhsA6B4egOg63ipvJFyGdNTVYKnYa7J8YASmdqoPAUw3DKFrr5Ax8z/EEgb7/tZcy8ac67/oUr9GNsMoLiJQ7fuREGJaNBWcR7JvdFl5r7DwwA1TN0eBh5MdwAA9wv2x4WwGb+f1KszDtwe+Q4c7l+Xb/qrRCJN7f4wsd+W1oXTBthx8xbodWflFWPjHP8gtUK5mrKkGS+WpsQ38XU22yrW2MvmqGDv/w9Kng1t6+wkhqlFwwxMuQxqmUCZhMGTDX/KfhQLp9GiZAJEjwgLccPTfp6zP2er+dazY9xX8X0oTVyUQYGV0fyx/Y7DK/BpvF3v0ax6IH0+ncy50KMvtSErL0TgcIAtOktJyVAY2MrKb1tLEm4ip66N2aMGUhc60lclnoLz+lrECLxlLnw5u6e0nhKhGwQ0P2C54aEovixWL6skyrUbHBKNzmD9ahXjhQnou6+Dm/64exv8OroQNI12z6qmzB6b2+ghnQpqp3H9uz1CMjAmBjVCAyCBPzPrtGp6/0r5GlIeTHf73dmMAUFotW1OlYLY3o5XHb2Pl8dsaz2XKQmey4TZ1vUemrjBr6dPBLb39hBDVKLjRE58LHhqTbGjmz+tZmNNTOoTBZRgkOTAURbb2cCktwrlaTTC593Q8dVU9bOThbCcPbADpDdvN0Q5DfvxL5f4VrRrSHC+KSjkvWMj1ZmTOix9q6z0y5XCJpa8wbentJ4SoRgnFejLEgofGUrkQHZupvTJp3jUxp+sELI0ZjKEDFqoNbADpKt6JqVkK26Jqe7MqitYy2EunBQu51q4x98UPzTWJ1dLrv1h6+wkhqlFwoydrGIuv+BxkwyBeLvbybUKJGMOS/4BDmeIMsD2NOmD5G4Mh0VK/Rpa8q0tRtMt3n7FO+KxIl9o1fFcdriosvf6LpbefEKKMhqX0ZA1j8ZWfQ7fwAMQ29EPU4iOwefwYy/d/gzb3rqJ+9j3M7fIB5/Orm3GiLZ+kW3gA9qY8ZPUYqoJMdefX5VxEM0tfYdrS208IUUTBjZ50ma5rLirnE1QuTvZDtRzU/t+H8CnMAwAMSjmI+Mg+uONdQ6fHUxeAaLqp6JvwWfH8Z28/xcrjaTqfi2hm6StMW3r7CSHlKLjRk6bpuuascj5BxansNhIxpp7Zig/O74Lw9bSqTFdvTOrzsc6BDaA+aNB0U+Ej4VN2/lYhXvgt+SEljxKjoWrGhJgG5dzwgM8FD42lYj6BbCp7Zl4R/POzsW37J5iY9Is8sHnStiMeHDuH0P49dXosbWs7acJnwicljxJjYrNeFSHEMGhtKR7Xlqr4LS0x9TH+uGqeH2IVa86IJYy8fsybaRfx3YGl8HqVDwAoFdrgm3bDsb/TIJye3QkX0nN5WS9LU00ZdVQVSdTlPHyfixBVuK5XRQhhh+39m4aleCQb/hBLGMzb94+pm6PWs8LyWU+yqext05Ox6dcF8u0P3KthUp8ZSA4MBV6U4EJ6Lqv8osqVj1XRpaYMnwmflDxKDEnbelXqlv4ghPCHghsDOH8nR6EkvrlZeTwNvyU/xLzeYSguk1YYPhvUFOdrhiPq/nUk1m2N6T2mIM/JTX7MkxdFWpcDAICVg5rB08VB57WdNOEz4ZOSR4mh0HpVhJge5dwYwDeH/jPo+fn4rifrPcnILgQASIQ2mNT7Y8ztPB7j+n2qENgA5cnA6vKLPF3sMDomGJ4uDmgV4gV/d0dWaztRTRlibWi9KkJMj3pueJZw9RGu3H9u0MfwFzkiPNAdialPdDreTlyKGSd/woGGbbHjoiP83R3wOL8YT9y88XPzXgr7qppBVHFYJzE1C3tSHiG3oAQbzmZgw9kMBIgc0T3cn1Vb6AOeWBtar4oQ06OeGx6JJQw+3Xvd4I/Tv0UNHNExsKnxPAu7ts7AuIt78P2+r1DwOBuDWtUCwG0GkY1QgLxXJYg/m4HcAsXKxVl5Rdh4NoNVe+gDnlgbbUt/yJYWoZIDhBgOBTc8upCeq3Eohi/rTt3RqZ5Ot//OImHTZERk3gIA+L7MQbNH/yHYx4Vz+Xk2SZOaUmks5QNeLGGQlJaDvSkPOU9jJ1UTlRwgxPRoWIpHxhpieVUq4bS/Q1kJPjm+ASOSD8i3pXsGYGLcLPzjVwfvujogpq4PpxlEbJImZUUG1CUfm/sHPE0ZJ7pis7QIIcRwKLjhkTkOsQQ9e4RVe5cg/HH5sgP7Qtvhk64T8dLBWbrhdeTBZQYR20BuTEwwEq5nWdwHvLo6JbpMYydVE5UcIMR0KLjhkWysncsijYbUO/Ukvjy0Em4lrwAARbb2WNDxXWxv2hUQlH/AZhcUcz4320CuU5g/PukZZlEf8FSnhPCFSg4QYhqUc8MjG6EAc3uGmboZAICA/Kf4JmGpPLBJ86qBvsO+xfaIbgqBDaBbjxOXpEnZB3xcRCCi63ibfUDApU4JIYQQ80PBDc88XexN3QQAQKZ7NSyKHQsA2N2oA3qPWIp/fUMU9tEnqdeakyapTgkhhFg2GpbimSlveAJGAkZQHq/+3Kwn7njVwNmgpkq9NXwEINaaNEl1SgghxLJRcMMzU9zwnEqKsODID8hx9sCSN0eW/0IgwNngCNk/UXGJVL4CEGtMmtS2hpaqwoaEEELMBwU3PJGtCP7o+SvYCoEybrO1OZnaqR4a+Lthwf5UuN7+D6v2LkH9nHsAgL9qhuNEnRZKxzCMdDVwHzcH3gMQa0uaZLOGlqUOuRFCSFVAwQ0PVNVDMaRgHxd0a+SPLn/9Cebz6bApkiYNF9g5wuV1ArEqPm4OiIsINEobLZ21DrkRQkhVQMGNntTVQzEkf2EZMHw4hFu2yLfdqBaMiXEzkeZdU+1xlCPCjTUOuRFCSFVAwY0eNNVDMQQBgJjCR2jV/yPgv/KVxyXvvov3A+Nwt1B1SyhHRHfWNuRGCCFVAU0F14O2eij6UJpezTAYlHIQP639EAJZYOPmBmzfDuHatZjVr5nq417/n3JECCGEVBUU3OjBENO+A0SOWD1YeRHLQDc7zHxwGjYlr6sJN28OJCcDAwcCKM8R4bL4JSGEEGKNaFhKD3znsAgAebJq13AVuR6D60mDmmHDgG++ARwcFI6nHBFCCCGEghu9aKuHwkXl1aZtBEC0iAHqVJjdVLs2cOMGEKC+F4ZyRAghhFR1NCylh4pLEOiqbT0fbB8XhTMzY8uHjp4/B/7v/4B27YCCAsUDNAQ2hBBCCKHgRm+yXJcAkW5DVG/Wr6a4mOTFi9Khp99+k/bSTJjAY2sJIYQQ60fDUjyomOuSlV+EqTtTWB87LDpY+g+GAZYvB2bMAEpLpds8PYF+/XhvLyGEEGLNqOeGJ7Jcl7eaBSpNx9bE3lYI5OYCffsCU6eWBzZRUcCVK0CfPoZoLiGEEGK1KLgxADuWV9VeCCApCYiIAPbtK//FjBnAqVNAUJAhmkcIIYRYNQpuDCDEy4nVfh9d+0OaNHz/vnSDtzdw4ACwZAlgZ2fAFhJCCCHWi3JuDKCE5cCUUFwGlJVJf3jjDWD7dqBGDQO2jBBCCLF+ZtFzs3r1aoSEhMDR0RGRkZE4ffq0xv1PnjyJyMhIODo6onbt2vjhhx+M1FJ2AtzZzZw60Wck0L07MGcOcPw4BTaEEEIID0we3OzcuRNTpkzBnDlzcOXKFbRt2xbdu3fHvXv3VO6fnp6OHj16oG3btrhy5Qo++eQTTJo0Cb/99puRW65eu3rVlLYJGAkiH6QqbGvbwA/Yvx9YtAiwpU40QgghhA8mD26+++47jBkzBmPHjkVoaCiWLVuGmjVrYs2aNSr3/+GHH1CrVi0sW7YMoaGhGDt2LEaPHo1vvvnGyC1Xz89DMefGp+AZfvplHn7ZNgut7l9X3M/GxtjNI4QQQqyaSYObkpISXL58GV26dFHY3qVLF5w7d07lMUlJSUr7d+3aFZcuXUKpbBq1iflXGJaKvnsVCfGT0C7jCmwYCZbu/xb2ZaVK+xFCCCGEHyYdC8nOzoZYLIafn5/Cdj8/P2RlZak8JisrS+X+ZWVlyM7ORoCK5QmKi4tRXFws/zk/P5+H1qvXKsQLgW52eOfPnzDp3A7YMBIAwBMXT0zvOQUltnYIEEkXtSSEEEIIv8wi0UMgUJxdxDCM0jZt+6vaLrN48WIsWLBAz1ayZ/M4C/v2fQ7vC2fl204FN8O0XtOQ4+IpX/2bVusmhBBC+GfSYSkfHx/Y2Ngo9dI8efJEqXdGxt/fX+X+tra28PZWvRr27NmzkZeXJ//vvqyujCEkJgIREfLARiwQ4uu2wzCi/wJku3jCX+SINUObly+SSQghhBBembTnxt7eHpGRkUhMTMRbb70l356YmIi4uDiVx0RHR2P//v0K2w4fPowWLVrATk3hOwcHBzg4OPDXcHVWrwYmTpSuEwUAgYHA1m14o0Yj1H9RBF836VAU9dgQQgghhmPyYalp06Zh2LBhaNGiBaKjo7Fu3Trcu3cP48ePByDtdXn48CE2b94MABg/fjxWrlyJadOmYdy4cUhKSsKGDRuwfft2Uz4NqfbtAUdH4NUraf2azZth4+ODaFO3ixBCCKlCTB7cDBgwADk5Ofj888+RmZmJ8PBwJCQkIOj1ukqZmZkKNW9CQkKQkJCAqVOnYtWqVahevTpWrFiBt99+21RPoVyjRsCaNcDjx8D06YDQ5DPtCSGEkCpHwMiycauQ/Px8iEQi5OXlwd3d3dTNIYQQQggLbO/f1LVACCGEEKtCwQ0hhBBCrAoFN4QQQgixKhTcEEIIIcSqUHBDCCGEEKtCwQ0hhBBCrAoFN4QQQgixKhTcEEIIIcSqUHBDCCGEEKtCwQ0hhBBCrAoFN4QQQgixKhTcEEIIIcSqUHBDCCGEEKtCwQ0hhBBCrIqtqRtgCgzDAJAunU4IIYQQyyC7b8vu4+pUyeDmxYsXAICaNWuauCWEEEII4erFixcQiURqfy9gtIU/VkgikeDRo0dwc3ODQCAw+OPl5+ejZs2auH//Ptzd3Q3+eEQZvQamRdff9Og1MD16DfTHMAxevHiB6tWrQyhUn1lTJXtuhEIhatSoYfTHdXd3pze0idFrYFp0/U2PXgPTo9dAP5p6bGQooZgQQgghVoWCG0IIIYRYFQpujMDBwQHz5s2Dg4ODqZtSZdFrYFp0/U2PXgPTo9fAeKpkQjEhhBBCrBf13BBCCCHEqlBwQwghhBCrQsENIYQQQqwKBTeEEEIIsSoU3PBk9erVCAkJgaOjIyIjI3H69GmN+588eRKRkZFwdHRE7dq18cMPPxippdaJy/XfvXs3OnfujGrVqsHd3R3R0dE4dOiQEVtrnbj+DcicPXsWtra2iIiIMGwDqwCur0FxcTHmzJmDoKAgODg4oE6dOti4caORWmuduL4GW7duRdOmTeHs7IyAgACMGjUKOTk5RmqtFWOI3nbs2MHY2dkx69evZ1JTU5nJkyczLi4uzN27d1Xuf+fOHcbZ2ZmZPHkyk5qayqxfv56xs7Njfv31VyO33Dpwvf6TJ09mlixZwly4cIG5efMmM3v2bMbOzo5JTk42csutB9fXQOb58+dM7dq1mS5dujBNmzY1TmOtlC6vQZ8+fZjWrVsziYmJTHp6OvPXX38xZ8+eNWKrrQvX1+D06dOMUChkli9fzty5c4c5ffo006hRI6Zv375Gbrn1oeCGB61atWLGjx+vsK1hw4bMrFmzVO4/Y8YMpmHDhgrb3nvvPSYqKspgbbRmXK+/KmFhYcyCBQv4blqVoetrMGDAAObTTz9l5s2bR8GNnri+Bn/++ScjEomYnJwcYzSvSuD6Gnz99ddM7dq1FbatWLGCqVGjhsHaWFXQsJSeSkpKcPnyZXTp0kVhe5cuXXDu3DmVxyQlJSnt37VrV1y6dAmlpaUGa6s10uX6VyaRSPDixQt4eXkZoolWT9fXID4+HmlpaZg3b56hm2j1dHkN9u3bhxYtWuCrr75CYGAg6tevj+nTp+PVq1fGaLLV0eU1aNOmDR48eICEhAQwDIPHjx/j119/Rc+ePY3RZKtWJRfO5FN2djbEYjH8/PwUtvv5+SErK0vlMVlZWSr3LysrQ3Z2NgICAgzWXmujy/Wv7Ntvv0VBQQH69+9viCZaPV1eg1u3bmHWrFk4ffo0bG3pY0hfurwGd+7cwZkzZ+Do6Ijff/8d2dnZ+OCDD5Cbm0t5NzrQ5TVo06YNtm7digEDBqCoqAhlZWXo06cPvv/+e2M02apRzw1PBAKBws8Mwyht07a/qu2EHa7XX2b79u2YP38+du7cCV9fX0M1r0pg+xqIxWIMHjwYCxYsQP369Y3VvCqBy9+BRCKBQCDA1q1b0apVK/To0QPfffcdNm3aRL03euDyGqSmpmLSpEn47LPPcPnyZRw8eBDp6ekYP368MZpq1egrk558fHxgY2OjFJk/efJEKYKX8ff3V7m/ra0tvL29DdZWa6TL9ZfZuXMnxowZg127dqFTp06GbKZV4/oavHjxApcuXcKVK1cwceJEANIbLcMwsLW1xeHDhxEbG2uUtlsLXf4OAgICEBgYCJFIJN8WGhoKhmHw4MED1KtXz6Bttja6vAaLFy9GTEwMPv74YwBAkyZN4OLigrZt22LRokXUi68H6rnRk729PSIjI5GYmKiwPTExEW3atFF5THR0tNL+hw8fRosWLWBnZ2ewtlojXa4/IO2xGTlyJLZt20bj23ri+hq4u7vj2rVrSElJkf83fvx4NGjQACkpKWjdurWxmm41dPk7iImJwaNHj/Dy5Uv5tps3b0IoFKJGjRoGba810uU1KCwshFCoeBu2sbEBUN6bT3RkqkxmayKb/rdhwwYmNTWVmTJlCuPi4sJkZGQwDMMws2bNYoYNGybfXzYVfOrUqUxqaiqzYcMGmgquB67Xf9u2bYytrS2zatUqJjMzU/7f8+fPTfUULB7X16Aymi2lP66vwYsXL5gaNWow77zzDvPPP/8wJ0+eZOrVq8eMHTvWVE/B4nF9DeLj4xlbW1tm9erVTFpaGnPmzBmmRYsWTKtWrUz1FKwGBTc8WbVqFRMUFMTY29szzZs3Z06ePCn/3YgRI5j27dsr7H/ixAmmWbNmjL29PRMcHMysWbPGyC22Llyuf/v27RkASv+NGDHC+A23Ilz/Biqi4IYfXF+DGzduMJ06dWKcnJyYGjVqMNOmTWMKCwuN3GrrwvU1WLFiBRMWFsY4OTkxAQEBzJAhQ5gHDx4YudXWR8Aw1PdFCCGEEOtBOTeEEEIIsSoU3BBCCCHEqlBwQwghhBCrQsENIYQQQqwKBTeEEEIIsSoU3BBCCCHEqlBwQwghhBCrQsENIcSsvPnmm5gyZYrJHj8jIwMCgQACgQAREREma4euRo4cKW//nj17TN0cQkyCghtCLJzsRqbuv5EjRxqlHb1791a7AGlSUhIEAgGSk5ON0hY+HDlyBEePHuXlXJs2bYKHhwcv55I5ceIEBAIBnj9/rrB9+fLlyMzM5PWxCLE0tCo4IRau4o1s586d+Oyzz/Dff//Jtzk5OSnsX1paapAFWseMGYN+/frh7t27CAoKUvjdxo0bERERgebNm/P+uIbi7e0Nb29vUzeDM5FIpLDSNyFVEfXcEGLh/P395f+JRCIIBAL5z0VFRfDw8MAvv/yCN998E46OjtiyZQvmz5+vNOSybNkyBAcHK2yLj49HaGgoHB0d0bBhQ6xevVptO3r16gVfX19s2rRJYXthYSF27tyJMWPGICcnB4MGDUKNGjXg7OyMxo0bY/v27Rqfn6rhFQ8PD4XHefjwIQYMGABPT094e3sjLi4OGRkZ8t+fOHECrVq1gouLCzw8PBATE4O7d+9qfNyKTp06BTs7O2RlZSls/+ijj9CuXTuNx544cQKjRo1CXl6evDdt/vz5AICSkhLMmDEDgYGBcHFxQevWrXHixAn5sXfv3kXv3r3h6ekJFxcXNGrUCAkJCcjIyECHDh0AAJ6enkbtoSPEElBwQ0gVMHPmTEyaNAk3btxA165dWR2zfv16zJkzB1988QVu3LiBL7/8EnPnzsVPP/2kcn9bW1sMHz4cmzZtQsUl63bt2oWSkhIMGTIERUVFiIyMxB9//IHr16/j3XffxbBhw/DXX3/p/NwKCwvRoUMHuLq64tSpUzhz5gxcXV3RrVs3lJSUoKysDH379kX79u1x9epVJCUl4d1334VAIGD9GO3atUPt2rXx888/y7eVlZVhy5YtGDVqlMZj27Rpg2XLlsHd3R2ZmZnIzMzE9OnTAQCjRo3C2bNnsWPHDly9ehX/93//h27duuHWrVsAgAkTJqC4uBinTp3CtWvXsGTJEri6uqJmzZr47bffAAD//fcfMjMzsXz5cq6XjhCrRcNShFQBU6ZMQb9+/Tgds3DhQnz77bfy40JCQpCamoq1a9dixIgRKo8ZPXo0vv76a5w4cULes7Bx40b069cPnp6e8PT0lN/YAeDDDz/EwYMHsWvXLrRu3Vqn57Zjxw4IhUL8+OOP8oAlPj4eHh4eOHHiBFq0aIG8vDz06tULderUAQCEhoZyfpwxY8YgPj4eH3/8MQDgwIEDKCwsRP/+/TUeZ29vr9CjJpOWlobt27fjwYMHqF69OgBg+vTpOHjwIOLj4/Hll1/i3r17ePvtt9G4cWMAQO3ateXHe3l5AQB8fX15z+chxNJRzw0hVUCLFi047f/06VPcv38fY8aMgaurq/y/RYsWIS0tTe1xDRs2RJs2bbBx40YA0hv46dOnMXr0aACAWCzGF198gSZNmsDb2xuurq44fPgw7t27p/Nzu3z5Mm7fvg03Nzd5O728vFBUVIS0tDR4eXlh5MiR6Nq1K3r37q1zwu3IkSNx+/ZtnD9/HoA0aOvfvz9cXFx0andycjIYhkH9+vUVrvHJkyfl13jSpElYtGgRYmJiMG/ePFy9elWnxyKkqqGeG0KqgMo3YKFQqDB0BEgTjWUkEgkA6dBU5R4VGxsbjY81ZswYTJw4EatWrUJ8fDyCgoLQsWNHAMC3336LpUuXYtmyZWjcuDFcXFwwZcoUlJSUqD2fQCDQ2tbIyEhs3bpV6dhq1aoBkPbkTJo0CQcPHsTOnTvx6aefIjExEVFRURqfS0W+vr7o3bs34uPjUbt2bSQkJCjkx3AlkUhgY2ODy5cvK11TV1dXAMDYsWPRtWtXHDhwAIcPH8bixYvx7bff4sMPP9T5cQmpCii4IaQKqlatGrKyssAwjHwoJyUlRf57Pz8/BAYG4s6dOxgyZAinc/fv3x+TJ0/Gtm3b8NNPP2HcuHHyxzh9+jTi4uIwdOhQANIb/K1btzQOE1WrVk2hp+XWrVsoLCyU/9y8eXPs3LkTvr6+cHd3V3ueZs2aoVmzZpg9ezaio6Oxbds2TsENIA02Bg4ciBo1aqBOnTqIiYlhdZy9vT3EYrFSe8RiMZ48eYK2bduqPbZmzZoYP348xo8fj9mzZ2P9+vX48MMPYW9vDwBK5yWE0LAUIVXSm2++iadPn+Krr75CWloaVq1ahT///FNhn/nz52Px4sVYvnw5bt68iWvXriE+Ph7fffedxnO7urpiwIAB+OSTT/Do0SOFWTx169ZFYmIizp07hxs3buC9995TmoFUWWxsLFauXInk5GRcunQJ48ePV5jKPmTIEPj4+CAuLg6nT59Geno6Tp48icmTJ+PBgwdIT0/H7NmzkZSUhLt37+Lw4cO4efOmTnk3Xbt2hUgkwqJFi7QmElcUHByMly9f4ujRo8jOzkZhYSHq16+PIUOGYPjw4di9ezfS09Nx8eJFLFmyBAkJCQCkuVKHDh1Ceno6kpOTcezYMXm7g4KCIBAI8Mcff+Dp06d4+fIl5+dDiNViCCFWIz4+nhGJRPKf09PTGQDMlStXlPZds2YNU7NmTcbFxYUZPnw488UXXzBBQUEK+2zdupWJiIhg7O3tGU9PT6Zdu3bM7t27tbbj3LlzDACmS5cuCttzcnKYuLg4xtXVlfH19WU+/fRTZvjw4UxcXJx8n/bt2zOTJ0+W//zw4UOmS5cujIuLC1OvXj0mISGBEYlETHx8vHyfzMxMZvjw4YyPjw/j4ODA1K5dmxk3bhyTl5fHZGVlMX379mUCAgIYe3t7JigoiPnss88YsVissu2arhnDMMzcuXMZGxsb5tGjR1qvQ0Xjx49nvL29GQDMvHnzGIZhmJKSEuazzz5jgoODGTs7u/9v7w5xFAaiMI5/o+pJSIpAEG7RG9TBAQiKUNETVHIBTlHdBI3DYUg4QQ0CggFT0+QhNmlgU8Nult0d/j/XNJ158svMm6mFYWjj8dj2+72ZmaVpasPh0IIgsG63a5PJxM7nczPmYrGwMAzNOWfT6fRhPklWFMVTNQK+cGafNrMB4I2VZanBYKDdbtf6+4XZbKbj8ajVavX64p7gnFNRFBqNRr9dCvBybEsBQIsoihRFUfN8uVy0Xq+V5/mfbuhNkqRpSAbeFSs3AHCnruvmduMgCNTv9yV99Cltt1vN53Mtl8uHb+I41mazaR0vyzJlWfajNd87nU66Xq+SpF6v9+Wj6sB/RrgBgG86HA6qqqr1XafTaS7cA/AahBsAAOAVem4AAIBXCDcAAMArhBsAAOAVwg0AAPAK4QYAAHiFcAMAALxCuAEAAF4h3AAAAK/cAPjnRA3x9wH8AAAAAElFTkSuQmCC",
      "text/plain": [
       "<Figure size 640x480 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "\n",
    "\n",
    "import numpy as np\n",
    "from sklearn.model_selection import train_test_split\n",
    "from sklearn.ensemble import RandomForestRegressor\n",
    "from sklearn.metrics import r2_score, mean_squared_error\n",
    "from scipy.stats import pearsonr\n",
    "import matplotlib.pyplot as plt\n",
    "\n",
    "input_array = []\n",
    "truth_array = []\n",
    "input_array, truth_array = prepare_data()\n",
    "input_array = np.concatenate(input_array, axis=0)\n",
    "truth_array = np.array(truth_array)/100\n",
    "\n",
    "'''\n",
    "# Find indices where NaNs are present in input_array and truth_array\n",
    "nan_indices_input = np.where(np.isnan(input_array).any(axis=(1, 2)))[0]\n",
    "nan_indices_truth = np.where(np.isnan(truth_array))[0]\n",
    "\n",
    "# Combine the indices\n",
    "nan_indices = np.union1d(nan_indices_input, nan_indices_truth)\n",
    "\n",
    "# Filter out rows with NaNs in both arrays\n",
    "input_array_filtered = np.delete(input_array, nan_indices, axis=0)\n",
    "truth_array_filtered = np.delete(truth_array, nan_indices)\n",
    "\n",
    "print(f\"Original input_array shape: {input_array.shape}\")\n",
    "print(f\"Filtered input_array shape: {input_array_filtered.shape}\")\n",
    "print(f\"Original truth_array shape: {truth_array.shape}\")\n",
    "print(f\"Filtered truth_array shape: {truth_array_filtered.shape}\")\n",
    "'''\n",
    "# featrue format[1-47]: 1: SASA, 2-6: HDMD,  7-15:ss one-hot, 16:rigidity,   17-46: HHblits, 47: log_time\n",
    "x = input_array.reshape(input_array.shape[0], -1, input_array.shape[1])\n",
    "#indices = list(range(1, 47))\n",
    "#x= np.delete(x, indices, axis=-1)\n",
    "\n",
    "y = truth_array\n",
    "x = x + 1e-10\n",
    "print(x.shape, y.shape)\n",
    "\n",
    "np.save('/Users/liyao/Desktop/Tsuda_Lab/Source_code/AI-HDX-main/HDX_MS_dataset/database_collection/feature/x_drop_duplicate.npy', x)\n",
    "np.save('/Users/liyao/Desktop/Tsuda_Lab/Source_code/AI-HDX-main/HDX_MS_dataset/database_collection/feature/y_drop_duplicate.npy', y)\n",
    "\n",
    "\n",
    "x = np.mean(x, axis=2)\n",
    "\n",
    "x_train, x_test, y_train, y_test = train_test_split(x, y, test_size=0.3, random_state=42)\n",
    "\n",
    "print('training x:', x_train.shape)\n",
    "print('training y:', y_train.shape)\n",
    "print('test x:', x_test.shape)\n",
    "print('test y:',y_test.shape)\n",
    "\n",
    "try:\n",
    "# Initialize the Random Forest Regressor\n",
    "    random_forest_model = RandomForestRegressor(n_estimators=100, random_state=50, max_features='sqrt')\n",
    "    random_forest_model.fit(x_train, y_train)\n",
    "    y_pred = random_forest_model.predict(x_test)\n",
    "\n",
    "    print('R2 score:', r2_score(y_test, y_pred))\n",
    "    print('RMSE:', np.sqrt(mean_squared_error(y_test, y_pred)))\n",
    "    print('Pearson:', pearsonr(y_test, y_pred))\n",
    "\n",
    "    plt.scatter(y_test, y_pred)\n",
    "    plt.xlabel('True Values [y_test]')\n",
    "    plt.ylabel('Predictions [y_pred]')\n",
    "    plt.title('True vs Predicted Values')\n",
    "\n",
    "    # Optional: add a line representing perfect predictions\n",
    "    max_value = max(y_test.max(), y_pred.max())\n",
    "    min_value = min(y_test.min(), y_pred.min())\n",
    "    plt.plot([min_value, max_value], [min_value, max_value], color='red', linestyle='--', lw=2)\n",
    "\n",
    "    plt.show()      \n",
    "except Exception as e:\n",
    "    print(e)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['/Users/liyao/Desktop/Tsuda_Lab/Source_code/AI-HDX-main/HDX_MS_dataset/database_collection/feature/model_files/RF_240123.joblib']"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "### save model ###\n",
    "from joblib import dump, load\n",
    "\n",
    "root_dir = \"/Users/liyao/Desktop/Tsuda_Lab/Source_code/AI-HDX-main/HDX_MS_dataset/database_collection\"\n",
    "model_path = f\"{root_dir}/feature/model_files/RF_240123.joblib\"\n",
    "dump(random_forest_model, model_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1077.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1077.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1077.h . . .\n",
      "Opened output file \"model_1077.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1078.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1078.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1078.h . . .\n",
      "Opened output file \"model_1078.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1079.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1079.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1079.h . . .\n",
      "Opened output file \"model_1079.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_108.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_108.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_108.h . . .\n",
      "Opened output file \"model_108.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1080.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1080.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1080.h . . .\n",
      "Opened output file \"model_1080.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1081.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1081.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1081.h . . .\n",
      "Opened output file \"model_1081.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1075.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1075.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1075.h . . .\n",
      "Opened output file \"model_1075.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1082.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1082.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1082.h . . .\n",
      "Opened output file \"model_1082.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1083.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1083.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1083.h . . .\n",
      "Opened output file \"model_1083.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1042.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1042.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1042.h . . .\n",
      "Opened output file \"model_1042.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1049.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1049.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1049.h . . .\n",
      "Opened output file \"model_1049.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_11.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_11.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_11.h . . .\n",
      "Opened output file \"model_11.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1043.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1043.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1043.h . . .\n",
      "Opened output file \"model_1043.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_105.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_105.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_105.h . . .\n",
      "Opened output file \"model_105.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_12.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_12.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_12.h . . .\n",
      "Opened output file \"model_12.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1044.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1044.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1044.h . . .\n",
      "Opened output file \"model_1044.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1050.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1050.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1050.h . . .\n",
      "Opened output file \"model_1050.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_13.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_13.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_13.h . . .\n",
      "Opened output file \"model_13.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1045.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1045.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1045.h . . .\n",
      "Opened output file \"model_1045.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1070.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1070.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1070.h . . .\n",
      "Opened output file \"model_1070.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_14.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_14.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_14.h . . .\n",
      "Opened output file \"model_14.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1046.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1046.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1046.h . . .\n",
      "Opened output file \"model_1046.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1071.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1071.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1071.h . . .\n",
      "Opened output file \"model_1071.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_15.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_15.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_15.h . . .\n",
      "Opened output file \"model_15.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1047.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1047.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1047.h . . .\n",
      "Opened output file \"model_1047.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1072.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1072.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1072.h . . .\n",
      "Opened output file \"model_1072.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_16.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_16.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_16.h . . .\n",
      "Opened output file \"model_16.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_104.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_104.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_104.h . . .\n",
      "Opened output file \"model_104.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1041.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1041.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1041.h . . .\n",
      "Opened output file \"model_1041.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1048.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1048.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1048.h . . .\n",
      "Opened output file \"model_1048.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1073.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1073.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1073.h . . .\n",
      "Opened output file \"model_1073.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1084.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1084.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1084.h . . .\n",
      "Opened output file \"model_1084.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1085.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1085.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1085.h . . .\n",
      "Opened output file \"model_1085.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1076.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1076.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1076.h . . .\n",
      "Opened output file \"model_1076.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1086.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1086.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1086.h . . .\n",
      "Opened output file \"model_1086.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1.h . . .\n",
      "Opened output file \"model_1.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_10.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_10.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_10.h . . .\n",
      "Opened output file \"model_10.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_100.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_100.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_100.h . . .\n",
      "Opened output file \"model_100.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1000.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1000.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1000.h . . .\n",
      "Opened output file \"model_1000.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1001.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1001.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1001.h . . .\n",
      "Opened output file \"model_1001.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1002.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1002.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1002.h . . .\n",
      "Opened output file \"model_1002.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1003.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1003.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1003.h . . .\n",
      "Opened output file \"model_1003.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1004.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1004.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1004.h . . .\n",
      "Opened output file \"model_1004.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1005.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1005.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1005.h . . .\n",
      "Opened output file \"model_1005.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1006.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1006.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1006.h . . .\n",
      "Opened output file \"model_1006.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1007.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1007.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1007.h . . .\n",
      "Opened output file \"model_1007.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1008.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1008.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1008.h . . .\n",
      "Opened output file \"model_1008.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1009.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1009.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1009.h . . .\n",
      "Opened output file \"model_1009.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_101.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_101.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_101.h . . .\n",
      "Opened output file \"model_101.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1010.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1010.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1010.h . . .\n",
      "Opened output file \"model_1010.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1011.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1011.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1011.h . . .\n",
      "Opened output file \"model_1011.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1012.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1012.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1012.h . . .\n",
      "Opened output file \"model_1012.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1013.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1013.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1013.h . . .\n",
      "Opened output file \"model_1013.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1014.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1014.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1014.h . . .\n",
      "Opened output file \"model_1014.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1015.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1015.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1015.h . . .\n",
      "Opened output file \"model_1015.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1016.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1016.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1016.h . . .\n",
      "Opened output file \"model_1016.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1017.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1017.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1017.h . . .\n",
      "Opened output file \"model_1017.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1018.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1018.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1018.h . . .\n",
      "Opened output file \"model_1018.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1019.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1019.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1019.h . . .\n",
      "Opened output file \"model_1019.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_102.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_102.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_102.h . . .\n",
      "Opened output file \"model_102.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1020.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1020.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1020.h . . .\n",
      "Opened output file \"model_1020.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1021.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1021.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1021.h . . .\n",
      "Opened output file \"model_1021.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1022.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1022.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1022.h . . .\n",
      "Opened output file \"model_1022.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1023.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1023.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1023.h . . .\n",
      "Opened output file \"model_1023.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1024.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1024.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1024.h . . .\n",
      "Opened output file \"model_1024.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1025.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1025.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1025.h . . .\n",
      "Opened output file \"model_1025.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1026.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1026.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1026.h . . .\n",
      "Opened output file \"model_1026.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1027.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1027.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1027.h . . .\n",
      "Opened output file \"model_1027.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1028.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1028.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1028.h . . .\n",
      "Opened output file \"model_1028.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1029.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1029.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1029.h . . .\n",
      "Opened output file \"model_1029.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_103.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_103.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_103.h . . .\n",
      "Opened output file \"model_103.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1030.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1030.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1030.h . . .\n",
      "Opened output file \"model_1030.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1031.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1031.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1031.h . . .\n",
      "Opened output file \"model_1031.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1032.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1032.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1032.h . . .\n",
      "Opened output file \"model_1032.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1033.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1033.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1033.h . . .\n",
      "Opened output file \"model_1033.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1034.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1034.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1034.h . . .\n",
      "Opened output file \"model_1034.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1035.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1035.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1035.h . . .\n",
      "Opened output file \"model_1035.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1036.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1036.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1036.h . . .\n",
      "Opened output file \"model_1036.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1037.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1037.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1037.h . . .\n",
      "Opened output file \"model_1037.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1038.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1038.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1038.h . . .\n",
      "Opened output file \"model_1038.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1039.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1039.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1039.h . . .\n",
      "Opened output file \"model_1039.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1040.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1040.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1040.h . . .\n",
      "Opened output file \"model_1040.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1051.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1051.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1051.h . . .\n",
      "Opened output file \"model_1051.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1052.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1052.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1052.h . . .\n",
      "Opened output file \"model_1052.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1053.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1053.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1053.h . . .\n",
      "Opened output file \"model_1053.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1054.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1054.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1054.h . . .\n",
      "Opened output file \"model_1054.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1055.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1055.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1055.h . . .\n",
      "Opened output file \"model_1055.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1056.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1056.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1056.h . . .\n",
      "Opened output file \"model_1056.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1057.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1057.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1057.h . . .\n",
      "Opened output file \"model_1057.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1058.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1058.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1058.h . . .\n",
      "Opened output file \"model_1058.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1059.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1059.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1059.h . . .\n",
      "Opened output file \"model_1059.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_106.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_106.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_106.h . . .\n",
      "Opened output file \"model_106.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1060.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1060.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1060.h . . .\n",
      "Opened output file \"model_1060.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1061.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1061.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1061.h . . .\n",
      "Opened output file \"model_1061.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1062.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1062.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1062.h . . .\n",
      "Opened output file \"model_1062.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1063.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1063.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1063.h . . .\n",
      "Opened output file \"model_1063.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1064.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1064.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1064.h . . .\n",
      "Opened output file \"model_1064.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1065.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1065.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1065.h . . .\n",
      "Opened output file \"model_1065.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1066.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1066.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1066.h . . .\n",
      "Opened output file \"model_1066.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1067.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1067.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1067.h . . .\n",
      "Opened output file \"model_1067.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1068.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1068.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1068.h . . .\n",
      "Opened output file \"model_1068.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1069.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1069.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1069.h . . .\n",
      "Opened output file \"model_1069.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_107.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_107.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_107.h . . .\n",
      "Opened output file \"model_107.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1074.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1074.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1074.h . . .\n",
      "Opened output file \"model_1074.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1087.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1087.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1087.h . . .\n",
      "Opened output file \"model_1087.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1088.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1088.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1088.h . . .\n",
      "Opened output file \"model_1088.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1089.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1089.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1089.h . . .\n",
      "Opened output file \"model_1089.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_109.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_109.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_109.h . . .\n",
      "Opened output file \"model_109.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1090.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1090.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1090.h . . .\n",
      "Opened output file \"model_1090.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1091.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1091.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1091.h . . .\n",
      "Opened output file \"model_1091.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1092.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1092.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1092.h . . .\n",
      "Opened output file \"model_1092.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1093.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1093.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1093.h . . .\n",
      "Opened output file \"model_1093.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1094.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1094.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1094.h . . .\n",
      "Opened output file \"model_1094.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1095.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1095.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1095.h . . .\n",
      "Opened output file \"model_1095.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1096.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1096.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1096.h . . .\n",
      "Opened output file \"model_1096.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1097.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1097.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1097.h . . .\n",
      "Opened output file \"model_1097.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1098.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1098.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1098.h . . .\n",
      "Opened output file \"model_1098.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1099.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1099.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1099.h . . .\n",
      "Opened output file \"model_1099.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_110.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_110.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_110.h . . .\n",
      "Opened output file \"model_110.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1100.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1100.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1100.h . . .\n",
      "Opened output file \"model_1100.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1101.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1101.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1101.h . . .\n",
      "Opened output file \"model_1101.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1102.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1102.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1102.h . . .\n",
      "Opened output file \"model_1102.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1103.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1103.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1103.h . . .\n",
      "Opened output file \"model_1103.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1104.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1104.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1104.h . . .\n",
      "Opened output file \"model_1104.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1105.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1105.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1105.h . . .\n",
      "Opened output file \"model_1105.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1106.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1106.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1106.h . . .\n",
      "Opened output file \"model_1106.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1107.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1107.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1107.h . . .\n",
      "Opened output file \"model_1107.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1108.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1108.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1108.h . . .\n",
      "Opened output file \"model_1108.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1109.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1109.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1109.h . . .\n",
      "Opened output file \"model_1109.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_111.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_111.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_111.h . . .\n",
      "Opened output file \"model_111.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1110.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1110.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1110.h . . .\n",
      "Opened output file \"model_1110.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1111.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1111.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1111.h . . .\n",
      "Opened output file \"model_1111.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1112.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1112.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1112.h . . .\n",
      "Opened output file \"model_1112.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1113.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1113.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1113.h . . .\n",
      "Opened output file \"model_1113.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1114.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1114.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1114.h . . .\n",
      "Opened output file \"model_1114.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1115.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1115.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1115.h . . .\n",
      "Opened output file \"model_1115.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1116.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1116.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1116.h . . .\n",
      "Opened output file \"model_1116.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1117.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1117.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1117.h . . .\n",
      "Opened output file \"model_1117.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1118.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1118.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1118.h . . .\n",
      "Opened output file \"model_1118.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1119.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1119.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1119.h . . .\n",
      "Opened output file \"model_1119.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_112.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_112.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_112.h . . .\n",
      "Opened output file \"model_112.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1120.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1120.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1120.h . . .\n",
      "Opened output file \"model_1120.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1121.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1121.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1121.h . . .\n",
      "Opened output file \"model_1121.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1122.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1122.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1122.h . . .\n",
      "Opened output file \"model_1122.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1123.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1123.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1123.h . . .\n",
      "Opened output file \"model_1123.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1124.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1124.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1124.h . . .\n",
      "Opened output file \"model_1124.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1125.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1125.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1125.h . . .\n",
      "Opened output file \"model_1125.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1126.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1126.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1126.h . . .\n",
      "Opened output file \"model_1126.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1127.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1127.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1127.h . . .\n",
      "Opened output file \"model_1127.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1128.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1128.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1128.h . . .\n",
      "Opened output file \"model_1128.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1129.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1129.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1129.h . . .\n",
      "Opened output file \"model_1129.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_113.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_113.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_113.h . . .\n",
      "Opened output file \"model_113.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1130.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1130.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1130.h . . .\n",
      "Opened output file \"model_1130.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1131.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1131.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1131.h . . .\n",
      "Opened output file \"model_1131.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1132.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1132.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1132.h . . .\n",
      "Opened output file \"model_1132.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1133.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1133.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1133.h . . .\n",
      "Opened output file \"model_1133.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1134.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1134.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1134.h . . .\n",
      "Opened output file \"model_1134.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1135.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1135.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1135.h . . .\n",
      "Opened output file \"model_1135.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1136.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1136.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1136.h . . .\n",
      "Opened output file \"model_1136.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1137.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1137.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1137.h . . .\n",
      "Opened output file \"model_1137.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1138.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1138.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1138.h . . .\n",
      "Opened output file \"model_1138.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1139.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1139.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1139.h . . .\n",
      "Opened output file \"model_1139.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_114.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_114.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_114.h . . .\n",
      "Opened output file \"model_114.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1140.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1140.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1140.h . . .\n",
      "Opened output file \"model_1140.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1141.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1141.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1141.h . . .\n",
      "Opened output file \"model_1141.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1142.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1142.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1142.h . . .\n",
      "Opened output file \"model_1142.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1143.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1143.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1143.h . . .\n",
      "Opened output file \"model_1143.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1144.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1144.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1144.h . . .\n",
      "Opened output file \"model_1144.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1145.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1145.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1145.h . . .\n",
      "Opened output file \"model_1145.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1146.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1146.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1146.h . . .\n",
      "Opened output file \"model_1146.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1147.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1147.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1147.h . . .\n",
      "Opened output file \"model_1147.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1148.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1148.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1148.h . . .\n",
      "Opened output file \"model_1148.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1149.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1149.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1149.h . . .\n",
      "Opened output file \"model_1149.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_115.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_115.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_115.h . . .\n",
      "Opened output file \"model_115.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1150.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1150.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1150.h . . .\n",
      "Opened output file \"model_1150.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1151.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1151.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1151.h . . .\n",
      "Opened output file \"model_1151.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1152.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1152.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1152.h . . .\n",
      "Opened output file \"model_1152.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1153.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1153.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1153.h . . .\n",
      "Opened output file \"model_1153.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1154.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1154.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1154.h . . .\n",
      "Opened output file \"model_1154.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1155.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1155.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1155.h . . .\n",
      "Opened output file \"model_1155.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1156.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1156.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1156.h . . .\n",
      "Opened output file \"model_1156.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1157.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1157.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1157.h . . .\n",
      "Opened output file \"model_1157.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1158.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1158.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1158.h . . .\n",
      "Opened output file \"model_1158.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1159.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1159.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1159.h . . .\n",
      "Opened output file \"model_1159.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_116.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_116.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_116.h . . .\n",
      "Opened output file \"model_116.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1160.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1160.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1160.h . . .\n",
      "Opened output file \"model_1160.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1161.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1161.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1161.h . . .\n",
      "Opened output file \"model_1161.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1162.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1162.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1162.h . . .\n",
      "Opened output file \"model_1162.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1163.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1163.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1163.h . . .\n",
      "Opened output file \"model_1163.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1164.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1164.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1164.h . . .\n",
      "Opened output file \"model_1164.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1165.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1165.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1165.h . . .\n",
      "Opened output file \"model_1165.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1166.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1166.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1166.h . . .\n",
      "Opened output file \"model_1166.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1167.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1167.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1167.h . . .\n",
      "Opened output file \"model_1167.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1168.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1168.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1168.h . . .\n",
      "Opened output file \"model_1168.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1169.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1169.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1169.h . . .\n",
      "Opened output file \"model_1169.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_117.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_117.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_117.h . . .\n",
      "Opened output file \"model_117.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1170.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1170.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1170.h . . .\n",
      "Opened output file \"model_1170.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1171.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1171.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1171.h . . .\n",
      "Opened output file \"model_1171.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1172.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1172.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1172.h . . .\n",
      "Opened output file \"model_1172.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1173.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1173.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1173.h . . .\n",
      "Opened output file \"model_1173.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1174.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1174.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1174.h . . .\n",
      "Opened output file \"model_1174.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1175.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1175.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1175.h . . .\n",
      "Opened output file \"model_1175.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1176.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1176.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1176.h . . .\n",
      "Opened output file \"model_1176.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1177.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1177.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1177.h . . .\n",
      "Opened output file \"model_1177.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1178.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1178.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1178.h . . .\n",
      "Opened output file \"model_1178.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1179.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1179.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1179.h . . .\n",
      "Opened output file \"model_1179.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_118.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_118.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_118.h . . .\n",
      "Opened output file \"model_118.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1180.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1180.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1180.h . . .\n",
      "Opened output file \"model_1180.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1181.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1181.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1181.h . . .\n",
      "Opened output file \"model_1181.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1182.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1182.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1182.h . . .\n",
      "Opened output file \"model_1182.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1183.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1183.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1183.h . . .\n",
      "Opened output file \"model_1183.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1184.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1184.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1184.h . . .\n",
      "Opened output file \"model_1184.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1185.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1185.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1185.h . . .\n",
      "Opened output file \"model_1185.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1186.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1186.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1186.h . . .\n",
      "Opened output file \"model_1186.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1187.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1187.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1187.h . . .\n",
      "Opened output file \"model_1187.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1188.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1188.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1188.h . . .\n",
      "Opened output file \"model_1188.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1189.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1189.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1189.h . . .\n",
      "Opened output file \"model_1189.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_119.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_119.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_119.h . . .\n",
      "Opened output file \"model_119.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1190.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1190.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1190.h . . .\n",
      "Opened output file \"model_1190.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1191.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1191.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1191.h . . .\n",
      "Opened output file \"model_1191.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1192.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1192.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1192.h . . .\n",
      "Opened output file \"model_1192.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1193.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1193.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1193.h . . .\n",
      "Opened output file \"model_1193.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1194.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1194.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1194.h . . .\n",
      "Opened output file \"model_1194.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1195.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1195.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1195.h . . .\n",
      "Opened output file \"model_1195.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1196.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1196.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1196.h . . .\n",
      "Opened output file \"model_1196.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1197.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1197.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1197.h . . .\n",
      "Opened output file \"model_1197.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1198.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1198.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1198.h . . .\n",
      "Opened output file \"model_1198.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1199.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1199.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1199.h . . .\n",
      "Opened output file \"model_1199.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_120.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_120.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_120.h . . .\n",
      "Opened output file \"model_120.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1200.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1200.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1200.h . . .\n",
      "Opened output file \"model_1200.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1201.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1201.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1201.h . . .\n",
      "Opened output file \"model_1201.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1202.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1202.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1202.h . . .\n",
      "Opened output file \"model_1202.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1203.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1203.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1203.h . . .\n",
      "Opened output file \"model_1203.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1204.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1204.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1204.h . . .\n",
      "Opened output file \"model_1204.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1205.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1205.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1205.h . . .\n",
      "Opened output file \"model_1205.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1206.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1206.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1206.h . . .\n",
      "Opened output file \"model_1206.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1207.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1207.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1207.h . . .\n",
      "Opened output file \"model_1207.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1208.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1208.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1208.h . . .\n",
      "Opened output file \"model_1208.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1209.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1209.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1209.h . . .\n",
      "Opened output file \"model_1209.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_121.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_121.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_121.h . . .\n",
      "Opened output file \"model_121.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1210.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1210.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1210.h . . .\n",
      "Opened output file \"model_1210.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1211.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1211.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1211.h . . .\n",
      "Opened output file \"model_1211.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1212.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1212.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1212.h . . .\n",
      "Opened output file \"model_1212.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1213.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1213.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1213.h . . .\n",
      "Opened output file \"model_1213.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1214.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1214.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1214.h . . .\n",
      "Opened output file \"model_1214.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1215.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1215.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1215.h . . .\n",
      "Opened output file \"model_1215.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1216.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1216.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1216.h . . .\n",
      "Opened output file \"model_1216.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1217.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1217.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1217.h . . .\n",
      "Opened output file \"model_1217.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1218.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1218.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1218.h . . .\n",
      "Opened output file \"model_1218.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1219.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1219.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1219.h . . .\n",
      "Opened output file \"model_1219.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_122.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_122.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_122.h . . .\n",
      "Opened output file \"model_122.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1220.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1220.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1220.h . . .\n",
      "Opened output file \"model_1220.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1221.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1221.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1221.h . . .\n",
      "Opened output file \"model_1221.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1222.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1222.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1222.h . . .\n",
      "Opened output file \"model_1222.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1223.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1223.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1223.h . . .\n",
      "Opened output file \"model_1223.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1224.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1224.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1224.h . . .\n",
      "Opened output file \"model_1224.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1225.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1225.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1225.h . . .\n",
      "Opened output file \"model_1225.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1226.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1226.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1226.h . . .\n",
      "Opened output file \"model_1226.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1227.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1227.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1227.h . . .\n",
      "Opened output file \"model_1227.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1228.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1228.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1228.h . . .\n",
      "Opened output file \"model_1228.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1229.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1229.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1229.h . . .\n",
      "Opened output file \"model_1229.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_123.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_123.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_123.h . . .\n",
      "Opened output file \"model_123.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1230.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1230.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1230.h . . .\n",
      "Opened output file \"model_1230.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1231.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1231.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1231.h . . .\n",
      "Opened output file \"model_1231.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1232.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1232.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1232.h . . .\n",
      "Opened output file \"model_1232.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1233.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1233.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1233.h . . .\n",
      "Opened output file \"model_1233.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1234.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1234.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1234.h . . .\n",
      "Opened output file \"model_1234.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1235.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1235.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1235.h . . .\n",
      "Opened output file \"model_1235.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1236.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1236.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1236.h . . .\n",
      "Opened output file \"model_1236.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1237.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1237.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1237.h . . .\n",
      "Opened output file \"model_1237.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1238.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1238.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1238.h . . .\n",
      "Opened output file \"model_1238.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1239.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1239.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1239.h . . .\n",
      "Opened output file \"model_1239.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_124.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_124.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_124.h . . .\n",
      "Opened output file \"model_124.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1240.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1240.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1240.h . . .\n",
      "Opened output file \"model_1240.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1241.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1241.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1241.h . . .\n",
      "Opened output file \"model_1241.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1242.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1242.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1242.h . . .\n",
      "Opened output file \"model_1242.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1243.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1243.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1243.h . . .\n",
      "Opened output file \"model_1243.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1244.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1244.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1244.h . . .\n",
      "Opened output file \"model_1244.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1245.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1245.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1245.h . . .\n",
      "Opened output file \"model_1245.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1246.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1246.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1246.h . . .\n",
      "Opened output file \"model_1246.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1247.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1247.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1247.h . . .\n",
      "Opened output file \"model_1247.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1248.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1248.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1248.h . . .\n",
      "Opened output file \"model_1248.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1249.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1249.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1249.h . . .\n",
      "Opened output file \"model_1249.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_125.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_125.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_125.h . . .\n",
      "Opened output file \"model_125.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1250.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1250.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1250.h . . .\n",
      "Opened output file \"model_1250.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1251.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1251.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1251.h . . .\n",
      "Opened output file \"model_1251.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1252.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1252.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1252.h . . .\n",
      "Opened output file \"model_1252.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1253.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1253.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1253.h . . .\n",
      "Opened output file \"model_1253.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1254.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1254.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1254.h . . .\n",
      "Opened output file \"model_1254.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1255.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1255.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1255.h . . .\n",
      "Opened output file \"model_1255.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1256.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1256.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1256.h . . .\n",
      "Opened output file \"model_1256.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1257.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1257.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1257.h . . .\n",
      "Opened output file \"model_1257.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1258.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1258.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1258.h . . .\n",
      "Opened output file \"model_1258.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1259.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1259.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1259.h . . .\n",
      "Opened output file \"model_1259.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_126.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_126.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_126.h . . .\n",
      "Opened output file \"model_126.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1260.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1260.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1260.h . . .\n",
      "Opened output file \"model_1260.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1261.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1261.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1261.h . . .\n",
      "Opened output file \"model_1261.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1262.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1262.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1262.h . . .\n",
      "Opened output file \"model_1262.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1263.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1263.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1263.h . . .\n",
      "Opened output file \"model_1263.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1264.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1264.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1264.h . . .\n",
      "Opened output file \"model_1264.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1265.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1265.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1265.h . . .\n",
      "Opened output file \"model_1265.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1266.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1266.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1266.h . . .\n",
      "Opened output file \"model_1266.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1267.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1267.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1267.h . . .\n",
      "Opened output file \"model_1267.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1268.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1268.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1268.h . . .\n",
      "Opened output file \"model_1268.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1269.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1269.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1269.h . . .\n",
      "Opened output file \"model_1269.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_127.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_127.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_127.h . . .\n",
      "Opened output file \"model_127.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1270.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1270.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1270.h . . .\n",
      "Opened output file \"model_1270.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1271.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1271.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1271.h . . .\n",
      "Opened output file \"model_1271.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1272.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1272.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1272.h . . .\n",
      "Opened output file \"model_1272.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1273.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1273.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1273.h . . .\n",
      "Opened output file \"model_1273.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1274.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1274.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1274.h . . .\n",
      "Opened output file \"model_1274.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1275.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1275.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1275.h . . .\n",
      "Opened output file \"model_1275.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1276.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1276.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1276.h . . .\n",
      "Opened output file \"model_1276.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1277.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1277.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1277.h . . .\n",
      "Opened output file \"model_1277.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1278.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1278.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1278.h . . .\n",
      "Opened output file \"model_1278.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1279.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1279.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1279.h . . .\n",
      "Opened output file \"model_1279.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_128.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_128.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_128.h . . .\n",
      "Opened output file \"model_128.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1280.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1280.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1280.h . . .\n",
      "Opened output file \"model_1280.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1281.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1281.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1281.h . . .\n",
      "Opened output file \"model_1281.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1282.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1282.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1282.h . . .\n",
      "Opened output file \"model_1282.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1283.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1283.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1283.h . . .\n",
      "Opened output file \"model_1283.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1284.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1284.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1284.h . . .\n",
      "Opened output file \"model_1284.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1285.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1285.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1285.h . . .\n",
      "Opened output file \"model_1285.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1286.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1286.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1286.h . . .\n",
      "Opened output file \"model_1286.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1287.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1287.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1287.h . . .\n",
      "Opened output file \"model_1287.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1288.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1288.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1288.h . . .\n",
      "Opened output file \"model_1288.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1289.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1289.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1289.h . . .\n",
      "Opened output file \"model_1289.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_129.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_129.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_129.h . . .\n",
      "Opened output file \"model_129.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1290.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1290.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1290.h . . .\n",
      "Opened output file \"model_1290.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1291.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1291.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1291.h . . .\n",
      "Opened output file \"model_1291.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1292.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1292.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1292.h . . .\n",
      "Opened output file \"model_1292.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1293.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1293.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1293.h . . .\n",
      "Opened output file \"model_1293.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1294.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1294.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1294.h . . .\n",
      "Opened output file \"model_1294.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1295.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1295.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1295.h . . .\n",
      "Opened output file \"model_1295.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1296.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1296.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1296.h . . .\n",
      "Opened output file \"model_1296.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1297.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1297.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1297.h . . .\n",
      "Opened output file \"model_1297.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1298.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1298.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1298.h . . .\n",
      "Opened output file \"model_1298.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1299.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1299.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1299.h . . .\n",
      "Opened output file \"model_1299.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_130.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_130.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_130.h . . .\n",
      "Opened output file \"model_130.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1300.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1300.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1300.h . . .\n",
      "Opened output file \"model_1300.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1301.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1301.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1301.h . . .\n",
      "Opened output file \"model_1301.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1302.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1302.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1302.h . . .\n",
      "Opened output file \"model_1302.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1303.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1303.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1303.h . . .\n",
      "Opened output file \"model_1303.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1304.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1304.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1304.h . . .\n",
      "Opened output file \"model_1304.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1305.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1305.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1305.h . . .\n",
      "Opened output file \"model_1305.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1306.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1306.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1306.h . . .\n",
      "Opened output file \"model_1306.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1307.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1307.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1307.h . . .\n",
      "Opened output file \"model_1307.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1308.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1308.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1308.h . . .\n",
      "Opened output file \"model_1308.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1309.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1309.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1309.h . . .\n",
      "Opened output file \"model_1309.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_131.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_131.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_131.h . . .\n",
      "Opened output file \"model_131.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1310.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1310.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1310.h . . .\n",
      "Opened output file \"model_1310.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1311.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1311.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1311.h . . .\n",
      "Opened output file \"model_1311.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1312.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1312.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1312.h . . .\n",
      "Opened output file \"model_1312.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1313.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1313.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1313.h . . .\n",
      "Opened output file \"model_1313.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1314.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1314.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1314.h . . .\n",
      "Opened output file \"model_1314.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1315.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1315.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1315.h . . .\n",
      "Opened output file \"model_1315.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1316.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1316.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1316.h . . .\n",
      "Opened output file \"model_1316.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1317.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1317.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1317.h . . .\n",
      "Opened output file \"model_1317.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1318.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1318.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1318.h . . .\n",
      "Opened output file \"model_1318.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1319.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1319.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1319.h . . .\n",
      "Opened output file \"model_1319.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_132.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_132.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_132.h . . .\n",
      "Opened output file \"model_132.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1320.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1320.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1320.h . . .\n",
      "Opened output file \"model_1320.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1321.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1321.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1321.h . . .\n",
      "Opened output file \"model_1321.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1322.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1322.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1322.h . . .\n",
      "Opened output file \"model_1322.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1323.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1323.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1323.h . . .\n",
      "Opened output file \"model_1323.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1324.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1324.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1324.h . . .\n",
      "Opened output file \"model_1324.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1325.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1325.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1325.h . . .\n",
      "Opened output file \"model_1325.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1326.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1326.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1326.h . . .\n",
      "Opened output file \"model_1326.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1327.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1327.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1327.h . . .\n",
      "Opened output file \"model_1327.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1328.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1328.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1328.h . . .\n",
      "Opened output file \"model_1328.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1329.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1329.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1329.h . . .\n",
      "Opened output file \"model_1329.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_133.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_133.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_133.h . . .\n",
      "Opened output file \"model_133.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1330.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1330.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1330.h . . .\n",
      "Opened output file \"model_1330.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1331.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1331.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1331.h . . .\n",
      "Opened output file \"model_1331.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1332.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1332.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1332.h . . .\n",
      "Opened output file \"model_1332.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1333.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1333.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1333.h . . .\n",
      "Opened output file \"model_1333.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1334.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1334.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1334.h . . .\n",
      "Opened output file \"model_1334.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1335.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1335.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1335.h . . .\n",
      "Opened output file \"model_1335.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1336.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1336.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1336.h . . .\n",
      "Opened output file \"model_1336.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1337.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1337.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1337.h . . .\n",
      "Opened output file \"model_1337.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1338.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1338.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1338.h . . .\n",
      "Opened output file \"model_1338.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1339.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1339.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1339.h . . .\n",
      "Opened output file \"model_1339.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_134.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_134.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_134.h . . .\n",
      "Opened output file \"model_134.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1340.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1340.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1340.h . . .\n",
      "Opened output file \"model_1340.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1341.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1341.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1341.h . . .\n",
      "Opened output file \"model_1341.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1342.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1342.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1342.h . . .\n",
      "Opened output file \"model_1342.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1343.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1343.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1343.h . . .\n",
      "Opened output file \"model_1343.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1344.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1344.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1344.h . . .\n",
      "Opened output file \"model_1344.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1345.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1345.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1345.h . . .\n",
      "Opened output file \"model_1345.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1346.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1346.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1346.h . . .\n",
      "Opened output file \"model_1346.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1347.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1347.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1347.h . . .\n",
      "Opened output file \"model_1347.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1348.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1348.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1348.h . . .\n",
      "Opened output file \"model_1348.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1349.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1349.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1349.h . . .\n",
      "Opened output file \"model_1349.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_135.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_135.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_135.h . . .\n",
      "Opened output file \"model_135.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1350.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1350.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1350.h . . .\n",
      "Opened output file \"model_1350.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1351.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1351.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1351.h . . .\n",
      "Opened output file \"model_1351.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1352.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1352.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1352.h . . .\n",
      "Opened output file \"model_1352.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1353.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1353.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1353.h . . .\n",
      "Opened output file \"model_1353.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1354.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1354.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1354.h . . .\n",
      "Opened output file \"model_1354.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1355.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1355.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1355.h . . .\n",
      "Opened output file \"model_1355.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1356.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1356.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1356.h . . .\n",
      "Opened output file \"model_1356.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1357.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1357.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1357.h . . .\n",
      "Opened output file \"model_1357.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1358.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1358.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1358.h . . .\n",
      "Opened output file \"model_1358.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1359.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1359.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1359.h . . .\n",
      "Opened output file \"model_1359.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_136.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_136.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_136.h . . .\n",
      "Opened output file \"model_136.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1360.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1360.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1360.h . . .\n",
      "Opened output file \"model_1360.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1361.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1361.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1361.h . . .\n",
      "Opened output file \"model_1361.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1362.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1362.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1362.h . . .\n",
      "Opened output file \"model_1362.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1363.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1363.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1363.h . . .\n",
      "Opened output file \"model_1363.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1364.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1364.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1364.h . . .\n",
      "Opened output file \"model_1364.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1365.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1365.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1365.h . . .\n",
      "Opened output file \"model_1365.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1366.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1366.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1366.h . . .\n",
      "Opened output file \"model_1366.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1367.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1367.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1367.h . . .\n",
      "Opened output file \"model_1367.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1368.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1368.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1368.h . . .\n",
      "Opened output file \"model_1368.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1369.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1369.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1369.h . . .\n",
      "Opened output file \"model_1369.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_137.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_137.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_137.h . . .\n",
      "Opened output file \"model_137.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1370.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1370.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1370.h . . .\n",
      "Opened output file \"model_1370.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1371.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1371.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1371.h . . .\n",
      "Opened output file \"model_1371.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1372.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1372.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1372.h . . .\n",
      "Opened output file \"model_1372.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1373.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1373.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1373.h . . .\n",
      "Opened output file \"model_1373.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1374.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1374.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1374.h . . .\n",
      "Opened output file \"model_1374.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1375.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1375.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1375.h . . .\n",
      "Opened output file \"model_1375.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1376.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1376.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1376.h . . .\n",
      "Opened output file \"model_1376.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1377.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1377.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1377.h . . .\n",
      "Opened output file \"model_1377.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1378.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1378.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1378.h . . .\n",
      "Opened output file \"model_1378.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1379.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1379.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1379.h . . .\n",
      "Opened output file \"model_1379.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_138.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_138.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_138.h . . .\n",
      "Opened output file \"model_138.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1380.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1380.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1380.h . . .\n",
      "Opened output file \"model_1380.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1381.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1381.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1381.h . . .\n",
      "Opened output file \"model_1381.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1382.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1382.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1382.h . . .\n",
      "Opened output file \"model_1382.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1383.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1383.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1383.h . . .\n",
      "Opened output file \"model_1383.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1384.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1384.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1384.h . . .\n",
      "Opened output file \"model_1384.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1385.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1385.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1385.h . . .\n",
      "Opened output file \"model_1385.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1386.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1386.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1386.h . . .\n",
      "Opened output file \"model_1386.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1387.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1387.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1387.h . . .\n",
      "Opened output file \"model_1387.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1388.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1388.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1388.h . . .\n",
      "Opened output file \"model_1388.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1389.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1389.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1389.h . . .\n",
      "Opened output file \"model_1389.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_139.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_139.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_139.h . . .\n",
      "Opened output file \"model_139.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1390.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1390.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1390.h . . .\n",
      "Opened output file \"model_1390.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1391.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1391.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1391.h . . .\n",
      "Opened output file \"model_1391.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1392.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1392.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1392.h . . .\n",
      "Opened output file \"model_1392.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1393.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1393.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1393.h . . .\n",
      "Opened output file \"model_1393.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1394.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1394.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1394.h . . .\n",
      "Opened output file \"model_1394.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1395.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1395.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1395.h . . .\n",
      "Opened output file \"model_1395.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1396.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1396.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1396.h . . .\n",
      "Opened output file \"model_1396.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1397.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1397.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1397.h . . .\n",
      "Opened output file \"model_1397.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1398.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1398.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1398.h . . .\n",
      "Opened output file \"model_1398.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1399.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1399.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1399.h . . .\n",
      "Opened output file \"model_1399.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_140.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_140.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_140.h . . .\n",
      "Opened output file \"model_140.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1400.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1400.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1400.h . . .\n",
      "Opened output file \"model_1400.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1401.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1401.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1401.h . . .\n",
      "Opened output file \"model_1401.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1402.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1402.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1402.h . . .\n",
      "Opened output file \"model_1402.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1403.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1403.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1403.h . . .\n",
      "Opened output file \"model_1403.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1404.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1404.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1404.h . . .\n",
      "Opened output file \"model_1404.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1405.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1405.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1405.h . . .\n",
      "Opened output file \"model_1405.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1406.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1406.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1406.h . . .\n",
      "Opened output file \"model_1406.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1407.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1407.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1407.h . . .\n",
      "Opened output file \"model_1407.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1408.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1408.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1408.h . . .\n",
      "Opened output file \"model_1408.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1409.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1409.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1409.h . . .\n",
      "Opened output file \"model_1409.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_141.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_141.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_141.h . . .\n",
      "Opened output file \"model_141.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1410.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1410.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1410.h . . .\n",
      "Opened output file \"model_1410.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1411.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1411.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1411.h . . .\n",
      "Opened output file \"model_1411.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1412.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1412.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1412.h . . .\n",
      "Opened output file \"model_1412.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1413.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1413.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1413.h . . .\n",
      "Opened output file \"model_1413.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1414.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1414.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1414.h . . .\n",
      "Opened output file \"model_1414.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1415.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1415.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1415.h . . .\n",
      "Opened output file \"model_1415.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1416.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1416.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1416.h . . .\n",
      "Opened output file \"model_1416.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1417.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1417.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1417.h . . .\n",
      "Opened output file \"model_1417.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1418.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1418.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1418.h . . .\n",
      "Opened output file \"model_1418.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1419.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1419.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1419.h . . .\n",
      "Opened output file \"model_1419.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_142.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_142.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_142.h . . .\n",
      "Opened output file \"model_142.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1420.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1420.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1420.h . . .\n",
      "Opened output file \"model_1420.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1421.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1421.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1421.h . . .\n",
      "Opened output file \"model_1421.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1422.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1422.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1422.h . . .\n",
      "Opened output file \"model_1422.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1423.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1423.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1423.h . . .\n",
      "Opened output file \"model_1423.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1424.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1424.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1424.h . . .\n",
      "Opened output file \"model_1424.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1425.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1425.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1425.h . . .\n",
      "Opened output file \"model_1425.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1426.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1426.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1426.h . . .\n",
      "Opened output file \"model_1426.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1427.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1427.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1427.h . . .\n",
      "Opened output file \"model_1427.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1428.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1428.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1428.h . . .\n",
      "Opened output file \"model_1428.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1429.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1429.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1429.h . . .\n",
      "Opened output file \"model_1429.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_143.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_143.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_143.h . . .\n",
      "Opened output file \"model_143.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1430.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1430.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1430.h . . .\n",
      "Opened output file \"model_1430.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1431.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1431.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1431.h . . .\n",
      "Opened output file \"model_1431.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1432.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1432.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1432.h . . .\n",
      "Opened output file \"model_1432.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1433.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1433.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1433.h . . .\n",
      "Opened output file \"model_1433.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1434.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1434.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1434.h . . .\n",
      "Opened output file \"model_1434.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1435.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1435.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1435.h . . .\n",
      "Opened output file \"model_1435.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1436.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1436.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1436.h . . .\n",
      "Opened output file \"model_1436.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1437.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1437.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1437.h . . .\n",
      "Opened output file \"model_1437.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1438.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1438.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1438.h . . .\n",
      "Opened output file \"model_1438.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1439.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1439.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1439.h . . .\n",
      "Opened output file \"model_1439.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_144.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_144.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_144.h . . .\n",
      "Opened output file \"model_144.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1440.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1440.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1440.h . . .\n",
      "Opened output file \"model_1440.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1441.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1441.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1441.h . . .\n",
      "Opened output file \"model_1441.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1442.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1442.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1442.h . . .\n",
      "Opened output file \"model_1442.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1443.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1443.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1443.h . . .\n",
      "Opened output file \"model_1443.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1444.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1444.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1444.h . . .\n",
      "Opened output file \"model_1444.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1445.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1445.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1445.h . . .\n",
      "Opened output file \"model_1445.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1446.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1446.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1446.h . . .\n",
      "Opened output file \"model_1446.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1447.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1447.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1447.h . . .\n",
      "Opened output file \"model_1447.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1448.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1448.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1448.h . . .\n",
      "Opened output file \"model_1448.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1449.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1449.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1449.h . . .\n",
      "Opened output file \"model_1449.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_145.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_145.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_145.h . . .\n",
      "Opened output file \"model_145.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1450.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1450.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1450.h . . .\n",
      "Opened output file \"model_1450.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1451.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1451.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1451.h . . .\n",
      "Opened output file \"model_1451.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1452.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1452.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1452.h . . .\n",
      "Opened output file \"model_1452.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1453.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1453.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1453.h . . .\n",
      "Opened output file \"model_1453.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1454.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1454.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1454.h . . .\n",
      "Opened output file \"model_1454.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1455.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1455.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1455.h . . .\n",
      "Opened output file \"model_1455.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1456.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1456.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1456.h . . .\n",
      "Opened output file \"model_1456.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1457.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1457.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1457.h . . .\n",
      "Opened output file \"model_1457.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1458.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1458.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1458.h . . .\n",
      "Opened output file \"model_1458.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1459.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1459.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1459.h . . .\n",
      "Opened output file \"model_1459.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_146.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_146.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_146.h . . .\n",
      "Opened output file \"model_146.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1460.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1460.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1460.h . . .\n",
      "Opened output file \"model_1460.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1461.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1461.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1461.h . . .\n",
      "Opened output file \"model_1461.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1462.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1462.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1462.h . . .\n",
      "Opened output file \"model_1462.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1463.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1463.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1463.h . . .\n",
      "Opened output file \"model_1463.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1464.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1464.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1464.h . . .\n",
      "Opened output file \"model_1464.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1465.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1465.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1465.h . . .\n",
      "Opened output file \"model_1465.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1466.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1466.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1466.h . . .\n",
      "Opened output file \"model_1466.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1467.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1467.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1467.h . . .\n",
      "Opened output file \"model_1467.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1468.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1468.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1468.h . . .\n",
      "Opened output file \"model_1468.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1469.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1469.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1469.h . . .\n",
      "Opened output file \"model_1469.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_147.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_147.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_147.h . . .\n",
      "Opened output file \"model_147.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1470.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1470.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1470.h . . .\n",
      "Opened output file \"model_1470.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1471.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1471.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1471.h . . .\n",
      "Opened output file \"model_1471.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1472.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1472.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1472.h . . .\n",
      "Opened output file \"model_1472.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1473.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1473.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1473.h . . .\n",
      "Opened output file \"model_1473.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1474.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1474.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1474.h . . .\n",
      "Opened output file \"model_1474.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1475.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1475.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1475.h . . .\n",
      "Opened output file \"model_1475.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1476.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1476.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1476.h . . .\n",
      "Opened output file \"model_1476.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1477.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1477.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1477.h . . .\n",
      "Opened output file \"model_1477.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1478.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1478.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1478.h . . .\n",
      "Opened output file \"model_1478.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1479.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1479.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1479.h . . .\n",
      "Opened output file \"model_1479.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_148.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_148.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_148.h . . .\n",
      "Opened output file \"model_148.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1480.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1480.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1480.h . . .\n",
      "Opened output file \"model_1480.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1481.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1481.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1481.h . . .\n",
      "Opened output file \"model_1481.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1482.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1482.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1482.h . . .\n",
      "Opened output file \"model_1482.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1483.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1483.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1483.h . . .\n",
      "Opened output file \"model_1483.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1484.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1484.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1484.h . . .\n",
      "Opened output file \"model_1484.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1485.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1485.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1485.h . . .\n",
      "Opened output file \"model_1485.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1486.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1486.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1486.h . . .\n",
      "Opened output file \"model_1486.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1487.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1487.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1487.h . . .\n",
      "Opened output file \"model_1487.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1488.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1488.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1488.h . . .\n",
      "Opened output file \"model_1488.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1489.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1489.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1489.h . . .\n",
      "Opened output file \"model_1489.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_149.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_149.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_149.h . . .\n",
      "Opened output file \"model_149.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1490.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1490.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1490.h . . .\n",
      "Opened output file \"model_1490.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1491.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1491.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1491.h . . .\n",
      "Opened output file \"model_1491.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1492.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1492.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1492.h . . .\n",
      "Opened output file \"model_1492.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1493.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1493.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1493.h . . .\n",
      "Opened output file \"model_1493.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1494.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1494.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1494.h . . .\n",
      "Opened output file \"model_1494.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1495.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1495.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1495.h . . .\n",
      "Opened output file \"model_1495.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1496.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1496.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1496.h . . .\n",
      "Opened output file \"model_1496.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1497.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1497.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1497.h . . .\n",
      "Opened output file \"model_1497.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1498.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1498.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1498.h . . .\n",
      "Opened output file \"model_1498.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1499.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1499.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1499.h . . .\n",
      "Opened output file \"model_1499.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_150.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_150.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_150.h . . .\n",
      "Opened output file \"model_150.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1500.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1500.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1500.h . . .\n",
      "Opened output file \"model_1500.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1501.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1501.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1501.h . . .\n",
      "Opened output file \"model_1501.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1502.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1502.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1502.h . . .\n",
      "Opened output file \"model_1502.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1503.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1503.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1503.h . . .\n",
      "Opened output file \"model_1503.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1504.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1504.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1504.h . . .\n",
      "Opened output file \"model_1504.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1505.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1505.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1505.h . . .\n",
      "Opened output file \"model_1505.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1506.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1506.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1506.h . . .\n",
      "Opened output file \"model_1506.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1507.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1507.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1507.h . . .\n",
      "Opened output file \"model_1507.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1508.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1508.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1508.h . . .\n",
      "Opened output file \"model_1508.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1509.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1509.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1509.h . . .\n",
      "Opened output file \"model_1509.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_151.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_151.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_151.h . . .\n",
      "Opened output file \"model_151.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1510.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1510.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1510.h . . .\n",
      "Opened output file \"model_1510.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1511.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1511.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1511.h . . .\n",
      "Opened output file \"model_1511.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1512.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1512.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1512.h . . .\n",
      "Opened output file \"model_1512.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1513.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1513.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1513.h . . .\n",
      "Opened output file \"model_1513.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1514.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1514.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1514.h . . .\n",
      "Opened output file \"model_1514.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1515.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1515.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1515.h . . .\n",
      "Opened output file \"model_1515.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1516.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1516.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1516.h . . .\n",
      "Opened output file \"model_1516.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1517.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1517.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1517.h . . .\n",
      "Opened output file \"model_1517.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1518.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1518.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1518.h . . .\n",
      "Opened output file \"model_1518.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1519.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1519.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1519.h . . .\n",
      "Opened output file \"model_1519.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_152.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_152.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_152.h . . .\n",
      "Opened output file \"model_152.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1520.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1520.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1520.h . . .\n",
      "Opened output file \"model_1520.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1521.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1521.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1521.h . . .\n",
      "Opened output file \"model_1521.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1522.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1522.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1522.h . . .\n",
      "Opened output file \"model_1522.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1523.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1523.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1523.h . . .\n",
      "Opened output file \"model_1523.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1524.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1524.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1524.h . . .\n",
      "Opened output file \"model_1524.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1525.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1525.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1525.h . . .\n",
      "Opened output file \"model_1525.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1526.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1526.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1526.h . . .\n",
      "Opened output file \"model_1526.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1527.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1527.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1527.h . . .\n",
      "Opened output file \"model_1527.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1528.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1528.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1528.h . . .\n",
      "Opened output file \"model_1528.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1529.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1529.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1529.h . . .\n",
      "Opened output file \"model_1529.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_153.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_153.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_153.h . . .\n",
      "Opened output file \"model_153.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1530.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1530.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1530.h . . .\n",
      "Opened output file \"model_1530.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1531.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1531.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1531.h . . .\n",
      "Opened output file \"model_1531.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1532.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1532.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1532.h . . .\n",
      "Opened output file \"model_1532.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1533.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1533.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1533.h . . .\n",
      "Opened output file \"model_1533.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1534.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1534.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1534.h . . .\n",
      "Opened output file \"model_1534.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1535.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1535.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1535.h . . .\n",
      "Opened output file \"model_1535.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1536.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1536.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1536.h . . .\n",
      "Opened output file \"model_1536.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1537.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1537.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1537.h . . .\n",
      "Opened output file \"model_1537.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1538.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1538.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1538.h . . .\n",
      "Opened output file \"model_1538.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1539.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1539.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1539.h . . .\n",
      "Opened output file \"model_1539.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_154.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_154.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_154.h . . .\n",
      "Opened output file \"model_154.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1540.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1540.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1540.h . . .\n",
      "Opened output file \"model_1540.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1541.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1541.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1541.h . . .\n",
      "Opened output file \"model_1541.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1542.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1542.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1542.h . . .\n",
      "Opened output file \"model_1542.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1543.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1543.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1543.h . . .\n",
      "Opened output file \"model_1543.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1544.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1544.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1544.h . . .\n",
      "Opened output file \"model_1544.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1545.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1545.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1545.h . . .\n",
      "Opened output file \"model_1545.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1546.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1546.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1546.h . . .\n",
      "Opened output file \"model_1546.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1547.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1547.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1547.h . . .\n",
      "Opened output file \"model_1547.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1548.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1548.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1548.h . . .\n",
      "Opened output file \"model_1548.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1549.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1549.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1549.h . . .\n",
      "Opened output file \"model_1549.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_155.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_155.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_155.h . . .\n",
      "Opened output file \"model_155.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1550.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1550.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1550.h . . .\n",
      "Opened output file \"model_1550.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1551.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1551.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1551.h . . .\n",
      "Opened output file \"model_1551.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1552.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1552.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1552.h . . .\n",
      "Opened output file \"model_1552.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1553.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1553.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1553.h . . .\n",
      "Opened output file \"model_1553.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1554.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1554.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1554.h . . .\n",
      "Opened output file \"model_1554.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1555.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1555.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1555.h . . .\n",
      "Opened output file \"model_1555.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1556.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1556.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1556.h . . .\n",
      "Opened output file \"model_1556.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1557.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1557.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1557.h . . .\n",
      "Opened output file \"model_1557.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1558.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1558.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1558.h . . .\n",
      "Opened output file \"model_1558.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1559.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1559.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1559.h . . .\n",
      "Opened output file \"model_1559.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_156.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_156.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_156.h . . .\n",
      "Opened output file \"model_156.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1560.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1560.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1560.h . . .\n",
      "Opened output file \"model_1560.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1561.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1561.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1561.h . . .\n",
      "Opened output file \"model_1561.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1562.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1562.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1562.h . . .\n",
      "Opened output file \"model_1562.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1563.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1563.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1563.h . . .\n",
      "Opened output file \"model_1563.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1564.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1564.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1564.h . . .\n",
      "Opened output file \"model_1564.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1565.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1565.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1565.h . . .\n",
      "Opened output file \"model_1565.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "280 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1566.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1566.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1566.h . . .\n",
      "Opened output file \"model_1566.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1567.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1567.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1567.h . . .\n",
      "Opened output file \"model_1567.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1568.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1568.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1568.h . . .\n",
      "Opened output file \"model_1568.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1569.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1569.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1569.h . . .\n",
      "Opened output file \"model_1569.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_157.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_157.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_157.h . . .\n",
      "Opened output file \"model_157.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1570.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1570.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1570.h . . .\n",
      "Opened output file \"model_1570.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1571.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1571.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1571.h . . .\n",
      "Opened output file \"model_1571.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1572.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1572.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1572.h . . .\n",
      "Opened output file \"model_1572.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1573.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1573.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1573.h . . .\n",
      "Opened output file \"model_1573.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1574.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1574.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1574.h . . .\n",
      "Opened output file \"model_1574.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1575.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1575.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1575.h . . .\n",
      "Opened output file \"model_1575.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1576.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1576.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1576.h . . .\n",
      "Opened output file \"model_1576.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1577.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1577.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1577.h . . .\n",
      "Opened output file \"model_1577.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1578.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1578.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1578.h . . .\n",
      "Opened output file \"model_1578.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1579.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1579.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1579.h . . .\n",
      "Opened output file \"model_1579.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_158.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_158.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_158.h . . .\n",
      "Opened output file \"model_158.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1580.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1580.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1580.h . . .\n",
      "Opened output file \"model_1580.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1581.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1581.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1581.h . . .\n",
      "Opened output file \"model_1581.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1582.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1582.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1582.h . . .\n",
      "Opened output file \"model_1582.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1583.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1583.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1583.h . . .\n",
      "Opened output file \"model_1583.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1584.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1584.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1584.h . . .\n",
      "Opened output file \"model_1584.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1585.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1585.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1585.h . . .\n",
      "Opened output file \"model_1585.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:44:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1586.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1586.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1586.h . . .\n",
      "Opened output file \"model_1586.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1587.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1587.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1587.h . . .\n",
      "Opened output file \"model_1587.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1588.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1588.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1588.h . . .\n",
      "Opened output file \"model_1588.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1589.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1589.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1589.h . . .\n",
      "Opened output file \"model_1589.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_159.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_159.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_159.h . . .\n",
      "Opened output file \"model_159.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1590.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1590.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1590.h . . .\n",
      "Opened output file \"model_1590.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1591.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1591.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1591.h . . .\n",
      "Opened output file \"model_1591.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1592.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1592.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1592.h . . .\n",
      "Opened output file \"model_1592.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1593.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1593.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1593.h . . .\n",
      "Opened output file \"model_1593.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1594.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1594.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1594.h . . .\n",
      "Opened output file \"model_1594.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1595.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1595.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1595.h . . .\n",
      "Opened output file \"model_1595.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1596.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1596.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1596.h . . .\n",
      "Opened output file \"model_1596.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1597.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1597.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1597.h . . .\n",
      "Opened output file \"model_1597.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1598.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1598.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1598.h . . .\n",
      "Opened output file \"model_1598.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1599.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1599.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1599.h . . .\n",
      "Opened output file \"model_1599.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_160.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_160.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_160.h . . .\n",
      "Opened output file \"model_160.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1600.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1600.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1600.h . . .\n",
      "Opened output file \"model_1600.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1601.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1601.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1601.h . . .\n",
      "Opened output file \"model_1601.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1602.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1602.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1602.h . . .\n",
      "Opened output file \"model_1602.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1603.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1603.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1603.h . . .\n",
      "Opened output file \"model_1603.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1604.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1604.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1604.h . . .\n",
      "Opened output file \"model_1604.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "280 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1605.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1605.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1605.h . . .\n",
      "Opened output file \"model_1605.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1606.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1606.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1606.h . . .\n",
      "Opened output file \"model_1606.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1607.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1607.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1607.h . . .\n",
      "Opened output file \"model_1607.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1608.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1608.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1608.h . . .\n",
      "Opened output file \"model_1608.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1609.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1609.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1609.h . . .\n",
      "Opened output file \"model_1609.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_161.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_161.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_161.h . . .\n",
      "Opened output file \"model_161.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1610.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1610.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1610.h . . .\n",
      "Opened output file \"model_1610.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1611.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1611.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1611.h . . .\n",
      "Opened output file \"model_1611.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1612.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1612.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1612.h . . .\n",
      "Opened output file \"model_1612.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1613.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1613.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1613.h . . .\n",
      "Opened output file \"model_1613.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1614.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1614.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1614.h . . .\n",
      "Opened output file \"model_1614.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1615.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1615.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1615.h . . .\n",
      "Opened output file \"model_1615.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1616.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1616.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1616.h . . .\n",
      "Opened output file \"model_1616.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1617.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1617.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1617.h . . .\n",
      "Opened output file \"model_1617.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1618.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1618.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1618.h . . .\n",
      "Opened output file \"model_1618.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1619.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1619.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1619.h . . .\n",
      "Opened output file \"model_1619.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_162.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_162.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_162.h . . .\n",
      "Opened output file \"model_162.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1620.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1620.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1620.h . . .\n",
      "Opened output file \"model_1620.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1621.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1621.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1621.h . . .\n",
      "Opened output file \"model_1621.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1622.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1622.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1622.h . . .\n",
      "Opened output file \"model_1622.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1623.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1623.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1623.h . . .\n",
      "Opened output file \"model_1623.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1624.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1624.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1624.h . . .\n",
      "Opened output file \"model_1624.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1625.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1625.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1625.h . . .\n",
      "Opened output file \"model_1625.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1626.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1626.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1626.h . . .\n",
      "Opened output file \"model_1626.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1627.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1627.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1627.h . . .\n",
      "Opened output file \"model_1627.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1628.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1628.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1628.h . . .\n",
      "Opened output file \"model_1628.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1629.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1629.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1629.h . . .\n",
      "Opened output file \"model_1629.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_163.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_163.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_163.h . . .\n",
      "Opened output file \"model_163.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1630.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1630.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1630.h . . .\n",
      "Opened output file \"model_1630.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1631.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1631.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1631.h . . .\n",
      "Opened output file \"model_1631.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1632.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1632.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1632.h . . .\n",
      "Opened output file \"model_1632.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1633.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1633.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1633.h . . .\n",
      "Opened output file \"model_1633.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1634.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1634.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1634.h . . .\n",
      "Opened output file \"model_1634.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1635.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1635.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1635.h . . .\n",
      "Opened output file \"model_1635.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1636.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1636.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1636.h . . .\n",
      "Opened output file \"model_1636.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1637.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1637.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1637.h . . .\n",
      "Opened output file \"model_1637.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1638.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1638.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1638.h . . .\n",
      "Opened output file \"model_1638.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1639.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1639.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1639.h . . .\n",
      "Opened output file \"model_1639.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_164.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_164.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_164.h . . .\n",
      "Opened output file \"model_164.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "281 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1640.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1640.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1640.h . . .\n",
      "Opened output file \"model_1640.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1641.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1641.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1641.h . . .\n",
      "Opened output file \"model_1641.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1642.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1642.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1642.h . . .\n",
      "Opened output file \"model_1642.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1643.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1643.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1643.h . . .\n",
      "Opened output file \"model_1643.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1644.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1644.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1644.h . . .\n",
      "Opened output file \"model_1644.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1645.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1645.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1645.h . . .\n",
      "Opened output file \"model_1645.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "283 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1646.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1646.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1646.h . . .\n",
      "Opened output file \"model_1646.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1647.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1647.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1647.h . . .\n",
      "Opened output file \"model_1647.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1648.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1648.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1648.h . . .\n",
      "Opened output file \"model_1648.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1649.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1649.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1649.h . . .\n",
      "Opened output file \"model_1649.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_165.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_165.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_165.h . . .\n",
      "Opened output file \"model_165.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1650.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1650.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1650.h . . .\n",
      "Opened output file \"model_1650.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1651.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1651.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1651.h . . .\n",
      "Opened output file \"model_1651.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1652.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1652.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1652.h . . .\n",
      "Opened output file \"model_1652.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1653.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1653.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1653.h . . .\n",
      "Opened output file \"model_1653.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1654.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1654.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1654.h . . .\n",
      "Opened output file \"model_1654.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1655.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1655.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1655.h . . .\n",
      "Opened output file \"model_1655.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1656.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1656.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1656.h . . .\n",
      "Opened output file \"model_1656.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1657.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1657.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1657.h . . .\n",
      "Opened output file \"model_1657.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1658.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1658.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1658.h . . .\n",
      "Opened output file \"model_1658.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1659.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1659.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1659.h . . .\n",
      "Opened output file \"model_1659.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_166.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_166.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_166.h . . .\n",
      "Opened output file \"model_166.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1660.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1660.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1660.h . . .\n",
      "Opened output file \"model_1660.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1661.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1661.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1661.h . . .\n",
      "Opened output file \"model_1661.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1662.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1662.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1662.h . . .\n",
      "Opened output file \"model_1662.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1663.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1663.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1663.h . . .\n",
      "Opened output file \"model_1663.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1664.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1664.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1664.h . . .\n",
      "Opened output file \"model_1664.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1665.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1665.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1665.h . . .\n",
      "Opened output file \"model_1665.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1666.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1666.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1666.h . . .\n",
      "Opened output file \"model_1666.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1667.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1667.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1667.h . . .\n",
      "Opened output file \"model_1667.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1668.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1668.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1668.h . . .\n",
      "Opened output file \"model_1668.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1669.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1669.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1669.h . . .\n",
      "Opened output file \"model_1669.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_167.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_167.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_167.h . . .\n",
      "Opened output file \"model_167.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1670.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1670.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1670.h . . .\n",
      "Opened output file \"model_1670.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1671.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1671.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1671.h . . .\n",
      "Opened output file \"model_1671.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1672.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1672.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1672.h . . .\n",
      "Opened output file \"model_1672.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1673.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1673.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1673.h . . .\n",
      "Opened output file \"model_1673.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1674.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1674.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1674.h . . .\n",
      "Opened output file \"model_1674.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1675.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1675.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1675.h . . .\n",
      "Opened output file \"model_1675.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1676.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1676.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1676.h . . .\n",
      "Opened output file \"model_1676.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1677.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1677.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1677.h . . .\n",
      "Opened output file \"model_1677.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1678.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1678.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1678.h . . .\n",
      "Opened output file \"model_1678.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1679.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1679.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1679.h . . .\n",
      "Opened output file \"model_1679.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_168.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_168.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_168.h . . .\n",
      "Opened output file \"model_168.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1680.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1680.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1680.h . . .\n",
      "Opened output file \"model_1680.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1681.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1681.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1681.h . . .\n",
      "Opened output file \"model_1681.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1682.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1682.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1682.h . . .\n",
      "Opened output file \"model_1682.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1683.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1683.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1683.h . . .\n",
      "Opened output file \"model_1683.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1684.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1684.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1684.h . . .\n",
      "Opened output file \"model_1684.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1685.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1685.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1685.h . . .\n",
      "Opened output file \"model_1685.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1686.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1686.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1686.h . . .\n",
      "Opened output file \"model_1686.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1687.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1687.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1687.h . . .\n",
      "Opened output file \"model_1687.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1688.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1688.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1688.h . . .\n",
      "Opened output file \"model_1688.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1689.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1689.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1689.h . . .\n",
      "Opened output file \"model_1689.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_169.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_169.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_169.h . . .\n",
      "Opened output file \"model_169.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1690.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1690.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1690.h . . .\n",
      "Opened output file \"model_1690.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1691.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1691.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1691.h . . .\n",
      "Opened output file \"model_1691.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1692.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1692.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1692.h . . .\n",
      "Opened output file \"model_1692.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1693.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1693.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1693.h . . .\n",
      "Opened output file \"model_1693.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1694.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1694.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1694.h . . .\n",
      "Opened output file \"model_1694.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1695.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1695.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1695.h . . .\n",
      "Opened output file \"model_1695.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1696.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1696.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1696.h . . .\n",
      "Opened output file \"model_1696.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1697.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1697.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1697.h . . .\n",
      "Opened output file \"model_1697.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1698.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1698.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1698.h . . .\n",
      "Opened output file \"model_1698.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1699.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1699.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1699.h . . .\n",
      "Opened output file \"model_1699.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_17.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_17.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_17.h . . .\n",
      "Opened output file \"model_17.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_170.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_170.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_170.h . . .\n",
      "Opened output file \"model_170.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1700.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1700.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1700.h . . .\n",
      "Opened output file \"model_1700.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1701.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1701.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1701.h . . .\n",
      "Opened output file \"model_1701.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1702.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1702.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1702.h . . .\n",
      "Opened output file \"model_1702.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1703.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1703.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1703.h . . .\n",
      "Opened output file \"model_1703.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1704.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1704.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1704.h . . .\n",
      "Opened output file \"model_1704.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1705.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1705.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1705.h . . .\n",
      "Opened output file \"model_1705.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1706.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1706.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1706.h . . .\n",
      "Opened output file \"model_1706.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1707.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1707.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1707.h . . .\n",
      "Opened output file \"model_1707.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1708.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1708.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1708.h . . .\n",
      "Opened output file \"model_1708.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1709.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1709.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1709.h . . .\n",
      "Opened output file \"model_1709.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_171.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_171.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_171.h . . .\n",
      "Opened output file \"model_171.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1710.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1710.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1710.h . . .\n",
      "Opened output file \"model_1710.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1711.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1711.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1711.h . . .\n",
      "Opened output file \"model_1711.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1712.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1712.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1712.h . . .\n",
      "Opened output file \"model_1712.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1713.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1713.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1713.h . . .\n",
      "Opened output file \"model_1713.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1714.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1714.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1714.h . . .\n",
      "Opened output file \"model_1714.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1715.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1715.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1715.h . . .\n",
      "Opened output file \"model_1715.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1716.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1716.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1716.h . . .\n",
      "Opened output file \"model_1716.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1717.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1717.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1717.h . . .\n",
      "Opened output file \"model_1717.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1718.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1718.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1718.h . . .\n",
      "Opened output file \"model_1718.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1719.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1719.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1719.h . . .\n",
      "Opened output file \"model_1719.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_172.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_172.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_172.h . . .\n",
      "Opened output file \"model_172.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1720.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1720.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1720.h . . .\n",
      "Opened output file \"model_1720.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1721.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1721.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1721.h . . .\n",
      "Opened output file \"model_1721.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1722.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1722.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1722.h . . .\n",
      "Opened output file \"model_1722.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1723.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1723.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1723.h . . .\n",
      "Opened output file \"model_1723.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1724.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1724.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1724.h . . .\n",
      "Opened output file \"model_1724.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1725.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1725.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1725.h . . .\n",
      "Opened output file \"model_1725.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1726.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1726.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1726.h . . .\n",
      "Opened output file \"model_1726.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1727.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1727.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1727.h . . .\n",
      "Opened output file \"model_1727.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1728.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1728.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1728.h . . .\n",
      "Opened output file \"model_1728.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1729.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1729.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1729.h . . .\n",
      "Opened output file \"model_1729.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_173.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_173.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_173.h . . .\n",
      "Opened output file \"model_173.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1730.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1730.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1730.h . . .\n",
      "Opened output file \"model_1730.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1731.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1731.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1731.h . . .\n",
      "Opened output file \"model_1731.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1732.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1732.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1732.h . . .\n",
      "Opened output file \"model_1732.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1733.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1733.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1733.h . . .\n",
      "Opened output file \"model_1733.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1734.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1734.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1734.h . . .\n",
      "Opened output file \"model_1734.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1735.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1735.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1735.h . . .\n",
      "Opened output file \"model_1735.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1736.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1736.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1736.h . . .\n",
      "Opened output file \"model_1736.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1737.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1737.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1737.h . . .\n",
      "Opened output file \"model_1737.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1738.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1738.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1738.h . . .\n",
      "Opened output file \"model_1738.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1739.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1739.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1739.h . . .\n",
      "Opened output file \"model_1739.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_174.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_174.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_174.h . . .\n",
      "Opened output file \"model_174.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1740.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1740.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1740.h . . .\n",
      "Opened output file \"model_1740.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1741.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1741.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1741.h . . .\n",
      "Opened output file \"model_1741.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1742.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1742.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1742.h . . .\n",
      "Opened output file \"model_1742.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1743.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1743.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1743.h . . .\n",
      "Opened output file \"model_1743.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1744.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1744.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1744.h . . .\n",
      "Opened output file \"model_1744.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1745.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1745.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1745.h . . .\n",
      "Opened output file \"model_1745.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1746.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1746.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1746.h . . .\n",
      "Opened output file \"model_1746.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1747.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1747.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1747.h . . .\n",
      "Opened output file \"model_1747.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1748.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1748.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1748.h . . .\n",
      "Opened output file \"model_1748.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1749.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1749.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1749.h . . .\n",
      "Opened output file \"model_1749.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_175.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_175.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_175.h . . .\n",
      "Opened output file \"model_175.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1750.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1750.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1750.h . . .\n",
      "Opened output file \"model_1750.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1751.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1751.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1751.h . . .\n",
      "Opened output file \"model_1751.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1752.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1752.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1752.h . . .\n",
      "Opened output file \"model_1752.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1753.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1753.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1753.h . . .\n",
      "Opened output file \"model_1753.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1754.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1754.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1754.h . . .\n",
      "Opened output file \"model_1754.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1755.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1755.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1755.h . . .\n",
      "Opened output file \"model_1755.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1756.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1756.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1756.h . . .\n",
      "Opened output file \"model_1756.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1757.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1757.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1757.h . . .\n",
      "Opened output file \"model_1757.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1758.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1758.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1758.h . . .\n",
      "Opened output file \"model_1758.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1759.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1759.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1759.h . . .\n",
      "Opened output file \"model_1759.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_176.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_176.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_176.h . . .\n",
      "Opened output file \"model_176.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1760.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1760.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1760.h . . .\n",
      "Opened output file \"model_1760.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1761.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1761.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1761.h . . .\n",
      "Opened output file \"model_1761.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1762.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1762.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1762.h . . .\n",
      "Opened output file \"model_1762.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1763.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1763.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1763.h . . .\n",
      "Opened output file \"model_1763.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1764.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1764.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1764.h . . .\n",
      "Opened output file \"model_1764.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1765.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1765.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1765.h . . .\n",
      "Opened output file \"model_1765.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1766.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1766.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1766.h . . .\n",
      "Opened output file \"model_1766.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1767.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1767.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1767.h . . .\n",
      "Opened output file \"model_1767.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1768.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1768.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1768.h . . .\n",
      "Opened output file \"model_1768.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1769.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1769.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1769.h . . .\n",
      "Opened output file \"model_1769.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_177.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_177.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_177.h . . .\n",
      "Opened output file \"model_177.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1770.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1770.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1770.h . . .\n",
      "Opened output file \"model_1770.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1771.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1771.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1771.h . . .\n",
      "Opened output file \"model_1771.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1772.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1772.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1772.h . . .\n",
      "Opened output file \"model_1772.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1773.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1773.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1773.h . . .\n",
      "Opened output file \"model_1773.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1774.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1774.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1774.h . . .\n",
      "Opened output file \"model_1774.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1775.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1775.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1775.h . . .\n",
      "Opened output file \"model_1775.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1776.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1776.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1776.h . . .\n",
      "Opened output file \"model_1776.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1777.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1777.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1777.h . . .\n",
      "Opened output file \"model_1777.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1778.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1778.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1778.h . . .\n",
      "Opened output file \"model_1778.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1779.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1779.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1779.h . . .\n",
      "Opened output file \"model_1779.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_178.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_178.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_178.h . . .\n",
      "Opened output file \"model_178.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1780.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1780.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1780.h . . .\n",
      "Opened output file \"model_1780.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1781.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1781.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1781.h . . .\n",
      "Opened output file \"model_1781.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1782.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1782.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1782.h . . .\n",
      "Opened output file \"model_1782.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1783.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1783.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1783.h . . .\n",
      "Opened output file \"model_1783.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1784.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1784.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1784.h . . .\n",
      "Opened output file \"model_1784.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1785.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1785.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1785.h . . .\n",
      "Opened output file \"model_1785.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1786.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1786.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1786.h . . .\n",
      "Opened output file \"model_1786.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1787.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1787.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1787.h . . .\n",
      "Opened output file \"model_1787.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1788.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1788.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1788.h . . .\n",
      "Opened output file \"model_1788.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1789.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1789.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1789.h . . .\n",
      "Opened output file \"model_1789.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_179.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_179.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_179.h . . .\n",
      "Opened output file \"model_179.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1790.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1790.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1790.h . . .\n",
      "Opened output file \"model_1790.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1791.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1791.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1791.h . . .\n",
      "Opened output file \"model_1791.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1792.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1792.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1792.h . . .\n",
      "Opened output file \"model_1792.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1793.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1793.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1793.h . . .\n",
      "Opened output file \"model_1793.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1794.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1794.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1794.h . . .\n",
      "Opened output file \"model_1794.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1795.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1795.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1795.h . . .\n",
      "Opened output file \"model_1795.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1796.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1796.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1796.h . . .\n",
      "Opened output file \"model_1796.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1797.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1797.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1797.h . . .\n",
      "Opened output file \"model_1797.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1798.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1798.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1798.h . . .\n",
      "Opened output file \"model_1798.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1799.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1799.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1799.h . . .\n",
      "Opened output file \"model_1799.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_18.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_18.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_18.h . . .\n",
      "Opened output file \"model_18.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_180.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_180.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_180.h . . .\n",
      "Opened output file \"model_180.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1800.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1800.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1800.h . . .\n",
      "Opened output file \"model_1800.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1801.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1801.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1801.h . . .\n",
      "Opened output file \"model_1801.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "280 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1802.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1802.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1802.h . . .\n",
      "Opened output file \"model_1802.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1803.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1803.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1803.h . . .\n",
      "Opened output file \"model_1803.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1804.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1804.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1804.h . . .\n",
      "Opened output file \"model_1804.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1805.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1805.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1805.h . . .\n",
      "Opened output file \"model_1805.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1806.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1806.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1806.h . . .\n",
      "Opened output file \"model_1806.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1807.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1807.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1807.h . . .\n",
      "Opened output file \"model_1807.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1808.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1808.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1808.h . . .\n",
      "Opened output file \"model_1808.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1809.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1809.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1809.h . . .\n",
      "Opened output file \"model_1809.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_181.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_181.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_181.h . . .\n",
      "Opened output file \"model_181.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1810.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1810.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1810.h . . .\n",
      "Opened output file \"model_1810.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1811.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1811.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1811.h . . .\n",
      "Opened output file \"model_1811.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1812.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1812.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1812.h . . .\n",
      "Opened output file \"model_1812.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1813.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1813.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1813.h . . .\n",
      "Opened output file \"model_1813.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1814.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1814.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1814.h . . .\n",
      "Opened output file \"model_1814.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1815.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1815.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1815.h . . .\n",
      "Opened output file \"model_1815.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1816.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1816.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1816.h . . .\n",
      "Opened output file \"model_1816.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1817.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1817.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1817.h . . .\n",
      "Opened output file \"model_1817.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1818.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1818.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1818.h . . .\n",
      "Opened output file \"model_1818.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1819.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1819.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1819.h . . .\n",
      "Opened output file \"model_1819.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_182.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_182.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_182.h . . .\n",
      "Opened output file \"model_182.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1820.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1820.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1820.h . . .\n",
      "Opened output file \"model_1820.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1821.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1821.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1821.h . . .\n",
      "Opened output file \"model_1821.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1822.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1822.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1822.h . . .\n",
      "Opened output file \"model_1822.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1823.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1823.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1823.h . . .\n",
      "Opened output file \"model_1823.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1824.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1824.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1824.h . . .\n",
      "Opened output file \"model_1824.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1825.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1825.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1825.h . . .\n",
      "Opened output file \"model_1825.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1826.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1826.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1826.h . . .\n",
      "Opened output file \"model_1826.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1827.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1827.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1827.h . . .\n",
      "Opened output file \"model_1827.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1828.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1828.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1828.h . . .\n",
      "Opened output file \"model_1828.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1829.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1829.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1829.h . . .\n",
      "Opened output file \"model_1829.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_183.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_183.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_183.h . . .\n",
      "Opened output file \"model_183.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "280 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1830.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1830.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1830.h . . .\n",
      "Opened output file \"model_1830.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1831.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1831.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1831.h . . .\n",
      "Opened output file \"model_1831.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1832.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1832.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1832.h . . .\n",
      "Opened output file \"model_1832.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1833.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1833.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1833.h . . .\n",
      "Opened output file \"model_1833.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1834.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1834.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1834.h . . .\n",
      "Opened output file \"model_1834.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1835.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1835.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1835.h . . .\n",
      "Opened output file \"model_1835.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1836.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1836.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1836.h . . .\n",
      "Opened output file \"model_1836.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1837.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1837.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1837.h . . .\n",
      "Opened output file \"model_1837.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1838.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1838.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1838.h . . .\n",
      "Opened output file \"model_1838.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1839.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1839.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1839.h . . .\n",
      "Opened output file \"model_1839.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_184.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_184.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_184.h . . .\n",
      "Opened output file \"model_184.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1840.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1840.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1840.h . . .\n",
      "Opened output file \"model_1840.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1841.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1841.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1841.h . . .\n",
      "Opened output file \"model_1841.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1842.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1842.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1842.h . . .\n",
      "Opened output file \"model_1842.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1843.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1843.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1843.h . . .\n",
      "Opened output file \"model_1843.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1844.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1844.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1844.h . . .\n",
      "Opened output file \"model_1844.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1845.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1845.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1845.h . . .\n",
      "Opened output file \"model_1845.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1846.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1846.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1846.h . . .\n",
      "Opened output file \"model_1846.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1847.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1847.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1847.h . . .\n",
      "Opened output file \"model_1847.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1848.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1848.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1848.h . . .\n",
      "Opened output file \"model_1848.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1849.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1849.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1849.h . . .\n",
      "Opened output file \"model_1849.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_185.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_185.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_185.h . . .\n",
      "Opened output file \"model_185.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1850.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1850.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1850.h . . .\n",
      "Opened output file \"model_1850.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1851.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1851.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1851.h . . .\n",
      "Opened output file \"model_1851.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1852.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1852.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1852.h . . .\n",
      "Opened output file \"model_1852.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1853.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1853.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1853.h . . .\n",
      "Opened output file \"model_1853.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1854.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1854.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1854.h . . .\n",
      "Opened output file \"model_1854.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1855.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1855.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1855.h . . .\n",
      "Opened output file \"model_1855.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1856.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1856.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1856.h . . .\n",
      "Opened output file \"model_1856.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1857.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1857.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1857.h . . .\n",
      "Opened output file \"model_1857.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1858.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1858.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1858.h . . .\n",
      "Opened output file \"model_1858.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1859.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1859.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1859.h . . .\n",
      "Opened output file \"model_1859.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_186.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_186.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_186.h . . .\n",
      "Opened output file \"model_186.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1860.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1860.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1860.h . . .\n",
      "Opened output file \"model_1860.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1861.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1861.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1861.h . . .\n",
      "Opened output file \"model_1861.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1862.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1862.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1862.h . . .\n",
      "Opened output file \"model_1862.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1863.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1863.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1863.h . . .\n",
      "Opened output file \"model_1863.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1864.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1864.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1864.h . . .\n",
      "Opened output file \"model_1864.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1865.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1865.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1865.h . . .\n",
      "Opened output file \"model_1865.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1866.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1866.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1866.h . . .\n",
      "Opened output file \"model_1866.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1867.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1867.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1867.h . . .\n",
      "Opened output file \"model_1867.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1868.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1868.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1868.h . . .\n",
      "Opened output file \"model_1868.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1869.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1869.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1869.h . . .\n",
      "Opened output file \"model_1869.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_187.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_187.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_187.h . . .\n",
      "Opened output file \"model_187.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1870.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1870.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1870.h . . .\n",
      "Opened output file \"model_1870.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1871.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1871.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1871.h . . .\n",
      "Opened output file \"model_1871.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1872.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1872.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1872.h . . .\n",
      "Opened output file \"model_1872.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1873.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1873.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1873.h . . .\n",
      "Opened output file \"model_1873.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1874.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1874.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1874.h . . .\n",
      "Opened output file \"model_1874.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1875.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1875.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1875.h . . .\n",
      "Opened output file \"model_1875.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1876.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1876.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1876.h . . .\n",
      "Opened output file \"model_1876.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1877.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1877.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1877.h . . .\n",
      "Opened output file \"model_1877.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1878.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1878.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1878.h . . .\n",
      "Opened output file \"model_1878.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1879.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1879.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1879.h . . .\n",
      "Opened output file \"model_1879.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_188.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_188.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_188.h . . .\n",
      "Opened output file \"model_188.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1880.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1880.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1880.h . . .\n",
      "Opened output file \"model_1880.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1881.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1881.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1881.h . . .\n",
      "Opened output file \"model_1881.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1882.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1882.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1882.h . . .\n",
      "Opened output file \"model_1882.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1883.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1883.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1883.h . . .\n",
      "Opened output file \"model_1883.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1884.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1884.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1884.h . . .\n",
      "Opened output file \"model_1884.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1885.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1885.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1885.h . . .\n",
      "Opened output file \"model_1885.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1886.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1886.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1886.h . . .\n",
      "Opened output file \"model_1886.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1887.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1887.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1887.h . . .\n",
      "Opened output file \"model_1887.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1888.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1888.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1888.h . . .\n",
      "Opened output file \"model_1888.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1889.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1889.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1889.h . . .\n",
      "Opened output file \"model_1889.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_189.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_189.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_189.h . . .\n",
      "Opened output file \"model_189.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1890.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1890.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1890.h . . .\n",
      "Opened output file \"model_1890.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1891.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1891.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1891.h . . .\n",
      "Opened output file \"model_1891.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1892.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1892.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1892.h . . .\n",
      "Opened output file \"model_1892.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1893.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1893.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1893.h . . .\n",
      "Opened output file \"model_1893.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1894.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1894.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1894.h . . .\n",
      "Opened output file \"model_1894.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1895.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1895.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1895.h . . .\n",
      "Opened output file \"model_1895.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1896.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1896.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1896.h . . .\n",
      "Opened output file \"model_1896.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1897.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1897.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1897.h . . .\n",
      "Opened output file \"model_1897.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1898.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1898.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1898.h . . .\n",
      "Opened output file \"model_1898.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1899.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1899.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1899.h . . .\n",
      "Opened output file \"model_1899.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_19.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_19.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_19.h . . .\n",
      "Opened output file \"model_19.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_190.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_190.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_190.h . . .\n",
      "Opened output file \"model_190.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1900.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1900.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1900.h . . .\n",
      "Opened output file \"model_1900.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1901.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1901.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1901.h . . .\n",
      "Opened output file \"model_1901.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1902.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1902.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1902.h . . .\n",
      "Opened output file \"model_1902.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1903.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1903.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1903.h . . .\n",
      "Opened output file \"model_1903.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1904.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1904.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1904.h . . .\n",
      "Opened output file \"model_1904.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1905.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1905.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1905.h . . .\n",
      "Opened output file \"model_1905.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1906.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1906.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1906.h . . .\n",
      "Opened output file \"model_1906.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1907.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1907.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1907.h . . .\n",
      "Opened output file \"model_1907.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1908.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1908.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1908.h . . .\n",
      "Opened output file \"model_1908.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1909.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1909.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1909.h . . .\n",
      "Opened output file \"model_1909.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_191.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_191.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_191.h . . .\n",
      "Opened output file \"model_191.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1910.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1910.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1910.h . . .\n",
      "Opened output file \"model_1910.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1911.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1911.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1911.h . . .\n",
      "Opened output file \"model_1911.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1912.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1912.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1912.h . . .\n",
      "Opened output file \"model_1912.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1913.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1913.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1913.h . . .\n",
      "Opened output file \"model_1913.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1914.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1914.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1914.h . . .\n",
      "Opened output file \"model_1914.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1915.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1915.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1915.h . . .\n",
      "Opened output file \"model_1915.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1916.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1916.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1916.h . . .\n",
      "Opened output file \"model_1916.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1917.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1917.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1917.h . . .\n",
      "Opened output file \"model_1917.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1918.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1918.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1918.h . . .\n",
      "Opened output file \"model_1918.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1919.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1919.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1919.h . . .\n",
      "Opened output file \"model_1919.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_192.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_192.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_192.h . . .\n",
      "Opened output file \"model_192.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1920.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1920.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1920.h . . .\n",
      "Opened output file \"model_1920.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1921.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1921.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1921.h . . .\n",
      "Opened output file \"model_1921.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1922.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1922.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1922.h . . .\n",
      "Opened output file \"model_1922.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1923.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1923.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1923.h . . .\n",
      "Opened output file \"model_1923.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1924.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1924.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1924.h . . .\n",
      "Opened output file \"model_1924.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1925.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1925.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1925.h . . .\n",
      "Opened output file \"model_1925.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1926.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1926.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1926.h . . .\n",
      "Opened output file \"model_1926.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1927.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1927.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1927.h . . .\n",
      "Opened output file \"model_1927.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1928.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1928.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1928.h . . .\n",
      "Opened output file \"model_1928.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1929.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1929.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1929.h . . .\n",
      "Opened output file \"model_1929.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_193.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_193.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_193.h . . .\n",
      "Opened output file \"model_193.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1930.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1930.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1930.h . . .\n",
      "Opened output file \"model_1930.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1931.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1931.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1931.h . . .\n",
      "Opened output file \"model_1931.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1932.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1932.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1932.h . . .\n",
      "Opened output file \"model_1932.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1933.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1933.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1933.h . . .\n",
      "Opened output file \"model_1933.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1934.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1934.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1934.h . . .\n",
      "Opened output file \"model_1934.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1935.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1935.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1935.h . . .\n",
      "Opened output file \"model_1935.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1936.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1936.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1936.h . . .\n",
      "Opened output file \"model_1936.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1937.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1937.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1937.h . . .\n",
      "Opened output file \"model_1937.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1938.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1938.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1938.h . . .\n",
      "Opened output file \"model_1938.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1939.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1939.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1939.h . . .\n",
      "Opened output file \"model_1939.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_194.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_194.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_194.h . . .\n",
      "Opened output file \"model_194.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1940.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1940.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1940.h . . .\n",
      "Opened output file \"model_1940.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1941.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1941.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1941.h . . .\n",
      "Opened output file \"model_1941.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1942.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1942.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1942.h . . .\n",
      "Opened output file \"model_1942.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1943.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1943.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1943.h . . .\n",
      "Opened output file \"model_1943.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1944.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1944.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1944.h . . .\n",
      "Opened output file \"model_1944.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1945.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1945.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1945.h . . .\n",
      "Opened output file \"model_1945.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1946.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1946.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1946.h . . .\n",
      "Opened output file \"model_1946.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1947.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1947.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1947.h . . .\n",
      "Opened output file \"model_1947.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1948.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1948.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1948.h . . .\n",
      "Opened output file \"model_1948.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1949.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1949.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1949.h . . .\n",
      "Opened output file \"model_1949.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_195.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_195.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_195.h . . .\n",
      "Opened output file \"model_195.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1950.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1950.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1950.h . . .\n",
      "Opened output file \"model_1950.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1951.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1951.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1951.h . . .\n",
      "Opened output file \"model_1951.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1952.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1952.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1952.h . . .\n",
      "Opened output file \"model_1952.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1953.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1953.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1953.h . . .\n",
      "Opened output file \"model_1953.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1954.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1954.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1954.h . . .\n",
      "Opened output file \"model_1954.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1955.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_1955.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_1955.h . . .\n",
      "Opened output file \"model_1955.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_196.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_196.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_196.h . . .\n",
      "Opened output file \"model_196.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_197.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_197.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_197.h . . .\n",
      "Opened output file \"model_197.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_198.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_198.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_198.h . . .\n",
      "Opened output file \"model_198.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_199.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_199.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_199.h . . .\n",
      "Opened output file \"model_199.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_2.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_2.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_2.h . . .\n",
      "Opened output file \"model_2.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_20.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_20.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_20.h . . .\n",
      "Opened output file \"model_20.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_200.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_200.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_200.h . . .\n",
      "Opened output file \"model_200.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_201.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_201.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_201.h . . .\n",
      "Opened output file \"model_201.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_202.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_202.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_202.h . . .\n",
      "Opened output file \"model_202.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_203.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_203.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_203.h . . .\n",
      "Opened output file \"model_203.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_204.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_204.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_204.h . . .\n",
      "Opened output file \"model_204.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_205.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_205.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_205.h . . .\n",
      "Opened output file \"model_205.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_206.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_206.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_206.h . . .\n",
      "Opened output file \"model_206.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_207.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_207.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_207.h . . .\n",
      "Opened output file \"model_207.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_208.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_208.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_208.h . . .\n",
      "Opened output file \"model_208.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_209.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_209.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_209.h . . .\n",
      "Opened output file \"model_209.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_21.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_21.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_21.h . . .\n",
      "Opened output file \"model_21.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_210.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_210.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_210.h . . .\n",
      "Opened output file \"model_210.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_211.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_211.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_211.h . . .\n",
      "Opened output file \"model_211.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_212.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_212.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_212.h . . .\n",
      "Opened output file \"model_212.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_213.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_213.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_213.h . . .\n",
      "Opened output file \"model_213.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_214.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_214.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_214.h . . .\n",
      "Opened output file \"model_214.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_215.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_215.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_215.h . . .\n",
      "Opened output file \"model_215.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_216.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_216.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_216.h . . .\n",
      "Opened output file \"model_216.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_217.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_217.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_217.h . . .\n",
      "Opened output file \"model_217.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_218.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_218.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_218.h . . .\n",
      "Opened output file \"model_218.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_219.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_219.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_219.h . . .\n",
      "Opened output file \"model_219.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_22.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_22.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_22.h . . .\n",
      "Opened output file \"model_22.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_220.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_220.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_220.h . . .\n",
      "Opened output file \"model_220.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_221.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_221.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_221.h . . .\n",
      "Opened output file \"model_221.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_222.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_222.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_222.h . . .\n",
      "Opened output file \"model_222.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_223.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_223.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_223.h . . .\n",
      "Opened output file \"model_223.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_224.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_224.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_224.h . . .\n",
      "Opened output file \"model_224.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_225.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_225.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_225.h . . .\n",
      "Opened output file \"model_225.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_226.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_226.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_226.h . . .\n",
      "Opened output file \"model_226.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_227.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_227.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_227.h . . .\n",
      "Opened output file \"model_227.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_228.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_228.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_228.h . . .\n",
      "Opened output file \"model_228.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_229.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_229.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_229.h . . .\n",
      "Opened output file \"model_229.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_23.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_23.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_23.h . . .\n",
      "Opened output file \"model_23.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_230.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_230.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_230.h . . .\n",
      "Opened output file \"model_230.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_231.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_231.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_231.h . . .\n",
      "Opened output file \"model_231.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_232.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_232.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_232.h . . .\n",
      "Opened output file \"model_232.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_233.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_233.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_233.h . . .\n",
      "Opened output file \"model_233.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_234.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_234.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_234.h . . .\n",
      "Opened output file \"model_234.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_235.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_235.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_235.h . . .\n",
      "Opened output file \"model_235.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_236.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_236.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_236.h . . .\n",
      "Opened output file \"model_236.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_237.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_237.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_237.h . . .\n",
      "Opened output file \"model_237.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_238.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_238.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_238.h . . .\n",
      "Opened output file \"model_238.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_239.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_239.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_239.h . . .\n",
      "Opened output file \"model_239.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_24.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_24.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_24.h . . .\n",
      "Opened output file \"model_24.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_25.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_25.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_25.h . . .\n",
      "Opened output file \"model_25.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_240.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_240.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_240.h . . .\n",
      "Opened output file \"model_240.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_241.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_241.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_241.h . . .\n",
      "Opened output file \"model_241.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_242.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_242.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_242.h . . .\n",
      "Opened output file \"model_242.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_243.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_243.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_243.h . . .\n",
      "Opened output file \"model_243.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_244.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_244.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_244.h . . .\n",
      "Opened output file \"model_244.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_245.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_245.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_245.h . . .\n",
      "Opened output file \"model_245.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_246.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_246.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_246.h . . .\n",
      "Opened output file \"model_246.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_247.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_247.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_247.h . . .\n",
      "Opened output file \"model_247.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_248.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_248.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_248.h . . .\n",
      "Opened output file \"model_248.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_249.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_249.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_249.h . . .\n",
      "Opened output file \"model_249.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_250.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_250.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_250.h . . .\n",
      "Opened output file \"model_250.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_251.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_251.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_251.h . . .\n",
      "Opened output file \"model_251.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_252.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_252.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_252.h . . .\n",
      "Opened output file \"model_252.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_253.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_253.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_253.h . . .\n",
      "Opened output file \"model_253.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_254.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_254.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_254.h . . .\n",
      "Opened output file \"model_254.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_255.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_255.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_255.h . . .\n",
      "Opened output file \"model_255.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_256.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_256.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_256.h . . .\n",
      "Opened output file \"model_256.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_257.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_257.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_257.h . . .\n",
      "Opened output file \"model_257.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_258.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_258.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_258.h . . .\n",
      "Opened output file \"model_258.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_259.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_259.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_259.h . . .\n",
      "Opened output file \"model_259.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_26.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_26.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_26.h . . .\n",
      "Opened output file \"model_26.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_260.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_260.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_260.h . . .\n",
      "Opened output file \"model_260.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_261.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_261.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_261.h . . .\n",
      "Opened output file \"model_261.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_262.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_262.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_262.h . . .\n",
      "Opened output file \"model_262.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_263.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_263.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_263.h . . .\n",
      "Opened output file \"model_263.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_264.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_264.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_264.h . . .\n",
      "Opened output file \"model_264.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_265.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_265.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_265.h . . .\n",
      "Opened output file \"model_265.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_266.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_266.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_266.h . . .\n",
      "Opened output file \"model_266.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_267.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_267.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_267.h . . .\n",
      "Opened output file \"model_267.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_268.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_268.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_268.h . . .\n",
      "Opened output file \"model_268.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_269.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_269.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_269.h . . .\n",
      "Opened output file \"model_269.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_27.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_27.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_27.h . . .\n",
      "Opened output file \"model_27.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_270.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_270.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_270.h . . .\n",
      "Opened output file \"model_270.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_271.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_271.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_271.h . . .\n",
      "Opened output file \"model_271.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_272.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_272.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_272.h . . .\n",
      "Opened output file \"model_272.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_273.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_273.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_273.h . . .\n",
      "Opened output file \"model_273.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_274.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_274.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_274.h . . .\n",
      "Opened output file \"model_274.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_275.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_275.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_275.h . . .\n",
      "Opened output file \"model_275.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_276.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_276.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_276.h . . .\n",
      "Opened output file \"model_276.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_277.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_277.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_277.h . . .\n",
      "Opened output file \"model_277.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_278.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_278.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_278.h . . .\n",
      "Opened output file \"model_278.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_279.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_279.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_279.h . . .\n",
      "Opened output file \"model_279.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_28.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_28.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_28.h . . .\n",
      "Opened output file \"model_28.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "280 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_280.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_280.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_280.h . . .\n",
      "Opened output file \"model_280.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_281.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_281.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_281.h . . .\n",
      "Opened output file \"model_281.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_282.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_282.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_282.h . . .\n",
      "Opened output file \"model_282.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_283.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_283.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_283.h . . .\n",
      "Opened output file \"model_283.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_284.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_284.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_284.h . . .\n",
      "Opened output file \"model_284.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_285.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_285.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_285.h . . .\n",
      "Opened output file \"model_285.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_286.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_286.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_286.h . . .\n",
      "Opened output file \"model_286.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_287.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_287.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_287.h . . .\n",
      "Opened output file \"model_287.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_288.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_288.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_288.h . . .\n",
      "Opened output file \"model_288.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_289.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_289.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_289.h . . .\n",
      "Opened output file \"model_289.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_29.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_29.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_29.h . . .\n",
      "Opened output file \"model_29.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_290.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_290.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_290.h . . .\n",
      "Opened output file \"model_290.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_291.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_291.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_291.h . . .\n",
      "Opened output file \"model_291.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_292.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_292.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_292.h . . .\n",
      "Opened output file \"model_292.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_293.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_293.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_293.h . . .\n",
      "Opened output file \"model_293.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_294.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_294.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_294.h . . .\n",
      "Opened output file \"model_294.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_295.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_295.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_295.h . . .\n",
      "Opened output file \"model_295.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_296.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_296.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_296.h . . .\n",
      "Opened output file \"model_296.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_297.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_297.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_297.h . . .\n",
      "Opened output file \"model_297.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_298.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_298.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_298.h . . .\n",
      "Opened output file \"model_298.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_299.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_299.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_299.h . . .\n",
      "Opened output file \"model_299.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_3.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_3.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_3.h . . .\n",
      "Opened output file \"model_3.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_30.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_30.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_30.h . . .\n",
      "Opened output file \"model_30.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_300.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_300.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_300.h . . .\n",
      "Opened output file \"model_300.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_301.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_301.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_301.h . . .\n",
      "Opened output file \"model_301.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_302.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_302.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_302.h . . .\n",
      "Opened output file \"model_302.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_303.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_303.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_303.h . . .\n",
      "Opened output file \"model_303.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_304.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_304.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_304.h . . .\n",
      "Opened output file \"model_304.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_305.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_305.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_305.h . . .\n",
      "Opened output file \"model_305.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_306.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_306.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_306.h . . .\n",
      "Opened output file \"model_306.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_307.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_307.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_307.h . . .\n",
      "Opened output file \"model_307.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_308.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_308.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_308.h . . .\n",
      "Opened output file \"model_308.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_309.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_309.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_309.h . . .\n",
      "Opened output file \"model_309.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_31.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_31.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_31.h . . .\n",
      "Opened output file \"model_31.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_310.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_310.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_310.h . . .\n",
      "Opened output file \"model_310.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_311.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_311.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_311.h . . .\n",
      "Opened output file \"model_311.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_312.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_312.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_312.h . . .\n",
      "Opened output file \"model_312.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_313.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_313.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_313.h . . .\n",
      "Opened output file \"model_313.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_314.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_314.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_314.h . . .\n",
      "Opened output file \"model_314.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_315.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_315.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_315.h . . .\n",
      "Opened output file \"model_315.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_316.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_316.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_316.h . . .\n",
      "Opened output file \"model_316.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_317.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_317.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_317.h . . .\n",
      "Opened output file \"model_317.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_318.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_318.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_318.h . . .\n",
      "Opened output file \"model_318.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_319.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_319.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_319.h . . .\n",
      "Opened output file \"model_319.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_32.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_32.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_32.h . . .\n",
      "Opened output file \"model_32.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_320.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_320.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_320.h . . .\n",
      "Opened output file \"model_320.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_321.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_321.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_321.h . . .\n",
      "Opened output file \"model_321.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_322.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_322.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_322.h . . .\n",
      "Opened output file \"model_322.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_323.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_323.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_323.h . . .\n",
      "Opened output file \"model_323.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_324.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_324.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_324.h . . .\n",
      "Opened output file \"model_324.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_325.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_325.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_325.h . . .\n",
      "Opened output file \"model_325.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_326.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_326.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_326.h . . .\n",
      "Opened output file \"model_326.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_327.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_327.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_327.h . . .\n",
      "Opened output file \"model_327.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_328.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_328.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_328.h . . .\n",
      "Opened output file \"model_328.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_329.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_329.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_329.h . . .\n",
      "Opened output file \"model_329.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_33.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_33.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_33.h . . .\n",
      "Opened output file \"model_33.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_330.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_330.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_330.h . . .\n",
      "Opened output file \"model_330.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_331.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_331.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_331.h . . .\n",
      "Opened output file \"model_331.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_332.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_332.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_332.h . . .\n",
      "Opened output file \"model_332.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_333.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_333.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_333.h . . .\n",
      "Opened output file \"model_333.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_334.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_334.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_334.h . . .\n",
      "Opened output file \"model_334.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_335.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_335.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_335.h . . .\n",
      "Opened output file \"model_335.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_336.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_336.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_336.h . . .\n",
      "Opened output file \"model_336.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_337.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_337.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_337.h . . .\n",
      "Opened output file \"model_337.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_338.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_338.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_338.h . . .\n",
      "Opened output file \"model_338.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_339.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_339.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_339.h . . .\n",
      "Opened output file \"model_339.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_34.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_34.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_34.h . . .\n",
      "Opened output file \"model_34.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_340.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_340.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_340.h . . .\n",
      "Opened output file \"model_340.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_341.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_341.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_341.h . . .\n",
      "Opened output file \"model_341.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_342.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_342.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_342.h . . .\n",
      "Opened output file \"model_342.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_343.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_343.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_343.h . . .\n",
      "Opened output file \"model_343.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_344.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_344.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_344.h . . .\n",
      "Opened output file \"model_344.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_345.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_345.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_345.h . . .\n",
      "Opened output file \"model_345.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_346.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_346.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_346.h . . .\n",
      "Opened output file \"model_346.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_347.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_347.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_347.h . . .\n",
      "Opened output file \"model_347.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_348.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_348.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_348.h . . .\n",
      "Opened output file \"model_348.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_349.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_349.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_349.h . . .\n",
      "Opened output file \"model_349.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_35.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_35.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_35.h . . .\n",
      "Opened output file \"model_35.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_350.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_350.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_350.h . . .\n",
      "Opened output file \"model_350.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_351.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_351.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_351.h . . .\n",
      "Opened output file \"model_351.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_352.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_352.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_352.h . . .\n",
      "Opened output file \"model_352.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_353.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_353.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_353.h . . .\n",
      "Opened output file \"model_353.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "280 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_354.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_354.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_354.h . . .\n",
      "Opened output file \"model_354.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_355.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_355.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_355.h . . .\n",
      "Opened output file \"model_355.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_356.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_356.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_356.h . . .\n",
      "Opened output file \"model_356.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_36.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_36.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_36.h . . .\n",
      "Opened output file \"model_36.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_357.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_357.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_357.h . . .\n",
      "Opened output file \"model_357.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_358.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_358.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_358.h . . .\n",
      "Opened output file \"model_358.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_359.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_359.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_359.h . . .\n",
      "Opened output file \"model_359.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_360.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_360.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_360.h . . .\n",
      "Opened output file \"model_360.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_361.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_361.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_361.h . . .\n",
      "Opened output file \"model_361.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_362.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_362.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_362.h . . .\n",
      "Opened output file \"model_362.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_363.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_363.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_363.h . . .\n",
      "Opened output file \"model_363.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_364.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_364.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_364.h . . .\n",
      "Opened output file \"model_364.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_365.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_365.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_365.h . . .\n",
      "Opened output file \"model_365.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_366.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_366.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_366.h . . .\n",
      "Opened output file \"model_366.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_367.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_367.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_367.h . . .\n",
      "Opened output file \"model_367.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_368.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_368.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_368.h . . .\n",
      "Opened output file \"model_368.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_369.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_369.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_369.h . . .\n",
      "Opened output file \"model_369.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_37.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_37.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_37.h . . .\n",
      "Opened output file \"model_37.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "280 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_370.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_370.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_370.h . . .\n",
      "Opened output file \"model_370.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_371.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_371.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_371.h . . .\n",
      "Opened output file \"model_371.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_372.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_372.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_372.h . . .\n",
      "Opened output file \"model_372.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_373.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_373.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_373.h . . .\n",
      "Opened output file \"model_373.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_374.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_374.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_374.h . . .\n",
      "Opened output file \"model_374.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_375.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_375.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_375.h . . .\n",
      "Opened output file \"model_375.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "280 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_376.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_376.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_376.h . . .\n",
      "Opened output file \"model_376.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_377.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_377.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_377.h . . .\n",
      "Opened output file \"model_377.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_378.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_378.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_378.h . . .\n",
      "Opened output file \"model_378.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_379.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_379.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_379.h . . .\n",
      "Opened output file \"model_379.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_38.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_38.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_38.h . . .\n",
      "Opened output file \"model_38.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_380.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_380.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_380.h . . .\n",
      "Opened output file \"model_380.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_381.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_381.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_381.h . . .\n",
      "Opened output file \"model_381.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_382.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_382.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_382.h . . .\n",
      "Opened output file \"model_382.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_383.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_383.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_383.h . . .\n",
      "Opened output file \"model_383.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_384.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_384.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_384.h . . .\n",
      "Opened output file \"model_384.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_385.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_385.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_385.h . . .\n",
      "Opened output file \"model_385.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_386.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_386.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_386.h . . .\n",
      "Opened output file \"model_386.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_387.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_387.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_387.h . . .\n",
      "Opened output file \"model_387.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "281 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_388.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_388.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_388.h . . .\n",
      "Opened output file \"model_388.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_389.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_389.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_389.h . . .\n",
      "Opened output file \"model_389.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_39.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_39.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_39.h . . .\n",
      "Opened output file \"model_39.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_390.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_390.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_390.h . . .\n",
      "Opened output file \"model_390.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_391.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_391.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_391.h . . .\n",
      "Opened output file \"model_391.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_392.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_392.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_392.h . . .\n",
      "Opened output file \"model_392.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_393.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_393.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_393.h . . .\n",
      "Opened output file \"model_393.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_394.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_394.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_394.h . . .\n",
      "Opened output file \"model_394.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_395.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_395.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_395.h . . .\n",
      "Opened output file \"model_395.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_396.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_396.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_396.h . . .\n",
      "Opened output file \"model_396.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_397.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_397.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_397.h . . .\n",
      "Opened output file \"model_397.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_398.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_398.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_398.h . . .\n",
      "Opened output file \"model_398.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_399.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_399.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_399.h . . .\n",
      "Opened output file \"model_399.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_4.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_4.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_4.h . . .\n",
      "Opened output file \"model_4.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_40.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_40.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_40.h . . .\n",
      "Opened output file \"model_40.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_400.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_400.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_400.h . . .\n",
      "Opened output file \"model_400.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_401.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_401.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_401.h . . .\n",
      "Opened output file \"model_401.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_402.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_402.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_402.h . . .\n",
      "Opened output file \"model_402.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_403.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_403.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_403.h . . .\n",
      "Opened output file \"model_403.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_404.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_404.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_404.h . . .\n",
      "Opened output file \"model_404.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_405.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_405.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_405.h . . .\n",
      "Opened output file \"model_405.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_406.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_406.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_406.h . . .\n",
      "Opened output file \"model_406.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_407.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_407.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_407.h . . .\n",
      "Opened output file \"model_407.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_408.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_408.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_408.h . . .\n",
      "Opened output file \"model_408.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_409.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_409.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_409.h . . .\n",
      "Opened output file \"model_409.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_41.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_41.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_41.h . . .\n",
      "Opened output file \"model_41.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_410.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_410.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_410.h . . .\n",
      "Opened output file \"model_410.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_411.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_411.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_411.h . . .\n",
      "Opened output file \"model_411.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_412.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_412.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_412.h . . .\n",
      "Opened output file \"model_412.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_413.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_413.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_413.h . . .\n",
      "Opened output file \"model_413.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_414.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_414.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_414.h . . .\n",
      "Opened output file \"model_414.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_415.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_415.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_415.h . . .\n",
      "Opened output file \"model_415.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_416.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_416.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_416.h . . .\n",
      "Opened output file \"model_416.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_417.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_417.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_417.h . . .\n",
      "Opened output file \"model_417.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_418.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_418.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_418.h . . .\n",
      "Opened output file \"model_418.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_419.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_419.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_419.h . . .\n",
      "Opened output file \"model_419.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:51 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_42.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_42.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_42.h . . .\n",
      "Opened output file \"model_42.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_420.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_420.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_420.h . . .\n",
      "Opened output file \"model_420.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_421.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_421.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_421.h . . .\n",
      "Opened output file \"model_421.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_422.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_422.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_422.h . . .\n",
      "Opened output file \"model_422.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_423.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_423.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_423.h . . .\n",
      "Opened output file \"model_423.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_424.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_424.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_424.h . . .\n",
      "Opened output file \"model_424.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_425.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_425.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_425.h . . .\n",
      "Opened output file \"model_425.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_426.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_426.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_426.h . . .\n",
      "Opened output file \"model_426.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_427.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_427.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_427.h . . .\n",
      "Opened output file \"model_427.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_428.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_428.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_428.h . . .\n",
      "Opened output file \"model_428.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_429.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_429.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_429.h . . .\n",
      "Opened output file \"model_429.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_43.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_43.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_43.h . . .\n",
      "Opened output file \"model_43.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:52 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_430.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_430.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_430.h . . .\n",
      "Opened output file \"model_430.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_431.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_431.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_431.h . . .\n",
      "Opened output file \"model_431.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_432.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_432.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_432.h . . .\n",
      "Opened output file \"model_432.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_433.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_433.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_433.h . . .\n",
      "Opened output file \"model_433.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_434.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_434.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_434.h . . .\n",
      "Opened output file \"model_434.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_435.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_435.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_435.h . . .\n",
      "Opened output file \"model_435.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_436.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_436.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_436.h . . .\n",
      "Opened output file \"model_436.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_437.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_437.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_437.h . . .\n",
      "Opened output file \"model_437.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_438.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_438.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_438.h . . .\n",
      "Opened output file \"model_438.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_439.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_439.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_439.h . . .\n",
      "Opened output file \"model_439.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_44.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_44.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_44.h . . .\n",
      "Opened output file \"model_44.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:53 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_440.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_440.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_440.h . . .\n",
      "Opened output file \"model_440.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_441.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_441.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_441.h . . .\n",
      "Opened output file \"model_441.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_442.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_442.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_442.h . . .\n",
      "Opened output file \"model_442.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_443.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_443.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_443.h . . .\n",
      "Opened output file \"model_443.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_444.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_444.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_444.h . . .\n",
      "Opened output file \"model_444.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_445.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_445.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_445.h . . .\n",
      "Opened output file \"model_445.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_446.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_446.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_446.h . . .\n",
      "Opened output file \"model_446.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_447.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_447.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_447.h . . .\n",
      "Opened output file \"model_447.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_448.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_448.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_448.h . . .\n",
      "Opened output file \"model_448.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_449.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_449.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_449.h . . .\n",
      "Opened output file \"model_449.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_45.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_45.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_45.h . . .\n",
      "Opened output file \"model_45.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_450.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_450.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_450.h . . .\n",
      "Opened output file \"model_450.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:54 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_451.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_451.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_451.h . . .\n",
      "Opened output file \"model_451.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_452.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_452.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_452.h . . .\n",
      "Opened output file \"model_452.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_453.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_453.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_453.h . . .\n",
      "Opened output file \"model_453.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_454.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_454.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_454.h . . .\n",
      "Opened output file \"model_454.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_455.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_455.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_455.h . . .\n",
      "Opened output file \"model_455.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_456.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_456.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_456.h . . .\n",
      "Opened output file \"model_456.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_457.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_457.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_457.h . . .\n",
      "Opened output file \"model_457.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_458.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_458.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_458.h . . .\n",
      "Opened output file \"model_458.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_459.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_459.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_459.h . . .\n",
      "Opened output file \"model_459.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_46.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_46.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_46.h . . .\n",
      "Opened output file \"model_46.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_460.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_460.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_460.h . . .\n",
      "Opened output file \"model_460.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_461.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_461.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_461.h . . .\n",
      "Opened output file \"model_461.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:55 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_462.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_462.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_462.h . . .\n",
      "Opened output file \"model_462.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_463.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_463.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_463.h . . .\n",
      "Opened output file \"model_463.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_464.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_464.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_464.h . . .\n",
      "Opened output file \"model_464.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_465.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_465.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_465.h . . .\n",
      "Opened output file \"model_465.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_466.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_466.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_466.h . . .\n",
      "Opened output file \"model_466.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_467.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_467.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_467.h . . .\n",
      "Opened output file \"model_467.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "280 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_468.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_468.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_468.h . . .\n",
      "Opened output file \"model_468.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_469.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_469.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_469.h . . .\n",
      "Opened output file \"model_469.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_47.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_47.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_47.h . . .\n",
      "Opened output file \"model_47.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_470.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_470.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_470.h . . .\n",
      "Opened output file \"model_470.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_471.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_471.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_471.h . . .\n",
      "Opened output file \"model_471.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:56 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_472.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_472.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_472.h . . .\n",
      "Opened output file \"model_472.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_473.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_473.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_473.h . . .\n",
      "Opened output file \"model_473.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_474.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_474.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_474.h . . .\n",
      "Opened output file \"model_474.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_475.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_475.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_475.h . . .\n",
      "Opened output file \"model_475.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_476.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_476.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_476.h . . .\n",
      "Opened output file \"model_476.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_477.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_477.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_477.h . . .\n",
      "Opened output file \"model_477.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_478.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_478.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_478.h . . .\n",
      "Opened output file \"model_478.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_479.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_479.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_479.h . . .\n",
      "Opened output file \"model_479.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_48.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_48.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_48.h . . .\n",
      "Opened output file \"model_48.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_480.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_480.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_480.h . . .\n",
      "Opened output file \"model_480.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_481.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_481.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_481.h . . .\n",
      "Opened output file \"model_481.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_482.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_482.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_482.h . . .\n",
      "Opened output file \"model_482.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:57 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_483.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_483.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_483.h . . .\n",
      "Opened output file \"model_483.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_484.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_484.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_484.h . . .\n",
      "Opened output file \"model_484.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_485.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_485.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_485.h . . .\n",
      "Opened output file \"model_485.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_486.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_486.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_486.h . . .\n",
      "Opened output file \"model_486.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_487.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_487.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_487.h . . .\n",
      "Opened output file \"model_487.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_488.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_488.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_488.h . . .\n",
      "Opened output file \"model_488.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_489.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_489.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_489.h . . .\n",
      "Opened output file \"model_489.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_49.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_49.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_49.h . . .\n",
      "Opened output file \"model_49.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_490.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_490.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_490.h . . .\n",
      "Opened output file \"model_490.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_491.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_491.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_491.h . . .\n",
      "Opened output file \"model_491.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_492.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_492.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_492.h . . .\n",
      "Opened output file \"model_492.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_493.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_493.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_493.h . . .\n",
      "Opened output file \"model_493.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:58 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_494.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_494.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_494.h . . .\n",
      "Opened output file \"model_494.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_495.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_495.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_495.h . . .\n",
      "Opened output file \"model_495.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_496.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_496.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_496.h . . .\n",
      "Opened output file \"model_496.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_497.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_497.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_497.h . . .\n",
      "Opened output file \"model_497.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_498.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_498.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_498.h . . .\n",
      "Opened output file \"model_498.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_499.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_499.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_499.h . . .\n",
      "Opened output file \"model_499.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_5.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_5.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_5.h . . .\n",
      "Opened output file \"model_5.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_50.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_50.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_50.h . . .\n",
      "Opened output file \"model_50.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_500.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_500.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_500.h . . .\n",
      "Opened output file \"model_500.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_501.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_501.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_501.h . . .\n",
      "Opened output file \"model_501.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_502.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_502.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_502.h . . .\n",
      "Opened output file \"model_502.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:45:59 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_503.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_503.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_503.h . . .\n",
      "Opened output file \"model_503.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_504.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_504.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_504.h . . .\n",
      "Opened output file \"model_504.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_505.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_505.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_505.h . . .\n",
      "Opened output file \"model_505.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_506.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_506.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_506.h . . .\n",
      "Opened output file \"model_506.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_507.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_507.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_507.h . . .\n",
      "Opened output file \"model_507.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_508.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_508.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_508.h . . .\n",
      "Opened output file \"model_508.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_509.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_509.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_509.h . . .\n",
      "Opened output file \"model_509.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_51.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_51.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_51.h . . .\n",
      "Opened output file \"model_51.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_510.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_510.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_510.h . . .\n",
      "Opened output file \"model_510.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_511.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_511.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_511.h . . .\n",
      "Opened output file \"model_511.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_512.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_512.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_512.h . . .\n",
      "Opened output file \"model_512.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:00 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_513.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_513.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_513.h . . .\n",
      "Opened output file \"model_513.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_514.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_514.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_514.h . . .\n",
      "Opened output file \"model_514.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_515.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_515.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_515.h . . .\n",
      "Opened output file \"model_515.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_516.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_516.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_516.h . . .\n",
      "Opened output file \"model_516.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_517.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_517.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_517.h . . .\n",
      "Opened output file \"model_517.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_518.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_518.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_518.h . . .\n",
      "Opened output file \"model_518.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_519.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_519.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_519.h . . .\n",
      "Opened output file \"model_519.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_52.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_52.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_52.h . . .\n",
      "Opened output file \"model_52.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_520.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_520.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_520.h . . .\n",
      "Opened output file \"model_520.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_521.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_521.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_521.h . . .\n",
      "Opened output file \"model_521.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_522.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_522.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_522.h . . .\n",
      "Opened output file \"model_522.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_523.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_523.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_523.h . . .\n",
      "Opened output file \"model_523.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:01 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_524.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_524.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_524.h . . .\n",
      "Opened output file \"model_524.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_525.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_525.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_525.h . . .\n",
      "Opened output file \"model_525.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_526.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_526.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_526.h . . .\n",
      "Opened output file \"model_526.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_527.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_527.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_527.h . . .\n",
      "Opened output file \"model_527.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_528.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_528.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_528.h . . .\n",
      "Opened output file \"model_528.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_529.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_529.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_529.h . . .\n",
      "Opened output file \"model_529.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_53.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_53.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_53.h . . .\n",
      "Opened output file \"model_53.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_530.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_530.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_530.h . . .\n",
      "Opened output file \"model_530.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_531.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_531.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_531.h . . .\n",
      "Opened output file \"model_531.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_532.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_532.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_532.h . . .\n",
      "Opened output file \"model_532.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_533.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_533.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_533.h . . .\n",
      "Opened output file \"model_533.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:02 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_534.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_534.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_534.h . . .\n",
      "Opened output file \"model_534.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_535.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_535.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_535.h . . .\n",
      "Opened output file \"model_535.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_536.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_536.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_536.h . . .\n",
      "Opened output file \"model_536.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_537.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_537.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_537.h . . .\n",
      "Opened output file \"model_537.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_538.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_538.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_538.h . . .\n",
      "Opened output file \"model_538.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_539.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_539.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_539.h . . .\n",
      "Opened output file \"model_539.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_54.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_54.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_54.h . . .\n",
      "Opened output file \"model_54.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_540.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_540.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_540.h . . .\n",
      "Opened output file \"model_540.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_541.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_541.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_541.h . . .\n",
      "Opened output file \"model_541.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_542.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_542.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_542.h . . .\n",
      "Opened output file \"model_542.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_543.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_543.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_543.h . . .\n",
      "Opened output file \"model_543.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_544.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_544.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_544.h . . .\n",
      "Opened output file \"model_544.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:03 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_545.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_545.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_545.h . . .\n",
      "Opened output file \"model_545.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_546.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_546.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_546.h . . .\n",
      "Opened output file \"model_546.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_547.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_547.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_547.h . . .\n",
      "Opened output file \"model_547.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_548.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_548.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_548.h . . .\n",
      "Opened output file \"model_548.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_549.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_549.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_549.h . . .\n",
      "Opened output file \"model_549.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_55.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_55.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_55.h . . .\n",
      "Opened output file \"model_55.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_550.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_550.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_550.h . . .\n",
      "Opened output file \"model_550.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_551.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_551.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_551.h . . .\n",
      "Opened output file \"model_551.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_552.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_552.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_552.h . . .\n",
      "Opened output file \"model_552.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_553.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_553.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_553.h . . .\n",
      "Opened output file \"model_553.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_554.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_554.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_554.h . . .\n",
      "Opened output file \"model_554.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_555.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_555.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_555.h . . .\n",
      "Opened output file \"model_555.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:04 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_556.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_556.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_556.h . . .\n",
      "Opened output file \"model_556.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_557.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_557.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_557.h . . .\n",
      "Opened output file \"model_557.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_558.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_558.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_558.h . . .\n",
      "Opened output file \"model_558.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_559.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_559.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_559.h . . .\n",
      "Opened output file \"model_559.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_56.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_56.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_56.h . . .\n",
      "Opened output file \"model_56.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_560.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_560.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_560.h . . .\n",
      "Opened output file \"model_560.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_561.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_561.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_561.h . . .\n",
      "Opened output file \"model_561.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_562.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_562.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_562.h . . .\n",
      "Opened output file \"model_562.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_563.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_563.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_563.h . . .\n",
      "Opened output file \"model_563.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_564.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_564.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_564.h . . .\n",
      "Opened output file \"model_564.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_565.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_565.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_565.h . . .\n",
      "Opened output file \"model_565.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_566.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_566.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_566.h . . .\n",
      "Opened output file \"model_566.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:05 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_567.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_567.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_567.h . . .\n",
      "Opened output file \"model_567.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_568.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_568.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_568.h . . .\n",
      "Opened output file \"model_568.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_569.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_569.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_569.h . . .\n",
      "Opened output file \"model_569.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_57.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_57.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_57.h . . .\n",
      "Opened output file \"model_57.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_570.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_570.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_570.h . . .\n",
      "Opened output file \"model_570.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_571.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_571.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_571.h . . .\n",
      "Opened output file \"model_571.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_572.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_572.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_572.h . . .\n",
      "Opened output file \"model_572.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_573.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_573.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_573.h . . .\n",
      "Opened output file \"model_573.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_574.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_574.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_574.h . . .\n",
      "Opened output file \"model_574.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_575.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_575.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_575.h . . .\n",
      "Opened output file \"model_575.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_576.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_576.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_576.h . . .\n",
      "Opened output file \"model_576.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_577.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_577.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_577.h . . .\n",
      "Opened output file \"model_577.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:06 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_578.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_578.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_578.h . . .\n",
      "Opened output file \"model_578.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_579.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_579.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_579.h . . .\n",
      "Opened output file \"model_579.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_58.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_58.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_58.h . . .\n",
      "Opened output file \"model_58.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_580.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_580.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_580.h . . .\n",
      "Opened output file \"model_580.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_581.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_581.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_581.h . . .\n",
      "Opened output file \"model_581.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_582.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_582.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_582.h . . .\n",
      "Opened output file \"model_582.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_583.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_583.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_583.h . . .\n",
      "Opened output file \"model_583.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_584.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_584.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_584.h . . .\n",
      "Opened output file \"model_584.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_585.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_585.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_585.h . . .\n",
      "Opened output file \"model_585.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_586.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_586.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_586.h . . .\n",
      "Opened output file \"model_586.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_587.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_587.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_587.h . . .\n",
      "Opened output file \"model_587.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:07 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_588.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_588.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_588.h . . .\n",
      "Opened output file \"model_588.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_589.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_589.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_589.h . . .\n",
      "Opened output file \"model_589.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_59.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_59.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_59.h . . .\n",
      "Opened output file \"model_59.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_590.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_590.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_590.h . . .\n",
      "Opened output file \"model_590.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_591.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_591.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_591.h . . .\n",
      "Opened output file \"model_591.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_592.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_592.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_592.h . . .\n",
      "Opened output file \"model_592.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_593.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_593.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_593.h . . .\n",
      "Opened output file \"model_593.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_594.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_594.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_594.h . . .\n",
      "Opened output file \"model_594.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_595.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_595.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_595.h . . .\n",
      "Opened output file \"model_595.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_596.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_596.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_596.h . . .\n",
      "Opened output file \"model_596.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_597.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_597.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_597.h . . .\n",
      "Opened output file \"model_597.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_598.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_598.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_598.h . . .\n",
      "Opened output file \"model_598.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:08 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_599.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_599.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_599.h . . .\n",
      "Opened output file \"model_599.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_6.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_6.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_6.h . . .\n",
      "Opened output file \"model_6.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_60.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_60.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_60.h . . .\n",
      "Opened output file \"model_60.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_600.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_600.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_600.h . . .\n",
      "Opened output file \"model_600.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_601.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_601.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_601.h . . .\n",
      "Opened output file \"model_601.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_602.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_602.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_602.h . . .\n",
      "Opened output file \"model_602.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_603.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_603.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_603.h . . .\n",
      "Opened output file \"model_603.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_604.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_604.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_604.h . . .\n",
      "Opened output file \"model_604.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_605.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_605.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_605.h . . .\n",
      "Opened output file \"model_605.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_606.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_606.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_606.h . . .\n",
      "Opened output file \"model_606.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_607.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_607.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_607.h . . .\n",
      "Opened output file \"model_607.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:09 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_608.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_608.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_608.h . . .\n",
      "Opened output file \"model_608.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_609.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_609.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_609.h . . .\n",
      "Opened output file \"model_609.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_61.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_61.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_61.h . . .\n",
      "Opened output file \"model_61.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_610.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_610.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_610.h . . .\n",
      "Opened output file \"model_610.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_611.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_611.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_611.h . . .\n",
      "Opened output file \"model_611.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_612.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_612.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_612.h . . .\n",
      "Opened output file \"model_612.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_613.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_613.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_613.h . . .\n",
      "Opened output file \"model_613.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_614.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_614.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_614.h . . .\n",
      "Opened output file \"model_614.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_615.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_615.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_615.h . . .\n",
      "Opened output file \"model_615.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_616.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_616.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_616.h . . .\n",
      "Opened output file \"model_616.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:10 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_617.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_617.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_617.h . . .\n",
      "Opened output file \"model_617.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_618.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_618.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_618.h . . .\n",
      "Opened output file \"model_618.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_619.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_619.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_619.h . . .\n",
      "Opened output file \"model_619.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_62.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_62.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_62.h . . .\n",
      "Opened output file \"model_62.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_620.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_620.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_620.h . . .\n",
      "Opened output file \"model_620.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_621.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_621.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_621.h . . .\n",
      "Opened output file \"model_621.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_622.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_622.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_622.h . . .\n",
      "Opened output file \"model_622.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_623.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_623.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_623.h . . .\n",
      "Opened output file \"model_623.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_624.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_624.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_624.h . . .\n",
      "Opened output file \"model_624.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_625.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_625.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_625.h . . .\n",
      "Opened output file \"model_625.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_626.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_626.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_626.h . . .\n",
      "Opened output file \"model_626.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_627.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_627.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_627.h . . .\n",
      "Opened output file \"model_627.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:11 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_628.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_628.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_628.h . . .\n",
      "Opened output file \"model_628.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_629.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_629.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_629.h . . .\n",
      "Opened output file \"model_629.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_63.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_63.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_63.h . . .\n",
      "Opened output file \"model_63.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_630.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_630.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_630.h . . .\n",
      "Opened output file \"model_630.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_631.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_631.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_631.h . . .\n",
      "Opened output file \"model_631.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_632.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_632.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_632.h . . .\n",
      "Opened output file \"model_632.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_633.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_633.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_633.h . . .\n",
      "Opened output file \"model_633.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_634.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_634.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_634.h . . .\n",
      "Opened output file \"model_634.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_635.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_635.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_635.h . . .\n",
      "Opened output file \"model_635.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_636.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_636.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_636.h . . .\n",
      "Opened output file \"model_636.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_637.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_637.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_637.h . . .\n",
      "Opened output file \"model_637.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:12 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_638.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_638.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_638.h . . .\n",
      "Opened output file \"model_638.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_639.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_639.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_639.h . . .\n",
      "Opened output file \"model_639.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_64.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_64.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_64.h . . .\n",
      "Opened output file \"model_64.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_640.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_640.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_640.h . . .\n",
      "Opened output file \"model_640.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_641.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_641.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_641.h . . .\n",
      "Opened output file \"model_641.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_642.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_642.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_642.h . . .\n",
      "Opened output file \"model_642.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_643.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_643.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_643.h . . .\n",
      "Opened output file \"model_643.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_644.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_644.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_644.h . . .\n",
      "Opened output file \"model_644.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_645.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_645.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_645.h . . .\n",
      "Opened output file \"model_645.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_646.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_646.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_646.h . . .\n",
      "Opened output file \"model_646.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_647.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_647.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_647.h . . .\n",
      "Opened output file \"model_647.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:13 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_648.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_648.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_648.h . . .\n",
      "Opened output file \"model_648.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_649.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_649.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_649.h . . .\n",
      "Opened output file \"model_649.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_65.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_65.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_65.h . . .\n",
      "Opened output file \"model_65.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_650.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_650.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_650.h . . .\n",
      "Opened output file \"model_650.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_651.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_651.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_651.h . . .\n",
      "Opened output file \"model_651.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_652.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_652.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_652.h . . .\n",
      "Opened output file \"model_652.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_653.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_653.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_653.h . . .\n",
      "Opened output file \"model_653.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_654.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_654.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_654.h . . .\n",
      "Opened output file \"model_654.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_655.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_655.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_655.h . . .\n",
      "Opened output file \"model_655.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_656.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_656.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_656.h . . .\n",
      "Opened output file \"model_656.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_657.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_657.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_657.h . . .\n",
      "Opened output file \"model_657.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:14 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_658.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_658.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_658.h . . .\n",
      "Opened output file \"model_658.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_659.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_659.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_659.h . . .\n",
      "Opened output file \"model_659.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_66.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_66.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_66.h . . .\n",
      "Opened output file \"model_66.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_660.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_660.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_660.h . . .\n",
      "Opened output file \"model_660.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_661.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_661.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_661.h . . .\n",
      "Opened output file \"model_661.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_662.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_662.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_662.h . . .\n",
      "Opened output file \"model_662.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_663.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_663.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_663.h . . .\n",
      "Opened output file \"model_663.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_664.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_664.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_664.h . . .\n",
      "Opened output file \"model_664.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_665.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_665.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_665.h . . .\n",
      "Opened output file \"model_665.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_666.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_666.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_666.h . . .\n",
      "Opened output file \"model_666.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_667.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_667.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_667.h . . .\n",
      "Opened output file \"model_667.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:15 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_668.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_668.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_668.h . . .\n",
      "Opened output file \"model_668.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_669.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_669.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_669.h . . .\n",
      "Opened output file \"model_669.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_67.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_67.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_67.h . . .\n",
      "Opened output file \"model_67.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_670.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_670.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_670.h . . .\n",
      "Opened output file \"model_670.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_671.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_671.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_671.h . . .\n",
      "Opened output file \"model_671.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_672.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_672.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_672.h . . .\n",
      "Opened output file \"model_672.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_673.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_673.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_673.h . . .\n",
      "Opened output file \"model_673.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_674.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_674.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_674.h . . .\n",
      "Opened output file \"model_674.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_675.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_675.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_675.h . . .\n",
      "Opened output file \"model_675.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_676.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_676.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_676.h . . .\n",
      "Opened output file \"model_676.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_677.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_677.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_677.h . . .\n",
      "Opened output file \"model_677.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:16 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_678.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_678.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_678.h . . .\n",
      "Opened output file \"model_678.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_679.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_679.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_679.h . . .\n",
      "Opened output file \"model_679.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_68.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_68.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_68.h . . .\n",
      "Opened output file \"model_68.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_680.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_680.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_680.h . . .\n",
      "Opened output file \"model_680.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_681.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_681.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_681.h . . .\n",
      "Opened output file \"model_681.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_682.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_682.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_682.h . . .\n",
      "Opened output file \"model_682.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_683.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_683.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_683.h . . .\n",
      "Opened output file \"model_683.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_684.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_684.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_684.h . . .\n",
      "Opened output file \"model_684.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_685.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_685.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_685.h . . .\n",
      "Opened output file \"model_685.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_686.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_686.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_686.h . . .\n",
      "Opened output file \"model_686.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_687.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_687.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_687.h . . .\n",
      "Opened output file \"model_687.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_688.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_688.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_688.h . . .\n",
      "Opened output file \"model_688.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:17 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_689.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_689.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_689.h . . .\n",
      "Opened output file \"model_689.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_69.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_69.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_69.h . . .\n",
      "Opened output file \"model_69.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_690.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_690.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_690.h . . .\n",
      "Opened output file \"model_690.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_691.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_691.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_691.h . . .\n",
      "Opened output file \"model_691.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_692.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_692.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_692.h . . .\n",
      "Opened output file \"model_692.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_693.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_693.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_693.h . . .\n",
      "Opened output file \"model_693.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_694.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_694.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_694.h . . .\n",
      "Opened output file \"model_694.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_695.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_695.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_695.h . . .\n",
      "Opened output file \"model_695.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_696.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_696.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_696.h . . .\n",
      "Opened output file \"model_696.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_697.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_697.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_697.h . . .\n",
      "Opened output file \"model_697.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_698.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_698.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_698.h . . .\n",
      "Opened output file \"model_698.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:18 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_699.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_699.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_699.h . . .\n",
      "Opened output file \"model_699.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_7.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_7.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_7.h . . .\n",
      "Opened output file \"model_7.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_70.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_70.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_70.h . . .\n",
      "Opened output file \"model_70.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_700.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_700.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_700.h . . .\n",
      "Opened output file \"model_700.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_701.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_701.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_701.h . . .\n",
      "Opened output file \"model_701.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_702.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_702.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_702.h . . .\n",
      "Opened output file \"model_702.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_703.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_703.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_703.h . . .\n",
      "Opened output file \"model_703.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_704.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_704.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_704.h . . .\n",
      "Opened output file \"model_704.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_705.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_705.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_705.h . . .\n",
      "Opened output file \"model_705.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_706.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_706.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_706.h . . .\n",
      "Opened output file \"model_706.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_707.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_707.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_707.h . . .\n",
      "Opened output file \"model_707.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:19 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_708.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_708.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_708.h . . .\n",
      "Opened output file \"model_708.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_709.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_709.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_709.h . . .\n",
      "Opened output file \"model_709.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_71.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_71.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_71.h . . .\n",
      "Opened output file \"model_71.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_710.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_710.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_710.h . . .\n",
      "Opened output file \"model_710.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_711.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_711.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_711.h . . .\n",
      "Opened output file \"model_711.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_712.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_712.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_712.h . . .\n",
      "Opened output file \"model_712.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_713.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_713.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_713.h . . .\n",
      "Opened output file \"model_713.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_714.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_714.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_714.h . . .\n",
      "Opened output file \"model_714.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_715.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_715.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_715.h . . .\n",
      "Opened output file \"model_715.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_716.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_716.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_716.h . . .\n",
      "Opened output file \"model_716.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_717.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_717.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_717.h . . .\n",
      "Opened output file \"model_717.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:20 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_718.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_718.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_718.h . . .\n",
      "Opened output file \"model_718.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_719.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_719.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_719.h . . .\n",
      "Opened output file \"model_719.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_72.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_72.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_72.h . . .\n",
      "Opened output file \"model_72.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_720.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_720.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_720.h . . .\n",
      "Opened output file \"model_720.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_721.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_721.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_721.h . . .\n",
      "Opened output file \"model_721.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_722.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_722.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_722.h . . .\n",
      "Opened output file \"model_722.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_723.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_723.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_723.h . . .\n",
      "Opened output file \"model_723.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "281 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_724.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_724.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_724.h . . .\n",
      "Opened output file \"model_724.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_725.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_725.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_725.h . . .\n",
      "Opened output file \"model_725.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_726.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_726.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_726.h . . .\n",
      "Opened output file \"model_726.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_727.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_727.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_727.h . . .\n",
      "Opened output file \"model_727.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:21 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_728.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_728.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_728.h . . .\n",
      "Opened output file \"model_728.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_729.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_729.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_729.h . . .\n",
      "Opened output file \"model_729.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_73.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_73.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_73.h . . .\n",
      "Opened output file \"model_73.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_730.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_730.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_730.h . . .\n",
      "Opened output file \"model_730.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_731.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_731.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_731.h . . .\n",
      "Opened output file \"model_731.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_732.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_732.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_732.h . . .\n",
      "Opened output file \"model_732.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_733.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_733.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_733.h . . .\n",
      "Opened output file \"model_733.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_734.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_734.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_734.h . . .\n",
      "Opened output file \"model_734.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_735.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_735.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_735.h . . .\n",
      "Opened output file \"model_735.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_736.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_736.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_736.h . . .\n",
      "Opened output file \"model_736.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_737.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_737.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_737.h . . .\n",
      "Opened output file \"model_737.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:22 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_738.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_738.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_738.h . . .\n",
      "Opened output file \"model_738.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_739.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_739.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_739.h . . .\n",
      "Opened output file \"model_739.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_74.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_74.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_74.h . . .\n",
      "Opened output file \"model_74.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_740.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_740.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_740.h . . .\n",
      "Opened output file \"model_740.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_741.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_741.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_741.h . . .\n",
      "Opened output file \"model_741.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_742.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_742.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_742.h . . .\n",
      "Opened output file \"model_742.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_743.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_743.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_743.h . . .\n",
      "Opened output file \"model_743.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_744.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_744.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_744.h . . .\n",
      "Opened output file \"model_744.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_745.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_745.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_745.h . . .\n",
      "Opened output file \"model_745.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_746.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_746.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_746.h . . .\n",
      "Opened output file \"model_746.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_747.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_747.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_747.h . . .\n",
      "Opened output file \"model_747.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:23 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_748.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_748.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_748.h . . .\n",
      "Opened output file \"model_748.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_749.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_749.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_749.h . . .\n",
      "Opened output file \"model_749.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_75.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_75.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_75.h . . .\n",
      "Opened output file \"model_75.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_750.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_750.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_750.h . . .\n",
      "Opened output file \"model_750.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_751.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_751.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_751.h . . .\n",
      "Opened output file \"model_751.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_752.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_752.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_752.h . . .\n",
      "Opened output file \"model_752.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_753.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_753.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_753.h . . .\n",
      "Opened output file \"model_753.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_754.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_754.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_754.h . . .\n",
      "Opened output file \"model_754.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_755.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_755.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_755.h . . .\n",
      "Opened output file \"model_755.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_756.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_756.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_756.h . . .\n",
      "Opened output file \"model_756.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:24 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_757.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_757.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_757.h . . .\n",
      "Opened output file \"model_757.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_758.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_758.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_758.h . . .\n",
      "Opened output file \"model_758.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_759.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_759.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_759.h . . .\n",
      "Opened output file \"model_759.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_76.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_76.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_76.h . . .\n",
      "Opened output file \"model_76.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_760.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_760.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_760.h . . .\n",
      "Opened output file \"model_760.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_761.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_761.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_761.h . . .\n",
      "Opened output file \"model_761.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_762.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_762.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_762.h . . .\n",
      "Opened output file \"model_762.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_763.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_763.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_763.h . . .\n",
      "Opened output file \"model_763.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_764.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_764.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_764.h . . .\n",
      "Opened output file \"model_764.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_765.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_765.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_765.h . . .\n",
      "Opened output file \"model_765.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:25 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_766.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_766.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_766.h . . .\n",
      "Opened output file \"model_766.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_767.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_767.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_767.h . . .\n",
      "Opened output file \"model_767.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_768.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_768.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_768.h . . .\n",
      "Opened output file \"model_768.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_769.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_769.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_769.h . . .\n",
      "Opened output file \"model_769.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_77.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_77.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_77.h . . .\n",
      "Opened output file \"model_77.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_770.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_770.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_770.h . . .\n",
      "Opened output file \"model_770.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_771.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_771.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_771.h . . .\n",
      "Opened output file \"model_771.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_772.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_772.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_772.h . . .\n",
      "Opened output file \"model_772.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_773.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_773.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_773.h . . .\n",
      "Opened output file \"model_773.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_774.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_774.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_774.h . . .\n",
      "Opened output file \"model_774.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_775.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_775.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_775.h . . .\n",
      "Opened output file \"model_775.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:26 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_776.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_776.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_776.h . . .\n",
      "Opened output file \"model_776.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_777.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_777.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_777.h . . .\n",
      "Opened output file \"model_777.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_778.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_778.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_778.h . . .\n",
      "Opened output file \"model_778.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_779.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_779.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_779.h . . .\n",
      "Opened output file \"model_779.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_78.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_78.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_78.h . . .\n",
      "Opened output file \"model_78.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_780.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_780.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_780.h . . .\n",
      "Opened output file \"model_780.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_781.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_781.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_781.h . . .\n",
      "Opened output file \"model_781.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_782.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_782.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_782.h . . .\n",
      "Opened output file \"model_782.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_783.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_783.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_783.h . . .\n",
      "Opened output file \"model_783.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_784.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_784.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_784.h . . .\n",
      "Opened output file \"model_784.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_785.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_785.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_785.h . . .\n",
      "Opened output file \"model_785.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:27 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_786.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_786.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_786.h . . .\n",
      "Opened output file \"model_786.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_787.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_787.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_787.h . . .\n",
      "Opened output file \"model_787.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_788.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_788.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_788.h . . .\n",
      "Opened output file \"model_788.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_789.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_789.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_789.h . . .\n",
      "Opened output file \"model_789.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_79.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_79.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_79.h . . .\n",
      "Opened output file \"model_79.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_790.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_790.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_790.h . . .\n",
      "Opened output file \"model_790.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_791.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_791.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_791.h . . .\n",
      "Opened output file \"model_791.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_792.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_792.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_792.h . . .\n",
      "Opened output file \"model_792.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_793.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_793.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_793.h . . .\n",
      "Opened output file \"model_793.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_794.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_794.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_794.h . . .\n",
      "Opened output file \"model_794.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:28 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_795.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_795.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_795.h . . .\n",
      "Opened output file \"model_795.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_796.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_796.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_796.h . . .\n",
      "Opened output file \"model_796.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_797.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_797.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_797.h . . .\n",
      "Opened output file \"model_797.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_798.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_798.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_798.h . . .\n",
      "Opened output file \"model_798.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_799.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_799.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_799.h . . .\n",
      "Opened output file \"model_799.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_8.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_8.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_8.h . . .\n",
      "Opened output file \"model_8.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_80.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_80.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_80.h . . .\n",
      "Opened output file \"model_80.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_800.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_800.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_800.h . . .\n",
      "Opened output file \"model_800.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_801.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_801.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_801.h . . .\n",
      "Opened output file \"model_801.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_802.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_802.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_802.h . . .\n",
      "Opened output file \"model_802.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_803.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_803.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_803.h . . .\n",
      "Opened output file \"model_803.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:29 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_804.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_804.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_804.h . . .\n",
      "Opened output file \"model_804.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_805.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_805.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_805.h . . .\n",
      "Opened output file \"model_805.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_806.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_806.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_806.h . . .\n",
      "Opened output file \"model_806.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_807.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_807.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_807.h . . .\n",
      "Opened output file \"model_807.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_808.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_808.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_808.h . . .\n",
      "Opened output file \"model_808.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_809.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_809.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_809.h . . .\n",
      "Opened output file \"model_809.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_81.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_81.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_81.h . . .\n",
      "Opened output file \"model_81.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_810.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_810.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_810.h . . .\n",
      "Opened output file \"model_810.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_811.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_811.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_811.h . . .\n",
      "Opened output file \"model_811.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_812.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_812.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_812.h . . .\n",
      "Opened output file \"model_812.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_813.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_813.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_813.h . . .\n",
      "Opened output file \"model_813.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:30 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_814.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_814.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_814.h . . .\n",
      "Opened output file \"model_814.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_815.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_815.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_815.h . . .\n",
      "Opened output file \"model_815.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_816.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_816.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_816.h . . .\n",
      "Opened output file \"model_816.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_817.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_817.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_817.h . . .\n",
      "Opened output file \"model_817.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_818.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_818.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_818.h . . .\n",
      "Opened output file \"model_818.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_819.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_819.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_819.h . . .\n",
      "Opened output file \"model_819.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_82.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_82.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_82.h . . .\n",
      "Opened output file \"model_82.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_820.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_820.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_820.h . . .\n",
      "Opened output file \"model_820.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_821.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_821.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_821.h . . .\n",
      "Opened output file \"model_821.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_822.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_822.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_822.h . . .\n",
      "Opened output file \"model_822.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:31 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_823.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_823.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_823.h . . .\n",
      "Opened output file \"model_823.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_824.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_824.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_824.h . . .\n",
      "Opened output file \"model_824.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_825.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_825.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_825.h . . .\n",
      "Opened output file \"model_825.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_826.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_826.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_826.h . . .\n",
      "Opened output file \"model_826.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_827.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_827.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_827.h . . .\n",
      "Opened output file \"model_827.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_828.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_828.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_828.h . . .\n",
      "Opened output file \"model_828.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_829.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_829.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_829.h . . .\n",
      "Opened output file \"model_829.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_83.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_83.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_83.h . . .\n",
      "Opened output file \"model_83.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_830.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_830.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_830.h . . .\n",
      "Opened output file \"model_830.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_831.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_831.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_831.h . . .\n",
      "Opened output file \"model_831.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_832.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_832.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_832.h . . .\n",
      "Opened output file \"model_832.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:32 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_833.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_833.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_833.h . . .\n",
      "Opened output file \"model_833.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_834.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_834.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_834.h . . .\n",
      "Opened output file \"model_834.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_835.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_835.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_835.h . . .\n",
      "Opened output file \"model_835.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_836.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_836.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_836.h . . .\n",
      "Opened output file \"model_836.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_837.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_837.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_837.h . . .\n",
      "Opened output file \"model_837.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_838.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_838.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_838.h . . .\n",
      "Opened output file \"model_838.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_839.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_839.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_839.h . . .\n",
      "Opened output file \"model_839.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "279 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_84.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_84.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_84.h . . .\n",
      "Opened output file \"model_84.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_840.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_840.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_840.h . . .\n",
      "Opened output file \"model_840.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_841.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_841.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_841.h . . .\n",
      "Opened output file \"model_841.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_842.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_842.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_842.h . . .\n",
      "Opened output file \"model_842.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:33 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_843.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_843.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_843.h . . .\n",
      "Opened output file \"model_843.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_844.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_844.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_844.h . . .\n",
      "Opened output file \"model_844.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_845.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_845.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_845.h . . .\n",
      "Opened output file \"model_845.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_846.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_846.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_846.h . . .\n",
      "Opened output file \"model_846.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_847.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_847.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_847.h . . .\n",
      "Opened output file \"model_847.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_848.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_848.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_848.h . . .\n",
      "Opened output file \"model_848.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_849.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_849.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_849.h . . .\n",
      "Opened output file \"model_849.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_85.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_85.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_85.h . . .\n",
      "Opened output file \"model_85.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_850.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_850.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_850.h . . .\n",
      "Opened output file \"model_850.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_851.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_851.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_851.h . . .\n",
      "Opened output file \"model_851.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_852.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_852.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_852.h . . .\n",
      "Opened output file \"model_852.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:34 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_853.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_853.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_853.h . . .\n",
      "Opened output file \"model_853.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_854.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_854.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_854.h . . .\n",
      "Opened output file \"model_854.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_855.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_855.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_855.h . . .\n",
      "Opened output file \"model_855.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_856.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_856.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_856.h . . .\n",
      "Opened output file \"model_856.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_857.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_857.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_857.h . . .\n",
      "Opened output file \"model_857.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_858.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_858.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_858.h . . .\n",
      "Opened output file \"model_858.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_859.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_859.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_859.h . . .\n",
      "Opened output file \"model_859.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_86.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_86.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_86.h . . .\n",
      "Opened output file \"model_86.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_860.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_860.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_860.h . . .\n",
      "Opened output file \"model_860.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_861.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_861.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_861.h . . .\n",
      "Opened output file \"model_861.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_862.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_862.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_862.h . . .\n",
      "Opened output file \"model_862.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:35 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_863.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_863.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_863.h . . .\n",
      "Opened output file \"model_863.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_864.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_864.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_864.h . . .\n",
      "Opened output file \"model_864.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_865.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_865.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_865.h . . .\n",
      "Opened output file \"model_865.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_866.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_866.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_866.h . . .\n",
      "Opened output file \"model_866.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_867.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_867.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_867.h . . .\n",
      "Opened output file \"model_867.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_868.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_868.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_868.h . . .\n",
      "Opened output file \"model_868.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_869.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_869.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_869.h . . .\n",
      "Opened output file \"model_869.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_87.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_87.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_87.h . . .\n",
      "Opened output file \"model_87.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_870.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_870.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_870.h . . .\n",
      "Opened output file \"model_870.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_871.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_871.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_871.h . . .\n",
      "Opened output file \"model_871.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_872.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_872.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_872.h . . .\n",
      "Opened output file \"model_872.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:36 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_873.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_873.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_873.h . . .\n",
      "Opened output file \"model_873.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_874.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_874.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_874.h . . .\n",
      "Opened output file \"model_874.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_875.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_875.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_875.h . . .\n",
      "Opened output file \"model_875.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_876.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_876.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_876.h . . .\n",
      "Opened output file \"model_876.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_877.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_877.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_877.h . . .\n",
      "Opened output file \"model_877.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_878.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_878.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_878.h . . .\n",
      "Opened output file \"model_878.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_879.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_879.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_879.h . . .\n",
      "Opened output file \"model_879.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_88.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_88.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_88.h . . .\n",
      "Opened output file \"model_88.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_880.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_880.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_880.h . . .\n",
      "Opened output file \"model_880.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_881.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_881.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_881.h . . .\n",
      "Opened output file \"model_881.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:37 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_882.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_882.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_882.h . . .\n",
      "Opened output file \"model_882.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_883.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_883.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_883.h . . .\n",
      "Opened output file \"model_883.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_884.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_884.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_884.h . . .\n",
      "Opened output file \"model_884.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_885.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_885.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_885.h . . .\n",
      "Opened output file \"model_885.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_886.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_886.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_886.h . . .\n",
      "Opened output file \"model_886.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_887.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_887.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_887.h . . .\n",
      "Opened output file \"model_887.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_888.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_888.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_888.h . . .\n",
      "Opened output file \"model_888.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_889.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_889.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_889.h . . .\n",
      "Opened output file \"model_889.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_89.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_89.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_89.h . . .\n",
      "Opened output file \"model_89.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_890.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_890.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_890.h . . .\n",
      "Opened output file \"model_890.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_891.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_891.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_891.h . . .\n",
      "Opened output file \"model_891.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:38 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_892.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_892.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_892.h . . .\n",
      "Opened output file \"model_892.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_893.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_893.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_893.h . . .\n",
      "Opened output file \"model_893.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_894.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_894.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_894.h . . .\n",
      "Opened output file \"model_894.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_895.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_895.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_895.h . . .\n",
      "Opened output file \"model_895.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_896.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_896.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_896.h . . .\n",
      "Opened output file \"model_896.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_897.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_897.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_897.h . . .\n",
      "Opened output file \"model_897.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_898.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_898.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_898.h . . .\n",
      "Opened output file \"model_898.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_899.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_899.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_899.h . . .\n",
      "Opened output file \"model_899.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_9.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_9.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_9.h . . .\n",
      "Opened output file \"model_9.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_90.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_90.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_90.h . . .\n",
      "Opened output file \"model_90.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_900.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_900.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_900.h . . .\n",
      "Opened output file \"model_900.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:39 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_901.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_901.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_901.h . . .\n",
      "Opened output file \"model_901.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_902.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_902.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_902.h . . .\n",
      "Opened output file \"model_902.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_903.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_903.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_903.h . . .\n",
      "Opened output file \"model_903.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_904.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_904.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_904.h . . .\n",
      "Opened output file \"model_904.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "281 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_905.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_905.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_905.h . . .\n",
      "Opened output file \"model_905.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_906.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_906.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_906.h . . .\n",
      "Opened output file \"model_906.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_907.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_907.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_907.h . . .\n",
      "Opened output file \"model_907.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_908.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_908.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_908.h . . .\n",
      "Opened output file \"model_908.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_909.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_909.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_909.h . . .\n",
      "Opened output file \"model_909.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_91.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_91.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_91.h . . .\n",
      "Opened output file \"model_91.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_910.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_910.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_910.h . . .\n",
      "Opened output file \"model_910.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:40 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_911.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_911.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_911.h . . .\n",
      "Opened output file \"model_911.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_912.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_912.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_912.h . . .\n",
      "Opened output file \"model_912.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_913.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_913.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_913.h . . .\n",
      "Opened output file \"model_913.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_914.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_914.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_914.h . . .\n",
      "Opened output file \"model_914.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_915.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_915.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_915.h . . .\n",
      "Opened output file \"model_915.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_916.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_916.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_916.h . . .\n",
      "Opened output file \"model_916.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_917.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_917.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_917.h . . .\n",
      "Opened output file \"model_917.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_918.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_918.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_918.h . . .\n",
      "Opened output file \"model_918.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_919.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_919.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_919.h . . .\n",
      "Opened output file \"model_919.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_92.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_92.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_92.h . . .\n",
      "Opened output file \"model_92.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_920.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_920.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_920.h . . .\n",
      "Opened output file \"model_920.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:41 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_921.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_921.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_921.h . . .\n",
      "Opened output file \"model_921.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_922.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_922.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_922.h . . .\n",
      "Opened output file \"model_922.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_923.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_923.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_923.h . . .\n",
      "Opened output file \"model_923.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_924.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_924.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_924.h . . .\n",
      "Opened output file \"model_924.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_925.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_925.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_925.h . . .\n",
      "Opened output file \"model_925.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_926.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_926.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_926.h . . .\n",
      "Opened output file \"model_926.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_927.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_927.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_927.h . . .\n",
      "Opened output file \"model_927.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_928.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_928.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_928.h . . .\n",
      "Opened output file \"model_928.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_929.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_929.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_929.h . . .\n",
      "Opened output file \"model_929.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_93.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_93.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_93.h . . .\n",
      "Opened output file \"model_93.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:42 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_930.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_930.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_930.h . . .\n",
      "Opened output file \"model_930.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_931.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_931.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_931.h . . .\n",
      "Opened output file \"model_931.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_932.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_932.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_932.h . . .\n",
      "Opened output file \"model_932.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_933.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_933.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_933.h . . .\n",
      "Opened output file \"model_933.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_934.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_934.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_934.h . . .\n",
      "Opened output file \"model_934.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_935.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_935.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_935.h . . .\n",
      "Opened output file \"model_935.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_936.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_936.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_936.h . . .\n",
      "Opened output file \"model_936.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_937.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_937.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_937.h . . .\n",
      "Opened output file \"model_937.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_938.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_938.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_938.h . . .\n",
      "Opened output file \"model_938.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_939.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_939.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_939.h . . .\n",
      "Opened output file \"model_939.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_94.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_94.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_94.h . . .\n",
      "Opened output file \"model_94.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:43 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_940.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_940.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_940.h . . .\n",
      "Opened output file \"model_940.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_941.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_941.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_941.h . . .\n",
      "Opened output file \"model_941.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_942.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_942.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_942.h . . .\n",
      "Opened output file \"model_942.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_943.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_943.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_943.h . . .\n",
      "Opened output file \"model_943.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_944.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_944.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_944.h . . .\n",
      "Opened output file \"model_944.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_945.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_945.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_945.h . . .\n",
      "Opened output file \"model_945.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "281 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_946.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_946.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_946.h . . .\n",
      "Opened output file \"model_946.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_947.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_947.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_947.h . . .\n",
      "Opened output file \"model_947.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_948.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_948.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_948.h . . .\n",
      "Opened output file \"model_948.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_949.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_949.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_949.h . . .\n",
      "Opened output file \"model_949.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:44 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_95.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_95.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_95.h . . .\n",
      "Opened output file \"model_95.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_950.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_950.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_950.h . . .\n",
      "Opened output file \"model_950.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "271 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_951.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_951.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_951.h . . .\n",
      "Opened output file \"model_951.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_952.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_952.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_952.h . . .\n",
      "Opened output file \"model_952.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_953.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_953.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_953.h . . .\n",
      "Opened output file \"model_953.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_954.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_954.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_954.h . . .\n",
      "Opened output file \"model_954.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_955.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_955.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_955.h . . .\n",
      "Opened output file \"model_955.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_956.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_956.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_956.h . . .\n",
      "Opened output file \"model_956.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_957.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_957.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_957.h . . .\n",
      "Opened output file \"model_957.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_958.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_958.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_958.h . . .\n",
      "Opened output file \"model_958.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_959.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_959.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_959.h . . .\n",
      "Opened output file \"model_959.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:45 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_96.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_96.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_96.h . . .\n",
      "Opened output file \"model_96.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_960.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_960.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_960.h . . .\n",
      "Opened output file \"model_960.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_961.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_961.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_961.h . . .\n",
      "Opened output file \"model_961.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_962.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_962.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_962.h . . .\n",
      "Opened output file \"model_962.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_963.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_963.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_963.h . . .\n",
      "Opened output file \"model_963.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_964.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_964.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_964.h . . .\n",
      "Opened output file \"model_964.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_965.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_965.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_965.h . . .\n",
      "Opened output file \"model_965.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_966.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_966.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_966.h . . .\n",
      "Opened output file \"model_966.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_967.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_967.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_967.h . . .\n",
      "Opened output file \"model_967.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_968.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_968.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_968.h . . .\n",
      "Opened output file \"model_968.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:46 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_969.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_969.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_969.h . . .\n",
      "Opened output file \"model_969.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_97.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_97.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_97.h . . .\n",
      "Opened output file \"model_97.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_970.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_970.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_970.h . . .\n",
      "Opened output file \"model_970.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_971.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_971.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_971.h . . .\n",
      "Opened output file \"model_971.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_972.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_972.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_972.h . . .\n",
      "Opened output file \"model_972.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_973.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_973.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_973.h . . .\n",
      "Opened output file \"model_973.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_974.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_974.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_974.h . . .\n",
      "Opened output file \"model_974.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_975.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_975.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_975.h . . .\n",
      "Opened output file \"model_975.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_976.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_976.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_976.h . . .\n",
      "Opened output file \"model_976.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_977.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_977.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_977.h . . .\n",
      "Opened output file \"model_977.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:47 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_978.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_978.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_978.h . . .\n",
      "Opened output file \"model_978.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_979.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_979.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_979.h . . .\n",
      "Opened output file \"model_979.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_98.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_98.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_98.h . . .\n",
      "Opened output file \"model_98.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_980.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_980.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_980.h . . .\n",
      "Opened output file \"model_980.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_981.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_981.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_981.h . . .\n",
      "Opened output file \"model_981.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "278 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_982.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_982.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_982.h . . .\n",
      "Opened output file \"model_982.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_983.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_983.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_983.h . . .\n",
      "Opened output file \"model_983.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_984.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_984.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_984.h . . .\n",
      "Opened output file \"model_984.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_985.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_985.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_985.h . . .\n",
      "Opened output file \"model_985.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_986.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_986.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_986.h . . .\n",
      "Opened output file \"model_986.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:48 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_987.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_987.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_987.h . . .\n",
      "Opened output file \"model_987.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_988.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_988.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_988.h . . .\n",
      "Opened output file \"model_988.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "277 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_989.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_989.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_989.h . . .\n",
      "Opened output file \"model_989.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_99.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_99.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_99.h . . .\n",
      "Opened output file \"model_99.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_990.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_990.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_990.h . . .\n",
      "Opened output file \"model_990.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_991.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_991.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_991.h . . .\n",
      "Opened output file \"model_991.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_992.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_992.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_992.h . . .\n",
      "Opened output file \"model_992.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_993.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_993.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_993.h . . .\n",
      "Opened output file \"model_993.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_994.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_994.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_994.h . . .\n",
      "Opened output file \"model_994.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "275 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_995.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_995.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_995.h . . .\n",
      "Opened output file \"model_995.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "274 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_996.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_996.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_996.h . . .\n",
      "Opened output file \"model_996.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "272 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:49 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_997.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_997.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_997.h . . .\n",
      "Opened output file \"model_997.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_998.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_998.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_998.h . . .\n",
      "Opened output file \"model_998.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "276 hydrogen bonds found.\n",
      "HBPLUS Hydrogen Bond Calculator v 3.2            Jun 30 17:46:50 JST 2024\n",
      "(c) I McDonald, D Naylor, D Jones and J Thornton 1993 All Rights Reserved.\n",
      "\n",
      "Configured for 100000 atoms and 24000 residues.\n",
      "\n",
      "Criteria\n",
      "\n",
      "Minimum Angles; DHA 90.00, HAAA 90.00, DAAA 90.00\n",
      "Maximum Distances; D-A 3.9, H-A 2.5, S-S 3.0\n",
      "Maximum angles at aromatic acceptors DAAX 20.00, HAAAX 20.00\n",
      "Minimum covalent separation 3 Covalent bonds\n",
      "\n",
      "\n",
      "Processing file \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_999.pdb\" . . .\n",
      "Opening \"/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus/model_999.pdb\" for protein co-ordinates. . .\n",
      "\n",
      "HEADER continuation line identified and ignored!\n",
      "HEADER continuation line identified and ignored!\n",
      "2485 atoms selected from 305 residues.\n",
      "\n",
      "Found 1 elements of secondary structure and 2 chains.\n",
      "0 CONRECS used.\n",
      "Checking for disulphide bridges . . . \n",
      "0 disulphide bonds found.\n",
      "\n",
      "\n",
      "Adding Polar Hydrogens.\n",
      "Outputting PDB format file into model_999.h . . .\n",
      "Opened output file \"model_999.hb2\".\n",
      "Checking for hydrogen bonds . . . \n",
      "273 hydrogen bonds found.\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "import shutil\n",
    "\n",
    "def hbplus(hbplus_dir, filename, save_dir):\n",
    "    os.environ['PATH'] += os.pathsep + hbplus_dir\n",
    "    os.chdir(hbplus_dir)\n",
    "\n",
    "    filename = filename.split('.')[0]\n",
    "    src_file = os.path.join(hbplus_dir, f'{filename}.pdb')\n",
    "    command = f'hbplus -O {src_file}'\n",
    "    os.system(command)\n",
    "    os.remove(src_file)\n",
    "\n",
    "    src_file = os.path.join(hbplus_dir, f'{filename}.h')\n",
    "    dst_file = os.path.join(save_dir, f'{filename}_Hplus.pdb')\n",
    "    shutil.move(src_file, dst_file)\n",
    "\n",
    "    src_file = os.path.join(hbplus_dir, f'{filename}.hb2')\n",
    "    dst_file = os.path.join(save_dir, f'{filename}.hb2')\n",
    "    shutil.move(src_file, dst_file)    \n",
    "\n",
    "hbplus_dir = '/home/lwang/AI-HDX-main/ProteinComplex_HDX_prediction/prog/hbplus'\n",
    "save_dir = '/home/lwang/models/HDX_LSTM/data/Latest_test/hdock/structure/1UGH_hbplus'\n",
    "\n",
    "for pdb_file in os.listdir(hbplus_dir):\n",
    "    if pdb_file.endswith('.pdb'):\n",
    "        pdb_name = pdb_file.split('.')[0]\n",
    "        hbplus(hbplus_dir, pdb_name, save_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "\n",
    "save_dir = '/home/lwang/models/HDX_LSTM/data/Latest_test/hdock/structure/1UGH_hbplus'\n",
    "output_file = '/home/lwang/models/HDX_LSTM/data/Latest_test/hdock/structure/pdb_paths.txt'\n",
    "\n",
    "with open(output_file, 'w') as f:\n",
    "    for pdb_file in os.listdir(save_dir):\n",
    "        if pdb_file.endswith('.pdb'):\n",
    "            pdb_path = os.path.join(save_dir, pdb_file)\n",
    "            f.write(pdb_path + '\\n')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "liyao_env",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
